Table 1. Number of blood samples collected from humans (male/female) and the number and prevalence of Salmonella spp. by sites.



<table><tr><td rowspan="2">Location</td><td colspan="3">No. blood samples collected</td><td colspan="3">No. (%) blood samples positive</td></tr><tr><td>Total</td><td>Male</td><td>Female</td><td>Total</td><td>Male</td><td>Female</td></tr><tr><td>Hospital A</td><td>204</td><td>116</td><td>88</td><td>9 (4)</td><td>8 (7)</td><td>1 (1)</td></tr><tr><td>Hospital B</td><td>119</td><td>66</td><td>53</td><td>12 (10)</td><td>10 (15)</td><td>2 (4)</td></tr><tr><td>Hospital C</td><td>120</td><td>66</td><td>54</td><td>5 (4)</td><td>1 (2)</td><td>4 (7)</td></tr><tr><td>Hospital D</td><td>102</td><td>52</td><td>50</td><td>4 (4)</td><td>2 (4)</td><td>2 (4)</td></tr><tr><td>Hospital E</td><td>104</td><td>54</td><td>50</td><td>3 (3)</td><td>1 (2)</td><td>2 (4)</td></tr><tr><td>Hospital F</td><td>107</td><td>38</td><td>69</td><td>2 (2)</td><td>2 (5)</td><td>0 (0)</td></tr><tr><td>Hospital G</td><td>157</td><td>83</td><td>74</td><td>3 (2)</td><td>2 (2)</td><td>1 (1)</td></tr><tr><td>Diagnostic lab.</td><td>78</td><td>44</td><td>34</td><td>1 (1)</td><td>0 (0)</td><td>1 (3)</td></tr><tr><td>Total</td><td>991</td><td>519</td><td>472</td><td>39 (4)</td><td>26 (5)</td><td>13 (3)</td></tr></table>

Copenhagen, Denmark (DTU-FOOD) on all Salmonella isolates as minimum inhibitory concentration (MIC) determinations according to the previously described method  $ [27] $ . In addition to the previously described method, the following antimicrobial and resistance epidemiological cut-off values were used in the study for cefotaxime; (R > 2 mg/mL) (www.eucast.org).

## Detection of plasmid mediated quinolone resistance genes

Strains showing reduced susceptibility to ciprofloxacin and susceptible to nalidixic acid were further characterized through the use of a PCR assay with primers specific for qnrA, qnrB, qnrC, qnrD, qnrS, and  $ aac(6')-lb $  [23,28,29]. PCR products were purified using the GFX $ ^{â„¢} $  PCR DNA kit (GE Healthcare, Chalfont St. Giles, UK) and submitted to Macrogen Inc. for sequencing. Sequence analysis and alignment was performed using the Vecton NTI suite 9 (InforMax Inc., Bethesda, Maryland, US) and the resulting nucleotide sequences were compared to sequences obtained from GenBank (http://www.lahey.org/studies/webt.html).

## Results

## Bacterial isolates from humans

Of the 991 blood samples collected, 519 samples originated from males. Thirty-nine samples yielded Salmonella, which gave an overall prevalence among humans submitting a blood sample at 4%. The prevalence ranged substantially between the different sample sites from 1% (diagnostic laboratory) to 10% (hospital B) (Table 1). The prevalence between males and females differed with 5% among males in contrast to 3% among females (not significant p = 0.07). The overall age of patients from whom the positive blood samples were collected ranged from seven days to 28 years; however, large differences were observed between the different hospitals: 7 days to 4 years in hospital A; 3 months to 2 years and 6 months in hospital B; 18 days to 2 years in hospital C; 2 years and 5 months to 7 years in hospital D; 1 year and 7 months to 10 years and 7 months in hospital E; 1 year and 3 months to 16 years in hospital F; 11 years to 28 years in hospital G; and 14 years at the diagnostic laboratory. The median age of the febrile patients that yielded Salmonella was 1 year and 7 months.



## Bacterial isolates from chickens

Out of the 641 chicken fecal samples collected, 70 were positive for Salmonella, giving an overall prevalence of 11%. The number of samples investigated and the number of samples positive for Salmonella fluctuated among the different farms, resulting in a prevalence ranging from 2% (farm C and G) to 26% (farm J) (Table 2).

## Serotyping of human isolates

The 39 human isolates revealed nine different Salmonella serovars of which 32 (82%) were NTS and seven (18%) were S. Typhi. The most commonly isolated serovars were S. Enteritidis (n = 13; 33%), S. Dublin (n = 7; 18%), S. Typhimurium (n = 7; 18), and S. Typhi (n = 7; 18%). Several uncommon serovars were among the isolated Salmonella, such as S. Jukestown (n = 1; 3%), S. Monschaui (n = 1; 3%), S. Oritamerin (n = 1; 3%), and S. Apapa (n = 1; 3%), which were all from different sample sites (Table 3).