156

W. Oueslati et al. / Meat Science 119 (2016) 154–159

disease (chronic gastroenteritis, traumatic peritonitis, intestinal obstruction, chronic feet diseases, chronic mastitis...) (Table 1).

Disposable sterile bags were used to collect approximately 100 g of meat from the inner parts of iliopsoas muscle. The meat surface was sterilized and the sample was transferred to a sterile stomacher bag.

As recommended by Codex Alimentarius (1987), samples were stored at  $ -18^{\circ} $ C until used in the Laboratory of Food Microbiology National School Veterinary of Medicine, Sidi Thabet (Tunisia).

##### 2.2. Salmonella spp. VIDAS detection

The detection of Salmonella spp. by alternative VIDAS Easy Salmonella (bioMérieux, SA Marcy l'Étoile, France) technique was performed in four steps: primary enrichment, selective enrichment, VIDAS and confirmation (Cheung & Kam, 2012). Twenty five grams of meat sample were homogenized in 225 ml of Buffered Peptone Water (Biokar Diagnostics, Beauvais, France). The mixture was incubated at 37 °C for 22 h. A volume of 100  $ \mu $ l of media was transferred to 10 ml Salmonella Xpress broth 2 (SX2) (bioMérieux SA, Marcy l'Étoile, France) and incubated at 41.5 °C for 24 h. One milliliter of the culture was heated for 15 min at 100 °C then cooled at room temperature. A volume of 0.5 ml SX2 broth was transferred into a SLM array and deposited in the VIDAS (bioMérieux SA, Marcy l'Étoile, France). Confirmation of positive samples was performed by isolation of Salmonella from unheated SX2 broth on selective agar (XLD and SS) (Biokar Diagnostics, Beauvais, France) and identification by urease test followed by API 20E system test (bioMérieux SA, Marcy l'Étoile, France) according to ISO 6579, 2002 reference technique.

#### 2.3. Serotypes of Salmonella strains

Serotypes of Salmonella isolates were determined by slide agglutination tests with specific immune sera against O, H and Vi Salmonella antigens (BioRad, Marne-La-Coquette, France) at the National Centre for Salmonella, Shigella and Vibrio, Pasteur Institute of Tunis (Tunisia). The interpretation of the results was performed according to the White–Kauffmann–Le Minor scheme (Popoff, Bockemühl, & Gheesling, 2001).

#### 2.4. Molecular study

InnuPREP DNA Mini Kit (Biometra, Les Ulis, France) was used for the extraction of Salmonella genomic DNA from the selective enrichment SX2 broth (bioMérieux SA, Marcy l'Étoile, France): one milliliter of SX2 broth was centrifuged for 10 min at 7500 rpm, the pellet was mixed with a washing solution and proteinase K (20 mg/ml). The mixture was heated at 50 °C for 10 min then centrifuged at 12,000 rpm for 2 min. The pellet was washed twice and DNA was eluted by a centrifugation at 8000 rpm for 1 min then stored at -20 °C until used. The molecular confirmation of Salmonella was carried out by performing a PCR detecting a 1 kb DNA fragment (Karraouan, Fassouane, El Ossmani, Cohen, & Bouchrif, 2010), using specific primers: F-5'ACCACGCTCTTTCGTCTGG3' and R-5GAACTGACTACGTAGACGCTC3' (Abouzeed, Hariharana, Poppe, & Kibengea, 2000). The invasion gene invA and virulence genes spvC and h-li respectively have sizes of 275, 669 and 173 bp (Karraouan et al., 2010). All Salmonella isolated strains were tested for three invasion and virulence genes (invA, h-li and spvC) using three specific primer sets: F-5'TATCGCCACGTTCGGGCAA3' and R-5'TCGCACCGTCCAAAGGAACC3', F-5'AGCCTCGGCTACTGGTCTTG3' and R-5'CCGCAGCAAGAGTCACCTCA3', F-5'CGGAAATACCATCAAA TA3' and R-5'CCCAAACCCATACTTACTCTG3' respectively (Abouzeed et al., 2000).



The PCR mixture consisted of 2.5  $ \mu $ l of 10× PCR buffer (20 mM Tris–HCl; pH 8.5; 50 mM KCl), 3 mM MgCl $ _{2} $ , 0.4 mM of each dNTP, 25  $ \mu $ M of each primer, 1.25 U Taq Polymerase (Vivantis, Oceanside, USA), sterile distilled water and 2.5  $ \mu $ l of DNA template.

The DNA amplification was performed using the following program: 1 min denaturation at  $ 95\ ^{\circ}C $ , followed by 35 cycles ( $ 95\ ^{\circ}C $  for 20 s,  $ 55\ ^{\circ}C $  for 20 s and  $ 72\ ^{\circ}C $  for 2 min) and a final extension at  $ 72\ ^{\circ}C $  for 4 min (Nayak, Stewart, & Wang, 2004). Amplification was performed in a thermocycler (Esco Swift Max Pro, Horsham PA, USA).

PCR products were electrophorized in 1.5% agarose gel and visualized with ultraviolet transillumination after ethidium bromide staining. Negative and positive controls were added for each PCR run.

#### 2.5. Antibiotic resistance

Resistance test to 12 antibiotics was screened by the diffusion method on Mueller-Hinton agar antibiotic discs (BioRad, Marne-La-Coquette, France). Interpretation of the results was done according to the Clinical and Laboratory of Standards Institute criteria (CLSI, 2008).

#### 2.6. Statistical analyses

The infection prevalence percentages were compared using Epi Info 6 (Dean et al., 2011) with Mantel Haenszel Chi-square test. Observed differences were considered significant when the p value was lower than 0.05 (Schwartz, 1993).

#### 2.7. Ethics statement

The study was performed in accordance with the Memorandum of the French General Directorate of Food No 2012-8056 March 13, 2012 on the authorization of slaughterhouses to derogate from the requirement to stun animals.

<div style="text-align: center;"><img src="imgs/img_in_chart_box_261_1134_907_1455.jpg" alt="Image" width="54%" /></div>


<div style="text-align: center;">Fig. 1. Seasonal distribution of Salmonella spp. serotypes.</div>
