Nguyen _et al. Gut Pathog (2016) 8:39_
DOI 10.1186/s13099-016-0121-5

### **RESEARCH**


## Gut Pathogens

### **Open Access**


# Antimicrobial resistance of Campylobacter isolates from small scale and backyard chicken in Kenya

Tuan Ngoc Minh Nguyen [1,2,3*], Helmut Hotzel [2], John Njeru [2,4], Joyce Mwituria [4], Hosny El‑Adawy [2,5],
Herbert Tomaso [2], Heinrich Neubauer [2] and Hafez M. Hafez [3]


**Abstract**
**Background:** Thermophilic _Campylobacter_ species are a major cause of bacterial foodborne diarrhoea in humans
worldwide. Poultry and their products are the predominant source for human campylobacteriosis. Resistance of
_Campylobacter_ to antibiotics is increasing worldwide, but little is known about the antibiotic resistance in _Campy-_
_lobacter_ isolated from chicken in Kenya. In this study, 35 suspected _Campylobacter_ strains isolated from faeces and
cloacal swabs of chicken were tested for their susceptibility to seven antibiotics using a broth microdilution assay and
molecular biological investigations.
**Results:** Overall, DNA of thermophilic _Campylobacter_ was identified in 53 samples by PCR (34 _C. jejuni_, 18 _C. coli_ and
one mix of both species) but only 35 _Campylobacter_ isolates (31 _C. jejuni_ and 4 _C. coli_ ) could be re-cultivated after
transportation to Germany. Isolates were tested for their susceptibility to antibiotics using a broth microdilution assay.
Additionally, molecular biological detection of antibiotic resistance genes was carried out. _C. jejuni_ isolates showed a
high rate of resistance to nalidixic acid, tetracycline and ciprofloxacin of 77.4, 71.0 and 71.0 %, respectively. Low resist‑
ance (25.8 %) was detected for gentamicin and chloramphenicol. Multidrug resistance in _C. jejuni_ could be detected
in 19 (61.3 %) isolates. Resistance pattern of _C. coli_ isolates was comparable. Resistance to ciprofloxacin was confirmed
by MAMA–PCR and PCR–RFLP in all phenotypically resistant isolates. The _tet_ (O) gene was detected only in 54.5 % of
tetracycline resistant _C. jejuni_ isolates. The _tet_ (A) gene, which is also responsible for tetracycline resistance, was found
in 90.3 % of _C. jejuni_ and in all _C. coli_ isolates. Thirteen phenotypically erythromycin-resistant isolates could not be char‑
acterised by using PCR–RFLP and MAMA–PCR.
**Conclusions:** To the best of our knowledge, this study is the first report about resistance to antibiotics in thermo‑
philic _Campylobacter_ originating from chicken in Kenya. _Campylobacter_ spp. show a high level of resistance to cipro‑
floxacin, nalidixic acid and tetracycline but also a remarkable one to chloramphenicol and gentamicin and they are
multidrug resistant. Resistance to antibiotics is a global public health concern. In Kenya, resistance surveillance needs
further attention in the future. Efforts to establish at least a National Laboratory with facilities for performing pheno‑
typic and genotypic characterization of thermophilic _Campylobacter_ is highly recommended.
**Keywords:** _Campylobacter_, Antibiotic resistance, Microdilution, Chicken, Kenya



**Background**
Thermophilic _Campylobacter_ ( _C._ ) species have become
the most frequent cause of bacterial gastroenteritis in


*Correspondence: tuan.nguyen@fli.de; minhtuannn@hvu.edu.vn
2 Friedrich-Loeffler-Institut, Institute of Bacterial Infections and Zoonoses,
Jena, Germany
Full list of author information is available at the end of the article



humans worldwide [1]. Campylobacteriosis exceed the
total number of those caused by _Salmonella, Shigella_,
and _Escherichia coli_ O157:H7 in humans, recently [2].
_Campylobacter_ infections are normally self-limiting in
adults but can cause diarrhoea or even mortality in children in developing as well as in developed countries [3,
4].



© 2016 The Author(s). This article is distributed under the terms of the Creative Commons Attribution 4.0 International License
[(http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium,](http://creativecommons.org/licenses/by/4.0/)
provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license,
[and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (http://creativecommons.org/](http://creativecommons.org/publicdomain/zero/1.0/)
[publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated.](http://creativecommons.org/publicdomain/zero/1.0/)




Nguyen _et al. Gut Pathog (2016) 8:39_ Page 2 of 9



A study from western Kenya showed that 20 % of
patients with diarrhoea were infected by _Campylobacter_
but in the group of children below 5 years _Campylobacter_
was represented with 42 % [5].
Commercial poultry and free-living birds are natural
reservoirs of thermophilic campylobacters. The organism
has been isolated from numerous bird species, including
_Columbiformes_ and domestic and free-living _Galliformes_
and _Anseriformes_ . _C. jejuni_ has been found in all areas of
commercial poultry production [6, 7]. Prevalence rates in
poultry, especially in slaughter-age broiler flocks, could
reach 100 % on some farms. Although, _Campylobacter_ is
insignificant for poultry health, it is a predominant cause
of foodborne gastroenteritis in humans worldwide, and
contaminated poultry meat is recognized as the main
source of human infections [7, 8]. In general, the knowledge about _Campylobacter_ in Kenya is limited. Most of
the published reports describe prevalence and antibiotic
resistance in _Campylobacter_ of human origin [4, 9–11].
Other reports gave information about _Campylobacter_
as cause of foodborne diseases [12] and contamination
of raw chicken and beef from butcheries and markets in
Nairobi [13]. Information on thermophilic _Campylobac-_
_ter_ of animal origin from Kenya is lacking.
Resistance against antibiotics in bacteria is of public
health concern. Most commonly used drugs in treatment
of campylobacteriosis in humans are erythromycin, fluoroquinolones or tetracycline [14]. Although, this antimicrobial treatment is usually not necessary, however the
misuse of antibiotics is widespread in Kenya [5]. Attention on resistance of _Campylobacter_ is raising and warning has been launched not to misuse antibiotics such as
macrolides, fluoroquinolones or alternative drugs [15].
Kenyan _Campylobacter_ isolates from humans showed a
high resistance rate against erythromycin (52 %), but only
low resistance to ciprofloxacin, tetracycline and nalidixic
acid with 6, 18 and 26 % in the past, respectively [5].
Clinical breakpoints of _Campylobacter_ susceptibility
based on epidemiological cut-off (ECOFF) values were
recommended. EUCAST MIC distributions incorporate
human and veterinary clinical data from several sources
worldwide [16]. The method of choice for testing antibiotic susceptibility and determination of minimum
inhibitory concentration (MIC) values of _Campylobacter_
isolates is the broth microdilution assay [17, 18].
In addition to phenotypical determination of antibiotic
resistance, genetic analysis of resistance determinants
in _Campylobacter_ can be carried out. A replacement of
threonine by isoleucine at amino acid 86 in the _gyr_ A gene

[19, 20] and a mutation at position 2074 and 2075 on the
23S rRNA gene are the main mechanisms for fluoroquinolone and erythromycin resistance, respectively [21].
Presence of _tet_ (O) and/or _tet_ (A) genes is responsible for



tetracycline resistance [22]. A mismatch amplification
mutation assay (MAMA–PCR) can be used for detection
of the mutations in _gyr_ A and 23S rRNA genes in _C. coli_
and _C. jejuni_ responsible for ciprofloxacin and erythromycin resistance, respectively [21, 23]. PCR-restriction
fragment length polymorphism (PCR–RFLP) technique

[24] is available for detection of erythromycin resistance
as well as specific PCR assays for _tet_ (O) and _tet_ (A) genes.
These methods allow the investigation of antibiotic
resistance of _Campylobacter_ even in samples from which
no _Campylobacter_ could be isolated.
To the best of our knowledge there is no report available about antibiotic resistance of thermophilic _Campylo-_
_bacter_ species isolated from chicken in Kenya. MICs and
results of molecular assays on the resistance of recent
Kenyan _C. coli_ and _C. jejuni_ are presented.


**Methods**
**Sample collection and** _**Campylobacter**_ **isolation**
In total, 35 geographically different native breed layer
flocks were sampled. The chickens were housed in backyards and homesteads of small scale farmers from the
outskirts of Thika, a town 40 km northeastern of Nairobi,
Kenya. Farmers kept between 10 and 1000 layers. The
birds were fed on commercially formulated ration from
different sources and sometimes supplied with the leftover and residual food. All the manufactures used antibiotics as part of the ingredients in the feed. During the
rearing of these chickens, antibiotics were used for prevention and treatment of diseases without any instructions. Ten to 30 cloacal swabs and faecal samples were
collected from each flock according to flock size. _Campy-_
_lobacter_ were isolated in Kenya Medical Research Institute, Nairobi according to the guidelines of ISO 10272-1

[25]. The isolates were preserved in 1.5 ml Eppendorf
tubes filled with skimmed milk medium for 1-week
transportation from Kenya to Friedrich–Loeffler-Institut,
Jena, Germany for further laboratory analysis. _Campylo-_
_bacter_ strains were re-cultivated on both Mueller–Hinton agar and CCDA (Oxoid GmbH, Wesel, Germany)
under microaerophilic conditions (5 % O2, 10 % CO2, and
85 % N2) at 37 °C for 48–72 h.


**DNA extraction**
DNA from viable bacteria was extracted using the High
Pure PCR Template Preparation Kit [™] (Roche Diagnostics, Mannheim, Germany) according to the manufacturer’s instructions. Skimmed milk samples of
_Campylobacter_ that could not be re-cultivated were
treated with phenol–chloroform to extract DNA. Briefly,
500 µl of skimmed milk medium was boiled for 5 min.
After cooling the liquid was mixed with 500 µl buffer-saturated phenol (Carl Roth GmbH, Karlsruhe, Germany)




Nguyen _et al. Gut Pathog (2016) 8:39_ Page 3 of 9



and centrifuged for 5 min. at 13,400 rpm (miniSpin,
Eppendorf, Hamburg, Germany). 500 µl chloroform/
isoamyl alcohol (24:1 vol/vol) was added to the aqueous
phase, mixed and centrifuged for 5 min. at 13,400 rpm.
DNA from the aqueous phase was precipitated by mixing with 0.6 volume of isopropanol at room temperature. After centrifugation, the supernatant was discarded
and the DNA was air dried and finally dissolved in 50 µl
10 mM Tris (Carl Roth GmbH).


**Multiplex PCR for identification of** _**Campylobacter**_ **species**
A mPCR assay was used to identify thermophilic
_Campylobacter_ species ( _C. jejuni_, _C. coli_, and _C. lari_ ) as
described by El-Adawy et al. [26]. Briefly, the PCR was
performed in a 50-μl reaction mixture containing 5.0 μl
of 10 × _Taq_ reaction buffer complete (Jena Bioscience
GmbH, Jena, Germany), 2.0 μl of dNTP mix (2 mM
each; Carl Roth GmbH), 2.0 μl of each primer (Jena
Bioscience GmbH), and 0.2 μl of _Taq_ Pol thermostable
DNA polymerase (Jena Bioscience GmbH). Amplification reactions were carried out in a TRIO Thermoblock
cycler (Biometra, Göttingen, Germany) using the following programme: one cycle of 1 min at 96 °C was
followed by 35 cycles each consisting of 60 s at 95 °C
of denaturation, 90 s at 59 °C of annealing, and 60 s at
72 °C of elongation. The PCR was terminated after a final
extension step of 5 min. at 72 °C. Amplification generated 857, 589, 522, and 462 base pair DNA fragments
specific for the genus _Campylobacter_ and the species _C._
_jejuni_, _C. lari_, and _C. coli_, respectively. For analysis, 20 μl
of PCR products were subjected to electrophoresis in a
1.5 % agarose gel for 1 h, stained with ethidium bromide
(0.5 μg/ml), and visualized under UV light. Results were
documented using BioImage system GeneGenius (Syngene, Synoptics Ltd., Cambridge, UK). Reference strains
_C. jejuni_ DSM 4688, _C. coli_ DSM 4689, and _C. lari_ DSM
11375 (Deutsche Sammlung von Mikroorganismen und
Zellkulturen GmbH, Braunschweig, Germany) were used
as positive controls.


**Antimicrobial susceptibility testing and determination**
**of MICs**
The antimicrobial susceptibility of _C. jejuni_ and _C. coli_
isolates was tested against seven antibiotic agents (chloramphenicol, erythromycin, ciprofloxacin, nalidixic
acid, gentamicin, streptomycin, and tetracycline) using
the Sensititre [™] Campylobacter plates—EUCAMP (Trek
Diagnostic Systems Ltd., East Grinstead, UK). The MIC
values were detected using different concentration ranges
as previously described [23]. Briefly, _Campylobacter_ isolates grown on Mueller–Hinton agar (Oxoid GmbH)
supplemented with 10 % bovine blood under microaerophilic conditions were suspended in NaCl solution



(0.9 %) to obtain a turbidity corresponding to a McFarland standard of 0.5 (Dr. Lange, CADAS photometer
30, Berlin, Germany). One-hundred and fifty milliliters
of the above suspension were diluted with 10 ml Mueller–Hinton broth (Oxoid GmbH) resulting in a concentration of approximately 10 [6] –10 [7] colony forming units
(cfu)/ml. One hundred milliliters of the inoculum was
filled in each well of the plate; the plates were sealed
and incubated at 37 °C for 24 h under microaerophilic
conditions. Results were read either visually or photometrically (Tecan Deutschland GmbH, Crailsheim,
Germany) using the computer program easyWIN fitting (version V6.1, 2000). _C. jejuni_ DSM 4688 and _C. coli_
DSM 4689 (Deutsche Sammlung von Mikroorganismen
und Zellkulturen GmbH) were included in each batch of
broth microdilution assay for quality control. The lowest
concentration of antibiotics that prevents visible growth
of the microorganism is defined as the MIC.


**Molecular biological detection of antibiotic resistance**
**determinants**
Extracted _Campylobacter_ DNA from all samples and
strains was used for molecular biological determination
of selected antibiotic resistance determinants by PCR.


**Erythromycin resistance**
Detection of mutations at positions 2074 and 2075 in
domain V of the 23S rRNA gene, which mediates resistance to erythromycin, was carried out by MAMA–PCR
and PCR–RFLP as described previously [21, 24]. Genes
responsible for resistance of erythromycin and ciprofloxacin were tested at two loci using MAMA–PCR and
PCR–RFLP. Primers and their sequences are given in
Table 1.


**Ciprofloxacin resistance**
A single point mutation (Thr-86-Ile) in the quinolone
resistance-determining region (QRDR) of _gyr_ A gene
was defined as source of high-level resistance to fluoroquinolones [23]. MAMA–PCR for _C. jejuni_ isolates was
carried out as described previously [27], for _C. coli_ a procedure according to Zirnstein et al. [28] was used. Primer
details are given in Table 1.


**Tetracycline resistance**
Primers DMT1 and DMT2 (Jena Bioscience GmbH) were
used for the detection of the _tet_ (O) gene which is strongly
associated with tetracycline resistance in _C. jejuni_ and _C._
_coli_ as described previously [29]. As a second gene locus
associated with tetracycline resistance the presence of
_tet_ (A) was examined by a previously described PCR assay

[22]. An alternative, in-house validated PCR assay was
created based on _tet_ (A) sequences (GenBank acc. no.




Nguyen _et al. Gut Pathog (2016) 8:39_ Page 4 of 9


**Table 1 List of primers and primer sequences used for detection of antimicrobial resistance genes**


**Antibiotic** **Method** **Primer** **Sequence (5’–3’)** **Amplicon length (bp)** **Reference**



Erythromycin MAMA–PCR [a] 23SRNA-F
ERY2075-R
ERY2074-R



TTA GCT AAT GTT GCC CGT ACC G
TAG TAA AGG TCC ACG GGG TCG C
AGT AAA GGT CCA CGG GGT CTG G



485 [17]



PCR–RFLP [a] F2-campy-23S AAT TGA TGG GGT TAG CAT TAG C 316 [20]
R2-campy-23S CAA CAA TGG CTC ATA TAC AAC TGG

Ciprofloxacin MAMA–PCR [b] CampyMAMAgyrA1 TTT TTA GCA AAG ATT CTG AT 265 [19]
CampyMAMAgyrA5 CAA AGC ATC ATA AAC TGC AA

MAMA–PCR [c] GZgyrACcoli3F TAT GAG CGA TAT TAT CGG TC 192 [24]
CanpyMAMAgyrA8 TAA GGC ATC GTA AAC AGC CA

Tetracycline _tet_ (O) PCR DMT 1 GGC GTT TTG TTT ATG TGC G 559 [25]
DMT 2 ATG GAC AAC CCG ACA GAA GC

_tet_ (A) PCR Tet(A)-F GTG AAA CCC AAC ATA CCC C 888 [18]
Tet(A)-R GAA GGC AAG CAG GAT GTA G

_tet_ (A) PCR tet-A-1 GCT CAC GTT GAC GCA GGA AAG 486 This study
tet-A-2 ATC GTC ATT GTC CGT TAC


a 23S rRNA gene mutation

b _gyr_ A gene mutation _Campylobacter jejuni_

c _gyr_ A gene mutation _Campylobacter coli_



JX891463 and JX891464)). Briefly, primers tet-A-1 and
tet-A-2 (Table 1; Jena Bioscience GmbH) were used with
the following PCR programme: An initial denaturation at
96 °C for 60 s was followed by 35 cycles of denaturation
(96 °C for 15 s), annealing (49 °C for 60 s) and extension
(72 °C for 30 s). PCR was terminated by final extension at
72 °C for 60 s. The PCR resulted in a 486 bp product.
All PCR products were analyzed by electrophoresis on
1.5 % agarose gels, staining with ethidium bromide and
visualization under UV light.


**DNA sequencing**
PCR products obtained by _tet_ (A) PCRs were sequenced
by cycle sequencing with BigDye Terminator v1.1 Cycle
Sequencing Kit (Applied Biosystems, Darmstadt, Germany) according to the instructions of the manufacturer.
In addition to the amplification primers of the Tet(A)-F/R
fragment tet-A-A (5’-AAT TTT CTT CAA ATA AGG-3’)
and tet-A-B (5’-GTC ATT CTT ATA TTA AGT GG-3’)
were used as sequencing primers. Sequencing products
were analyzed with an ABI PRISM 3130 genetic analyzer.


**MALDI‑TOF mass spectrometry**
Cultured bacteria were suspended in 300 µl of bi-distilled water and mixed with 900 ml of ethanol (Carl Roth
GmbH). Further treatment of samples and measurement
were described by El-Ashker et al. [30].


**Results**
**Identification of bacteria**
In total, 58 isolates suspected as _Campylobacter_ were
recovered from faeces and cloacal swabs of chicken flocks
in Kenya. After storage in skimmed milk medium and



transportation to Germany only 40 of these isolates could
be re-cultivated. Four _C. coli_ and 31 _C. jejuni_ were identified by mPCR (Table 2). Five other isolates were identified by MALDI-TOF mass spectrometry as members of
genera _Bacillus_, _Staphylococcus_, _Ochrobactrum_ as well as
two _Bordetella_ isolates.
Eighteen skimmed milk tubes contained _Campylobac-_
_ter_ DNA [14 _C. coli_, 3 _C. jejuni_ and one sample harboured
both _C. coli_ and _C. jejuni_ (Table 2)].


**Antimicrobial susceptibility profiles and multidrug**
**resistance**
The results of antimicrobial susceptibility testing of _C._
_jejuni_ and _C. coli_ isolates and the rate of resistance to
seven antimicrobial agents are given in Tables 3 and 4,
respectively. The _C. jejuni_ isolates showed a high rate of
resistance to nalidixic acid, tetracycline and ciprofloxacin
with 77.4, 71.0 and 71.0 %, respectively. Low resistance
rates were detected for gentamicin and chloramphenicol,
both with 25.8 % of the isolates. For the low number of _C._
_coli_ isolates (n = 4) a similar pattern was observed. Only
two isolates were susceptible to all tested antimicrobial
agents, one isolate was resistant to all tested antibiotics.
The multidrug resistance profiles of 31 _C. jejuni_ isolates
are shown in Table 5. Multidrug resistance to three or
more classes of antibiotics was found in 19 isolates (61.3 %)
and was observed in a range between 5.3 and 26.3 %.


**Molecular biological detection of antibiotic resistance**
**determinants**
DNA of 35 viable _Campylobacter_ isolates and of 18 nongrowing samples was investigated by PCR to detect antibiotic resistance. Mismatch amplification mutation assay




Nguyen _et al. Gut Pathog (2016) 8:39_ Page 5 of 9



**Table 2 Results of cultivation and multiplex PCR identifi-**
**cation of** _**Campylobacter**_ **isolates**


**Cultivation** **mPCR identification of** _**Campylobacter**_ **Total**
**n (%)**
_**C. jejuni C. coli**_ **Not identi‑** _**C. coli/C.**_
**fied** _**jejuni**_


Positive (n) 31 4 5 0 40 (69.0)

Negative (n) 3 14 0 1 18 (31.0)


(MAMA–PCR) was used to characterize a _gyr_ A gene
mutation associated with ciprofloxacin resistance as well
as mutations in 23S rRNA genes as cause of erythromycin
resistance. The molecular biological detection of resistance to ciprofloxacin in both _C. coli_ and _C. jejuni_ was also
confirmed by change of amino acid 86 from threonine to
isoleucine in the _gyr_ A gene. Additional to the ciprofloxacin resistant _Campylobacter_ isolates (Tables 3, 4), three



_Campylobacter_ DNAs were detected harbouring the _gyr_ A
gene mutation. The results were confirmed by PCR–RFLP
according to Vacher et al. [24]. Mutations at positions
2074 and 2075 of the 23S rRNA genes in 13 phenotypically erythromycin-resistant isolates could neither be
detected by using PCR–RFLP nor MAMA–PCR.
The _tet_ (O) gene which is mainly responsible for tetracycline-resistance was detected by PCR in 12 out of 22
resistant _C. jejuni_ isolates (54.5 %) and in all tetracycline
resistant _C. coli_ isolates. _tet_ (O) gene was not detected in
DNA extracted from the non-growing samples. Additionally, a newly developed PCR assay was used for the
detection of the _tet_ (A) gene. _tet_ (A) was detected in 28
out of 31 _C. jejuni_ (90.3 %) and in all 4 _C. coli_ isolates.
In 3 out of 14 non-growing samples which harboured
_C. coli_ DNA, _tet_ (A) gene could be found as well as in
the one sample where both _C. jejuni_ and _C. coli_ were
detected.



















|Table 3 Results of M|MIC determination|n and resistance rates of Kenyan Campylobacter jejuni isolate Antibiotic susceptibility of Campylobacter jejuni|Col4|Col5|Col6|Col7|Col8|Col9|Col10|Col11|Col12|Col13|es R*|
|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
|Class|Antibiotic<br>(µg/ml)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~Antibiotic susceptibility of~~~~_Campylobacter jejuni_~~<br>(n=31)|~~R*~~<br>(%)|
|Class|Antibiotic<br>(µg/ml)|0.06|0.12|0.25|0.5|1|2|4|8|16|32|64||
|Macrolides<br>|Chloramphenicol||||||9|3|6|5<br>**_8_**|5<br>**_8_**||~~25.8~~<br>|
|Macrolides<br>|Erythromycin<br>||||14|||1<br>**_2_**|1<br>**_2_**|**_2_**|**_12_**||~~51.6~~<br>|
|~~Fluoroquinolones ~~<br>|Ciprofloxacin|5|2|2|**_2_**|**_2_**|**_1_**|**_19_**|||||~~71.0~~<br>|
|~~Quinolones~~|Nalidixic acid|||||||4|1|2<br>**_2_**|2<br>**_2_**|**_22_**|~~77.4~~<br>|
|Aminoglycosides<br>|Gentamicin||2|1|4|11|5<br>**_2_**|5<br>**_2_**|**_1_**|**_5_**|||~~25.8~~<br>|
|Aminoglycosides<br>|Streptomycin|||||12|2|4<br>**_1_**|4<br>**_1_**|**_12_**|||~~41.9~~<br>|
|~~Tetracyclines~~|Tetracycline|||6|1|2|**_1_**|**_1_**|**_1_**|**_20_**|||~~71.0~~|


Boldface in italic type indicates the number of resistant isolates. A thick black line indicates the break point between clinically sensitive and resistant strains

R* resistance rate

















|Table 4 Results of M|MIC determination|n and resistance rates of Kenyan Campylobacter coli isolates Antibiotic susceptibility of Campylobacter coli|Col4|Col5|Col6|Col7|Col8|Col9|Col10|Col11|Col12|Col13|R*|
|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
|Class|Antibiotic<br>(µg/ml)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~Antibiotic susceptibility of~~~~_Campylobacter coli_~~<br>(n=4)|~~R*~~<br>(%)|
|Class|Antibiotic<br>(µg/ml)|0.06|0.12|0.25|0.5|1|2|4|8|16|32|64||
|Macrolides<br>|Chloramphenicol||||||2|||**_2_**|**_2_**||~~50.0~~<br>|
|Macrolides<br>|Erythromycin<br>||||4||||||||~~0~~<br>|
|~~Fluoroquinolones ~~<br>|Ciprofloxacin|1||||||**_3_**|||||~~75.0~~<br>|
|~~Quinolones~~|Nalidixic acid||||||||1|||**_3_**|~~75.0~~<br>|
|Aminoglycosides<br>|Gentamicin||1||1|1|**_1_**|**_1_**|||||~~25.0~~<br>|
|Aminoglycosides<br>|Streptomycin|||||2||2|2||||~~0~~<br>|
|~~Tetracyclines~~|Tetracycline|||1|1|1||||**_1_**|||~~25.0~~|


Boldface in italic type indicates the number of resistant isolates. A thick black line indicates the break point between clinically sensitive and resistant strains

R* resistance rate




Nguyen _et al. Gut Pathog (2016) 8:39_ Page 6 of 9



**Table 5 Multidrug resistance profiles of 19** _**Campylobacter**_
_**jejuni**_ **isolates**


**Antibiotic resistance profile** **No. of resistant**
**isolates (%)**


TET, CIP, NAL, ERY, GEN, STR, CHL 1 (5.3)

TET, CIP, NAL, ERY, STR, CHL 1 (5.3)

TET, CIP, NAL, ERY, GEN, STR 3 (15.8)

TET, CIP, NAL, GEN, STR, CHL 1 (5.3)

TET, CIP, NAL, ERY, STR 2 (10.5)

TET, CIP, NAL, ERY, CHL 1 (5.3)

TET, CIP, NAL, ERY 2 (10.5)

TET, CIP, NAL, STR 1 (5.3)

TET, CIP, NAL 5 (26.3)

CIP, NAL, ERY, STR, CHL 1 (5.3)

CIP, NAL, STR 1 (5.3)


_TET_ tetracycline, _CIP_ ciprofloxacin, _NAL_ nalidixic acid, _ERY_ erythromycin, _GEN_
gentamicin, _STR_ streptomycin, _CHL_ chloramphenicol


**Discussion**
The antimicrobial susceptibility patterns among _Campy-_
_lobacter_ isolates originating from chicken in Kenya were
analyzed according to the guidelines of CLSI for _Enterobac-_
_teriaceae_ which had been guided by previous reports [17,
31–34]. Clinical breakpoints for interpretation of MIC values of _C. jejuni_ and _C. coli_ from chicken are available [31,
35]. In this study a commercially available broth microdilution assay was used for the determination of MIC values for
seven antibiotics. The assay already proved to be suitable in
previous investigations [21, 27, 34, 36, 37].
In this study, only 40 out of 58 suspected _Campylobac-_
_ter_ samples could be re-cultivated. The storage conditions (temperature, microaerophilic atmosphere) using
skimmed milk medium were possibly not ideal. However,
it had been demonstrated that _C. jejuni_ can survive up to
14 days at 1 °C or 2.5 days at 20 °C in sterile skimmed milk

[38, 39]. Alternative storage of _Campylobacter_ cultures
using transport medium (for example Amies medium) or
cryovials is recommended for future investigations.
In 53 out of 58 collected samples, _Campylobacter_ DNA
was identified by mPCR assay [26]. The majority of the
cultures proved to be _C. jejuni_ (88.6 %) which is in agreement with previous studies in chicken [6, 7]. In 18 DNA
extracts of non-viable samples, 3 _C. jejuni_ (16.7 %), one
mixed population of _C. jejuni/C. coli_ and 14 _C. coli_ were
identified. These findings are in agreement with those of
a previous study that found a longer viability of _C. jejuni_
in comparison to _C. coli_ in biological milieu [40]. It may
be possible that some _C. coli_ isolates had been lost during the shipment period. In summary, _C. jejuni_ was identified much more often than _C. coli_ (64.2 %) by mPCR
investigation. In agreement with other studies, the findings highlighted the usefulness of mPCR as a reliable,



sensitive, time and cost saving method for identification
of thermophilic _Campylobacter_ [26].
The antibiotic susceptibility of 35 _Campylobacter_ isolates from Kenyan chicken was investigated using European Committee on Antimicrobial Susceptibility Testing
and epidemiological cut-off values (EUCAST–ECOFFS)

[16]. A broth microdilution assay was used as a standardized, easy, and reliable method for the determination of
MIC of seven antibiotics [17, 31–34]. High resistance rates
were obtained for ciprofloxacin, nalidixic acid and tetracycline with more than 70 % which is in agreement with a
recent European Food Safety Authority (EFSA) report [41].
These results are in contrast to those of Brooks et al. [5]
who reported resistance rates for _Campylobacter_ recovered from humans with diarrhoea in Western Kenya for
ciprofloxacin, nalidixic acid and tetracycline with 6, 26 and
18 % in 2006, respectively. The general high rates of resistance in the chicken isolates may be caused by availability
and uncontrolled use of antibiotics by small farmers [42].
Resistance to chloramphenicol is remarkable with
25.8 % in this investigation. Use of chloramphenicol
is banned in animal breeding in Europe for more than
20 years, but still it is often used in many third world
countries [43]. It is easy to obtain antibiotics over-thecounter and other unregulated venues and injudicious
use promotes the development of resistance to antimicrobial agents. Resistance to gentamicin in the isolates
obtained from chicken was low in this study (25.8 %), but
_Campylobacter_ isolated from broilers and turkeys were
totally susceptible to gentamicin [37, 41, 44, 45]. Erythromycin resistance rates found in this study correspond to
those of similar studies elsewhere [41, 44, 45].
Multidrug resistance was detected in 61.3 % of the
_Campylobacter_ isolates. Eleven different combinations
were found (Table 5). Frequent, resistance to ciprofloxacin, nalidixic acid and tetracycline was identified (17 out
of 19 multidrug resistant isolates) which is in agreement
with previous investigation using Vietnamese _Campy-_
_lobacter_ isolates [45]. However, EFSA [41] reported low
level of multidrug resistance in _C. jejuni_ from broilers of
the member states of the EU.
The emerging of antibiotic resistance has been attributed to the overuse and misuse of antimicrobial agents
in both the developed and developing world. Antibiotics
are widely used as growth supplements in livestock and
to prevent infections [46]. The emerging of multidrug
resistance may reflect acquisition of different resistance
determinants on the same DNA molecule or single determinants, such as multidrug pumps, that specify efflux
activity against different antimicrobial agents [47]. The
mechanisms of genetic resistance might be chromosomal
or plasmid-borne, and represent a combination of endogenous and acquired genes. In general, mechanisms of




Nguyen _et al. Gut Pathog (2016) 8:39_ Page 7 of 9



antibiotic resistance as modification of the antibiotic by
aminoglycoside-modifying enzymes (AphA, AadE, Sat),
enzymatic inactivation of the antibiotic by β-lactamase
and modification of the DNA gyrase target, mutations
in 23S rRNA genes were included for aminoglycosides,
beta-lactams, fluoroquinolones, macrolides and tetracyclines, respectively [48, 49]. The multidrug efflux pump
CmeABC has been involved in the resistance mechanisms of _C. jejuni_ and _C. coli_ to tetracyclines, fluoroquinolones, macrolides and beta-lactams [49].
Molecular biological methods were used for detection
of antibiotic resistance determinants either using DNA
isolated from cultures or that of non-cultured bacteria

[27, 37]. All isolates of this study which were resistant to
ciprofloxacin carried a mutation of the amino acid 86 of
the _gyr_ A resulting in a change from threonine to isoleucine. This mutation was detected also in 3 DNA samples extracted from skimmed milk. The MAMA–PCR
protocol allowed the detection of the _gyr_ A mutation and
PCR–RFLP was confirming the mutation from (ACA to
ATA) of amino acid 86. This result was in agreement with
previous reports showing that both methods are simple,
reliable, rapid tools that can be used as screening methods [27, 37]. In _Campylobacter_, resistance to erythromycin is chromosomally encoded by an alteration of the
23S rRNA gene. High level resistance to erythromycin is
caused by mutations at position 2074 and/or 2075 of the
domain V of this gene. In this study the mutations were
neither detected by MAMA–PCR nor by PCR–RFLP.
The _tet_ (O) gene is known to be responsible for tetracycline resistance in _Campylobacter_ isolates [29]. In this
study, only 54.5 % of the tetracycline resistant isolates
harboured the _tet_ (O) gene. The _tet_ (A) gene also plays role
in resistance to tetracycline [22]. The efflux gene _tet_ (A)
is coding for an approximately 46 kDa membrane-bound
efflux protein for membrane-associated proteins and is
involved in the export of tetracycline from the cell [50].
In this study, using the recommended primers for _tet_ (A)
amplification [18] PCR products of 696 bp instead of
888 bp were obtained. DNA sequencing of amplicons and
database search resulted in 99.0 % homology to a partial
putative integral membrane protein and a putative periplasmic protein. Hence, a new PCR assay based on _tet_ (A)
gene sequences for _C. jejuni_ (acc. no. JX891464) and
_C. coli_ (acc. no. JX891463) was developed. Parameters
such as limit of detection, limit of quantification, PCR
efficiency and specificity were considered during an inhouse validation process. Amplicon length was 486 and
the amplicons were sequenced to confirm the identity.
The _tet_ (A) gene was much more frequently identified in
the Kenyan _Campylobacter_ isolates than _tet_ (O) (35 vs 13).
To the best of our knowledge this is the first report
on the status of antibiotic susceptibility of thermophilic



_Campylobacter_ from chicken in Kenya. High level of
resistance to ciprofloxacin, erythromycin and nalidixic
acid as well as multidrug resistance was detected previously in Kenya. In Kenya, this problem is reported to be
caused by the increasing rate of unregulated over-thecounter sale without prescriptions of these antibiotics,
mainly to humans self-treatment of suspected infections
and to a lesser extent for use in animals [51]. These findings also demonstrate the potential for antibiotic-resistant
bacteria to spread through the food chain from animals
treated with antibiotics for humans. Such misuse and
overuse may have resulted in the selection of resistant
mutants or acquisition of antibiotic resistance genes from
other organisms through the process of genetic exchange.
It is recommendable that a long-term local surveillance
programme is adopted for monitoring changes in resistance among _Campylobacter_ isolates. Efforts to establish
at least a National Laboratory with facilities for performing phenotyping and genotyping methods is highly recommended. Emphasis should be given on educational
advertising to reduce the input of antibiotics in animal
breeding to minimize the potential hazard for humans.


**Authors’ contributions**
TNMN, HH, JN, HE, HT and HMH participated in the conception and design
of the study. TNMN, HH, JN, JM and HE performed farm and laboratory work.
TNMN, HH, JN, HE, HT, HN and HMH analysed the data and wrote the manu‑
script. All authors contributed to the analysis and supported the manuscript
discussion. All authors read and approved the final manuscript.


**Author details**
1 Hung Vuong University, Phu Tho, Vietnam. 2 Friedrich-Loeffler-Institut, Insti‑
tute of Bacterial Infections and Zoonoses, Jena, Germany. [3] Institute of Poultry
Diseases, Free University Berlin, Berlin, Germany. [4] Centre for Microbiology
Research (CMR), Kenya Medical Research Institute, Nairobi, Kenya. [5] Depart‑
ment of Poultry Diseases, Faculty of Veterinary Medicine, Kafrelsheikh Univer‑
sity, Kafrelsheikh, Egypt.


**Acknowledgements**
The authors thank Byrgit Hofmann and Ronald Ng’etich for excellent technical
assistance.


**Competing interests**
The authors declare that they have no competing interests.


**Availability of data and material**
The data supporting the findings of this study are contained within the
manuscript.


**Ethics approval and consent to participate**
This study was carried out in strict accordance with the recommendations of
the National Guidelines on the Care and Use of Animals in Research, Education
and Training in Kenya, Consortium for National Health Research (CNHR) which
complies with the international laws and regulation regarding ethical consid‑
erations, transport, housing and experimental use of animals in research.


**Funding**
This study was supported by Friedrich–Loeffler-Institut, Institute of Bacterial
Infections and Zoonoses, Germany and Vietnam Ministry of Agriculture and
Rural Development.


Received: 9 June 2016  Accepted: 12 August 2016




Nguyen _et al. Gut Pathog (2016) 8:39_ Page 8 of 9



**References**
1. Man SM. The clinical importance of emerging _Campylobacter_ species. Nat
Rev Gastroenterol Hepatol. 2011;8:669–85.
2. European Food Safety Authority (EFSA). Manual for reporting of foodborne outbreaks in accordance with Directive 2003/99/EC from the year
2010/2011. European Food Safety Authority (EFSA). Parma. 2011; 138.
3. Coker AO, Isokpehi RD, Thomas BN, Amisu KO, Obi CL. Human campylo‑
bacteriosis in developing countries. Emerg Infect Dis. 2002;8:237–44.
4. Saidi SM, Iijima Y, Sang WK, Mwangudza AK, Oundo JO, Taga K, et al.
Epidemiological study on infectious diarrheal diseases in children in a
coastal rural area of Kenya. Microbiol Immunol. 1997;41:773–8.
5. Brooks JT, Ochieng JB, Kumar L, Okoth G, Shapiro RL, Wells JG, et al.
Surveillance for bacterial diarrhea and antimicrobial resistance in rural
Western Kenya, 1997-2003. Clin Infect Dis. 2006;43:393–401.
6. Agunos A, Waddell L, Léger D, Taboada E. A systematic review character‑
izing on-farm sources of _Campylobacter_ spp. for broiler chickens. PLoS
ONE. 2014;9(8):e104905.
7. Sahin O, Kassem II, Shen Z, Lin J, Rajashekara G, Zhang Q. _Campy-_
_lobacter_ in poultry: ecology and potential interventions. Avian Dis.
2015;59:185–200.
8. Hermans D, Pasmans F, Messens W, Martel A, Van Immerseel F, Rasschaert
G, et al. Poultry as a host for the zoonotic pathogen _Campylobacter jejuni_ .
Vector Borne Zoonotic Dis. 2012;12:89–98.
9. Joyce T, McGuigan KG, Elmore-Meegan M, Conroy RM. Prevalence of enter‑
opathogens in stools of rural Maasai children under five years of age in the
Maasailand region of the Kenyan Rift Valley. East Afr Med J. 1996;73:59–62.
10. Shapiro RL, Kumar L, Phillips-Howard P, Wells JG, Adcock P, Brooks J, et al.
Antimicrobial-resistant bacterial diarrhea in rural western Kenya. J Infect
Dis. 2001;183:1701–4.
11. Swierczewski BE, Odundo EA, Koech MC, Ndonye JN, Kirera RK, Odhia‑
mbo CP, et al. Surveillance for enteric pathogens in a case–control
study of acute diarrhea in Western Kenya. Trans R Soc Trop Med Hyg.
2013;107:83–90.
12. Ombui JN, Kagiko MM, Arimi SM. Foodborne diseases in Kenya. East Afr
Med J. 2001;78:40–4.
13. Osano O, Arimi SM. Retail poultry and beef as sources of _Campylobacter_
_jejuni_ . East Afr Med J. 1999;76:141–3.
14. Blaser B, Engberg J. Clinical aspects of _Campylobacter jejuni_ and _Campy-_
_lobacter coli_ infections. In: Nachamkin I, Szymanski CM, Blaser MJ, editors.
_Campylobacter_ . Washington DC: American Society for Microbiology; 2008.
p. 99–121.
15. EFSA: Summary Report on antimicrobial resistance in zoonotic and indi‑
cator bacteria from humans, animals and food. EFSA J.2015;13(2):4036.
[doi:10.2903/j.efsa.2015.4036.](http://dx.doi.org/10.2903/j.efsa.2015.4036)
16. EUCAST. Breakpoint tables for interpretation of MICs and zone diameters.

[http://www.eucast.org, Version 3.1. 2013.](http://www.eucast.org)
17. Ge B, Wang F, Sjölund-Karlsson M, McDermott PF. Antimicrobial resistance
in _Campylobacter_ : susceptibility testing methods and resistance trends. J
Microbiol Methods. 2013;95:57–67.
18. Luber P, Bartelt E, Genschow E, Wagner J, Hahn H. Comparison of broth
microdilution, E test, and agar dilution methods for antibiotic suscep‑
tibility testing of _Campylobacter jejuni_ and _Campylobacter coli_ . J Clin
Microbiol. 2003;41:1062–8.
19. Wang Y, Huang WM, Taylor DE. Cloning and nucleotide sequence of the
_Campylobacter jejuni gyr_ A gene and characterization of quinolone resist‑
ance mutations. Antimicrob Agents Chemother. 1993;37:457–63.
20. Alonso R, Mateo E, Girbau C, Churruca E, Martinez I, Fernandez-Astorga
A. PCR-restriction fragment length polymorphism assay for detection of
_gyr_ A mutations associated with fluoroquinolone resistance in _Campylo-_
_bacter coli_ . Antimicrob Agents Chemother. 2004;48:4886–8.
21. Alonso R, Mateo E, Churruca E, Martinez I, Girbau C, Fernandez-Astorga A.
MAMA–PCR assay for the detection of point mutations associated with
high-level erythromycin resistance in _Campylobacter jejuni_ and _Campylo-_
_bacter coli_ strains. J Microbiol Methods. 2005;63:99–103.
22. Abdi-Hachesoo B, Khoshbakht R, Sharifiyazdi H, Tabatabaei M, Hosseinzadeh
S, Asasi K. Tetracycline resistance genes in _Campylobacter jejuni_ and C. _coli_
isolated from poultry carcasses. Jundishapur J Microbiol. 2014;7:e12129.
23. Zirnstein G, Li Y, Swaminathan B, Angulo F. Ciprofloxacin resistance in
_Campylobacter jejuni_ isolates: detection of _gyr_ A resistance mutations by
mismatch amplification mutation assay PCR and DNA sequence analysis.
J Clin Microbiol. 1999;37:3276–80.



24. Vacher S, Menard A, Bernard E, Megraud F. PCR-restriction fragment
length polymorphism analysis for detection of point mutations associ‑
ated with macrolide resistance in _Campylobacter_ spp. Antimicrob Agents
Chemother. 2003;47:1125–8.
25. International Standards Organization. [ISO] 10272-1. Microbiology of food
and animal feeding stuffs-horizontal method for detection and enumera‑
tion of _Campylobacter_ spp. Part 1: detection method. 2006.
26. El-Adawy H, Hotzel H, Tomaso H, Neubauer H, Hafez HM. Elucidation of
colonization time and prevalence of thermophilic _Campylobacter_ species
during turkey rearing using multiplex polymerase chain reaction. Poult
Sci. 2012;91:454–9.
27. El-Adawy H, Hotzel H, Düpre S, Tomaso H, Neubauer H, Hafez HM. Deter‑
mination of antimicrobial sensitivities of _Campylobacter jejuni_ isolated
from commercial turkey farms in Germany. Avian Dis. 2012;56:685–92.
28. Zirnstein G, Helsel L, Li Y, Swaminathan B, Besser J. Characterization of
_gyr_ A mutations associated with fluoroquinolone resistance in _Campylo-_
_bacter coli_ by DNA sequence analysis and MAMA–PCR. FEMS Microbiol
Lett. 2000;190:1–7.
29. Mazi W, Senok AC, Al-Mahmeed A, Arzese A, Bindayna K, Botta G. Trends
in antibiotic sensitivity pattern and molecular detection of _tet_ (O)-medi‑
ated tetracycline resistance in _Campylobacter jejuni_ isolates from human
and poultry sources. Jpn J Infect Dis. 2008;61:82–4.
30. El-Ashker M, Gwida M, Tomaso H, Monecke S, Ehricht R, El-Gohary F, et al.
Staphylococci in cattle and buffaloes with mastitis in Dakahlia Governo‑
rate, Egypt. J Dairy Sci. 2015;98:7450–9.
31. Clinical and Laboratory Standards Institute (CLSI). Performance standards
for antimicrobial susceptibility testing; 20th informal supplement M100–
S20. 2010; 30(1).
32. Hariharan H, Sharma S, Chikweto A, Matthew V, DeAllie C. Antimicrobial
drug resistance as determined by the E-test in _Campylobacter jejuni_, C.
_coli_, and C. lari isolates from the ceca of broiler and layer chickens in
Grenada. Comp Immunol Microbiol Infect Dis. 2009;32:21–8.
33. Luangtongkum T, Morishita TY, El-Tayeb AB, Ison AJ, Zhang Q. Com‑
parison of antimicrobial susceptibility testing of _Campylobacter_ spp. by
the agar dilution and the agar disk diffusion methods. J Clin Microbiol.
2007;45:590–4.
34. Luber P, Wagner J, Hahn H, Bartelt E. Antimicrobial resistance in
_Campylobacter jejuni_ and _Campylobacter coli_ strains isolated in 1991 and
2001–2002 from poultry and humans in Berlin Germany. Antimicrob
Agents Chemother. 2003;47:3825–30.
35. The European Committee on Antimicrobial Susceptibility Testing
(EUCAST). Breakpoint tables for interpretation of MICs and zone diam‑
eters. Version 4.0. 2014.
36. Clinical and Laboratory Standards Institute (CLSI). Performance standards
for antimicrobial disk and dilution susceptibility tests for bacterial isolates
from animals; approved standards-3rd ed. M31–A3 2008; 28(8).
37. El-Adawy H, Ahmed MFE, Hotzel H, Tomaso H, Tenhagen BA, Hartung J, et al.
Antimicrobial susceptibilities of _Campylobacter jejuni_ and _Campylobacter coli_
recovered from organic turkey farms in Germany. Poult Sci. 2015;94:2831–7.
38. Christopher FM, Smith GC, Vanderzant C. Effect of temperature and pH
on the survival of _Campylobacter_ fetus. J Food Prot. 1982;45:253–9.
39. Doyle MP, Roman DJ. Prevalence and survival of _Campylobacter jejuni_ in
unpasteurized milk. Appl Environ Microbiol. 1982;44:1154–8.
40. Blaser MJ, Hardesty HL, Powers B, Wang WL. Survival of _Campylobacter_
_fetus_ subsp. _jejuni_ in biological milieus. J Clin Microbiol. 1980;11:309–13.
41. European Food Safety Authority (EFSA)/European Centre for Disease
Prevention and Control (ECDC). EU summary report an antimicrobial
resistance in zoonotic and indicator bacteria from humans, animals and
food in 2013. EFSA J. 2015; 13:4036.
42. Center for Disease Dynamics, Economics & Policy. 2015. State of the
World´s antibiotics. 2015. CDDEP: Washington DC, 38–48.
43. European Food Safety Authority (EFSA). Scientific opinion on chloram‑
phenicol in food and feed. EFSA J. 2014; 12:3907.
44. Luangtongkum T, Morishita TY, Ison AJ, Huang S, McDermott PF, Zhang
Q. Effect of conventional and organic production practices on the preva‑
lence and antimicrobial resistance of _Campylobacter_ spp. in poultry. Appl
Environ Microbiol. 2006;72:3600–7.
45. Nguyen TNM, Hotzel H, El-Adawy H, Tran HT, Le MTH, Tomaso H, et al.
Genotyping and antibiotic resistance of thermophilic _Campylobac-_
_ter_ isolated from chicken and pig meat in Vietnam. Gut Pathogens.
2016;8:19.




Nguyen _et al. Gut Pathog (2016) 8:39_ Page 9 of 9



46. Ventola CL. The antibiotic resistance crisis: part 1: causes and threats.
Pharm Ther. 2015;40:277–83.
47. Levy SB. Factors impacting on the problem of antibiotic resistance. J
Antimicrob Chemother. 2002;49:25–30.
48. Zhang Q, Plummer P. Mechanisms of antibiotic resistance in _Campylobac-_
_ter_ . In: Nachamkin I, Szymanski C, Blaser M, editors. _Campylobacter_ . 3rd ed.
Washington DC: ASM Press; 2008. p. 263–76.
49. Iovine NM. Resistance mechanisms in _Campylobacter jejuni_ . Virulence.
2013;4:230–40.



50. Chopra I, Roberts M. Tetracyline antibiotics: mode of action, applications,
molecular biology, and epidemiology of bacterial resistance. Microbiol
Mol Rev. 2001;62:232–60.
51. Kariuki S, Revathi G, Kariuki N, Kiiru J, Mwiruria J, Hart CA. Characterisation
of community acquired non-typhoidal _Salmonella_ from bacteraemia and
diarrhoeal infection in children admitted to hospital in Nairobi Kenya.
BMC Microbiol. 2006;6:101.




