Nguyen et al. Gut Pathog (2016) 8:39
DOI 10.1186/s13099-016-0121-5

Gut Pathogens

RESEARCH

Open Access

# Antimicrobial resistance of Campylobacter isolates from small scale and backyard chicken in Kenya

<div style="text-align: center;"><img src="imgs/img_in_image_box_971_217_1081_260.jpg" alt="Image" width="9%" /></div>


Tuan Ngoc Minh Nguyen $ ^{1,2,3,*} $ , Helmut Hotzel $ ^{2} $ , John Njeru $ ^{2,4} $ , Joyce Mwituria $ ^{4} $ , Hosny El-Adawy $ ^{2,5} $ , Herbert Tomaso $ ^{2} $ , Heinrich Neubauer $ ^{2} $  and Hafez M. Hafez $ ^{3} $ 

## Abstract

Background: Thermophilic Campylobacter species are a major cause of bacterial foodborne diarrhoea in humans worldwide. Poultry and their products are the predominant source for human campylobacteriosis. Resistance of Campylobacter to antibiotics is increasing worldwide, but little is known about the antibiotic resistance in Campylobacter isolated from chicken in Kenya. In this study, 35 suspected Campylobacter strains isolated from faeces and cloacal swabs of chicken were tested for their susceptibility to seven antibiotics using a broth microdilution assay and molecular biological investigations.

Results: Overall, DNA of thermophilic Campylobacter was identified in 53 samples by PCR (34 C. jejuni, 18 C. coli and one mix of both species) but only 35 Campylobacter isolates (31 C. jejuni and 4 C. coli) could be re-cultivated after transportation to Germany. Isolates were tested for their susceptibility to antibiotics using a broth microdilution assay. Additionally, molecular biological detection of antibiotic resistance genes was carried out. C. jejuni isolates showed a high rate of resistance to nalidixic acid, tetracycline and ciprofloxacin of 77.4, 71.0 and 71.0%, respectively. Low resistance (25.8%) was detected for gentamicin and chloramphenicol. Multidrug resistance in C. jejuni could be detected in 19 (61.3%) isolates. Resistance pattern of C. coli isolates was comparable. Resistance to ciprofloxacin was confirmed by MAMA–PCR and PCR–RFLP in all phenotypically resistant isolates. The tet(O) gene was detected only in 54.5% of tetracycline resistant C. jejuni isolates. The tet(A) gene, which is also responsible for tetracycline resistance, was found in 90.3% of C. jejuni and in all C. coli isolates. Thirteen phenotypically erythromycin-resistant isolates could not be characterised by using PCR–RFLP and MAMA–PCR.

Conclusions: To the best of our knowledge, this study is the first report about resistance to antibiotics in thermophilic Campylobacter originating from chicken in Kenya. Campylobacter spp. show a high level of resistance to ciprofloxacin, nalidixic acid and tetracycline but also a remarkable one to chloramphenicol and gentamicin and they are multidrug resistant. Resistance to antibiotics is a global public health concern. In Kenya, resistance surveillance needs further attention in the future. Efforts to establish at least a National Laboratory with facilities for performing phenotypic and genotypic characterization of thermophilic Campylobacter is highly recommended.

Keywords: Campylobacter, Antibiotic resistance, Microdilution, Chicken, Kenya

## Background

Thermophilic Campylobacter (C.) species have become the most frequent cause of bacterial gastroenteritis in humans worldwide  $ [1] $ . Campylobacteriosis exceeds the total number of those caused by Salmonella, Shigella, and Escherichia coli O157:H7 in humans, recently  $ [2] $ . Campylobacter infections are normally self-limiting in adults but can cause diarrhoea or even mortality in children in developing as well as in developed countries  $ [3, 4] $ .



 $ ^{*} $ Correspondence: tuan.nguyen@fli.de; minhtuannn@hvu.edu.vn
 $ ^{2} $  Friedrich-Loeffler-Institut, Institute of Bacterial Infections and Zoonoses, Jena, Germany

Full list of author information is available at the end of the article

© 2016 The Author(s). This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated.

Nguyen et al. Gut Pathog (2016) 8:39

Page 2 of 9

A study from western Kenya showed that 20 % of patients with diarrhoea were infected by Campylobacter but in the group of children below 5 years Campylobacter was represented with 42 %  $ [5] $ .

Commercial poultry and free-living birds are natural reservoirs of thermophilic campylobacteres. The organism has been isolated from numerous bird species, including Columbiformes and domestic and free-living Galliformes and Anseriformes. C. jejuni has been found in all areas of commercial poultry production  $ [6, 7] $ . Prevalence rates in poultry, especially in slaughter-age broiler flocks, could reach 100 % on some farms. Although, Campylobacter is insignificant for poultry health, it is a predominant cause of foodborne gastroenteritis in humans worldwide, and contaminated poultry meat is recognized as the main source of human infections  $ [7, 8] $ . In general, the knowledge about Campylobacter in Kenya is limited. Most of the published reports describe prevalence and antibiotic resistance in Campylobacter of human origin  $ [4, 9–11] $ . Other reports gave information about Campylobacter as cause of foodborne diseases  $ [12] $  and contamination of raw chicken and beef from butcheries and markets in Nairobi  $ [13] $ . Information on thermophilic Campylobacter of animal origin from Kenya is lacking.

Resistance against antibiotics in bacteria is of public health concern. Most commonly used drugs in treatment of campylobacteriosis in humans are erythromycin, fluoroquinolones or tetracycline  $ [14] $ . Although, this antimicrobial treatment is usually not necessary, however the misuse of antibiotics is widespread in Kenya  $ [5] $ . Attention on resistance of Campylobacter is raising and warning has been launched not to misuse antibiotics such as macrolides, fluoroquinolones or alternative drugs  $ [15] $ . Kenyan Campylobacter isolates from humans showed a high resistance rate against erythromycin (52%), but only low resistance to ciprofloxacin, tetracycline and nalidixic acid with 6, 18 and 26 % in the past, respectively  $ [5] $ .

Clinical breakpoints of Campylobacter susceptibility based on epidemiological cut-off (ECOFF) values were recommended. EUCAST MIC distributions incorporate human and veterinary clinical data from several sources worldwide  $ [16] $ . The method of choice for testing antibiotic susceptibility and determination of minimum inhibitory concentration (MIC) values of Campylobacter isolates is the broth microdilution assay  $ [17, 18] $ .

In addition to phenotypical determination of antibiotic resistance, genetic analysis of resistance determinants in Campylobacter can be carried out. A replacement of threonine by isoleucine at amino acid 86 in the gyrA gene  $ [19, 20] $  and a mutation at position 2074 and 2075 on the 23S rRNA gene are the main mechanisms for fluoroquinolone and erythromycin resistance, respectively  $ [21] $ . Presence of  $ \text{tet(O)} $  and/or  $ \text{tet(A)} $  genes is responsible for tetracycline resistance  $ [22] $ . A mismatch amplification mutation assay (MAMA–PCR) can be used for detection of the mutations in gyrA and 23S rRNA genes in C. coli and C. jejuni responsible for ciprofloxacin and erythromycin resistance, respectively  $ [21, 23] $ . PCR-restriction fragment length polymorphism (PCR–RFLP) technique  $ [24] $  is available for detection of erythromycin resistance as well as specific PCR assays for  $ \text{tet(O)} $  and  $ \text{tet(A)} $  genes. These methods allow the investigation of antibiotic resistance of Campylobacter even in samples from which no Campylobacter could be isolated.



To the best of our knowledge there is no report available about antibiotic resistance of thermophilic Campylobacter species isolated from chicken in Kenya. MICs and results of molecular assays on the resistance of recent Kenyan C. coli and C. jejuni are presented.

## Methods

## Sample collection and Campylobacter isolation

In total, 35 geographically different native breed layer flocks were sampled. The chickens were housed in backyards and homesteads of small scale farmers from the outskirts of Thika, a town 40 km northeastern of Nairobi, Kenya. Farmers kept between 10 and 1000 layers. The birds were fed on commercially formulated ration from different sources and sometimes supplied with the leftover and residual food. All the manufactures used antibiotics as part of the ingredients in the feed. During the rearing of these chickens, antibiotics were used for prevention and treatment of diseases without any instructions. Ten to 30 cloacal swabs and faecal samples were collected from each flock according to flock size. Campylobacter were isolated in Kenya Medical Research Institute, Nairobi according to the guidelines of ISO 10272-1 [25]. The isolates were preserved in 1.5 ml Eppendorf tubes filled with skimmed milk medium for 1-week transportation from Kenya to Friedrich–Loeffler-Institut, Jena, Germany for further laboratory analysis. Campylobacter strains were re-cultivated on both Mueller–Hinton agar and CCDA (Oxoid GmbH, Wesel, Germany) under microaerophilic conditions (5 % O₂, 10 % CO₂, and 85 % N₂) at 37 °C for 48–72 h.

## DNA extraction

DNA from viable bacteria was extracted using the High Pure PCR Template Preparation Kit™ (Roche Diagnostics, Mannheim, Germany) according to the manufacturer's instructions. Skimmed milk samples of Campylobacter that could not be re-cultivated were treated with phenol–chloroform to extract DNA. Briefly, 500  $ \mu $ l of skimmed milk medium was boiled for 5 min. After cooling the liquid was mixed with 500  $ \mu $ l buffer-saturated phenol (Carl Roth GmbH, Karlsruhe, Germany)

Nguyen et al. Gut Pathog (2016) 8:39

Page 3 of 9

and centrifuged for 5 min. at 13,400 rpm (miniSpin, Eppendorf, Hamburg, Germany). 500  $ \mu $ l chloroform/isoamyl alcohol (24:1 vol/vol) was added to the aqueous phase, mixed and centrifuged for 5 min. at 13,400 rpm. DNA from the aqueous phase was precipitated by mixing with 0.6 volume of isopropanol at room temperature. After centrifugation, the supernatant was discarded and the DNA was air dried and finally dissolved in 50  $ \mu $ l 10 mM Tris (Carl Roth GmbH).

## Multiplex PCR for identification of Campylobacter species

A mPCR assay was used to identify thermophilic Campylobacter species (C. jejuni, C. coli, and C. lari) as described by El-Adawy et al. [26]. Briefly, the PCR was performed in a 50- $ \mu $ l reaction mixture containing 5.0  $ \mu $ l of 10  $ \times $  Taq reaction buffer complete (Jena Bioscience GmbH, Jena, Germany), 2.0  $ \mu $ l of dNTP mix (2 mM each; Carl Roth GmbH), 2.0  $ \mu $ l of each primer (Jena Bioscience GmbH), and 0.2  $ \mu $ l of Taq Pol thermostable DNA polymerase (Jena Bioscience GmbH). Amplification reactions were carried out in a TRIO Thermoblock cycler (Biometra, Göttingen, Germany) using the following programme: one cycle of 1 min at 96  $ ^{\circ} $ C was followed by 35 cycles each consisting of 60 s at 95  $ ^{\circ} $ C of denaturation, 90 s at 59  $ ^{\circ} $ C of annealing, and 60 s at 72  $ ^{\circ} $ C of elongation. The PCR was terminated after a final extension step of 5 min. at 72  $ ^{\circ} $ C. Amplification generated 857, 589, 522, and 462 base pair DNA fragments specific for the genus Campylobacter and the species C. jejuni, C. lari, and C. coli, respectively. For analysis, 20  $ \mu $ l of PCR products were subjected to electrophoresis in a 1.5 % agarose gel for 1 h, stained with ethidium bromide (0.5  $ \mu $ g/ml), and visualized under UV light. Results were documented using BioImage system GeneGenius (Syngene, Synoptics Ltd., Cambridge, UK). Reference strains C. jejuni DSM 4688, C. coli DSM 4689, and C. lari DSM 11375 (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Braunschweig, Germany) were used as positive controls.

## Antimicrobial susceptibility testing and determination of MICs

The antimicrobial susceptibility of C. jejuni and C. coli isolates was tested against seven antibiotic agents (chloramphenicol, erythromycin, ciprofloxacin, nalidixic acid, gentamicin, streptomycin, and tetracycline) using the Sensititre $ ^{™} $  Campylobacter plates—EUCAMP (Trek Diagnostic Systems Ltd., East Grinstead, UK). The MIC values were detected using different concentration ranges as previously described  $ [23] $ . Briefly, Campylobacter isolates grown on Mueller–Hinton agar (Oxoid GmbH) supplemented with 10 % bovine blood under microaerophilic conditions were suspended in NaCl solution (0.9 %) to obtain a turbidity corresponding to a McFarland standard of 0.5 (Dr. Lange, CADAS photometer 30, Berlin, Germany). One-hundred and fifty milliliters of the above suspension were diluted with 10 ml Mueller–Hinton broth (Oxoid GmbH) resulting in a concentration of approximately  $ 10^{6}-10^{7} $  colony forming units (cfu)/ml. One hundred milliliters of the inoculum was filled in each well of the plate; the plates were sealed and incubated at 37 °C for 24 h under microaerophilic conditions. Results were read either visually or photometrically (Tecan Deutschland GmbH, Crailsheim, Germany) using the computer program easyWIN fitting (version V6.1, 2000). C. jejuni DSM 4688 and C. coli DSM 4689 (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH) were included in each batch of broth microdilution assay for quality control. The lowest concentration of antibiotics that prevents visible growth of the microorganism is defined as the MIC.



## Molecular biological detection of antibiotic resistance determinants

Extracted Campylobacter DNA from all samples and strains was used for molecular biological determination of selected antibiotic resistance determinants by PCR.

## Erythromycin resistance

Detection of mutations at positions 2074 and 2075 in domain V of the 23S rRNA gene, which mediates resistance to erythromycin, was carried out by MAMA–PCR and PCR–RFLP as described previously [21, 24]. Genes responsible for resistance of erythromycin and ciprofloxacin were tested at two loci using MAMA–PCR and PCR–RFLP. Primers and their sequences are given in Table 1.

## Ciprofloxacin resistance

A single point mutation (Thr-86-Ile) in the quinolone resistance-determining region (QRDR) of gyrA gene was defined as source of high-level resistance to fluoroquinolones [23]. MAMA–PCR for C. jejuni isolates was carried out as described previously [27], for C. coli a procedure according to Zirnstein et al. [28] was used. Primer details are given in Table 1.

## Tetracycline resistance

Primers DMT1 and DMT2 (Jena Bioscience GmbH) were used for the detection of the tet(O) gene which is strongly associated with tetracycline resistance in C. jejuni and C. coli as described previously [29]. As a second gene locus associated with tetracycline resistance the presence of tet(A) was examined by a previously described PCR assay [22]. An alternative, in-house validated PCR assay was created based on tet(A) sequences (GenBank acc. no.

Nguyen et al. Gut Pathog (2016) 8:39

Page 4 of 9

<div style="text-align: center;">Table 1 List of primers and primer sequences used for detection of antimicrobial resistance genes</div>



<table border=1 style='margin: auto; width: max-content;'><tr><td style='text-align: center;'>Antibiotic</td><td style='text-align: center;'>Method</td><td style='text-align: center;'>Primer</td><td style='text-align: center;'>Sequence (5′-3′)</td><td style='text-align: center;'>Amplicon length (bp)</td><td style='text-align: center;'>Reference</td></tr><tr><td rowspan="2">Erythromycin</td><td style='text-align: center;'>MAMA-PCR $ ^{a} $</td><td style='text-align: center;'>23SRNA-F ERY2075-R ERY2074-R</td><td style='text-align: center;'>TTA GCT AAT GTT GCC CGT ACC G TAG TAA AGG TCC ACG GGGTCG C AGT AAA GGT CCA CGG GGT CTG G</td><td style='text-align: center;'>485</td><td style='text-align: center;'>[17]</td></tr><tr><td style='text-align: center;'>PCR-RFLP $ ^{a} $</td><td style='text-align: center;'>F2-campy-23S R2-campy-23S</td><td style='text-align: center;'>AAT TGA TGG GGT TAG CAT TAG C CAA CAA TGG CTC ATA TAC AACTGG</td><td style='text-align: center;'>316</td><td style='text-align: center;'>[20]</td></tr><tr><td rowspan="2">Ciprofloxacin</td><td style='text-align: center;'>MAMA-PCR $ ^{b} $</td><td style='text-align: center;'>CampyMAMAgyrA1 CampyMAMAgyrA5</td><td style='text-align: center;'>TTT TTA GCA AAG ATT CTG AT CAA AGC ATC ATA AAC TGC AA</td><td style='text-align: center;'>265</td><td style='text-align: center;'>[19]</td></tr><tr><td style='text-align: center;'>MAMA-PCR $ ^{c} $</td><td style='text-align: center;'>GZgyrACcoli3F CanpyMAMAgyrA8</td><td style='text-align: center;'>TAT GAG CGA TAT TAT CGG TC TAA GGC ATC GTA AAC AGC CA</td><td style='text-align: center;'>192</td><td style='text-align: center;'>[24]</td></tr><tr><td rowspan="3">Tetracycline</td><td style='text-align: center;'>tet(O) PCR</td><td style='text-align: center;'>DMT 1 DMT 2</td><td style='text-align: center;'>GGC GTT TTG TTT ATG TGC G ATG GAC AAC CCG ACA GAA GC</td><td style='text-align: center;'>559</td><td style='text-align: center;'>[25]</td></tr><tr><td style='text-align: center;'>tet(A) PCR</td><td style='text-align: center;'>Tet(A)-F Tet(A)-R</td><td style='text-align: center;'>GTG AAA CCC AAC ATA CCC C GAA GGC AAG CAG GAT GTA G</td><td style='text-align: center;'>888</td><td style='text-align: center;'>[18]</td></tr><tr><td style='text-align: center;'>tet(A) PCR</td><td style='text-align: center;'>tet-A-1 tet-A-2</td><td style='text-align: center;'>GCT CAC GTT GAC GCA GGA AAG ATC GTC ATT GTC CGT TAC</td><td style='text-align: center;'>486</td><td style='text-align: center;'>This study</td></tr></table>

 $ ^{a} $  23S rRNA gene mutation

 $ ^{b} $  gyrA gene mutation Campylobacter jejuni

c gyrA gene mutation Campylobacter coli

JX891463 and JX891464)). Briefly, primers tet-A-1 and tet-A-2 (Table 1; Jena Bioscience GmbH) were used with the following PCR programme: An initial denaturation at 96 °C for 60 s was followed by 35 cycles of denaturation (96 °C for 15 s), annealing (49 °C for 60 s) and extension (72 °C for 30 s). PCR was terminated by final extension at 72 °C for 60 s. The PCR resulted in a 486 bp product.

All PCR products were analyzed by electrophoresis on 1.5 % agarose gels, staining with ethidium bromide and visualization under UV light.

## DNA sequencing

PCR products obtained by tet(A) PCRs were sequenced by cycle sequencing with BigDye Terminator v1.1 Cycle Sequencing Kit (Applied Biosystems, Darmstadt, Germany) according to the instructions of the manufacturer. In addition to the amplification primers of the Tet(A)-F/R fragment tet-A-A (5'-AAT TTT CTT CAA ATA AGG-3') and tet-A-B (5'-GTC ATT CTT ATA TTA AGT GG-3') were used as sequencing primers. Sequencing products were analyzed with an ABI PRISM 3130 genetic analyzer.

## MALDI-TOF mass spectrometry

Cultured bacteria were suspended in 300  $ \mu $ l of bi-distilled water and mixed with 900 ml of ethanol (Carl Roth GmbH). Further treatment of samples and measurement were described by El-Ashker et al. [30].

## Results

## I dentification of bacteria

In total, 58 isolates suspected as Campylobacter were recovered from faeces and cloacal swabs of chicken flocks in Kenya. After storage in skimmed milk medium and transportation to Germany only 40 of these isolates could be re-cultivated. Four C. coli and 31 C. jejuni were identified by mPCR (Table 2). Five other isolates were identified by MALDI-TOF mass spectrometry as members of genera Bacillus, Staphylococcus, Ochrobactrum as well as two Bordetella isolates.



Eighteen skimmed milk tubes contained Campylobacter DNA  $ [14\ C.\ coli,\ 3\ C.\ jejuni\ and\ one\ sample\ harboured\ both\ C.\ coli\ and\ C.\ jejuni\ (Table\ 2)] $ .

## Antimicrobial susceptibility profiles and multidrug resistance

The results of antimicrobial susceptibility testing of C. jejuni and C. coli isolates and the rate of resistance to seven antimicrobial agents are given in Tables 3 and 4, respectively. The C. jejuni isolates showed a high rate of resistance to nalidixic acid, tetracycline and ciprofloxacin with 77.4, 71.0 and 71.0 %, respectively. Low resistance rates were detected for gentamicin and chloramphenicol, both with 25.8 % of the isolates. For the low number of C. coli isolates (n = 4) a similar pattern was observed. Only two isolates were susceptible to all tested antimicrobial agents, one isolate was resistant to all tested antibiotics.

The multidrug resistance profiles of 31 C. jejuni isolates are shown in Table 5. Multidrug resistance to three or more classes of antibiotics was found in 19 isolates (61.3 %) and was observed in a range between 5.3 and 26.3 %.

## Molecular biological detection of antibiotic resistance determinants

DNA of 35 viable Campylobacter isolates and of 18 non-growing samples was investigated by PCR to detect antibiotic resistance. Mismatch amplification mutation assay

Nguyen et al. Gut Pathog (2016) 8:39

Page 5 of 9

<div style="text-align: center;">Table 2 Results of cultivation and multiplex PCR identification of Campylobacter isolates</div>



<table border=1 style='margin: auto; width: max-content;'><tr><td style='text-align: center;'>Cultivation</td><td colspan="4">mPCR identification of Campylobacter</td><td rowspan="2">Total n (%)</td></tr><tr><td style='text-align: center;'></td><td style='text-align: center;'>C. jejuni</td><td style='text-align: center;'>C. coli</td><td style='text-align: center;'>Not identified</td><td style='text-align: center;'>C. coli/C. jejuni</td></tr><tr><td style='text-align: center;'>Positive (n)</td><td style='text-align: center;'>31</td><td style='text-align: center;'>4</td><td style='text-align: center;'>5</td><td style='text-align: center;'>0</td><td style='text-align: center;'>40 (69.0)</td></tr><tr><td style='text-align: center;'>Negative (n)</td><td style='text-align: center;'>3</td><td style='text-align: center;'>14</td><td style='text-align: center;'>0</td><td style='text-align: center;'>1</td><td style='text-align: center;'>18 (31.0)</td></tr></table>

(MAMA–PCR) was used to characterize a gyrA gene mutation associated with ciprofloxacin resistance as well as mutations in 23S rRNA genes as cause of erythromycin resistance. The molecular biological detection of resistance to ciprofloxacin in both C. coli and C. jejuni was also confirmed by change of amino acid 86 from threonine to isoleucine in the gyrA gene. Additional to the ciprofloxacin resistant Campylobacter isolates (Tables 3, 4), three Campylobacter DNAs were detected harbouring the gyrA gene mutation. The results were confirmed by PCR–RFLP according to Vacher et al. [24]. Mutations at positions 2074 and 2075 of the 23S rRNA genes in 13 phenotypically erythromycin-resistant isolates could neither be detected by using PCR–RFLP nor MAMA–PCR.



The  $ tet(O) $  gene which is mainly responsible for tetracycline-resistance was detected by PCR in 12 out of 22 resistant C. jejuni isolates (54.5 %) and in all tetracycline resistant C. coli isolates.  $ tet(O) $  gene was not detected in DNA extracted from the non-growing samples. Additionally, a newly developed PCR assay was used for the detection of the  $ tet(A) $  gene.  $ tet(A) $  was detected in 28 out of 31 C. jejuni (90.3 %) and in all 4 C. coli isolates. In 3 out of 14 non-growing samples which harboured C. coli DNA,  $ tet(A) $  gene could be found as well as in the one sample where both C. jejuni and C. coli were detected.

<div style="text-align: center;">Table 3 Results of MIC determination and resistance rates of Kenyan Campylobacter jejuni isolates</div>



<table border=1 style='margin: auto; width: max-content;'><tr><td rowspan="2">Class</td><td rowspan="2">Antibiotic ( $ \mu $ g/ml)</td><td colspan="12">Antibiotic susceptibility of Campylobacter jejuni (n=31)</td><td rowspan="9">R $ ^{*} $  (%)</td></tr><tr><td style='text-align: center;'>0.06</td><td style='text-align: center;'>0.12</td><td style='text-align: center;'>0.25</td><td style='text-align: center;'>0.5</td><td style='text-align: center;'>1</td><td style='text-align: center;'>2</td><td style='text-align: center;'>4</td><td style='text-align: center;'>8</td><td style='text-align: center;'>16</td><td style='text-align: center;'>32</td><td style='text-align: center;'>64</td><td style='text-align: center;'></td></tr><tr><td rowspan="2">Macrolides</td><td style='text-align: center;'>Chloramphenicol</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>9</td><td style='text-align: center;'>3</td><td style='text-align: center;'>6</td><td style='text-align: center;'>5</td><td style='text-align: center;'>8</td><td style='text-align: center;'></td><td style='text-align: center;'>25.8</td></tr><tr><td style='text-align: center;'>Erythromycin</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>14</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>1</td><td style='text-align: center;'>2</td><td style='text-align: center;'>2</td><td style='text-align: center;'>12</td><td style='text-align: center;'></td><td style='text-align: center;'>51.6</td></tr><tr><td style='text-align: center;'>Fluoroquinolones</td><td style='text-align: center;'>Ciprofloxacin</td><td style='text-align: center;'>5</td><td style='text-align: center;'>2</td><td style='text-align: center;'>2</td><td style='text-align: center;'></td><td style='text-align: center;'>2</td><td style='text-align: center;'>1</td><td style='text-align: center;'>19</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>71.0</td></tr><tr><td style='text-align: center;'>Quinolones</td><td style='text-align: center;'>Nalidixic acid</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>4</td><td style='text-align: center;'>1</td><td style='text-align: center;'>2</td><td style='text-align: center;'>2</td><td style='text-align: center;'>22</td><td style='text-align: center;'>77.4</td></tr><tr><td rowspan="2">Aminoglycosides</td><td style='text-align: center;'>Gentamicin</td><td style='text-align: center;'></td><td style='text-align: center;'>2</td><td style='text-align: center;'>1</td><td style='text-align: center;'>4</td><td style='text-align: center;'>11</td><td style='text-align: center;'>5</td><td style='text-align: center;'>2</td><td style='text-align: center;'>1</td><td style='text-align: center;'>5</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>25.8</td></tr><tr><td style='text-align: center;'>Streptomycin</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>12</td><td style='text-align: center;'>2</td><td style='text-align: center;'>4</td><td style='text-align: center;'>1</td><td style='text-align: center;'>12</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>41.9</td></tr><tr><td style='text-align: center;'>Tetracyclines</td><td style='text-align: center;'>Tetracycline</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>6</td><td style='text-align: center;'>1</td><td style='text-align: center;'>2</td><td style='text-align: center;'></td><td style='text-align: center;'>1</td><td style='text-align: center;'>1</td><td style='text-align: center;'>20</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>71.0</td></tr></table>

Boldface in italic type indicates the number of resistant isolates. A thick black line indicates the break point between clinically sensitive and resistant strains  $ R^{*} $  resistance rate

<div style="text-align: center;">Table 4 Results of MIC determination and resistance rates of Kenyan Campylobacter coli isolates</div>



<table border=1 style='margin: auto; width: max-content;'><tr><td rowspan="2">Class</td><td rowspan="2">Antibiotic ( $ \mu $ g/ml)</td><td colspan="12">Antibiotic susceptibility of Campylobacter coli (n=4)</td><td rowspan="4">R $ ^{*} $  (%)</td></tr><tr><td style='text-align: center;'>0.06</td><td style='text-align: center;'>0.12</td><td style='text-align: center;'>0.25</td><td style='text-align: center;'>0.5</td><td style='text-align: center;'>1</td><td style='text-align: center;'>2</td><td style='text-align: center;'>4</td><td style='text-align: center;'>8</td><td style='text-align: center;'>16</td><td style='text-align: center;'>32</td><td style='text-align: center;'>64</td><td style='text-align: center;'></td></tr><tr><td rowspan="2">Macrolides</td><td style='text-align: center;'>Chloramphenicol</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>2</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>2</td><td style='text-align: center;'></td><td style='text-align: center;'>50.0</td></tr><tr><td style='text-align: center;'>Erythromycin</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>4</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>0</td></tr><tr><td style='text-align: center;'>Fluoroquinolones</td><td style='text-align: center;'>Ciprofloxacin</td><td style='text-align: center;'>1</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>3</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>75.0</td></tr><tr><td style='text-align: center;'>Quinolones</td><td style='text-align: center;'>Nalidixic acid</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>1</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>3</td><td style='text-align: center;'>75.0</td></tr><tr><td rowspan="2">Aminoglycosides</td><td style='text-align: center;'>Gentamicin</td><td style='text-align: center;'></td><td style='text-align: center;'>1</td><td style='text-align: center;'></td><td style='text-align: center;'>1</td><td style='text-align: center;'>1</td><td style='text-align: center;'></td><td style='text-align: center;'>1</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>25.0</td></tr><tr><td style='text-align: center;'>Streptomycin</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>2</td><td style='text-align: center;'></td><td style='text-align: center;'>2</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>0</td></tr><tr><td style='text-align: center;'>Tetracyclines</td><td style='text-align: center;'>Tetracycline</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>1</td><td style='text-align: center;'>1</td><td style='text-align: center;'>1</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>1</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'>25.0</td></tr></table>

Boldface in italic type indicates the number of resistant isolates. A thick black line indicates the break point between clinically sensitive and resistant strains  $ R^{*} $  resistance rate

Nguyen et al. Gut Pathog (2016) 8:39

Page 6 of 9

<div style="text-align: center;">Table 5 Multidrug resistance profiles of 19 Campylobacter jejuni isolates</div>



<table border=1 style='margin: auto; width: max-content;'><tr><td style='text-align: center;'>Antibiotic resistance profile</td><td style='text-align: center;'>No. of resistant isolates (%)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL, ERY, GEN, STR, CHL</td><td style='text-align: center;'>1 (5.3)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL, ERY, STR, CHL</td><td style='text-align: center;'>1 (5.3)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL, ERY, GEN, STR</td><td style='text-align: center;'>3 (15.8)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL, GEN, STR, CHL</td><td style='text-align: center;'>1 (5.3)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL, ERY, STR</td><td style='text-align: center;'>2 (10.5)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL, ERY, CHL</td><td style='text-align: center;'>1 (5.3)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL, ERY</td><td style='text-align: center;'>2 (10.5)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL, STR</td><td style='text-align: center;'>1 (5.3)</td></tr><tr><td style='text-align: center;'>TET, CIP, NAL</td><td style='text-align: center;'>5 (26.3)</td></tr><tr><td style='text-align: center;'>CIP, NAL, ERY, STR, CHL</td><td style='text-align: center;'>1 (5.3)</td></tr><tr><td style='text-align: center;'>CIP, NAL, STR</td><td style='text-align: center;'>1 (5.3)</td></tr></table>

TET tetracycline, CIP ciprofloxacin, NAL nalidixic acid, ERY erythromycin, GEN gentamicin, STR streptomycin, CHL chloramphenicol

## Discussion

The antimicrobial susceptibility patterns among Campylobacter isolates originating from chicken in Kenya were analyzed according to the guidelines of CLSI for Enterobacteriaceae which had been guided by previous reports  $ [17, 31–34] $ . Clinical breakpoints for interpretation of MIC values of C. jejuni and C. coli from chicken are available  $ [31, 35] $ . In this study a commercially available broth microdilution assay was used for the determination of MIC values for seven antibiotics. The assay already proved to be suitable in previous investigations  $ [21, 27, 34, 36, 37] $ .

In this study, only 40 out of 58 suspected Campylobacter samples could be re-cultivated. The storage conditions (temperature, microaerophilic atmosphere) using skimmed milk medium were possibly not ideal. However, it had been demonstrated that C. jejuni can survive up to 14 days at 1  $ {}^{\circ} $ C or 2.5 days at 20  $ {}^{\circ} $ C in sterile skimmed milk [38, 39]. Alternative storage of Campylobacter cultures using transport medium (for example Amies medium) or cryovials is recommended for future investigations.

In 53 out of 58 collected samples, Campylobacter DNA was identified by mPCR assay  $ [26] $ . The majority of the cultures proved to be C. jejuni (88.6 %) which is in agreement with previous studies in chicken  $ [6, 7] $ . In 18 DNA extracts of non-viable samples, 3 C. jejuni (16.7 %), one mixed population of C. jejuni/C. coli and 14 C. coli were identified. These findings are in agreement with those of a previous study that found a longer viability of C. jejuni in comparison to C. coli in biological milieu  $ [40] $ . It may be possible that some C. coli isolates had been lost during the shipment period. In summary, C. jejuni was identified much more often than C. coli (64.2 %) by mPCR investigation. In agreement with other studies, the findings highlighted the usefulness of mPCR as a reliable, sensitive, time and cost saving method for identification of thermophilic Campylobacter  $ [26] $ .



The antibiotic susceptibility of 35 Campylobacter isolates from Kenyan chicken was investigated using European Committee on Antimicrobial Susceptibility Testing and epidemiological cut-off values (EUCAST–ECOFFS) $ ^{[16]} $ . A broth microdilution assay was used as a standardized, easy, and reliable method for the determination of MIC of seven antibiotics  $ [17, 31–34] $ . High resistance rates were obtained for ciprofloxacin, nalidixic acid and tetracycline with more than 70 % which is in agreement with a recent European Food Safety Authority (EFSA) report  $ [41] $ . These results are in contrast to those of Brooks et al.  $ [5] $  who reported resistance rates for Campylobacter recovered from humans with diarrhoea in Western Kenya for ciprofloxacin, nalidixic acid and tetracycline with 6, 26 and 18 % in 2006, respectively. The general high rates of resistance in the chicken isolates may be caused by availability and uncontrolled use of antibiotics by small farmers  $ [42] $ .

Resistance to chloramphenicol is remarkable with 25.8 % in this investigation. Use of chloramphenicol is banned in animal breeding in Europe for more than 20 years, but still it is often used in many third world countries [43]. It is easy to obtain antibiotics over-the-counter and other unregulated venues and injudicious use promotes the development of resistance to antimicrobial agents. Resistance to gentamicin in the isolates obtained from chicken was low in this study (25.8 %), but Campylobacter isolated from broilers and turkeys were totally susceptible to gentamicin [37, 41, 44, 45]. Erythromycin resistance rates found in this study correspond to those of similar studies elsewhere [41, 44, 45].

Multidrug resistance was detected in 61.3 % of the Campylobacter isolates. Eleven different combinations were found (Table 5). Frequent, resistance to ciprofloxacin, nalidixic acid and tetracycline was identified (17 out of 19 multidrug resistant isolates) which is in agreement with previous investigation using Vietnamese Campylobacter isolates [45]. However, EFSA [41] reported low level of multidrug resistance in C. jejuni from broilers of the member states of the EU.

The emerging of antibiotic resistance has been attributed to the overuse and misuse of antimicrobial agents in both the developed and developing world. Antibiotics are widely used as growth supplements in livestock and to prevent infections  $ [46] $ . The emerging of multidrug resistance may reflect acquisition of different resistance determinants on the same DNA molecule or single determinants, such as multidrug pumps, that specify efflux activity against different antimicrobial agents  $ [47] $ . The mechanisms of genetic resistance might be chromosomal or plasmid-borne, and represent a combination of endogenous and acquired genes. In general, mechanisms of

Nguyen et al. Gut Pathog (2016) 8:39

Page 7 of 9

antibiotic resistance as modification of the antibiotic by aminoglycoside-modifying enzymes (AphA, AadE, Sat), enzymatic inactivation of the antibiotic by  $ \beta $ -lactamase and modification of the DNA gyrase target, mutations in 23S rRNA genes were included for aminoglycosides, beta-lactams, fluoroquinolones, macrolides and tetracyclines, respectively [48, 49]. The multidrug efflux pump CmeABC has been involved in the resistance mechanisms of C. jejuni and C. coli to tetracyclines, fluoroquinolones, macrolides and beta-lactams [49].

Molecular biological methods were used for detection of antibiotic resistance determinants either using DNA isolated from cultures or that of non-cultured bacteria  $ [27, 37] $ . All isolates of this study which were resistant to ciprofloxacin carried a mutation of the amino acid 86 of the gyrA resulting in a change from threonine to isoleucine. This mutation was detected also in 3 DNA samples extracted from skimmed milk. The MAMA–PCR protocol allowed the detection of the gyrA mutation and PCR–RFLP was confirming the mutation from (ACA to ATA) of amino acid 86. This result was in agreement with previous reports showing that both methods are simple, reliable, rapid tools that can be used as screening methods  $ [27, 37] $ . In Campylobacter, resistance to erythromycin is chemosomally encoded by an alteration of the 23S rRNA gene. High level resistance to erythromycin is caused by mutations at position 2074 and/or 2075 of the domain V of this gene. In this study the mutations were neither detected by MAMA–PCR nor by PCR–RFLP.

The tet(O) gene is known to be responsible for tetracycline resistance in Campylobacter isolates  $ [29] $ . In this study, only 54.5 % of the tetracycline resistant isolates harboured the tet(O) gene. The tet(A) gene also plays a role in resistance to tetracycline  $ [22] $ . The efflux gene tet(A) is coding for an approximately 46 kDa membrane-bound efflux protein for membrane-associated proteins and is involved in the export of tetracycline from the cell  $ [50] $ . In this study, using the recommended primers for tet(A) amplification  $ [18] $  PCR products of 696 bp instead of 888 bp were obtained. DNA sequencing of amplicons and database search resulted in 99.0 % homology to a partial putative integral membrane protein and a putative periplasmic protein. Hence, a new PCR assay based on tet(A) gene sequences for C. jejuni (acc. no. JX891464) and C. coli (acc. no. JX891463) was developed. Parameters such as limit of detection, limit of quantification, PCR efficiency and specificity were considered during an in-house validation process. Amplicon length was 486 and the amplicons were sequenced to confirm the identity. The tet(A) gene was much more frequently identified in the Kenyan Campylobacter isolates than tet(O) (35 vs 13).

To the best of our knowledge this is the first report on the status of antibiotic susceptibility of thermophilic Campylobacter from chicken in Kenya. High level of resistance to ciprofloxacin, erythromycin and nalidixic acid as well as multidrug resistance was detected previously in Kenya. In Kenya, this problem is reported to be caused by the increasing rate of unregulated over-the-counter sale without prescriptions of these antibiotics, mainly to humans self-treatment of suspected infections and to a lesser extent for use in animals  $ [51] $ . These findings also demonstrate the potential for antibiotic-resistant bacteria to spread through the food chain from animals treated with antibiotics for humans. Such misuse and overuse may have resulted in the selection of resistant mutants or acquisition of antibiotic resistance genes from other organisms through the process of genetic exchange.



It is recommendable that a long-term local surveillance programme is adopted for monitoring changes in resistance among Campylobacter isolates. Efforts to establish at least a National Laboratory with facilities for performing phenotyping and genotyping methods is highly recommended. Emphasis should be given on educational advertising to reduce the input of antibiotics in animal breeding to minimize the potential hazard for humans.

## Authors' contributions

TNMN, HH, JN, HE, HT and HMH participated in the conception and design of the study. TNMN, HH, JN, JM and HE performed farm and laboratory work. TNMN, HH, JN, HE, HT, HN and HMH analysed the data and wrote the manuscript. All authors contributed to the analysis and supported the manuscript discussion. All authors read and approved the final manuscript.

## Author details

 $ ^{1} $  Hung Vuong University, Phu Tho, Vietnam.  $ ^{2} $  Friedrich-Loeffler-Institut, Institute of Bacterial Infections and Zoonoses, Jena, Germany.  $ ^{3} $  Institute of Poultry Diseases, Free University Berlin, Berlin, Germany.  $ ^{4} $  Centre for Microbiology Research (CMR), Kenya Medical Research Institute, Nairobi, Kenya.  $ ^{5} $  Department of Poultry Diseases, Faculty of Veterinary Medicine, Kafrelsheikh University, Kafrelsheikh, Egypt.

## Acknowledgements

The authors thank Byrgit Hofmann and Ronald Ng'etich for excellent technical assistance.

## Competing interests

The authors declare that they have no competing interests.

## Availability of data and material

The data supporting the findings of this study are contained within the manuscript.

## Ethics approval and consent to participate

This study was carried out in strict accordance with the recommendations of the National Guidelines on the Care and Use of Animals in Research, Education and Training in Kenya, Consortium for National Health Research (CNHR) which complies with the international laws and regulations regarding ethical considerations, transport, housing and experimental use of animals in research.

## Funding

This study was supported by Friedrich–Loeffler-Institut, Institute of Bacterial Infections and Zoonoses, Germany and Vietnam Ministry of Agriculture and Rural Development.

Received: 9 June 2016 Accepted: 12 August 2016 Published online: 27 August 2016



Nguyen et al. Gut Pathog (2016) 8:39

Page 8 of 9

