Gharieb _et al. Gut Pathog (2015) 7:34_
DOI 10.1186/s13099-015-0081-1

### **RESEARCH**


## Gut Pathogens

### **Open Access**


# Non‑Typhoidal Salmonella in poultry meat and diarrhoeic patients: prevalence, antibiogram, virulotyping, molecular detection and sequencing of class I integrons in multidrug resistant strains

Rasha M. Gharieb [1*], Yasmine H. Tartor [2] and Mariam H. E. Khedr [3]


**Abstract**
**Background:** The worldwide increase of food-borne infections with antibiotic resistant pathogens constitutes a
major public health problem. Therefore, this study aimed to determine the prevalence, antibiogram, virulence genes
profiles and integron characteristics of non-typhoidal _Salmonella_ spp. isolated from poultry meat and diarrhoeic
patients in Egypt.
**Methods:** A total of 150 samples comprising (100 poultry meat and 50 diarrhoeic patients’ stool) were examined for
the presence of _Salmonella_ spp. using culture methods followed by biochemical and serological identification of the
isolates. All _Salmonella_ strains were tested for their susceptibility to the antibiotics using disk diffusion method and
screened for the presence of virulence genes and class I integrons using PCR.
**Results:** The overall prevalence of _Salmonella_ spp. in poultry meat samples was 10 % compared to 4 % in diarrhoeic
patients. All the isolates were serologically identified into _Salmonella_ Typhimurium (seven isolates), _S._ Derby, _S._ Kiel,
_S._ Rubislaw (one isolate, each) and untypable strains (two isolates). Antibiotic susceptibility testing showed a higher
resistance of the total isolates to erythromycin and tetracycline (100 %, each), followed by amoxicillin-clavulanic acid
(91.7 %), trimethoprim-sulfamethoxazole (83.3 %), streptomycin, nalidixic acid, ampicillin-sulbactam (75 %, each), gentamycin, ampicillin (66.7 %, each), chloramphenicol (58.3 %), ciprofloxacin (25 %) and ceftriaxone (16.7 %). Virulence
genes profiles revealed the presence of _sop_ B gene in five _Salmonella_ strains isolated from poultry meat (n = 3) and
humans (n = 2). Moreover, _pef_ A was only identified in three isolates from poultry meat. On the other hand, _S._ Kiel and
_S._ Typhimurium (one isolate, each) were harboring _hilA_ and _stn_ genes, respectively. Class 1 integrons were detected in
all _Salmonella_ spp. with variable amplicon sizes ranged from 650–3000 bp. Sequencing of these amplicons revealed
the presence of gene cassettes harboring _aac(3)_  - _Id_, _aadA2_, _aadA4_, _aadA7_, _sat_, _dfrA15_, _lnuF_ and _estX_ resistance genes.
Nucleotide sequence analysis showed point mutations in the _aac(3)_  - _Id_ of _S._ Derby, _aadA2_, _estX_  - _sat_ genes of _S._ Typhimurium. Meanwhile, frame shift mutation was observed in _aad_ A7 genes of _S._ Typhimurium.
**Conclusions:** Increasing rate of antimicrobial resistance and class 1 integrons among multidrug resistant _Salmonella_
spp. has prompted calls for the reduction of antimicrobial use in livestock to prevent future emergence of resistance.
**Keywords:** _Salmonella_ spp., Zoonoses, multidrug resistance, Virulence, Class I integrons


*Correspondence: ghariebrasha@gmail.com
1 Depatment of Zoonoses, Faculty of Veterinary Medicine, Zagazig
University, 44511 Zagazig, Egypt
Full list of author information is available at the end of the article


© 2015 Gharieb et al. This article is distributed under the terms of the Creative Commons Attribution 4.0 International License
[(http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium,](http://creativecommons.org/licenses/by/4.0/)
provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license,
[and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (http://creativecommons.org/](http://creativecommons.org/publicdomain/zero/1.0/)
[publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated.](http://creativecommons.org/publicdomain/zero/1.0/)




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 2 of 11



**Background**
Non-typhoidal _Salmonella_ (NTS) is one of the major
zoonotic food-borne pathogens representing an important public health problem worldwide. It can cause a
variety of clinical manifestations ranging from mild gastroenteritis to bacteraemia and extra intestinal localized
infections involving many organs. The global burden of
NTS gastroenteritis has been estimated to be 93.8 million cases of gastroenteritis each year with 155, 000
deaths annually [1]. Food of animal origin especially
poultry meat is considered to be one of the major vehicles of _Salmonella_ infections in humans and has been
implicated in outbreaks of human salmonellosis [2].
Antimicrobial resistance and virulence of _Salmonella_
strains play a vital role in systemic infections. _Salmonella_
pathogenicity is dictated by an array of factors encoded
by virulence genes that assist the organism to express
its virulence in the host cells and ultimately manifest in
the typical symptoms of salmonellosis. Some genes are
known to be involved in adhesion and invasion including; plasmid encoded fimbriae ( _pefA_ ) and hyper invasive
locus ( _hilA_ ) [3, 4]. _Salmonella_ outer proteins (sop A-E)
encoded by _sop_ gene and ( _stn_ ) codes for enterotoxin productions are associated with the actual manifestation of
pathogenic processes [5]. In addition to virulence factors, the emergence of antimicrobial resistance among
NTS has become a public health threat. Since food of
animal origin is a major source of _Salmonella_ spp., it
has been suggested that abundant use of antimicrobials
in food animals’ production may contribute to the presence of antimicrobial resistance in these species and subsequently transferred to humans through food chain [6].
Factors contributed to the resistance and virulence of
_Salmonella_ may be located on chromosomes, plasmids,
transposon and integrons. Integrons are DNA elements
that can transfer antibiotic resistance genes among bacteria. Class I integrons are the most common type of
integrons recognized among the multidrug resistant



(MDR) _Salmonella_ and have conserved regions (5′-CS
and 3′-CS) which often contain gene cassettes [7]. These
cassettes contain genes that confer resistance to antimicrobial agents including aminoglycosides, b-lactams,
chloramphenicol and trimethoprim as well as genes that
confer resistance to antiseptics and disinfectants [8]. The
present study was undertaken to (i) trace the prevalence
and antimicrobial susceptibility patterns of NTS _enter-_
_ica_ serotypes isolated from poultry meat and diarrhoeic
patients (ii) ascertain the presence of virulence genes
and class I integrons using PCR (iii) sequencing of the
amplified DNA fragments of class I integrons in order
to identify the resistance genes located in integron gene
cassettes.


**Results and discussion**
**Prevalence of** _**Salmonella**_ **serovars in poultry meat**
**and diarrhoeic patients**
In the present study, the overall prevalence of _Salmonella_
spp. in the total examined poultry meat samples was
10 % (Table 1). This result was compatible with previous
studies in Nepal [9]. Moreover, our results regarding the
prevalence of _Salmonella_ spp. in broiler chicken meat
(14 %) substantiate the findings of others in Canada [10].
Serotyping of _Salmonella_ spp. isolated from poultry
meat in this study revealed the predominance of _S._ Typhimurium in chicken meat and this was in concordance
with previous studies from India [11]. Additionally, other
serovars such as _S._ Rubislaw, _S._ Kiel (chicken meat) and _S._
Derby (duck meat) were identified.
It was obvious from Table 1 that 4 % (2 out of 50) of
stool samples from diarrhoeic patients were positive
for _Salmonella_ spp. and _S._ Typhimurium was the only
serotype identified. Nearly similar prevalence in diarrhoeic patients was reported in India [12]. The isolation
of _S._ Derby from poultry meat and humans has been
previously recorded in Netherlands [13]. Moreover, _S._
Rubislaw was isolated from chicken meat in Senegal and



**Table 1 Frequency distribution of** _**Salmonella**_ **serovars in poultry meat and diarrhoeic patients**

**Source** **Poultry meat** **Diarrhoeic patients (n = 50)** **Total (150)**

**Chicken (n = 50)** **Duck (n = 50)** **Total (n = 100)**

**No. (%)** **No. (%)** **No. (%)** **No. (%)** **No. (%)**


Serotype

_S._ Typhimurium (O1,4,[5],12 H1: i H2: 1,2) 5 (10) 0 (0) 5 (5) 2 (4) 7 (4.7)

_S._ Rubislaw (O11 H1: r H2: e,n,x) 1 (2) 0 (0) 1 (1) 0 (0) 1(0.7)

_S._ Kiel (O1,2,12 H1: g, p H2:–) 1 (2) 0 (0) 1 (1) 0 (0) 1 (0.7)

_S._ Derby (O1,4,[5],12 H1: f,g H2: [1,2]) 0 (0) 1 (2) 1 (1) 0 (0) 1 (0.7)

Untypable 0 (0) 2 (4) 2 (2) 0 (0) 2 (1.3)

Total 7 (14) 3 (6) 10 (10) 2 (4) 12 (8)




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 3 of 11



diarrhoeic children in Gambia [14] and this confirms the
zoonotic importance of the former serotypes.


**Antibiotic susceptibility testing**
All _Salmonella_ isolates were tested for their susceptibility towards antimicrobial drugs (Table 2). The results
highlighted the higher resistance of the isolates to
erythromycin and tetracycline (100 %, each), followed
by amoxicillin-clavulanic acid (91.7 %), trimethoprimsulfamethoxazole (83.3 %), streptomycin, nalidixic acid,
ampicillin-sulbactam (75 %, each), gentamycin, ampicillin (66.7 %, each), chloramphenicol (58.3 %), ciprofloxacin (25 %) and ceftriaxone (16.7 %). This isn’t surprising
because these antibiotics are cheap, easily affordable and
commonly used in humans and poultry without prescription. In poultry, these drugs are used either for therapeutic purposes or as growth promoters added to the feed
leading to the development of resistance in the enteric
bacterial flora of poultry. Subsequently the pathogenic
bacteria such as _Salmonella_ may acquire resistance from
this enteric flora and transfer this resistance to human’s
strains through food chain leading to the emergence of
MDR _Salmonella_ strains that constitute a public health
risk and potentially affect the efficacy of drug treatment
in humans. It was obvious from Table 2 that 100 % of _Sal-_
_monella_ spp. isolated from poultry meat were resistant
to each of erythromycin and tetracycline, 9 (90 %) were
resistant to amoxicillin-clavulanic acid, 8 (80 %) showing
resistance to trimethoprim-sulfamethoxazole, 7 (70 %)
exhibiting resistance to each of streptomycin, ampicillinsulbactam, nalidixic acid and gentamycin. Moreover, the
lower resistance rates to chloramphenicol (50 %), ciprofloxacin (30 %) and ceftriaxone (10 %) were observed.



Nearly similar resistance rate to ampicillin, streptomycin,
nalidixic acid, tetracycline and trimethoprim-sulfamethoxazole was recorded in _Salmonella_ spp. isolated from
chicken meat in Pakistan [15]. On the contrary, the lower
resistance rate to ceftriaxone and ciprofloxacin corroborates with others in India [11].
Table 2 also showed that all _Salmonella_ isolates of
human origin were resistant to ampicillin, erythromycin, streptomycin, tetracycline, ampicillin-sulbactam,
trimethoprim- sulfamethoxazole, chloramphenicol, nalidixic acid and amoxicillin-clavulanic acid. Conversely,
the resistance to ceftriaxone and gentamycin was found
in 50 % of the isolates. On the other hand, none of the
isolates was resistant to ciprofloxacin. Consistent with
our results, a higher resistance rate (100 %) to ampicillin, chloramphenicol, streptomycin, trimethoprimsulfamethoxazole and tetracyclines was recorded in _S._
Typhimurium strains isolated from diarrhoeic patients in
Kenya [16]. Another worrisome situation in this study is
that, 50 % of _Salmonella_ isolates from diarrhoeic patients
were resistant to ceftriaxone and gentamycin. This is
of particular concern because the extended spectrum
cephalosporins such as ceftriaxone are the antibiotics
of choice for treatment of invasive salmonellosis in children and the emergence of resistance toward these drugs
could be attributed to inappropriate use of them in treatment of _Salmonella_ infections in humans. The higher
susceptibility of _S_ . Typhimurium isolated from human
source to ciprofloxacin (100 %) in this study may be due
to discriminate use of this drug because it is relatively
expensive, not easily affordable to all people, not sold in
private pharmacies without prescription. This result corroborates the findings of other investigators in Kenya



**Table 2 Antibiogram of** _**Salmonella**_ **serovars isolated from poultry meat and diarrhoeic patients**


**Sources/serovars** **Antibiotic (concentration µg)**


**A (10)** **E (15)** **S (10)** **Cx (5)** **T (30)** **Co (30)** **As (20)** **Sxt (25)** **C (30)** **Na (30)** **G (10)** **Amc (30)**


Chicken meat (7) 4 (57.1) 7 (100) 5 (71.4) 2 (28.6) 7 (100) 1 (14.3) 5 (71.4) 6 (85.7) 4 (57.1) 5 (71.4) 6 (85.7) 6 (85.7)

_S._ Typhimurium (5) 2 (20) 5 (100) 4 (80) 2 (40) 5 (100) 1 (20) 4 (80) 5 (100) 3 (60) 4 (80) 5 (100) 4 (80)

_S._ Kiel (1) 1 (100) 1 (100) 0 0 1 (100) 0 0 0 0 0 0 1 (100)

_S._ Rubislaw (1) 1 (100) 1 (100) 1 (100) 0 1 (100) 0 1 (100) 1 (100) 1 (100) 1 (100) 1 (100) 1 (100)

Duck meat (3) 2 (66.7) 3 (100) 2 (66.7) 1 (33.3) 3 (100) 0 2 (66.7) 2 (66.7) 1 (33.3) 2 (66.7) 1 (33.3) 3 (100)

_S._ Derby(1) 0 1 (100) 1 (100) 0 1 (100) 0 0 0 0 0 0 1 (100)

Untypable(2) 2 (100) 2 (100) 1 (50) 1 (50) 2 (100) 0 2 (100) 2 (100) 1 (50) 2 (100) 1 (50) 2 (100)

Total (10) 6 (60) 10 (100) 7 (70) 3 (30) 10 (100) 1 (10) 7 (70) 8 (80) 5 (50) 7 (70) 7 (70) 9 (90)

Human (2) _S._ Typhimurium 2 (100) 2 (100) 2 (100) 0 2 (100) 1 (50) 2 (100) 2 (100) 2 (100) 2 (100) 1 (50) 2 (100)

Total sensitive (%) 2 (16.7) 0 2 (16.7) 4 (33.3) 0 4 (33.3) 1 (8.3) 2 (16.7) 2 (16.7) 0 2 (16.7) 0

Total intermediate (%) 2 (16.7) 0 1 (8.3) 5 (41.7) 0 6 (50) 2 (16.7) 0 3 (25) 3 (25) 2 (16.7) 1 (8.3)

Total resistant (%) 8 (66.7) 12 (100) 9 (75) 3 (25) 12 (100) 2 (16.7) 9 (75) 10 (83.3) 7 (58.3) 9 (75) 8 (66.7) 11 (91.7)


_A_ Ampicillin, _E_ Erythromycin, _S_ Streptomycin, _Cx_ Ciprofloxacin, _T_ Tetracycline, _Co_ Ceftriaxone, _As_ Ampicillin-Sulbactam, _Sxt_ Trimethoprim-Sulfamethoxazole, _C_
Chloramphenicol, _Na_ Nalidixic acid, _G_ Gentmycin, _Amc_ Amoxicillin-Clavulanic acid




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 4 of 11




[16]. Therefore, ciprofloxacin still to have a high potency
against NTS strains in humans and continue to be the
most successful drug used for treatment of septicaemic
salmonellosis in adult humans. The increased resistance
of _S._ Typhimurium isolated from humans to quinolones
(nalidixic acid) in this study (100 %) is a matter of concern and could be attributed to indiscriminate use of
these antibiotics by human patients. Moreover, the presence of these drug residues in food of animal origin may
results in increased resistance among human’s isolates.
Table 3 verified that all _Salmonella_ spp. isolated from
poultry and humans were MDR, exhibiting resistance to
at least four or more antibiotics displayed by ten resistance patterns.


**Molecular detection of** _**Salmonella**_ **virulence genes**
**(virulotyping)**
Table 3 revealed the presence of _pef_ A gene among three
_Salmonella_ strains isolated from poultry meat, belonged
to _S._ Rubislaw, _S._ Typhimurium and untypable strain
(one isolate, each) (Fig. 1). Moreover, _hil_ A _gene_ was only
identified in one isolate of _S._ Kiel (Fig. 2). Furthermore,
three _Salmonella_ isolates from poultry meat comprising
untypable strains (two isolates) and _S._ Typhimurium (one



isolate) were harboring _sop_ B gene (Fig. 3). The detection
of _pef_ A gene in _S._ Rubislaw in Gambia and Senegal [14]
and _S._ Typhimurium in India [4] was previously reported.
On the other hand, _hil_ A gene was identified in _Salmo-_
_nella_ serotypes isolated from chickens in Iran [17].
It was obvious from Table 3 that all _S._ Typhimurium
isolates from humans were harboring _sop_ B compared to
one isolate had _stn_ gene (Figs. 3, 4). Consistent with our
findings, _sop_ B gene was previously identified in _S._ Typhimurium isolated from diarrhoeic patients and birds in
India [18]. Meanwhile, _stn_ gene was detected in _S._ Typhimurium isolated from humans in India [4].


**Molecular detection and sequencing of class 1 integrons**
A notable feature in this study is that all _Salmonella_
strains isolated from poultry meat and humans were
MDR and harboring class I integrons that are chromosomally located (Fig. 5). Ten different profiles of class I
integrons with variable amplicon sizes ranged from 650–
3000 bp were observed (Table 3). Similarily, class I integrons were detected in 97 % of _S._ Typhimurium isolated
from Norwegian diarrhoeic patients [19]. On the contrary, a lower detection rate (9 %) was recorded in _Sal-_
_monella_ spp. isolated from poultry meat [20]. Sequencing



**Table 3 Resistance pattern, virulence genes and class I integron profiles among multidrug resistant** _**Salmonella**_ **serotypes**
**in this study**



**Strain** **Serovar (source** **Resistance pat-**
**no.** **of isolate)** **tern**



**Virulence genes** **Integron** **Integron** **Genes in the** **Accession**
**amplicon** **profile (IP)** **cassettes** **numbers**

_**Pef**_ **A** _**hil**_ **A** _**Sop**_ **B** **Stn** **size (bp)**



1 S. Derby (duck) ST E Amc - - - - 937 I _aac (3)-Idaac_ KT581257
1600 _(3)-Id, aadA7_ KT427378

2 S. Kiel (chicken) A T E Amc - + - - 937 II

3 Untypable (duck) A C S Sxt T E Cx As + - + - 1500 III _aac (3)-Id, aadA7_ KT581256
Na G Amc

4 S. Rubislaw A C S Sxt T E As Na + - - - 1500 III (chicken) G Amc

5 S. Typhimurium A C S Sxt T E As Na - - - - 1100 IV - KT581255
(chicken) G Amc 1500 _estX-sat_

6 S. Typhimurium S Sxt T E Cx Co - - + - 650 V _sat_ KT449570
(chicken) Na G 1550 ac (3)-Id, aadA7at KT598359

7 Untypable (duck) A Sxt T E As Na - - + - 1600 VI Amc 3000

8 S. Typhimurium A C S Sxt T E Cro - - + - 1000 VII (human) As Na G Amc



9 S. Typhimurium A C S Sxt T E As Na - - + + 650
(human) Amc 750
1600



VIII _adA4_ KT581253
adfrA15 KT449571



10 S. Typhimurium S Sxt T E As G Amc - - - - 1900 IX
(chicken)

11 S. Typhimurium A C S Sxt T E As Na - - - - 1900 IX _aadA2, lnuF_ KT449569
(chicken) G Amc

12 S. Typhimurium C Sxt T E Cx As Na + - - - 1550 X _aac (3)-Id, aadA7_ KT581254
(chicken) G Amc




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 5 of 11


**Fig. 1** _Agarose gel electrophoresis_ showing an amplification of _pefA_ gene (700 bp) in _Salmonella_ isolates from poultry meat and humans. Lane N:
negative control ( _pef_ A [−] _Salmonella_ strain), lane L: DNA ladder (100 bp), lane P: positive control ( _pef_ A [+] _Salmonella_ strain), lane 1: _pef_ A [−] ( _S._ Derby,
duck), lane 2: _pef_ A [−] ( _S._ Kiel, chicken), lane3: _pef_ A [+] (untypable, duck), lane 4: _pef_ A [+] ( _S._ Rubislaw, chicken), lanes 5,6, 10, 11: _pef_ A [−] ( _S._ Typhimurium,
chicken), lane 7: _pef_ A [−] (untypable, duck), lanes 8,9: _pef_ A [−] ( _S._ Typhimurium, humans), lane12: _pef_ A [+] ( _S._ Typhimurium, chicken)


**Fig. 2** _Agarose gel electrophoresis_ showing an amplification of _hil_ A gene (854 bp) in _Salmonella_ isolates from poultry meat and humans. Lane
N: negative control ( _hilA_ [−] _Salmonella_ strain), lane L: DNA ladder (100 bp), lane P: Positive control ( _hil_ A [+] _Salmonella_ strain), lane 1: _hilA_ [−] ( _S._ Derby,
duck), lane 2: _hil_ A [+] ( _S._ Kiel, chicken), lane 3,7: _hil_ A [−] (untypable, duck), lane 4: _hil_ A [−] ( _S._ Rubislaw, chicken), lanes 5, 6, 10, 11,12: _hil_ A [−] ( _S._ Typhimurium,
chicken), lanes 8,9: _hil_ A [−] ( _S._ Typhimurium, humans)



of the variable amplicons of class I integrons denoted by
asterisks (Fig. 5) revealed the presence of gene cassettes
containing aminoglycoside acetyltransferase [ _aac(3)_ - _Id_
or _aacCA5_ ] gene which confers resistance against gentamycin, aminoglycoside adenyltransferase ( _aadA2_, _aadA4_,
_aadA7_ ) genes that confer resistance to streptomycin
and spectinomycin, streptothricin acetyltransferase ( _sat_ )
encoding resistance against streptothricin, dihydrofolate reductase type 15 ( _dfrA15_ ) that confers resistance
to trimethoprim, _lnuF_ gene that codes for lincosamides
resistance (lincomycin and clindamycin) and putative



esterase ( _estX_ ). Interestingly, the _aac (3)_ - _Id_ plus _aadA7_
were the most predominant resistance genes identified
in class I integron gene cassettes of 1600 bp ( _S._ Derby,
duck), 1500 bp (untypable strain, duck) and 1550 bp ( _S._
Typhimurium, chicken). The same cassettes harboring _aac (3)_ - _Id_ and _aadA7_ genes were previously identified in class I integron of _S._ Haifa (GenBank accession
no. AY563051) [21], _S._ Newport (AY458224) [22] and _S._
Kentucky (AY463797) [23]. Despite the presence of these
genes, some strains such as _S._ Derby was susceptible to
gentamycin and _S._ Typhimurium strains (chicken) were




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 6 of 11


**Fig. 3** _Agarose gel electrophoresis_ showing an amplification of _sop_ B gene (517 bp) in _Salmonella_ isolates from poultry meat and humans. Lane N:
negative control ( _sop_ B [−] _Salmonella_ strain), lane L: DNA ladder (100 bp), lane P: Positive control ( _sop_ B [+] _Salmonella_ strain), lane 1: _sop_ B [−] ( _S._ Derby,
duck), lane 2: _sop_ B [−] ( _S._ Kiel, chicken), lanes 3, 7: _sop_ B [+] (untypable, duck), lane 4: _sop_ B [−] ( _S._ Rubislaw, chicken), lanes 5: _sop_ B [−] ( _S._ Typhimurium,
chicken), lane 6: _sop_ B [+] ( _S._ Typhimurium, chicken), lanes 8,9: _sop_ B [+] ( _S._ Typhimurium, humans), lane 10, 11, 12: _sop_ B [−] ( _S._ Typhimurium,chicken)


**Fig. 4** _Agarose gel electrophoresis_ showing an amplification of _stn_ gene (617 bp) in _Salmonella_ isolates from poultry meat and humans. Lane N:
negative control ( _stn_ [−] _Salmonella_ strain), lane L: DNA ladder (100 bp), lane P: Positive control ( _stn_ [+] _Salmonella_ strain), lane 1: _stn_ [−] ( _S._ Derby, duck),
lane 2: _stn_ [−] ( _S._ Kiel, chicken), lanes 3,7: _stn_ [−] (untypable, duck), lane 4: _stn_ [−] ( _S._ Rubislaw, chicken), lanes 5,6,10,11,12: _stn_ [−] ( _S._ Typhimurium, chicken),
lane 8: _stn_ [−] ( _S._ Typhimurium, humans), lane 9: _stn_ [+] ( _S._ Typhimurium, humans)



sensitive to streptomycin and this could be attributed to
point mutations in the _aac(3)_ - _Id_ gene of _S._ Derby [24]
and frame shift mutation in _S._ Typhimurium [25, 26].
Nucleotide sequence analysis revealed point mutations
in the _aac(3)_ - _Id_ gene of _S._ Derby leading to the amino
acid changes at codon 36 (glutamine CAG→proline
CCG), codon 166,172 (arginine AGA→threonine ACA),
codon 187 (arginine CGA→CTA leucine) and codon
189 (cysteine TGC→arginine CGC).C-to-T transition
at position 208 of the deduced polypeptide resulted
in a leucine to phenylalanine substitution and A to C



transversion at codon 214 resulted in asparagine to histidine substitution. Silent mutation was recorded only at
nucleotide 138 and didn’t result in amino acid substitution (CCA→CCG, both are proline). Moreover, sequence
analysis of _aadA7_ gene showed frame shift mutation due
to additional cytosine at position 969 of the deposited
sequence of _S._ Typhimurium from chicken.
In the present study, only one isolate of _S._ Typhimurium from human was found to harbor class I integrons
of 650, 750 and 1600 bp. Sequence analysis of 650 bp
revealed the presence of gene cassette containing _aadA4_




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 7 of 11


**Fig. 5** Amplification of conserved regions (5′-CS and 3′-CS) of class 1 integron and integron profiles (IPs) detected in _Salmonella_ isolates from poultry meat and humans. _Asterisks_ represent the bands selected for sequencing. Lane N: negative control, lane L: DNA ladder (100 bp), lane P: Positive
control, lane 1: IP I _(S._ Derby, duck), lane 2: IP II ( _S._ Kiel, chicken), lanes 3,4: IP III (untypable, duck & _S._ Rubislaw, chicken), lanes 5, 6,10,11, 12: IP IV, V, IX,
X ( _S._ Typhimurium, chicken), lanes 8, 9: IPs VII, VIII ( _S._ Typhimurium, humans)



which showed 100 % identity with _aadA4_ previously
identified in class I integron of _S._ Newport (CP006631)
and plasmid of _S._ Indiana (ref|NG_041636). In addition,
the 1600 bp integron carried gene cassette containing
_dfrA15_ gene that showed 100 % amino acid and nucleotide homology with _dfrA15_ gene previously reported in _S._
_enterica_ strain (KM823524).
The _sat_ gene detected in the gene cassette of class
I integron (650 bp) of _S._ Typhimurium in this study
was previously identified in _S._ Kedougou (DQ284538)

[27] and plasmid of _S._ Choleraesuis (AY509004) [28].
Moreover, the 1500 bp amplicon identified in _S._ Typhimurium contained _estX_ - _sat_ gene cassette that is
identical to _est_ X gene previously identified in _S._ Typhimurium (EF051039), _sat_ gene of _S._ Choleraesuis plasmid
(ref|NG_036624) and SC-B67 strain (AY509004). Point
mutations were observed in _estX_ - _sat_ gene cassette in this
study compared to the classical _estX_ and _sat_ genes (99 %
nucleotide identity); these genes displayed both missense and silent mutations. The nucleotide number 763
of _estX_ was exchanged from T to C resulting in exchange
of amino acid (phenylalanine) into leucine. Furthermore,
at codon 288 (histidine TAC→tyrosine CAC) and 290
(threonine ACA→alanine GCA). Silent mutations were
recorded at nucleotides 894 (GTC→GTA, alanine), 903
(TAT→TAC, tyrosine) and 927 (GTC→GTA, both are
valine). While in _sat_ gene, silent mutations were detected
at codon 418 (ACT→ACC, tyrosine), 428 (GTC→GTA,
valine), 451 and 453 (GGC→GGT, GGA→GGG both
are glycine). Amino acids replacement was recorded
in _sat_ gene at codon 459 (lysine AAA→glutamic acid



GAA), 462 (proline CCG→leucine CTG) and 465 (isoleucine ATC→threonine ACC). Nonsense mutation
was predicted at nucleotide 1418 (serine TCA→stop
codon TAA). Antimicrobial drug resistance can occur by
point mutations in the bacterial genome [29] or through
mobile genetic elements called integrons which are able
to disseminate the antimicrobial resistance among the
enterobacteriaceae by horizontal transfer [30].
In fact, this study reports for the first time the presence
of _aadA2_ plus _lnuF_ genes within the same integron gene
cassette of 1900 bp in _S.typhimurium_ from chicken. The
nucleotide sequence of _aadA2_ and _lnuF_ genes, respectively showed 99 and 100 % identity with _aadA2_ and _lnuF_
genes previously reported in _S. enterica_ serovar Stanley
(EU118119). Moreover, the _aadA2_ gene had 99 % amino
acid and nucleotide homology with _aadA2_ previously
identified in _S._ Typhimurium isolated from food-animals
and humans in Mexico (FJ460233) [31]. On the other hand,
the 1900 bp of class I integron was found to harbor _aadA2_
plus _dfrA12_ in _S. enterica_ serovar Typhimurium isolated
from food-animals, chicken and humans in Malaysia [32].
The nucleotide sequence of _aadA2_ showed four
base differences from other _aadA2_ in GenBank
reported cassettes and this caused point mutation
which are responsible for an alteration in the codon
60 (glycine GGA→arginine CGA), codon 61(isoleucine ATC→threonine ACC), codon 62 (asparagine
AAC→aspartic acid GAC) and codon 105 (proline
CCT→TCT serine) (Fig. 6). Similarly, point mutation
in the _aadA2_ gene of NTS _enterica_ isolates was reported

[29].




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 8 of 11


**Fig. 6** Amino acid sequence similarities for _S._ Typhimurium _aadA2_ gene of the strain under study ( _S._ Typhimurium strain MASR STKT449569) and
the reference strains. _Dots_ indicate amino acid positions that are identical to the corresponding _S._ Typhimurium _aadA2_ sequence. The glycine 60,
isoleucine 61, asparagine 62 and proline 105 in which mutations occur are indicated by the _solid bars_




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 9 of 11



**Conclusions**
The higher resistance rates displayed by _Salmonella_ isolates in this study to the antibiotics and the detection of
class 1 integron gene cassettes harboring resistance genes
among MDR strains shows the potential of integrons to
carry and spread resistance genes to other _S. enterica_ isolates or to other bacteria. Therefore, a more prudent use
of antibiotics in both humans and animals is required.


**Methods**
**Ethical approval**
This study was approved by Zagazig University, Egypt
ethical board (Protocol No. 10122). Permission to collect
stool samples was obtained from the administrators of
Al-Ahrar General hospital.


**Specimen collection**
The present work was conducted during the period from
March, 2014 to February, 2015. A total of one hundred
fresh poultry carcasses comprising broiler chicken and
ducks (50, each) were randomly purchased from poultry retail outlets at Zagazig City, Egypt. In addition, fifty
stool samples were collected from the patients attending
the outpatients’ clinic of Al-Ahrar General Hospital. The
inclusion criteria were restricted to the patients who suffered from diarrhoea, fever and gave informed written
consent to participate in the study. Moreover, patients
received antibiotic treatments were excluded. A swab was
obtained from each freshly passed stool sample, labeled,
ice packed and transported to the Bacteriology Laboratory, Faculty of Veterinary Medicine, Zagazig University,
Egypt within 24 h for bacteriological analysis.


**Isolation and identification of** _**Salmonella**_ **spp**
_Salmonella_ isolation was carried out according to the
standard methods recommended by ISO 6579 [33].
Twenty-five grams of poultry meat were excised from
each sample and minced with 225 ml of buffered peptone water (BPW) enrichment broth. Also, each stool
swab was inserted in sterile tube containing 9 ml BPW.
All samples were incubated at 37 °C for 24 h. Aliquots
(0.1 ml) of the pre-enriched culture was inoculated into
10 ml of Rappaport Vasiliadis (RV) enrichment broth
and incubated at 42 °C for 24 h. A loopful of the enriched
culture was streaked on Xylose-Lysine-Deoxycholate
(XLD) agar plates (Oxoid, Basingstoke, UK) and incubated at 37 °C for 24 h. Suspected colonies with typical
_Salmonella_ morphology were confirmed biochemically
by indole, citrate utilization, urease, triple sugar iron
(TSI) and lysine iron agar (LIA) tests. All biochemically
confirmed _Salmonella_ isolates were serologically identified on the basis of somatic (O) and flagellar (H) antigens
by slide agglutination using commercial antisera (SISIN,



Berlin) following Kauffman-White scheme [34] in collaboration with Serology Unit, Animal Health Research
Institute, Dokki, Egypt.


**Antimicrobial susceptibility testing (antibiogram)**
Susceptibility of _Salmonella_ isolates to various routine
antimicrobial drugs was tested by the standard disc diffusion technique [35] using Mueller–Hinton agar and
commercial antibiotic discs (Oxoid, UK). The antibiotics
used were ampicillin (10 μg), erythromycin (15 μg), streptomycin (10 μg), ciprofloxacin (5 μg), tetracycline (30 μg),
ceftriaxone (30 μg), ampicillin-sulbactam (20 μg), trimethoprim-sulfamethoxazole (25 μg), chloramphenicol
(30 μg), nalidixic acid (30 μg), gentamicin (10 μg), amoxicillin-clavulanic acid (30 μg). The inhibition zones in mm
were measured and scored as sensitive, intermediate and
resistant categories in accordance with the critical breakpoints recommended by the Clinical and Laboratory
Standards Institute [36].


**Molecular detection of** _**Salmonella**_ **virulence genes using**
**uniplex PCR**
A total of twelve _Salmonella_ isolates were screened for
the presence of virulence genes using PCR. These genes
including; plasmid encoded fimbriae ( _pef_ A), hyper invasive locus ( _hil_ A), _Salmonella_ outer protein ( _sop_ B) and
_Salmonella_ enterotoxin gene ( _stn_ ). Briefly, DNA was
extracted from the overnight bacterial culture using
ABIO pure Genomic DNA extraction kit with modifications from the manufacturer’s recommendation. DNA
amplification was carried out using specific primers supplied from Metabion, Germany (Table 4). Primers were
utilized in a 25 µl reaction containing 12.5 µl of Emerald
Amp Max PCR Master Mix (Takara, Japan), one µl of
each primer (20 pmol concentration), 4.5 µl of nuclease
free water, and 6 µl of DNA template. The reaction was
performed in a T3 Biometra thermal cycler. Positive and
negative controls were included in each reaction. Aliquot
of each amplicon along with a 100 bp molecular weight
DNA ladder (Fermentas, USA) were separated by electrophoresis on 1.5 % agarose gel (Applichem, Germany,
GmbH) stained with 0.5 μg/ml ethidium bromide (Sigma,
USA) in 1 × TBE buffer on a mini slab horizontal electrophoresis unit (Bio-Rad, USA) at 100 V for 30 min. The gel
was photographed by a gel documentation system (Alpha
Innotech, Biometra) and the data was analyzed through
computer software.


**Molecular detection and sequencing of class 1 integrons**
All MDR _Salmonella_ isolates (n = 12) were tested for
the presence of conserved regions (5′-CS and 3′-CS) of
class Ι integrons using specific primers with the following sequences: 5′ CS-GGC ATC CAA GCA GCA AG and




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 10 of 11


**Table 4 Primers’ sequences and amplification’s conditions used for detection of** _**Salmonella**_ **virulence genes**



**Target** **Primers’ sequences 5′–3′** **Amplified**
**gene** **segment**
**(bp)**



**Primary**
**denaturation**



**Amplification (35 cycles)** **Final** **References**
**extension**

**Secondary** **Annealing Extension**
**denaturation**



_pef_ A TGTTTCCGGGCTTGTGCT 700 94 °C 10 min 94 °C 45 s 55 °C 45 s 72 °C 45 s 72 °C 10 min [4]

CAGGGCATTTGCTGATTCTTCC

_hil_ A CGGAAGCTTATTTGCGCCATGCT GAGGTAG 854 94 °C 10 min 94 °C 45 s 60 °C 45 s 72 °C 45 s 72 °C 10 min [37]

GCATGGATCCCCGCCGGCGAGAT TGTG

_sop_ B TCAGAAGRCGTCTAACCACTC 517 94 °C 5 min 94 °C 30 s 58 °C 30 s 72 °C 30 s 72 °C 7 min [38]

TACCGTCCTCATGCACACTC

_stn_ TTGTGTCGCTATCACTGGCAACC 617 94 °C 10 min 94 °C 45 s 59 °C 45 s 72 °C 45 s 72 °C 10 min [4]

ATTCGTAACCCGCTCTCGTCC



3′CS- AAG CAG ACT TGA CCT GA [30]. Chromosomal
DNA extraction, amplification and gel analysis were performed as mentioned before except to amplify the DNA
in the thermal cycler we used a three-step profile (cycling
conditions): 1 min of denaturation at 94 °C, 1 min of
annealing at 55 °C, and 5 min of extension at 72 °C for a
total of 35 cycles. Five seconds were added to the extension time at each cycle. Amplified products with variable
sizes were purified using QIA quick PCR Product extraction kit (Qiagen, Valencia). Each purified amplicon was
sequenced in both forward and reverse directions using
the amplification primers. DNA sequences were obtained
by Applied Biosystems 3130 genetic analyzer (HITACHI,
Japan).Nucleotide sequence analysis was compared with
published GenBank DNA sequences using NCBI BLAST
program [(http://www.ncbi.nlm.nih.gov/BLAST/)](http://www.ncbi.nlm.nih.gov/BLAST/) and
alignment was performed usingMEGA6 program.


**Nucleotide sequence accession numbers**
Nucleotide sequences from this study were deposited
into the GenBank under accession numbers [KT427378,
KT449569, KT449570, KT449571, KT581253, KT581254,
KT581255, KT581256, KT581257 and KT598359].


**Authors’ contributions**
RMG, YHT and MHE participated in the design of the study, collected and analyzed the samples. YHT participated in the sequence alignment. RMG and YHT
drafted the manuscript. All authors read and approved the final manuscript.


**Author details**
1 Depatment of Zoonoses, Faculty of Veterinary Medicine, Zagazig University,
44511 Zagazig, Egypt. [2] Department of Bacteriology, Mycology and Immunology, Faculty of Veterinary Medicine, Zagazig University, 44511 Zagazig, Egypt.
3 Department of Veterinary Public Health, Faculty of Veterinary Medicine,
Zagazig University, 44511 Zagazig, Egypt.


**Aknowledgements**
We are grateful to thank all administrators and clinicians in Al-Ahrar General
Hospital, Zagazig, Egypt for their assistance in collecting stool samples.


**Competing interests**
The authors declare that they have no competing interests.



Received: 17 September 2015  Accepted: 10 December 2015


**References**
1. Majowicz SE, Musto J, Scallan E, Angulo FJ, Kirk M, O’Brien SJ, et al. The
global burden of non-typhoidal _Salmonella_ gastroenteritis. Clin Infect Dis.
2010;50:882–9.
2. EFSA (European Food Safety Authority). The community summary report
on trends and sources of zoonoses and zoonotic agents in the European
[Union in 2007. EFSA J. 2009;223:312. doi:10.2903/j.efsa.2009.223r.](http://dx.doi.org/10.2903/j.efsa.2009.223r)
3. Lostroh CP, Bajaj V, Lee CA. The cis requirements for transcriptional activation by HilA, a virulence determinant encoded on SPI-1. Mol Microbiol.
2000;37:300–15.
4. Murugkar HV, Rahman H, Dutta PK. Distribution of virulence genes in
_Salmonella_ serovars isolated from man and animals. Indian J Med Res.
2003;117:66–70.
5. Van Asten AJ, van Dijk JE. Distribution of classic virulence factors among
_Salmonella_ spp. FEMS Immunol Med Microbiol. 2005;44:251–9.
6. Threlfall EJ. Antimicrobial drug resistance in _Salmonella_ : problems and
perspectives in food- and water-borne infections. FEMS Microbiol Rev.
2002;26:141–8.
7. Chu C, Chiu CH. Evolution of the virulence plasmids of non-typhoidal
_Salmonella_ and its association with antimicrobial resistance. Microbes
Infect. 2006;8(7):1931–6.
8. Rowe-Magnus DA, Mazel D. The role of integrons in antibiotic resistance
gene capture. Int J Med Microbiol. 2002;292(2):115–25.
9. Maharjan M, Joshi V, Joshi DD, Manandhar P. Prevalence of _Salmo-_
_nella_ species in various raw meat samples of a local market in Kathmandu. Trends in the study of disease agents. Ann N Y Acad Sci.
2006;1081:249–56.
10. Arsenault J, Letellier A, Quessy S, Boulianne M. Prevalence and risk factors
for _Salmonella_ and _Campylobacter_ spp. carcass contamination in broiler
chickens slaughtered in Quebec, Canada. J Food Prot. 2007;70(8):1820–8.
11. Kaushik P, Anjay Kumari S, Bharti SK, Dayal S. Isolation and prevalence
of _Salmonella_ from chicken meat and cattle milk collected from local
markets of Patna, India. Vet World. 2014;7(2):62–5.
12. Murugkar HV, Rahman H, Kumar A, Bhattacharyya D. Isolation, phage typing & antibiogram of _Salmonella_ from man and animals in northeastern
India. Indian J Med Res. 2005;122:237–42.
13. Vo ATT, Duijkeren EV, Fluit AC, Wannet WJB, Verbruggen AJ, Maas HME,
et al. Antibiotic resistance, integrons and _Salmonella_ genomic island
1 among non-typhoidal _Salmonella_ serovars in the Netherlands. Int J
Antimicrob Agents. 2006;28(3):172–9.
14. Dione MM, Ikumapayi U, Saha D, Mohammed NI, Adegbola RA, Geerts
S, et al. Antimicrobial resistance and virulence genes of non-typhoidal
_Salmonella_ isolates in the Gambia and Senegal. J Infect Dev Ctries.
2011;5(11):765–75.




Gharieb _et al. Gut Pathog (2015) 7:34_ Page 11 of 11



15. Soomro AH, Khaskheli M, Bhutto MB, Shah G, Memon A, Dewani
P. Prevalence and antimicrobial resistance of _Salmonella_ serovars
isolated from poultry meat in Hyderabad, Pakistan. Turk J Vet Anim Sci.
2010;34(5):455–60.
16. Onyango D, Machoni F, Kakai R, Waindi EN. Multidrug resistance of
_Salmonella enterica_ serovars Typhi and Typhimurium isolated from clinical samples at two rural hospitals in Western Kenya. J Infect Dev Ctries.
2008;2(2):106–11.
17. Akbarmehr J. Isolation of _Salmonella_ spp. from poultry (ostrich, pigeon,
and chicken) and detection of their hilA gene by PCR method. Afr J
Microbiol Res. 2010;4(24):2678–81.
18. Rahman H. Prevalence & phenotypic expression of sopB gene among
clinical isolates of _Salmonella enterica_ . Indian J Med Res. 2006;123:83–8.
19. Lindstedt BA, Heir E, Nygard I, Kapperud G. Characterization of class I
integrons in clinical strains of _Salmonella enterica_ subsp. enterica serovars
Typhimurium and Enteritidis from Norwegian hospitals. J Med Microbiol.
2003;52:141–9.
20. Antunes P, Machado J, Peixe L. Characterization of antimicrobial resistance and class 1 and 2 integrons in _Salmonella enterica_ isolates from
different sources in Portugal. J Antimicrob Chemother. 2006;58:297–304.
21. Cabrera R, Ruiz J, Sanchez- Cespedes J, Gómez-Lus R, Jiménez de Anta
MT, Gascón J, et al. Characterization of the enzyme aac (3)-Id in a clinical
isolate of _Salmonella enterica_ serovar Haifa causing traveler’s diarrhea.
Enferm Infecc Microbiol Clin. 2009;27(8):453–6.
22. Doublet B, Weill FX, Fabre L, Chaslus-Dancla E, Cloeckaert A. Variant
_Salmonella_ genomic island 1 antibiotic resistance gene cluster containing
a novel 3-N-aminoglycoside acetyltransferase gene cassette, aac (3)-Id
in _Salmonella enterica_ serovar Newport. Antimicrob Agents Chemother.
2004;48(10):3806–12.
23. Levings RS, Partridge SR, Lightfoot D, Hall RM, Djordjevic SP. New
integron-associated gene cassette encoding a 3-N- aminoglycoside
acetyltransferase. Antimicrob Agents Chemother. 2005;49(3):1238–41.
24. Casin I, Hanau-Bercot B, Podglajen I, Vahaboglu H, Collatz E. _Salmonella_
_enterica_ serovar Typhimurium blaPER-1-carrying plasmid pSTI1 encodes
an extended-spectrum aminoglycoside 6-N-acetyltransferase of type Ib.
Antimicrob Agents Chemother. 2003;47(2):697–703.
25. Sabtcheva S, Galimand M, Gerbaud G, Courvalin P, Lambert T. Aminoglycoside resistance gene ant(4)-IIb of _Pseudomonas_ _aeruginosa_
BM4492, a clinical isolate from Bulgaria. Antimicrob Agents Chemother.
2003;47(5):1584–8.



26. Michael GB, Cardoso M, Schwarz S. Class 1 integron-associated gene cassettes in _Salmonella enterica_ subsp. enterica serovar Agona isolated from
pig carcasses in Brazil. J Antimicrob Chemother. 2005;55:776–9.
27. Vo A T T, van Duijkeren E, Gaastra W, Fluit AC. Antimicrobial Resistance,
class 1 integrons, and genomic island 1 in _Salmonella_ isolates from Vietnam. PLoS One. 2010;5:e9440.
28. Chiu CH, Tang P, Chu C, Hu S, Bao Q, Yu J, et al. The genome sequence of
_Salmonella enterica_ serovar Choleraesuis, a highly invasive and resistant
zoonotic pathogen. Nucleic Acids Res. 2005;33:1690–8.
29. Krauland MG, Marsh JW, Paterson DL, Harrison LH. Integron-mediated
multidrug resistance in a global collection of non-typhoidal _Salmonella_ .
Emerg Infect Dis. 2009;15(3):388–96.
30. Lévesque C, Piché L, Larose C, Roy PH. PCR mapping of integrons reveals
several novel combinations of resistance genes. Antimicrob Agents
Chemother. 1995;39(1):185.
31. Wiesner M, Zaidi MB, Calva E, Fernández-Mora M, Calva JJ, Silva C.
Association of virulence plasmid and antibiotic resistance determinants
with chromosomal multilocus genotypes in Mexican _Salmonella enterica_
serovar Typhimurium strains. BMC Microbiol. 2009;9:131.
32. Douadi B, Thong KL, Watanabe H, Puthucheary SD. Characterization of
drug resistant _Salmonella enterica_ serotype Typhimurium by antibiograms, plasmids, integrons, resistance genes and PFGE. J Microbiol
Biotechnol. 2010;20(6):1042–52.
33. ISO (6579). Microbiology of food and animal feeding stuffs-horizontal
method for the detection of _Salmonella_ spp. Geneva: ISO; 2002.
34. Popoff MY, Bockemuhl J, Gheesling LL. Supplement 2002 (no. 46) to the
Kauffmann-White scheme. Res Microbiol. 2004;155:568–70.
35. Bauer AW, Kirby WMM, Sherris JC. Antibiotic susceptibility testing by a
standardized single disc. Am J Clin Pathol. 1966;45:493–6.
36. Clinical and Laboratory Standards Institute (CLSI). Performance standards
for antimicrobial susceptibility testing; Eighteenth informational supplement. Wayne: CLSI document M100–S20; 2010.
37. Cardona- Castro N, Restrepo-Pineda E, Correa-Ochoa M. Detection of hilA
gene sequences in serovars of _Salmonella enterica_ subspecies enterica.
Mem Inst Oswaldo Cruz. 2002;97:1153–6.
38. Huehn S, La Ragione RM, Anjum M, Saunders M, Woodward MJ, Bunge
C, et al. Virulotyping and antimicrobial resistance typing of _Salmonella_
_enterica_ serovars relevant to human health in Europe. Food borne Pathog
Dis. 2010;7:523–35.




