# Characterization of integrons and resistance genes in multidrug-resistant *Salmonella enterica* isolated from meat and dairy products in Egypt

**Authors:** Ashraf M. Ahmed, Toshi Shimamoto, Tadashi Shimamoto
**Journal:** International Journal of Food Microbiology 189 (2014) 39–44

## Abstract

Foodborne pathogens are a leading cause of illness and death, especially in developing countries. The problem is exacerbated if bacteria attain multidrug resistance. Little is currently known about the extent of antibiotic resistance in foodborne pathogens and the molecular mechanisms underlying this resistance in Africa. Therefore, the current study was carried out to characterize, at the molecular level, the mechanism of multidrug resistance in *Salmonella enterica* isolated from 1600 food samples (800 meat products and 800 dairy products) collected from different street venders, butchers, retail markets and slaughterhouses in Egypt. Forty-seven out of 69 isolates (68.1%) showed multidrug resistance phenotypes to at least three classes of antimicrobials. The incidence of multidrug-resistant isolates was higher in meat products (37, 69.8%) than in dairy products (10, 62.5%). The multidrug-resistant serovars included, *S. enterica* serovar Typhimurium (24 isolates, 34.8%), *S. enterica* serovar Enteritidis, (15 isolates, 21.8%), *S. enterica* serovar Infantis (7 isolates, 10.1%) and *S. enterica* non-typable serovar (1 isolate, 1.4%). The highest resistance was to ampicillin (95.7%), then to kanamycin (93.6%), spectinomycin (93.6%), streptomycin (91.5%) and sulfamethoxazole/trimethoprim (91.5%). PCR and DNA sequencing were used to screen and characterize integrons and antibiotic resistance genes and 39.1% and 8.7% of isolates were positive for class 1 and class 2 integrons, respectively. -lactamase-encoding genes were identified in 75.4% of isolates and plasmid-mediated quinolone resistance genes were identified in 27.5% of isolates. Finally, the florphenicol resistance gene, *floR*, was identified in 18.8% of isolates. PCR screening identified *S. enterica* serovar Typhimurium DT104 in both meat and dairy products. This is the first study to report many of these resistance genes in dairy products. This study highlights the high incidence of multidrug-resistant *S. enterica* in meat and dairy products in Egypt, with the possibility of their transfer to humans leading to therapeutic failure. Therefore, the overuse of antibiotics in animals should be drastically reduced in developing countries.

## 1. Introduction

Increasing resistance to antimicrobial agents remains a major challenge to public health professionals in both developed and developing countries. However, in developing countries, antimicrobial resistance is exacerbated by over-prescription of antibiotics and increased use in both human and animal healthcare. Furthermore, strategies to combat and prevent resistance are not at the top of the list of priorities in these countries (da Costa et al., 2013). More recently, the Centers for Disease Control and Prevention of the USA estimates that, in the USA, more than two million people are made ill every year with antibiotic-resistant infections, with at least 23,000 dying as a result of these infections (CDC, 2013). Food may act as a vector for the transfer of antimicrobial resistant bacteria and antimicrobial resistance genes to humans (Verraes et al., 2013). Multidrug-resistant (MDR) bacteria can spread to humans either via the food supply (e.g., meat, fish, eggs and dairy products), direct contact with animals or, more indirectly, through environmental pathways (Angulo et al., 2004). The consequences of antimicrobial resistance are particularly important when pathogens are resistant to antimicrobials that are critically important in the treatment of human disease. This concern includes infections acquired in hospitals, community infections acquired in outpatient care settings, and also, resistant foodborne disease associated with drug use in food-producing animals (WHO, 2009).

The spread of MDR bacteria via meat and dairy products poses serious public health concerns. In the United States, the newly-released 10th National Antimicrobial Resistance Monitoring System (NARMS) report describes alarming increases in antibiotic-resistant bacteria found on retail meats. The report also confirmed that 80% of all antibiotics used in the United States are used not on humans but on food animals, most of which are perfectly healthy (NARMS, 2011). The increasing prevalence of multidrug resistance among *Salmonella*, not only against the first-line antibiotics, ampicillin, chloramphenicol and trimethoprim/sulfamethoxazole, but also, against clinically important antimicrobial agents, such as fluoroquinolones and third generation cephalosporins, is also an emerging problem (Lunguya et al., 2013). In developed countries, many surveys have been conducted at the molecular level to monitor the incidence of MDR *Salmonella enterica* in meat and dairy products (Ahmed et al., 2009a; Barlow and Gobius, 2006; Clemente et al., 2013; Li et al., 2013; Mohamed et al., 2014; Thong and Modarressi, 2011; Van Keesel et al., 2013; Wong and Chen, 2013; Zhao et al., 2009). However, the extent of antibiotic resistance to *S. enterica* in many developing countries and the molecular mechanisms underlying this resistance remain unclear. The misuse of antibiotics in humans and animals in these countries is not uncommon. Therefore, the aim of this study was to perform a large scale survey to determine the incidence of MDR strains of *S. enterica* isolated from meat and dairy products in Egypt and to characterize the molecular mechanisms of antimicrobial resistance.

## 2. Materials and methods

### 2.1. Bacterial isolates

A total of 69 isolates of *S. enterica* (28 isolates of *S. enterica* serovar Typhimurium, 22 isolates of *S. enterica* serovar Enteritidis, 16 isolates of *S. enterica* serovar Infantis and 3 isolates of non-typable serovars) were used in this study. They were isolated in Egypt from 800 meat products (480 beef and 320 chickens) and 800 dairy products (480 milk, 240 cheese and 80 yogurt) as previously described (Ahmed and Shimamoto, 2014).

### 2.2. Antimicrobial susceptibility testing

The antimicrobial sensitivity phenotypes of bacterial isolates were determined using a Kirby-Bauer disk diffusion assay according to the standards and interpretive criteria described by the Clinical and Laboratory Standards Institute (CLSI, 2011). The following antibiotics were used: ampicillin (AMP), 10 µg; amoxicillin-clavulanic acid (AMC), 20/10 µg; cefoxitin (FOX), 30 µg; cefotetan (CTT), 30 µg; cefotaxime (CTX), 30 µg; cefpodoxime (CPD), 10 µg; ceftriaxone (CRO), 30 µg; aztreonam (ATM), 30 µg; nalidixic acid (NAL), 30 µg; ciprofloxacin (CIP), 5 µg; chloramphenicol (CHL), 30 µg; gentamicin (GEN), 10 µg; kanamycin (KAN), 30 µg; oxacillin (OXA) 30 µg; streptomycin (STR), 10 µg; spectinomycin (SPX), 10 µg; sulfamethoxazole/trimethoprim (SXT), 23.75/1.25 µg and tetracycline (TET), 30 µg. The disks were purchased from Oxoid, UK, and the results were recorded based on CLSI guidelines (CLSI, 2011). The reference strain, *Escherichia coli* ATCC 25922, was included as a quality control.

### 2.3. Bacterial DNA preparation

DNA was prepared using boiled lysates, as previously described (Ahmed et al., 2009a). All selected single colonies were subcultured in LB broth. An overnight bacterial culture (200 µl) was mixed with 800 µl of distilled water and boiled for 10 min. The resulting solution was centrifuged and the supernatant used as the DNA template, DNA template was stored at -20°C until use.

### 2.4. PCR screening for integrons and antimicrobial resistance genes

Conserved primers were used for the detection and identification of class 1 and class 2 integrons as previously described (Ahmed et al., 2013). PCR screening for TEM, SHV, CTX-M, OXA and CMY -lactamase-encoding genes was performed using universal primers for the TEM, SHV, OXA, CTX-M and CMY families, as described previously (Ahmed et al., 2013). Furthermore, PCR amplification was used to screen for plasmid-mediated quinolone resistance genes, *qnrA*, *qnrB*, *qnrS* and *aac(6')-Ib-cr*, using previously described primers (Ahmed et al., 2013). Finally, the florfenicol resistance gene, *floR*, was detected using StCM-L and StCM-R primers, as described previously (Ahmed et al., 2013). The PCR reaction products were subjected to electrophoresis in a 1.0% agarose gel, stained with ethidium bromide and visualized under UV light. PCR fragments were then purified from the agarose gel using a QIAquick Gel Extraction Kit (Qiagen, Japan). Both DNA strands of the PCR product were sequenced using an ABI automatic DNA sequencer (Model 373; Perkin-Elmer).

### 2.5. Identification of *S. enterica* serovar Typhimurium DT104 by PCR

The PCR amplification of an internal segment of the 16S-to-23S spacer region of bacterial rRNA genes (size = 162 bp) was used to identify *S. enterica* serovar Typhimurium DT104 as previously described (Pritchett et al., 2000).

### 2.6. Transconjugation experiments and plasmid incompatibility grouping

Transferability of plasmids was determined by mating-out assay using *S. enterica* isolates as donors and a rifampicin-resistant mutant of *E. coli* HB101 as a recipient as described previously (Ahmed et al., 2013). Some *S. enterica* isolates which have no potent resistance gene marker were not tested. Transconjugants were selected on agar supplemented with 250 mg/l rifampicin and 100 mg/l ampicillin. Plasmid DNA was extracted from both *S. enterica* isolates and *E. coli* transconjugants using the Kado and Liu method (Kado and Liu, 1981). Plasmid incompatibility grouping was determined by PCR-based replicon typing as previously described (Carattoli et al., 2005). The transfer of integrons and resistance genes was confirmed by PCR assays on the transconjugants as described previously (Ahmed et al., 2013).

### 2.7. Computer analysis of the sequence data

A similarity search was carried out using the BLAST program available at the NCBI BLAST homepage ([http://blast.ncbi.nlm.nih.gov/Blast.cgi](http://blast.ncbi.nlm.nih.gov/Blast.cgi)).

## 3. Results and discussion

### 3.1. Incidence of MDR *S. enterica* in meat and dairy products

The emergence and spread of antimicrobial resistance in bacteria constitute a threat to human health and present a major financial burden. Food is generally considered to be the most important vector for the spread of resistance between humans and animals (WHO, 2007). In this study 47 out of 69 isolates (68.1%) showed multidrug resistance phenotypes to at least three classes of antimicrobials (Table 1). The incidence of MDR isolates was higher in meat products (69.8%) than in dairy products (62.5%). The MDR *S. enterica* serovars were: *S. enterica* serovar Typhimurium (34.8%), *S. enterica* serovar Enteritidis, (21.8%), *S. enterica* serovar Infantis (10.1%) and *S. enterica* non-typable serovar (1.4%) (Table 1). The highest resistance was to ampicillin (95.7%), then to kanamycin (93.6%), spectinomycin (93.6%), streptomycin (91.5%) and sulfamethoxazole/trimethoprim (91.5%) (Table 1). Interestingly, the results of multidrug resistance phenotypes in this study are quite similar to that we previously reported in *S. enterica* isolated from diarrheic calves in Egypt as 66.7% of *S. enterica* showed multidrug resistance phenotypes to ampicillin, amoxicillin, streptomycin, spectinomycin, sulfamethoxazole/trimethoprim and kanamycin (Ahmed et al., 2009b). Our results confirm the high level of resistance to the traditional antibiotics used widely in Egypt. Unfortunately, in developing countries, these antibiotics are widely used to treat diarrhea because of their low cost and availability (Van et al., 2012).

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 1: Resistance phenotypes and incidence of integrons and resistance genes in Salmonella enterica isolated from meat and dairy products.</strong> </caption>
<thead>
<tr>
<th>No.</th>
<th>Isolate / Source</th>
<th>Serovar / ID</th>
<th>Resistance phenotype</th>
<th>Integrons/resistance genes</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M1</td>
<td>AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaCMY-2, blaCTX-M-3, qnrB, aac(6')-Ib-cr, floR</td>
</tr>
<tr>
<td>2</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M2</td>
<td>AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaCMY-2, blaCTX-M-15, blaSHV-12, qnrA, floR</td>
</tr>
<tr>
<td>3</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M3</td>
<td>AMC, AMP, ATM, CHL, CIP, CRO, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaSHV-12, qnrB, floR</td>
</tr>
<tr>
<td>4</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M4</td>
<td>AMP, ATM, CHL, CIP, CTT, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA2 and blaPSE-1), qnrS, floR</td>
</tr>
<tr>
<td>5</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M5</td>
<td>AMP, ATM, CIP, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA12-orf-aadA2), blaCMY-2, aac(6')-Ib-cr</td>
</tr>
<tr>
<td>6</td>
<td>Chicken</td>
<td>S. Typhimurium



ST-M6</td>
<td>AMP, ATM, CHL, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT</td>
<td>Class 1 (aadB-catB3), blaTEM-1, blaOXA-1, floR</td>
</tr>
<tr>
<td>7</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M7</td>
<td>AMC, AMP, ATM, CHL, CIP, CTT, CTX, FOX, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1, blaSHV-12, aac(6')-Ib-cr</td>
</tr>
<tr>
<td>8</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M8</td>
<td>AMC, AMP, ATM, CPD, CRO, CTT, CTX, FOX, KAN, GEN, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA12-orf-aadA2), blaTEM-1, blaCTX-M-3</td>
</tr>
<tr>
<td>9</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M9</td>
<td>AMC, AMP, ATM, CHL, CPD, CRO, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, blaCTX-M-3, floR</td>
</tr>
<tr>
<td>10</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M10</td>
<td>AMP, ATM, CHL, CIP, CTT, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, qnrS</td>
</tr>
<tr>
<td>11</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M11</td>
<td>AMP, ATM, CIP, FOX, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1, qnrB</td>
</tr>
<tr>
<td>12</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M12</td>
<td>AMP, ATM, CHL, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1</td>
</tr>
<tr>
<td>13</td>
<td>Chicken</td>
<td>S. Typhimurium



ST-M13</td>
<td>AMP, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA2), blaTEM-1</td>
</tr>
<tr>
<td>14</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M14</td>
<td>AMC, AMP, CPD, CRO, CTT, CTX, FOX, GEN, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA15), blaTEM-1, blaCTX-M-15</td>
</tr>
<tr>
<td>15</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M15</td>
<td>AMC, AMP, CHL, CTT, CTX, FOX, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 2 (dfrA1-sat2-aadA1), blaCMY-2</td>
</tr>
<tr>
<td>16</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M16</td>
<td>AMP, CHL, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 2 (dfrA1-sat2-aadA1)</td>
</tr>
<tr>
<td>17</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M17</td>
<td>CHL, CIP, GEN, NAL, SPX, STR, SXT, TET</td>
<td>qnrB</td>
</tr>
<tr>
<td>18</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M18</td>
<td>AMP, CIP, CTT, FOX, NAL, OXA, SPX, STR, SXT, TET</td>
<td>blaTEM-1, aac(6')-Ib-cr</td>
</tr>
<tr>
<td>19</td>
<td>Beef</td>
<td>S. Typhimurium



ST-M19</td>
<td>AMP, CHL, KAN, NAL, SXT, TET</td>
<td>floR</td>
</tr>
<tr>
<td>20</td>
<td>Beef</td>
<td>S. Enteriditis



SE-M1</td>
<td>AMC, AMP, ATM, CHL, CIP, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA1-aadA1), blaCMY-2, qnrB, floR</td>
</tr>
<tr>
<td>21</td>
<td>Chicken</td>
<td>S. Enteriditis



SE-M2</td>
<td>AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA2), blaTEM-1, blaCTX-M-3, qnrS</td>
</tr>
<tr>
<td>22</td>
<td>Chicken</td>
<td>S. Enteriditis



SE-M3</td>
<td>AMP, ATM, CHL, CIP, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, aac(6')-Ib-cr</td>
</tr>
<tr>
<td>23</td>
<td>Beef</td>
<td>S. Enteriditis



SE-M4</td>
<td>AMP, ATM, CTT, FOX, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA5), blaTEM-1</td>
</tr>
<tr>
<td>24</td>
<td>Beef</td>
<td>S. Enteriditis



SE-M5</td>
<td>AMP, CTT, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (estX-aadA1), blaTEM-1</td>
</tr>
<tr>
<td>25</td>
<td>Beef</td>
<td>S. Enteriditis



SE-M6</td>
<td>AMP, CHL, OXA, SPX, STR, SXT</td>
<td>blaTEM-1, floR</td>
</tr>
<tr>
<td>26</td>
<td>Chicken</td>
<td>S. Enteriditis



SE-M7</td>
<td>AMP, CTT, FOX, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 2 (estX-sat2-aadA1), blaTEM-1</td>
</tr>
<tr>
<td>27</td>
<td>Beef</td>
<td>S. Enteriditis



SE-M8</td>
<td>AMC, AMP, ATM, CHL, CPD, CRO, CTT, CTX, FOX, GEN, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 2 (estX-sat2-aadA1), blaSHV-12</td>
</tr>
<tr>
<td>28</td>
<td>Chicken</td>
<td>S. Enteriditis



SE-M9</td>
<td>AMP, GEN, KAN, SPX, STR, SXT, TET</td>
<td>Class 2 (estX-sat2-aadA1)</td>
</tr>
<tr>
<td>29</td>
<td>Beef</td>
<td>S. Enteriditis



SE-M10</td>
<td>AMP, KAN, OXA, SPX, STR, SXT, TET</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>30</td>
<td>Beef</td>
<td>S. Enteriditis



SE-M11</td>
<td>AMC, AMP, ATM, CHL, CPD, CRO, CTT, CTX, FOX, GEN, KAN, OXA, SPX, STR, SXT, TET</td>
<td>blaCMY-2</td>
</tr>
<tr>
<td>31</td>
<td>Beef</td>
<td>S. Enteriditis



SE-M12</td>
<td>AMP, ATM, KAN, OXA, SPX, STR</td>
<td>blaOXA-1</td>
</tr>
<tr>
<td>32</td>
<td>Chicken</td>
<td>S. Infantis



SI-M1</td>
<td>AMP, ATM, CTT, CTX, FOX, GEN, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA1), blaTEM-1</td>
</tr>
<tr>
<td>33</td>
<td>Chicken</td>
<td>S. Infantis



SI-M2</td>
<td>AMP, ATM, CHL, GEN, KAN, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA1)</td>
</tr>
<tr>
<td>34</td>
<td>Chicken</td>
<td>S. Infantis



SI-M3</td>
<td>AMP, CHL, CIP, NAL, OXA, SPX, STR, TET</td>
<td>floR</td>
</tr>
<tr>
<td>35</td>
<td>Chicken</td>
<td>S. Infantis



SI-M4</td>
<td>AMP, CIP, GEN, KAN, SPX, STR, SXT, TET</td>
<td>qnrB</td>
</tr>
<tr>
<td>36</td>
<td>Beef</td>
<td>S. Infantis



SI-M5</td>
<td>AMP, CHL, KAN, OXA, SPX, STR, SXT, TET</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>37</td>
<td>Beef</td>
<td>S. non-typable



SN-M1</td>
<td>AMP, CIP, GEN, KAN, NAL, OXA</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>38</td>
<td>Cheese</td>
<td>S. Typhimurium



ST-D1</td>
<td>AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaSHV-12, aac(6')-Ib-cr, qnrB, floR</td>
</tr>
<tr>
<td>39</td>
<td>Milk</td>
<td>S. Typhimurium



ST-D2</td>
<td>AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, blaCTX-M-3, qnrS</td>
</tr>
<tr>
<td>40</td>
<td>Milk</td>
<td>S. Typhimurium



ST-D4</td>
<td>AMC, AMP, CHL, CTT, FOX, GEN, KAN, NAL, OXA, STR, SPX, SXT, TET</td>
<td>Class 1 (dfrA15b-cmlA4-aadA2), blaTEM-1</td>
</tr>
<tr>
<td>41</td>
<td>Milk</td>
<td>S. Typhimurium



ST-D3</td>
<td>AMC, AMP, ATM, CTT, FOX, OXA, SPX, STR, SXT, TET</td>
<td>Class 2 (dfrA1-sat2), blaCMY-2</td>
</tr>
<tr>
<td>42</td>
<td>Milk</td>
<td>S. Typhimurium



ST-D5</td>
<td>AMC, AMP, GEN, NAL, SPX, STR, SXT</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>43</td>
<td>Milk</td>
<td>S. Enteriditis



SE-D1</td>
<td>AMP, ATM, CHL, CTT, FOX, GEN, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aac(3)-Id-aadA7), blaTEM-1, blaOXA-1, floR</td>
</tr>
<tr>
<td>44</td>
<td>Milk</td>
<td>S. Enteriditis



SE-D2</td>
<td>AMC, AMP, ATM, CPD, CRO, CTT, CTX, FOX, KAN, NAL, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (dfrA12-orf-aadA2), blaCMY-2</td>
</tr>
<tr>
<td>45</td>
<td>Cheese</td>
<td>S. Enteriditis



SE-D3</td>
<td>CHL, CIP, GEN, NAL, SXT, TET</td>
<td>qnrS</td>
</tr>
<tr>
<td>46</td>
<td>Milk</td>
<td>S. Infantis



SI-D1</td>
<td>AMP, CTT, FOX, KAN, OXA, SPX, STR, SXT, TET</td>
<td>Class 1 (aadA1), blaTEM-1</td>
</tr>
<tr>
<td>47</td>
<td>Milk</td>
<td>S. Infantis



SI-D2</td>
<td>AMP, CHL, GEN, KAN, NAL, SPX, TET</td>
<td>floR</td>
</tr>
</tbody>
</table>

**Abbreviations:** AMC, amoxicillin-clavulanic acid; AMP, ampicillin; ATM, aztreonam; CHL, chloramphenicol; CIP, ciprofloxacin; CPD, cefpodoxime; CRO, ceftriaxone; CTT, cefotetan; CTX, cefotaxime; FOX, cefoxitin; GEN, gentamicin; KAN, kanamycin; NAL, nalidixic acid; OXA, oxacillin; SPX, spectinomycin; STR, streptomycin; SXT, sulfamethoxazole-trimethoprim; TET, tetracycline.

### 3.2. Incidence of class 1 and class 2 integrons in *S. enterica* from meat and dairy products

It is well known that genes encoding antimicrobial resistance are often linked to mobile genetic elements. One of the most important genetic elements responsible for spreading antimicrobial resistance genes among bacteria is the integron (Mazel, 2006). Most integrons were found to be located on plasmids and could be transferred to other strains by conjugation (Van et al., 2012). In this study, PCR identified class 1 integrons in 39.1% *S. enterica* isolates (Table 1). The incidence of class 1 integrons was higher in meat products (39.6%) than in dairy products (37.5%) (Table 1). DNA-sequencing results for the inserted gene cassettes identified 12 types of class 1 integron with 16 different antimicrobial resistance gene cassettes (Table 1). Our results relating to the incidence of class 1 integrons are higher than those reported in the USA, where only 15% of *S. enterica* serovars (including Typhimurium and Enteritidis) isolated from retail meats harbored class 1 integrons (Zhao et al., 2009). This incidence of class 1 integrons in dairy products is significantly higher than that recently reported for the USA by Van Keesel et al. (2013), as class 1 integrons were identified in only 2.8% of the MDR *S. enterica* isolates isolated from bulk milk and milk filters.

Class 2 integron is also an important vehicle for spreading antimicrobial resistance genes in *S. enterica* strains (Ahmed et al., 2005). In this study, PCR identified class 2 integrons in only 8.7% *S. enterica* isolates as follows: *S. enterica* serovar Typhimurium (4.4%) and *S. enterica* serovar Enteritidis (4.4%) (Table 1). The incidence of class 2 integrons was higher in meat products (9.4%) than in dairy products (6.2%), DNA-sequencing results for the inserted gene cassettes identified three types of class 2 integron (Table 1). In Egypt, class 2 integrons have been reported previously in 5.4% of Gram-negative bacteria isolated from bovine mastitis (Ahmed and Shimamoto, 2011). Interestingly, to the best of our knowledge, this is the first report of the detection and identification of class 2 integrons in raw milk.

### 3.3. Incidence of -lactamase-encoding genes in *S. enterica* from meat and dairy products

Production of -lactamases is considered the main mechanism of resistance in Gram-negative bacteria to overcome penicillin-derived (-lactam) antibiotics. Resistance to expanded-spectrum cephalosporins in *S. enterica* is a special concern, since these antimicrobials are a front-line therapeutic for the treatment of numerous Gram-negative infections (Bradford, 2001). In this study, PCR identified -lactamase-encoding genes in 75.4% *S. enterica* isolates. The incidence of -lactamase-encoding genes was very similar in meat products (75.5%) and dairy products (75.0%) (Table 1). These genes include: *blaTEM-1* (41.5%), *blaCMY-2* (11.3%), *blaCTX-M-3* and *blaCTX-M-15* (11.3%), *blaSHV-12* (7.5%) and *blaOXA-1* (3.7%) (Table 1). In China, *blaOXA-1* was the most commonly identified -lactamase gene in *S. enterica* isolates from pigs, ducks and chickens from abattoirs and retail markets, followed by *blaTEM-1*, *blaPSE-1* and *blaCMY-2* (Li et al., 2013). In Portugal, *blaCTX-M-1*, *blaCTX-M-14*, *blaCTX-M-15* and *blaCTX-M-32*, *blaSHV-12* and *blaTEM-1* genes were detected in *S. enterica* isolates from poultry, swine and food products of animal origin (bovine, swine and poultry) (Clemente et al., 2013). In the USA, 11.4% of *S. enterica* isolates isolated from bulk milk and milk filters were positive for *blaCMY* (Van Keesel et al., 2013).

### 3.4. Incidence of plasmid-mediated quinolone resistance genes in *S. enterica* from meat and dairy products

Quinolones are used extensively in veterinary practice worldwide to combat bacterial diseases. *S. enterica* with reduced susceptibility to ciprofloxacin is of serious concern as fluoroquinolone (e.g., ciprofloxacin) is the drug of first choice for the treatment of invasive and systemic salmonellosis that occurs in humans and animals (Dimitrov et al., 2007). In this study, the plasmid-mediated quinolone resistance genes: *qnrA*, *qnrB*, *qnrS* and *aac(6')-Ib-cr* were identified in 28.3% of tested *S. enterica* isolates from meat products. In Hong Kong, it was reported that *S. enterica* strains isolated from retail meats with different resistance profiles harbored plasmid-mediated quinolone resistance genes, one carrying the *qnrS* gene and the other carrying *qnrS* and *aac(6')-Ib-cr* genes (Wong and Chen, 2013). In China, *qnrA*, *qnrB*, *qnrS* and *aac(6')-Ib-cr* genes were identified in *S. enterica* strains isolated from retail foods (including chicken meat) with the incidence of 46.6%, 12.7%, 19.5% and 13.6%, respectively (Yang et al., 2013). In Colombia, *qnrB* was identified in *S. enterica* strains isolated from retail meats (including ground and chicken meats) (Karczmarczyk et al., 2010). Also *qnrB*, *qnrS* and *aac(6')-Ib-cr* were identified in 25% of tested *S. enterica* isolates from dairy products. In Egypt, *qnrA*, *qnrB*, *qnrS* and *aac(6')-Ib-cr* genes were identified in 14.3% of Gram-negative bacteria isolated from bovine mastitis (Ahmed and Shimamoto, 2011). It is well known that *qnr* genes confer only low-level resistance to fluoroquinolones and accumulations of quinolone resistance-determining region (QRDR) mutations is necessary for *S. enterica* to be resistant to fluoroquinolone especially ciprofloxacin (Eaves et al., 2004; Robicsek et al., 2006). To the best of our knowledge, this is the first report of the detection and identification of plasmid-mediated quinolone resistance genes in *S. enterica* isolates from dairy products.

### 3.5. Incidence of florfenicol resistance gene, *floR*, in *S. enterica* from meat and dairy products

Florfenicol was approved by the FDA in 1996 for veterinary use in food animals in the United States. Florfenicol is not approved for human use; however, it is related to chloramphenicol and can select for cross-resistance among bacterial pathogens. Florfenicol resistance is mediated by the *floR* gene, which confers non-enzymatic cross-resistance to chloramphenicol (White et al., 2000). In this study, PCR-and DNA-sequence screening identified *floR* in 18.8% *S. enterica* isolates. The incidence of *floR* was higher in dairy products (25.0%) than in meat products (17.0%) (Table 1). In Malaysia, *floR* was detected in MDR *S. enterica* strains isolated from meat products (raw beef, chicken meat and street foods) (Thong and Modarressi, 2011). In Egypt, *floR* has been detected in 1% of *S. enterica* strains isolated from diseased broilers (Ahmed and Shimamoto, 2012). Also, in this study, *floR* was identified in 25.0% of *S. enterica* strains isolated from dairy products. In Egypt, *floR* has been detected in 6.3% of Gram-negative bacteria isolated from cases of bovine mastitis (Ahmed and Shimamoto, 2011). To the best of our knowledge, this is the first report of the detection and identification of *floR* in dairy products.

### 3.6. Incidence of *S. enterica* serovar Typhimurium DT104 in meat and dairy products

*S. enterica* serovar Typhimurium DT104 is a truly international multiresistant clone of *S. enterica* that emerged in the UK in the early 1990s and then spread worldwide (Threlfall, 2000). *S. enterica* serovar Typhimurium DT104 usually shows a pentadrug-resistance phenotype to ampicillin, chloramphenicol, streptomycin, sulphonamides and tetracyclines (ACSSuT resistance type). In this study, *S. enterica* serovar Typhimurium DT104 was identified at high levels in beef (7.5%) and also in cheese (6.3%) (Table 1). Our findings are very important from the public health point of view as a beef-associated outbreak of *S. enterica* serovar Typhimurium DT104 occurred in September-November 2005 in the Netherlands (Kivi et al., 2007). Also, an outbreak of human salmonellosis caused by *S. enterica* serovar Typhimurium DT104 occurred in Denmark between July and August 2005 due to imported beef served as carpaccio (Ethelberg et al., 2007). Furthermore, in the USA, numerous outbreaks of *S. enterica* serovar Typhimurium DT104 have been associated with the consumption of raw-milk cheese (Cody et al., 1999).

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 2: Results of conjugation experiments and plasmid replicon typing for Salmonella enterica isolated from meat and dairy products.</strong> </caption>
<thead>
<tr>
<th>No.</th>
<th>Isolate / Serovar</th>
<th>Resistance genotype</th>
<th>Plasmid replicon</th>
<th>Conjugable</th>
<th>Transconjugant resistance genotype</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>ST-M1



S. Typhimurium</td>
<td>Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaCMY-2, blaCTX-M-3, qnrB, aac(6')-Ib-cr, floR</td>
<td>N</td>
<td>Yes</td>
<td>blaTEM-1, blaCMY-2, blaCTX-M-3, aac(6')-Ib-cr</td>
</tr>
<tr>
<td>2</td>
<td>ST-M2



S. Typhimurium</td>
<td>Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaCMY-2, blaCTX-M-15, blaSHV-12, qnrA, floR</td>
<td>A/C</td>
<td>Yes</td>
<td>blaTEM-1, blaCMY-2, blaCTX-M-15, blaSHV-12, qnrA</td>
</tr>
<tr>
<td>3</td>
<td>ST-M3



S. Typhimurium</td>
<td>Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaSHV-12, qnrB, floR</td>
<td>I1</td>
<td>Yes</td>
<td>blaTEM-1, blaSHV-12, qnrB</td>
</tr>
<tr>
<td>4</td>
<td>ST-M4



S. Typhimurium</td>
<td>Class 1 (aadA2 and blaPSE-1), qnrS, floR</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>5</td>
<td>ST-M5



S. Typhimurium</td>
<td>Class 1 (dfrA12-orf-aadA2), blaCMY-2, aac(6')-Ib-cr</td>
<td>A/C</td>
<td>Yes</td>
<td>Class 1 (dfrA12-orf-aadA2), blaCMY-2, aac(6')-Ib-cr</td>
</tr>
<tr>
<td>6</td>
<td>ST-M6



S. Typhimurium</td>
<td>Class 1 (aadB-catB3), blaTEM-1, blaOXA-1, floR</td>
<td>I1</td>
<td>Yes</td>
<td>Class 1 (aadB-catB3), blaTEM-1</td>
</tr>
<tr>
<td>7</td>
<td>ST-M7



S. Typhimurium</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1, blaSHV-12, aac(6')-Ib-cr</td>
<td>HI1</td>
<td>Yes</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1, blaSHV-12, aac(6')-Ib-cr</td>
</tr>
<tr>
<td>8</td>
<td>ST-M8



S. Typhimurium</td>
<td>Class 1 (dfrA12-orf-aadA2), blaTEM-1, blaCTX-M-3</td>
<td>L/M</td>
<td>Yes</td>
<td>Class 1 (dfrA12-orf-aadA2), blaTEM-1, blaCTX-M-3</td>
</tr>
<tr>
<td>9</td>
<td>ST-M9



S. Typhimurium</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, blaCTX-M-3, floR</td>
<td>L/M</td>
<td>Yes</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, blaCTX-M-3</td>
</tr>
<tr>
<td>10</td>
<td>ST-M10



S. Typhimurium</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, qnrS</td>
<td>HI1</td>
<td>No</td>
<td>NA</td>
</tr>
<tr>
<td>11</td>
<td>ST-M11



S. Typhimurium</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1, qnrB</td>
<td>I1</td>
<td>Yes</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1, qnrB</td>
</tr>
<tr>
<td>12</td>
<td>ST-M12



S. Typhimurium</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1</td>
<td>I1</td>
<td>Yes</td>
<td>Class 1 (dfrA1-aadA1), blaTEM-1</td>
</tr>
<tr>
<td>13</td>
<td>ST-M13



S. Typhimurium</td>
<td>Class 1 (aadA2), blaTEM-1</td>
<td>I1</td>
<td>Yes</td>
<td>Class 1 (aadA2), blaTEM-1</td>
</tr>
<tr>
<td>14</td>
<td>ST-M14



S. Typhimurium</td>
<td>Class 1 (dfrA15), blaTEM-1, blaCTX-M-15</td>
<td>I1</td>
<td>Yes</td>
<td>Class 1 (dfrA15), blaTEM-1, blaCTX-M-15</td>
</tr>
<tr>
<td>15</td>
<td>ST-M15



S. Typhimurium</td>
<td>Class 2 (dfrA1-sat2-aadA1), blaCMY-2</td>
<td>A/C</td>
<td>Yes</td>
<td>Class 2 (dfrA1-sat2-aadA1), blaCMY-2</td>
</tr>
<tr>
<td>16</td>
<td>ST-M16



S. Typhimurium</td>
<td>Class 2 (dfrA1-sat2-aadA1)</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>17</td>
<td>ST-M17



S. Typhimurium</td>
<td>qnrB</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>18</td>
<td>ST-M18



S. Typhimurium</td>
<td>blaTEM-1, aac(6')-Ib-cr</td>
<td>N</td>
<td>No</td>
<td>NA</td>
</tr>
<tr>
<td>19</td>
<td>ST-M19



S. Typhimurium</td>
<td>floR</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>20</td>
<td>SE-M1



S. Enteriditis</td>
<td>Class 1 (dfrA1-aadA1), blaCMY-2, qnrB, floR</td>
<td>A/C</td>
<td>Yes</td>
<td>Class 1 (dfrA1-aadA1), blaCMY-2, qnrB</td>
</tr>
<tr>
<td>21</td>
<td>SE-M2



S. Enteriditis</td>
<td>Class 1 (aadA2), blaTEM-1, blaCTX-M-3, qnrS</td>
<td>N</td>
<td>Yes</td>
<td>Class 1 (aadA2), blaTEM-1, blaCTX-M-3, qnrS</td>
</tr>
<tr>
<td>22</td>
<td>SE-M3



S. Enteriditis</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, aac(6')-Ib-cr</td>
<td>A/C</td>
<td>Yes</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, aac(6')-Ib-cr</td>
</tr>
<tr>
<td>23</td>
<td>SE-M4



S. Enteriditis</td>
<td>Class 1 (dfrA5), blaTEM-1</td>
<td>A/C</td>
<td>Yes</td>
<td>Class 1 (dfrA5), blaTEM-1</td>
</tr>
<tr>
<td>24</td>
<td>SE-M5



S. Enteriditis</td>
<td>Class 1 (estX-aadA1), blaTEM-1</td>
<td>N</td>
<td>Yes</td>
<td>Class 1 (estX-aadA1), blaTEM-1</td>
</tr>
<tr>
<td>25</td>
<td>SE-M6



S. Enteriditis</td>
<td>blaTEM-1, floR</td>
<td>I1</td>
<td>Yes</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>26</td>
<td>SE-M7



S. Enteriditis</td>
<td>Class 2 (estX-sat2-aadA1), blaTEM-1</td>
<td>A/C</td>
<td>Yes</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>27</td>
<td>SE-M8



S. Enteriditis</td>
<td>Class 2 (estX-sat2-aadA1), blaSHV-12</td>
<td>A/C</td>
<td>Yes</td>
<td>blaSHV-12</td>
</tr>
<tr>
<td>28</td>
<td>SE-M9



S. Enteriditis</td>
<td>Class 2 (estX-sat2-aadA1)</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>29</td>
<td>SE-M10



S. Enteriditis</td>
<td>blaTEM-1</td>
<td>I1</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>30</td>
<td>SE-M11



S. Enteriditis</td>
<td>blaCMY-2</td>
<td>A/C</td>
<td>Yes</td>
<td>blaCMY-2</td>
</tr>
<tr>
<td>31</td>
<td>SE-M12



S. Enteriditis</td>
<td>blaOXA-1</td>
<td>A/C</td>
<td>Yes</td>
<td>blaOXA-1</td>
</tr>
<tr>
<td>32</td>
<td>SI-M1



S. Infantis</td>
<td>Class 1 (aadA1), blaTEM-1</td>
<td>HI1</td>
<td>No</td>
<td>NA</td>
</tr>
<tr>
<td>33</td>
<td>SI-M2



S. Infantis</td>
<td>Class 1 (aadA1)</td>
<td>HI1</td>
<td>No</td>
<td>NA</td>
</tr>
<tr>
<td>34</td>
<td>SI-M3



S. Infantis</td>
<td>floR</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>35</td>
<td>SI-M4



S. Infantis</td>
<td>qnrB</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>36</td>
<td>SI-M5



S. Infantis</td>
<td>blaTEM-1</td>
<td>I1</td>
<td>Yes</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>37</td>
<td>SN-M1



S. non-typable</td>
<td>blaTEM-1</td>
<td>I1</td>
<td>Yes</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>38</td>
<td>ST-D1



S. Typhimurium</td>
<td>Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaSHV-12, aac(6')-Ib-cr, qnrB, floR</td>
<td>N</td>
<td>Yes</td>
<td>blaTEM-1, blaSHV-12, aac(6')-Ib-cr, qnrB</td>
</tr>
<tr>
<td>39</td>
<td>ST-D2



S. Typhimurium</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, blaCTX-M-3, qnrS</td>
<td>L/M</td>
<td>Yes</td>
<td>Class 1 (dfrA17-aadA5), blaTEM-1, blaCTX-M-3, qnrS</td>
</tr>
<tr>
<td>40</td>
<td>ST-D4



S. Typhimurium</td>
<td>Class 1 (dfrA15b-cmlA4-aadA2), blaTEM-1</td>
<td>I1</td>
<td>Yes</td>
<td>Class 1 (dfrA15b-cmlA4-aadA2), blaTEM-1</td>
</tr>
<tr>
<td>41</td>
<td>ST-D3



S. Typhimurium</td>
<td>Class 2 (dfrA1-sat2), blaCMY-2</td>
<td>A/C</td>
<td>Yes</td>
<td>Class 2 (dfrA1-sat2), blaCMY-2</td>
</tr>
<tr>
<td>42</td>
<td>ST-D5



S. Typhimurium</td>
<td>blaTEM-1</td>
<td>I1</td>
<td>Yes</td>
<td>blaTEM-1</td>
</tr>
<tr>
<td>43</td>
<td>SE-D1



S. Enteriditis</td>
<td>Class 1 (aac(3)-Id-aadA7), blaTEM-1, blaOXA-1, floR</td>
<td>I1</td>
<td>Yes</td>
<td>Class 1 (aac(3)-Id-aadA7), blaTEM-1</td>
</tr>
<tr>
<td>44</td>
<td>SE-D2



S. Enteriditis</td>
<td>Class 1 (dfrA12-orf-aadA2), blaCMY-2</td>
<td>A/C</td>
<td>Yes</td>
<td>Class 1 (dfrA12-orf-aadA2), blaCMY-2</td>
</tr>
<tr>
<td>45</td>
<td>SE-D3



S. Enteriditis</td>
<td>qnrS</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>46</td>
<td>SI-D1



S. Infantis</td>
<td>Class 1 (aadA1), blaTEM-1</td>
<td>HI1</td>
<td>No</td>
<td>NA</td>
</tr>
<tr>
<td>47</td>
<td>SI-D2



S. Infantis</td>
<td>floR</td>
<td>NT</td>
<td>NA</td>
<td>NA</td>
</tr>
</tbody>
</table>

NT, not tested.
NA, not applicable.

### 3.7. Transferability and replicon typing of plasmids

In this study, plasmid analysis showed that most of plasmid carrying integrons and resistance genes were conjugable with replicon types: IncI1, IncA/C, IncHI1, IncN and IncL/M (Table 2). These types of plasmid replicon are more frequently detected in plasmids among *Enterobacteriaceae* and play a crucial role in spreading of specific resistance genes especially extended-spectrum -lactamase genes and acquired AmpC genes (Carattoli, 2011). Also, these types of plasmid are considered to be "epidemic resistance plasmids" that are being worldwide detected in *Enterobacteriaceae* of different origins and sources (Carattoli, 2011).

## 4. Conclusions

Our study highlights the high incidence of MDR *S. enterica* in meat and dairy products from Egypt and provides molecular characterization of different mechanisms of antimicrobial resistance. Also, some resistance genes were identified in dairy products for the first time. Information on antibiotic resistance phenotypes and genotypes of foodborne pathogens in different countries and geographic regions is necessary to track the change in resistance pattern and to follow changes in antimicrobial sensitivity patterns that may require a reassessment of treatment and control strategy.