# Antimicrobial resistance of 100 *Salmonella* strains isolated from *Gallus gallus* in 4 wilayas of Algeria

**Authors:** S. Bounar-Kechih, T. M. Hamdi, L. Mezali, F. Assaous, and K. Rahal
**Journal:** Poultry Science 91:1179–1185 (2012)

## Abstract

This study aims at identifying serotypes and surveying the antimicrobial resistance and plasmid support of resistance of 100 *Salmonella* strains, which were isolated from 96 out of 506 (18.97%) samples taken from different production farms in the wilayas (i.e., Algerian states) of Tizi-Ouzou, Bouira, Bejaïa, and Boumerdes in 2007. The highest percentage of *Salmonella* (48%) was recorded in Bouira. Thirteen serotypes were identified among the 100 *Salmonella* strains used in this study. The most prevalent ones were *Salmonella* Heidelberg (24%), *Salmonella* Enteritidis (20%), *Salmonella* Albany (16%), and *Salmonella* Typhimurium (9%). The strains showed resistance to 8 of the 34 antibiotics tested. Fifty-three percent of strains were resistant to at least one antibiotic, among which 15.09% were multiresistant. The most frequently observed resistance was to quinolones (58.49%), with a contribution of 94.74% of *Salmonella* Heidelberg resistant strains. The plasmid transfer performed on 53 strains showed that only 11 exhibited one or more markers of resistance, the most frequent being ampicillin, followed by tetracycline, then cotrimoxazole, sulphonamides, and kanamycin, in that order. The tetracycline characteristics were present in 72.72% of transconjugants, those of the -lactams and sulphonamides in 27.27% each and those of the aminosides in 9.09%. The incompatibility groups of plasmids belong to the Flme and Com1 classes, and the molecular weight of the plasmid DNA was greater than 100 kb. The phenotypic and genotypic results indicate a clonal dissemination in the *Gallus gallus* species in this particular study; this phenomenon could generate resistant bacteria and transferable genes of resistance to humans.

**Key words:** *Salmonella*, *Gallus gallus*, antimicrobial resistance profile, plasmid profile, incompatibility group

## Introduction

The production and consumption of poultry has increased considerably worldwide. Poultry production has left the farmyard and is now industrialized and conducted in a more rational manner (Delpech, 1992). In Algeria, since the 1980s, the emergence of the poultry industries increased the consumption of animal proteins at a much affordable cost (Ferrah et al., 2003). Poultry represents one of the main reservoirs of *Salmonella* that are responsible for a major zoonosis with serious economic and public health consequences (Brisabois, 2001). Antimicrobial agents played a dominant role for the treatment and the control of the salmonellosis. Their overconsumption contributed to the emergence of resistances. In fact, despite the decrease of salmonellosis incidence in the world, multiresistant strains appeared in humans and became increasingly alarming in both industrialized and developing countries (Brisabois, 2001). The surveillance of the current bacterial resistances should not be exclusively based on the determination of the resistant phenotypes. The genetic characterization by conjugation is currently the most used tool in Algeria to better understand the spread of antibiotic resistance in zoonotic bacteria, until the introduction of molecular biology techniques. Our objectives were the phenotypic characterization of 100 *Salmonella* strains by determining the circulating serotypes in the study’s area, the antimicrobial resistance profiles, as well as the genotypic characterization.

## Materials and Methods

**Sampling**
As part of official control of poultry in 4 wilayas of Algeria, from January to December 2007, 100 *Salmonella* strains were isolated from 506 avian samples during autopsies of slain live subjects. This was conducted at the medical bacteriology department of the Regional Veterinary Laboratory of Draa-Ben-Khedda.

**Isolation and Identification of *Salmonella* Strains**
The NF U 47-100/2007 (Afnor, 2007) standard method was adopted for the isolation and the biochemical and serological identification of *Salmonella*. The serotyping was carried out at the enterobacteria service of the Institut Pasteur of Algeria, using *Salmonella* antiserums anti-O and anti-H (BioMérieux, Marcy l’Etoile, France).

**Antimicrobial Sensitivity Testing**
The phenotypic study of the antimicrobial resistance was performed at the Bacteriology and Antibiotherapy Department of the Institut Pasteur of Algeria. Thirty-four antimicrobial agents of 9 different chemical classes were used: ampicillin (10 µg), amoxicillin/clavulanic acid (20 µg/10 µg), mecillinam (10 µg), ticarcillin (75 µg), piperacillin (100 µg), latamoxef (30 µg), imepenem (10 µg), aztreonam (30 µg), cefazolin (30 µg), cefalexin (30 µg), cefoxitin (30 µg), ceftazidim (30 µg), cefepim (30 µg), cefuroxim (30 µg), cefotaxim (30 µg), ceftiofur (30 µg), cephalotin (30 µg), gentamicin (10 µg), amikacin (30 µg), netilmicin (30 µg), kanamycin (30 µg), isepamicin (30 µg), nitrofurantoïne (300 µg), nalidixic acid (30 µg), ciprofloxacin (5 µg), enrofloxacin (5 µg), levofloxacin (5 µg), tetracyclines (30 µg), chloramphenicol (30 µg), sulphonamides (300 µg), trimethoprim (5 µg), trimethoprim/sulfamethoxazole (1.25 µg/23.75 µg) colistin (10 µg), and fosfomycin (200 µg; Bio-Rad, Hercules, CA). Micrograms indicates the antibiotic load in one disk; the unit of this load (expressed in micrograms) for each antibiotic used for each bacterial species is specified by the supplier (Bio-Rad). The minimum inhibitory concentration was established using the E-test (AB-Biodisk). *Escherichia coli* ATCC 25922 (Institut Pasteur Paris, Paris, France) was used for the control antibiograms. The method adopted for the study of the antimicrobial resistance is based on the diffusion in Mueller-Hinton agar media as described by Kirby-Bauer and interpreted according to the Clinical Laboratory Standards Institute (CLSI, 2001) guidelines, which are recommended by the World Health Organization (WHO, 1999). In our study, the multiresistant strains are the ones that confer a resistance to 2 or more antimicrobials. The study of genetic support required the use of antimicrobial powders: tetracycline (Sigma, Woodstock, VA), nalidixic acid (Serva, Heidelberg, Germany), rifampicin (Life-pharmacy, Tuen Mun, Hong Kong, China), nitrofurantoïnes (Saïdal, Annaba, Algeria), kanamycin (Bristol-Meyers Squibb, New York, NY), trimethoprim (Roche, Basel, Switzerland), injectable sodic ampicillin (Biopharm, Bothell, WA), chloramphenicol sodic ampicillin (Serva Peinbiochemica). Also, several reference strains and plasmids were used (Institut Pasteur Paris): *Escherichia coli* K12Nal (nalidixic acid resistance) and *Escherichia coli* C600Rif (rifampicin resistance) for the plasmid transfer by conjugation and Com1 (K) Rif, Com1 PPED 178 (ACTp) Rif, Flme PPED 181/4(T) Nal, and Flme PPED 181/3(CTp) Rif for the compatibility test, as well as the DNA of the  phage digested by the Hind III enzyme as a reference to determine the size of the plasmids. The genetic transfer by conjugation that was used for the detection of plasmids carrying the resistance characters was described by Courvalin et al. (1985).

## Results

Nineteen percent of the total samples were positive for *Salmonella*. The distribution of *Salmonella* per wilaya showed 48% of cases from Bouira, 21% from Bejaïa, 16% from Boumerdes, and 15% from Tizi-Ouzou. Thirteen different serotypes were identified, among which *Salmonella* Heidelberg was the most prevalent (24%). *Salmonella* Heidelberg, *Salmonella* Enteritidis, *Salmonella* Albany, *Salmonella* Infantis, *Salmonella* Blockley, and *Salmonella* Livingstone were present in all the wilayas. Besides, *Salmonella* Enteritidis prevailed in Bouira, *Salmonella* Albany in Bejaïa and Tizi-Ouzou, and *Salmonella* Heidelberg in Boumerdes (Table 1).

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 1. Distribution of Salmonella serotypes isolated from Gallus gallus on 4 wilayas of the center of Algeria</strong> </caption>
<thead>
<tr>
<th rowspan="2">Salmonella serotype</th>
<th colspan="4">Wilaya (n)</th>
<th rowspan="2">Total (%)</th>
</tr>
<tr>
<th>Bouira</th>
<th>Bejaia</th>
<th>Boumerdès</th>
<th>Tizi-Ouzou</th>
</tr>
</thead>
<tbody>
<tr>
<td>Heidelberg</td>
<td>9</td>
<td>4</td>
<td>8</td>
<td>3</td>
<td>24</td>
</tr>
<tr>
<td>Enteritidis</td>
<td>15</td>
<td>2</td>
<td>1</td>
<td>2</td>
<td>20</td>
</tr>
<tr>
<td>Albany</td>
<td>5</td>
<td>6</td>
<td>1</td>
<td>4</td>
<td>16</td>
</tr>
<tr>
<td>Typhimurium</td>
<td>7</td>
<td>2</td>
<td>0</td>
<td>0</td>
<td>9</td>
</tr>
<tr>
<td>Blockley</td>
<td>3</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>6</td>
</tr>
<tr>
<td>Infantis</td>
<td>1</td>
<td>3</td>
<td>1</td>
<td>1</td>
<td>6</td>
</tr>
<tr>
<td>Hadar</td>
<td>3</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>5</td>
</tr>
<tr>
<td>Livingstone</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>2</td>
<td>5</td>
</tr>
<tr>
<td>Pullorum</td>
<td>2</td>
<td>0</td>
<td>1</td>
<td>1</td>
<td>4</td>
</tr>
<tr>
<td>Indiana</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>2</td>
</tr>
<tr>
<td>Kedougou</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>1</td>
</tr>
<tr>
<td>Newport</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>Montevideo</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>1</td>
</tr>
<tr>
<td><strong>Total (%)</strong></td>
<td><strong>48</strong></td>
<td><strong>21</strong></td>
<td><strong>16</strong></td>
<td><strong>15</strong></td>
<td><strong>100</strong></td>
</tr>
</tbody>
</table>

### Antibiogram and Plasmid Profiles of *Salmonella*

The study of the antimicrobial resistance performed on 100 *Salmonella* strains revealed a resistance to 8 of them from 6 different classes. Fifty-three percent of the strains showed resistance to at least one antimicrobial, among which 15.09% were multiresistant. Forty-seven percent of strains were susceptible to all antimicrobials tested. The resistance to quinolones was the most frequent (58.49%) and included 94.74% of the resistant strains of the prevalent serotype in our study (Table 2). The plasmid transfer assay was performed on 53 strains with resistance to -lactams (4%), aminoglycosides (2%), tetracyclines (15%), sulphonamides (13%), nitrofurans (19%), and to quinolones (47%). In total, 11 strains have transferred one or several resistance markers, of which the most frequent were ampicillin, tetracycline, cotrimoxazole, sulphonamides, and kanamycin. The tetracycline character was present in 72.72% of transconjugants, those of the sulphonamides and -lactams in 27.27% each, and that of the aminoglycosides in 9.09%. The compatibility test revealed incompatibility groups of plasmids in Com1 and Flme classes (Table 3). The plasmid DNA analysis resulted in a molecular weight that is greater than 100 kb.

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 2. Rates and antimicrobial resistance profiles of Salmonella strains isolated from Gallus gallus</strong> </caption>
<thead>
<tr>
<th>Salmonella serotype</th>
<th>n1 (Resistant/Total)</th>
<th>n2 (No. of strains)</th>
<th>Resistance profile</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3">Heidelberg</td>
<td rowspan="3">19/24</td>
<td>1</td>
<td>AMP, TCY</td>
</tr>
<tr>
<td>1</td>
<td>NAL</td>
</tr>
<tr>
<td>17</td>
<td>TCY, NAL</td>
</tr>
<tr>
<td rowspan="2">Enteritidis</td>
<td rowspan="2">12/20</td>
<td>9</td>
<td>FT</td>
</tr>
<tr>
<td>3</td>
<td>NAL</td>
</tr>
<tr>
<td>Albany</td>
<td>2/16</td>
<td>2</td>
<td>NAL</td>
</tr>
<tr>
<td rowspan="5">Typhimurium</td>
<td rowspan="5">5/9</td>
<td>1</td>
<td>FT, SSS</td>
</tr>
<tr>
<td>1</td>
<td>TMP, SXT</td>
</tr>
<tr>
<td>1</td>
<td>SSS, TCY, NAL</td>
</tr>
<tr>
<td>1</td>
<td>NAL</td>
</tr>
<tr>
<td>1</td>
<td>NAL, SSS</td>
</tr>
<tr>
<td rowspan="2">Pullorum</td>
<td rowspan="2">4/4</td>
<td>3</td>
<td>FT</td>
</tr>
<tr>
<td>1</td>
<td>SSS</td>
</tr>
<tr>
<td rowspan="2">Blockley</td>
<td rowspan="2">3/6</td>
<td>1</td>
<td>TCY</td>
</tr>
<tr>
<td>2</td>
<td>AMP, TCY</td>
</tr>
<tr>
<td rowspan="2">Hadar</td>
<td rowspan="2">3/5</td>
<td>1</td>
<td>NAL</td>
</tr>
<tr>
<td>2</td>
<td>K, TCY</td>
</tr>
<tr>
<td rowspan="2">Indiana</td>
<td rowspan="2">2/2</td>
<td>1</td>
<td>FT</td>
</tr>
<tr>
<td>1</td>
<td>TMP, SXT, SSS, TCY</td>
</tr>
<tr>
<td>Infantis</td>
<td>1/6</td>
<td>1</td>
<td>NAL</td>
</tr>
<tr>
<td>Livingstone</td>
<td>1/5</td>
<td>1</td>
<td>TCY, SXT, TMP</td>
</tr>
<tr>
<td>Newport</td>
<td>1/1</td>
<td>1</td>
<td>AMP, TCY, NAL</td>
</tr>
<tr>
<td>Kedougou</td>
<td>0/1</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>Montevideo</td>
<td>0/1</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td><strong>% Total</strong></td>
<td><strong>53/100</strong></td>
<td><strong>53</strong></td>
<td></td>
</tr>
</tbody>
</table>

*AMP: ampicillin; K: kanamycin; FT: nitrofurantoine; NAL: nalidixic acid; TCY: tetracycline; SSS: sulphonamides, TMP: trimethoprim, SXT: cotrimoxazole.*

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 3. Determination of incompatibility groups and plasmid profiles of Salmonella strains that transferred resistance markers</strong> </caption>
<thead>
<tr>
<th>Salmonella serotype</th>
<th>n1 (Donor strains)</th>
<th>Resistance profile</th>
<th>n2 (Transferred)</th>
<th>Transferred resistance</th>
<th>Incompatibility group</th>
<th>DNA extraction</th>
</tr>
</thead>
<tbody>
<tr>
<td>Heidelberg</td>
<td>19</td>
<td>AMP, TCY, NAL</td>
<td>1</td>
<td>(AMP) Rif</td>
<td>ND</td>
<td>+</td>
</tr>
<tr>
<td rowspan="2">Typhimurium</td>
<td rowspan="2">5</td>
<td>TCY</td>
<td>1</td>
<td>TCY(NAL)</td>
<td>Com1</td>
<td>+</td>
</tr>
<tr>
<td>SXT, TMP, SSS, TCY</td>
<td>1</td>
<td>(TCY, SSS, TMP)NAL</td>
<td>Com1</td>
<td>+</td>
</tr>
<tr>
<td>Blockley</td>
<td>3</td>
<td>TCY</td>
<td>2</td>
<td>(TCY)NAL</td>
<td>Com1</td>
<td>+</td>
</tr>
<tr>
<td rowspan="3">Hadar</td>
<td rowspan="3">3</td>
<td>AMP, TCY, NAL</td>
<td rowspan="3">3</td>
<td>(AMP) Rif</td>
<td>ND</td>
<td>+</td>
</tr>
<tr>
<td>TCY</td>
<td>TCY(NAL)</td>
<td>Com1</td>
<td>+</td>
</tr>
<tr>
<td>K</td>
<td>K(NAL)</td>
<td>ND</td>
<td>+</td>
</tr>
<tr>
<td>Livingstone</td>
<td>1</td>
<td>TCY, SSS, TMP, SXT</td>
<td>1</td>
<td>(TCY, SSS, TMP)NAL</td>
<td>Com1</td>
<td>NV</td>
</tr>
<tr>
<td>Indiana</td>
<td>2</td>
<td>TCY, SSS, TMP, SXT</td>
<td>1</td>
<td>(TCY, SSS, TMP)NAL</td>
<td>Com1</td>
<td>+</td>
</tr>
<tr>
<td>Newport</td>
<td>-</td>
<td>AMP, TCY, NAL</td>
<td>1</td>
<td>(AMP, TCY) Rif</td>
<td>Flme</td>
<td>NE</td>
</tr>
</tbody>
</table>

*ND: not determined; NV: not visualized; NE: no extract.*

## Discussion

Bouira was characterized by the most important number of isolated strains because most avian samples were from this region, which has more than one-thousand poultry farms. *Salmonella* Heidelberg prevailed in our study. Our results were supported by the study of Barnhart et al. (1991) in Northern Ireland, on 42 poultry samples (56.5%), and that of Mammina et al. (2003) in Italy, on laying hens and the environment of chicken farms (20.3%). Whereas in a national study, *Salmonella* Heidelberg represents only 1.81% (Elgroud et al., 2009). The study of the distribution of *Salmonella* serotypes within 4 wilayas taught us about their dynamic expansion across the adjacent wilayas, or even their possible spread in the national territory. Bouira was the wilaya that counted most of the serotypes, with a high prevalence of *Salmonella* Enteritidis, *Salmonella* Heidelberg, and *Salmonella* Typhimurium. Bejaïa had less diversity than that recorded in Bouira; the predominant serotypes were *Salmonella* Albany, *Salmonella* Heidelberg, and *Salmonella* Infantis. Although Tizi-Ouzou and Boumerdes had only 15 and 16 strains, diversity was observed with a predominance of *Salmonella* Albany and *Salmonella* Heidelberg, respectively. The homogeneous distribution of all serotypes in the study’s area could be the consequence of the uncontrolled movement of poultry products.

There is a positive correlation between antimicrobial use and bacterial resistance rates; indeed, each exposure eliminates the susceptible bacteria and promotes the growth of resistant strains (Jaecklin, 2002; Fluit, 2005). Even though the treatment of avian salmonellosis is banned in Algeria by ministerial order (JORA, 2003), the high rate of antimicrobial resistance that we recorded (53%) could be explained by the indiscriminate use of these drugs for therapeutic, prophylactic purposes or as growth promoters. The observed resistance to first generation quinolones (nalidixic acid) was the most frequent. This is likely due to their repeated use in poultry production. Fortunately, there is no resistance to fluoroquinolones tested (ciprofloxacin, enrofloxacin, levofloxacin) as reported by Mc Kenzie and Nadeau (2006).

**Genetic Support of Resistance**
The *Salmonella* strains resistant to nalidixic acid and nitrofurantoïnes were not transferable, confirming the chromosomal support of these resistances. The majority of resistance markers were transferred; the most common were ampicillin, tetracycline, cotrimoxazole, sulphonamides, and kanamycin. In our study, a plasmid grouping identified the same classes (Com1 and Flme) as described by Chaslus-Dancla (1980) in France and found in a national study on enterobacteria isolated in western Algeria (Barka, 2002). The results obtained by physical analysis of plasmid content of transconjugants implied that the spread of resistance characteristics of studied *Salmonella* strains is provided by high molecular weight plasmids.