# Antimicrobial susceptibility and serovars of *Salmonella* from chickens and humans in Ibadan, Nigeria

**Authors:** Kayode Fashae, Folasade Ogunsola, Frank M. Aarestrup, Rene S. Hendriksen
**Journal:** J Infect Dev Ctries 2010; 4(8):484-494.

## Abstract

**Background:** This study determines the prevalence and antibiotic resistance of *Salmonella* serovars from humans and chickens in Ibadan, Nigeria, in 2004-2007.

**Methodology:** A total of 991 blood samples were collected from patients in 2004 to 2005 and 641 fecal samples were collected from poultry farms in 2007. All *Salmonella* isolates were serotyped and tested for antimicrobial susceptibility.

**Results:** Thirty-nine (4%) *Salmonella* isolates were obtained from human blood and 70 (11%) from chicken fecal samples. The human isolates revealed nine different serovars; 82% were non-typhoidal *Salmonella* and 18% were (*S.* Typhi). The majority of serovars from humans were *S.* Enteritidis (33%), *S.* Dublin (18%), and *S.* Typhimurium (18%). Resistance to chloramphenicol, sulfamethoxazole, trimethoprim, and ampicillin ranged from 36% to 59% for the human isolates. Eight different serovars were obtained from chickens; *S.* Virchow (71%) predominated. A high frequency (87%) of reduced susceptibility to ciprofloxacin was observed among the chicken isolates. A high frequency of resistance to tetracycline (93%), nalidixic acid (81%), and sulfamethoxazole (87%) was observed. Rare serovars such as *S.* Apapa, *S.* Mouschaui, *S.* Jukestown, *S.* Oritamerin, and *S.* Onireke were isolated from both humans and chickens. Identical serovars were not found among human and chicken isolates.

**Conclusions:** This study indicates that chickens are not a reservoir of *Salmonella* causing bacteraemia among humans in Ibadan, Nigeria. Studies locating the reservoirs responsible for invasive salmonellosis in humans are needed. Controls and targeted interventions against *S.* Virchow and the frequent occurrence of antimicrobial resistance in chickens should be initiated to prevent the spread of this serovar.

**Key words:** *Salmonella*, Nigeria, antimicrobial resistance, human, chicken, bacteriaemia, plasmid mediated quinoline resistance

## Introduction

*Salmonella enterica* is an important food-borne pathogen that is currently divided into 2,587 serotypes. The typhiodal *Salmonella* serovars are host-restricted human pathogens and include *Salmonella enterica* serotype Typhi and *S.* Paratyphi. Non-typhoidal *Salmonella* (NTS) are zoonotic agents and a wide variety of animals have been identified as reservoirs. Poultry are considered as one of the most common vehicles of human salmonellosis.

Salmonellosis is often associated with gastroenteritis, which is usually self-limiting. In some cases, particularly in children, the elderly, and immunocompromised patients, *Salmonella* infection can lead to invasive and focal infections that can be severe. The ability of *Salmonella* to cause invasive infection varies with the serovar, the age of patient, and region. *S.* Typhi and *S.* Paratyphi A are highly invasive for both children and adults. Certain serovars of NTS, such as *S.* Choleraesuis, *S.* Dublin, and *S.* Virchow, have higher abilities than other serovars to cause invasive salmonellosis. In sub-Saharan Africa, NTS invasive infections are a public health concern for infants, young children, and young adults suffering from malnutrition, malaria, and HIV infection. Most human infections are caused by a limited number of serovars, which may vary from country to country and over time. The predominant serovars in some African countries include *S.* Enteritidis, *S.* Typhimurium, *S.* Concord, and *S.* Isangi.

In Nigeria, there is limited data of the prevalence in *Salmonella* serovars causing human salmonellosis and the importance of transmitting food-borne pathogens from the food chain to humans is not well understood. Decades ago, a study reported the important role of *S.* Typhi and NTS in septicemia in humans in Ibadan, Nigeria; however, the specific NTS serotypes were not revealed, probably due to the lack of resources and high-quality antisera that are often lacking from many African countries.

Chicken is an important food source to man; chicken meat dishes are special delicacies, particularly during festive periods in Nigeria, but the poultry industry is largely unregulated. The link between human salmonellosis and consumption of chicken is not known nor is the prevalence of the serovars in Nigeria. A previous study found the prevalence of *S.* Enteritidis in chicken meat in Maiduguri, Nigeria, to be 27%. A recent study from Maiduguri, in the northern part of Nigeria, reported *S.* Hiduddify to be the predominant serovar in free-range chickens and poultry meat.

Treatment with antimicrobials is crucial for the proper management of severe or invasive human salmonellosis. Fluoroquinolones and third-generation cephalosporins are now commonly used in adults for treatment due to widespread resistance to chloramphenicol, ampicillin, and cotrimoxazole. Fluoroquinolones are often the last resort for treatment of children and are listed by the World Health Organization as critically important antimicrobials for human health. Multi-drug resistance and reduced susceptibility to ciprofloxacin have been reported from some African countries. Increasing multi-drug resistance associated with NTS in humans has been reported from Malawi and Ethiopia; multi-drug resistance rates of 10% or less with decreased susceptibility to ciprofloxacin have also been reported from Senegal. A recent study reported high rates of resistance to ciprofloxacin in *S.* Hiduddify in chickens in Maiduguri, Nigeria; otherwise, there are limited data on antimicrobial resistance on *Salmonella* in both humans and food animals in Nigeria.

This study was conducted to determine the prevalence of *Salmonella* serovars in chickens and from invasive extraintestinal salmonellosis in humans from the city of Ibadan, Nigeria, in various periods between 2004 and 2007. The susceptibility to antimicrobial resistance in the *Salmonella* isolates was also investigated and the presence of *qnr* genes was investigated in the isolates with a resistance phenotype indicating the presence of *qnr* genes. The serovars and susceptibility patterns of the isolates from the different reservoirs were compared.

## Materials and methods

### Isolation of bacteria from humans

During 2004 and 2005, blood samples were collected from 991 febrile patients attending seven major hospitals and a private diagnostic laboratory in Ibadan province. Blood samples of 10 to 15 mL and 2 to 5 mL of were collected from adults and children, respectively. Bacteria were isolated from each blood sample following standard procedure by using Brain-Heart infusion broth (Lab M, Lancashire, UK) followed by plating on Brain-Heart Infusion agar supplemented with 5% sheep blood, Salmonella-Shigella agar (Lab M, Lancashire, UK), and McConkey agar (Lab M, Lancashire, UK). The isolates were incubated aerobically at 37°C for 18 to 24 hours. Presumptive *Salmonella* colonies were identified by standard methods.

### Isolation of bacteria from chicken

In 2007, a total of 641 fecal samples were collected at 12 intensive poultry farms. Five grams of each chicken fecal sample were enriched in 25 mL of selenite-F broth (Lab M, Lancashire, UK) and incubated aerobically at 37°C for 18 to 24 hours followed by plating on Salmonella-Shigella agar (Lab M, Lancashire, UK) and incubated aerobically at 37°C for 18 to 24 hours. Presumptive *Salmonella* colonies were identified following standard methods.

### Statistical analysis

SAS version 9.1.3 (SAS Institute Inc.) was used to assess the association of the prevalence in males compared to the prevalence in females using a Fisher’s Exact test. The criteria for evaluating the significance level in the model was a P-value of < 0.05.

### Serotyping

Serotyping of all the presumptive biochemical positive *Salmonella* isolates was performed at the National Salmonella and Shigella Centre, Bangkok, Thailand, according to the previously described method.

### Antimicrobial susceptibility testing

Susceptibility to antimicrobial agents was performed at the National Food Institute, Copenhagen, Denmark (DTU-FOOD) on all *Salmonella* isolates as minimum inhibitory concentration (MIC) determinations according to the previously described method. In addition to the previously described method, the following antimicrobial and resistance epidemiological cut-off values were used in the study for cefotaxime; (R > 2 mg/mL).

### Detection of plasmid mediated quinolone resistance genes

Strains showing reduced susceptibility to ciprofloxacin and susceptible to nalidixic acid were further characterized through the use of a PCR assay with primers specific for *qnrA*, *qnrB*, *qnrC*, *qnrD*, *qnrS*, and *aac(6')-Ib*. PCR products were purified using the GFX™ PCR DNA kit (GE Healthcare, Chalfont St. Giles, UK) and submitted to Macrogen Inc. for sequencing. Sequence analysis and alignment was performed using the Vecton NTI suite 9 (InforMax Inc., Bethesda, Maryland, US) and the resulting nucleotide sequences were compared to sequences obtained from GenBank.

## Results

### Bacterial isolates from humans

Of the 991 blood samples collected, 519 samples originated from males. Thirty-nine samples yielded *Salmonella*, which gave an overall prevalence among humans submitting a blood sample at 4%. The prevalence ranged substantially between the different sample sites from 1% (diagnostic laboratory) to 10% (hospital B) (Table 1). The prevalence between males and females differed with 5% among males in contrast to 3% among females (not significant P = 0.07). The overall age of patients from whom the positive blood samples were collected ranged from seven days to 28 years; however, large differences were observed between the different hospitals. The median age of the febrile patients that yielded *Salmonella* was 1 year and 7 months.

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 1. Number of blood samples collected from humans (male/female) and the number and prevalence of Salmonella spp. by sites.</strong></caption>
<thead>
<tr>
<th rowspan="2">Location</th>
<th colspan="3">No. blood samples collected</th>
<th colspan="3">No. (%) blood samples positive</th>
</tr>
<tr>
<th>Total</th>
<th>Male</th>
<th>Female</th>
<th>Total</th>
<th>Male</th>
<th>Female</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hospital A</td>
<td>204</td>
<td>116</td>
<td>88</td>
<td>9 (4)</td>
<td>8 (7)</td>
<td>1 (1)</td>
</tr>
<tr>
<td>Hospital B</td>
<td>119</td>
<td>66</td>
<td>53</td>
<td>10 (15)</td>
<td>12 (10)</td>
<td>2 (4)</td>
</tr>
<tr>
<td>Hospital C</td>
<td>120</td>
<td>66</td>
<td>54</td>
<td>1 (2)</td>
<td>5 (4)</td>
<td>4 (7)</td>
</tr>
<tr>
<td>Hospital D</td>
<td>102</td>
<td>52</td>
<td>50</td>
<td>4 (4)</td>
<td>2 (4)</td>
<td>2 (4)</td>
</tr>
<tr>
<td>Hospital E</td>
<td>104</td>
<td>54</td>
<td>50</td>
<td>3 (3)</td>
<td>1 (2)</td>
<td>2 (4)</td>
</tr>
<tr>
<td>Hospital F</td>
<td>107</td>
<td>38</td>
<td>69</td>
<td>2 (2)</td>
<td>2 (5)</td>
<td>0 (0)</td>
</tr>
<tr>
<td>Hospital G</td>
<td>157</td>
<td>83</td>
<td>74</td>
<td>3 (2)</td>
<td>2 (2)</td>
<td>1 (1)</td>
</tr>
<tr>
<td>Diagnostic lab.</td>
<td>78</td>
<td>44</td>
<td>34</td>
<td>1 (1)</td>
<td>0 (0)</td>
<td>1 (3)</td>
</tr>
<tr>
<td><strong>Total</strong></td>
<td><strong>991</strong></td>
<td><strong>519</strong></td>
<td><strong>472</strong></td>
<td><strong>39 (4)</strong></td>
<td><strong>26 (5)</strong></td>
<td><strong>13 (3)</strong></td>
</tr>
</tbody>
</table>

### Bacterial isolates from chickens

Out of the 641 chicken fecal samples collected, 70 were positive for *Salmonella*, giving an overall prevalence of 11%. The number of samples investigated and the number of samples positive for *Salmonella* fluctuated among the different farms, resulting in a prevalence ranging from 2% (farm C and G) to 26% (farm J) (Table 2).

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 2. Number of samples investigated and the occurrence of Salmonella serovars by farms</strong></caption>
<thead>
<tr>
<th rowspan="2">Farm no.</th>
<th rowspan="2">No. samples per farm</th>
<th colspan="8">Number and percentages of isolates per serovars</th>
<th rowspan="2">Total no. serovars per farm (%)</th>
</tr>
<tr>
<th>S. Bredeney</th>
<th>S. Derby</th>
<th>S. Haifa</th>
<th>S. Havana</th>
<th>S. Muenster</th>
<th>S. Mbandaka</th>
<th>S. Onireke</th>
<th>S. Virchow</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td>35</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (25)</td>
<td>1 (25)</td>
<td>0 (0)</td>
<td>2 (50)</td>
<td>4 (11)</td>
</tr>
<tr>
<td>B</td>
<td>38</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>1 (3)</td>
</tr>
<tr>
<td>C</td>
<td>41</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>1 (2)</td>
</tr>
<tr>
<td>D</td>
<td>38</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>4 (100)</td>
<td>4 (11)</td>
</tr>
<tr>
<td>E</td>
<td>44</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (25)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>3 (75)</td>
<td>4 (9)</td>
</tr>
<tr>
<td>F</td>
<td>44</td>
<td>0 (0)</td>
<td>1 (17)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (17)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>4 (67)</td>
<td>6 (14)</td>
</tr>
<tr>
<td>G</td>
<td>47</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (2)</td>
</tr>
<tr>
<td>H</td>
<td>42</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (25)</td>
<td>2 (50)</td>
<td>0 (0)</td>
<td>1 (25)</td>
<td>4 (10)</td>
</tr>
<tr>
<td>I</td>
<td>80</td>
<td>0 (0)</td>
<td>2 (33)</td>
<td>0 (0)</td>
<td>1 (17)</td>
<td>2 (33)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (17)</td>
<td>6 (8)</td>
</tr>
<tr>
<td>J</td>
<td>92</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>2 (8)</td>
<td>0 (0)</td>
<td>22 (92)</td>
<td>24 (26)</td>
</tr>
<tr>
<td>K</td>
<td>80</td>
<td>1 (8)</td>
<td>1 (8)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>10 (83)</td>
<td>12 (15)</td>
</tr>
<tr>
<td>L</td>
<td>60</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (33)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (33)</td>
<td>1 (33)</td>
<td>3 (5)</td>
</tr>
<tr>
<td><strong>Total no. (%)</strong></td>
<td><strong>641</strong></td>
<td><strong>1 (1%)</strong></td>
<td><strong>4 (6%)</strong></td>
<td><strong>2 (3%)</strong></td>
<td><strong>1 (1%)</strong></td>
<td><strong>5 (7%)</strong></td>
<td><strong>6 (9%)</strong></td>
<td><strong>1 (1%)</strong></td>
<td><strong>50 (71%)</strong></td>
<td><strong>70 (11%)</strong></td>
</tr>
</tbody>
</table>

### Serotyping of human isolates

The 39 human isolates revealed nine different *Salmonella* serovars of which 32 (82%) were NTS and seven (18%) were *S.* Typhi. The most commonly isolated serovars were *S.* Enteritidis (n = 13; 33%), *S.* Dublin (n = 7; 18%), *S.* Typhimurium (n = 7; 18), and *S.* Typhi (n = 7; 18%). Several uncommon serovars were among the isolated *Salmonella*, such as *S.* Jukestown (n = 1; 3%), *S.* Monschaui (n = 1; 3%), *S.* Oritamerin (n = 1; 3%), and *S.* Apapa (n = 1; 3%), which were all from different sample sites (Table 3). Hospitals A and B accounted for 21 of the isolates. These two hospitals also accounted for four out of seven (57%) *S.* Dublin, 10 out of 13 (77%) *S.* Enteritidis, and six out of seven (86%) *S.* Typhimurium (Table 3). None of the serovars found in humans were observed among the isolates from chicken.

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 3. Frequency of and occurrence of Salmonella serovars from human blood samples by sites.</strong></caption>
<thead>
<tr>
<th rowspan="2">Sample site</th>
<th rowspan="2">No. samples</th>
<th colspan="9">Number and percentages of isolates per serovars</th>
<th rowspan="2">Total no. (%) serovars</th>
</tr>
<tr>
<th>S. Apapa</th>
<th>S. Dublin</th>
<th>S. Enteritidis</th>
<th>S. Infantis</th>
<th>S. Jukestown</th>
<th>S. Monschaui</th>
<th>S. Oritamerin</th>
<th>S. Typhi</th>
<th>S. Typhimurium</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hospital A</td>
<td>204</td>
<td>0 (0)</td>
<td>2 (22)</td>
<td>7 (78)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>9 (23)</td>
</tr>
<tr>
<td>Hospital B</td>
<td>119</td>
<td>0 (0)</td>
<td>3 (25)</td>
<td>2 (17)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (8)</td>
<td>0 (0)</td>
<td>6 (50)</td>
<td>12 (31)</td>
</tr>
<tr>
<td>Hospital C</td>
<td>120</td>
<td>0 (0)</td>
<td>2 (40)</td>
<td>2 (40)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (20)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>5 (13)</td>
</tr>
<tr>
<td>Hospital D</td>
<td>102</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>3 (75)</td>
<td>1 (25)</td>
<td>4 (10)</td>
</tr>
<tr>
<td>Hospital E</td>
<td>104</td>
<td>1 (33)</td>
<td>1 (33)</td>
<td>0 (0)</td>
<td>1 (33)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>3 (8)</td>
</tr>
<tr>
<td>Hospital F</td>
<td>107</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (50)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (50)</td>
<td>0 (0)</td>
<td>2 (5)</td>
</tr>
<tr>
<td>Hospital G</td>
<td>157</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (33)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>2 (67)</td>
<td>0 (0)</td>
<td>3 (8)</td>
</tr>
<tr>
<td>Diag. lab.</td>
<td>78</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>0 (0)</td>
<td>1 (3)</td>
</tr>
<tr>
<td><strong>Total (%)</strong></td>
<td><strong>39 (4)</strong></td>
<td><strong>1 (3)</strong></td>
<td><strong>7 (18)</strong></td>
<td><strong>13 (33)</strong></td>
<td><strong>1 (3)</strong></td>
<td><strong>1 (3)</strong></td>
<td><strong>1 (3)</strong></td>
<td><strong>1 (3)</strong></td>
<td><strong>7 (18)</strong></td>
<td><strong>7 (18)</strong></td>
<td><strong>39 (100)</strong></td>
</tr>
</tbody>
</table>

### Serotyping of chicken isolates

The 70 *Salmonella* isolates revealed eight different serovars with *S.* Virchow being the most frequent, accounting for 71% (n = 50) of all serovars isolated from chickens. *S.* Virchow was found in chickens from 11 of the 12 farms, accounting for 92% (n = 22) and 83% (n = 10) of all serovars found on farms J and K. Additionally, it was the only serovar originating from chickens in farms B, C, and D (Table 2). Two uncommon serovars from chickens were isolated: *S.* Haifa (n = 2; 3%) and *S.* Onireke (n = 1; 1%) (Table 2).

### Antimicrobial susceptibility testing of human isolates

The *Salmonella* isolates originating from humans were all susceptible to apramycin, ceftiofur, colistin, florfenicol, cefotaxime, gentamicin, and neomycin. Low frequency of resistance was observed to nalidixic acid (3%) and ciprofloxacin (3%). Antimicrobial resistance to spectinomycin, chloramphenicol, tetracycline, and streptomycin ranged from 18% to 44%. A higher frequency of antimicrobial resistance was observed to sulfamethoxazole (54%), trimethoprim (54%), amoxicillin+clavulanic (54%), and ampicillin (59%) (Table 4).

Three of the seven *S.* Dublin were resistant to ampicillin, streptomycin, sulfamethoxazole, tetracycline, and trimethoprim; the remaining four isolates were pansusceptible. Antimicrobial resistance was observed in eleven of the 13 *S.* Enteritidis isolates. Four *S.* Enteritidis isolates conferred resistance to ampicillin, chloramphenicol, sulfamethoxazole, and trimethoprim. Two additional isolates also conferred resistance to streptomycin. One of the remaining five *S.* Enteritidis isolates was resistant to ampicillin, trimethoprim, streptomycin, amoxicillin+clavulanic acid, nalidixic acid, and tetracycline. None of the 13 *S.* Enteritidis isolates were pansusceptible. Six of seven *S.* Typhi isolates were pansusceptible while the remaining one isolate was resistant only to streptomycin.

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 4. Frequency of antimicrobial resistance in Salmonella isolates from human blood samples</strong></caption>
<thead>
<tr>
<th rowspan="2">Serovars</th>
<th colspan="9">Number and (percentages) of antimicrobial resistance</th>
</tr>
<tr>
<th>Amox + Clav</th>
<th>Ampicillin</th>
<th>Chloramphenicol</th>
<th>Ciprofloxacin</th>
<th>Nalidixic acid</th>
<th>Spectinomycin</th>
<th>Streptomycin</th>
<th>Sulfamethoxazole</th>
<th>Tetracycline</th>
<th>Trimethoprim</th>
</tr>
</thead>
<tbody>
<tr>
<td>S. Apapa (n=1)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>1 (100)</td>
</tr>
<tr>
<td>S. Dublin (n=7)</td>
<td>0 (0)</td>
<td>3 (43)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>3 (43)</td>
<td>3 (43)</td>
<td>3 (43)</td>
<td>3 (43)</td>
</tr>
<tr>
<td>S. Enteritidis (n=13)</td>
<td>13 (100)</td>
<td>12 (93)</td>
<td>7 (54)</td>
<td>0 (0)</td>
<td>1 (8)</td>
<td>0 (0)</td>
<td>4 (31)</td>
<td>13 (100)</td>
<td>1 (8)</td>
<td>4 (31)</td>
</tr>
<tr>
<td>S. Infantis (n=1)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
</tr>
<tr>
<td>S. Jukestown (n=1)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
</tr>
<tr>
<td>S. Monschaui (n=1)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
</tr>
<tr>
<td>S. Oritamerin (n=1)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
</tr>
<tr>
<td>S. Typhi (n=7)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (14)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
</tr>
<tr>
<td>S. Typhimurium (n=7)</td>
<td>7 (100)</td>
<td>7 (100)</td>
<td>7 (100)</td>
<td>1 (14)</td>
<td>0 (0)</td>
<td>7 (100)</td>
<td>7 (100)</td>
<td>7 (100)</td>
<td>4 (57)</td>
<td>7 (100)</td>
</tr>
<tr>
<td><strong>Total (n=39)</strong></td>
<td><strong>21 (54)</strong></td>
<td><strong>23 (59)</strong></td>
<td><strong>14 (36)</strong></td>
<td><strong>1 (3)</strong></td>
<td><strong>1 (3)</strong></td>
<td><strong>7 (18)</strong></td>
<td><strong>17 (44)</strong></td>
<td><strong>21 (54)</strong></td>
<td><strong>12 (31)</strong></td>
<td><strong>21 (54)</strong></td>
</tr>
</tbody>
</table>
** All isolates were susceptible to: apramycin, ceftiofur, colistin, florfenicol and cefotaxime, gentamicin, and neomycin.*

### Antimicrobial susceptibility testing of chicken isolates

All 70 chicken isolates were susceptible to apramycin, ceftiofur, chloramphenicol, colistin, florfenicol, and cefotaxime. In contrast, we observed a high level of antimicrobial resistance to tetracycline (n = 65; 93%), ciprofloxacin (n = 61, 87%), nalidixic acid (n = 57; 81%), sulfamethoxazole (n = 61; 87%), and trimethoprim (n = 53; 76%) respectively (Table 5). None of the chicken isolates showed high levels of resistance to ciprofloxacin (MIC  2 g/mL); however, 87% (n = 61) of the isolates exhibited reduced susceptibility to ciprofloxacin. The reduced susceptibility to ciprofloxacin was observed in all of the following serovars: *S.* Virchow (n = 50; 100%), *S.* Derby (n = 4; 100%), *S.* Haifa (n = 2; 100%), *S.* Bredeney (n = 1; 100%), *S.* Onireke (n = 1; 100%), *S.* Mbandaka (n = 2; 33%), and *S.* Muenster (n = 1; 20%), respectively (Table 5). The four isolates of *S.* Derby were all susceptible to nalidixic acid but exhibited reduced susceptibility to ciprofloxacin, indicating a plasmid-mediated quinolone resistance phenotype.

Antimicrobial susceptibility of the three most commonly isolated serovars revealed that *S.* Virchow (n = 50), *S.* Mbandaka (n = 6), and *S.* Muenster (n = 5) were also the serovars with the most resistant susceptibility patterns. Those serovars showed resistance to ten (*S.* Virchow), nine (*S.* Muenster), and eight (*S.* Mbandaka) of the seventeen tested antimicrobials, respectively (Table 5). Six resistance patterns were observed among *S.* Virchow; most (n = 31; 62%) were resistant to nalidixic acid, sulfamethoxazole, tetracycline, and trimethoprim, while only 8% (n = 4) showed multidrug resistance to seven antimicrobials, namely ampicillin, gentamicin, nalidixic acid, neomycin, sulfamethoxazole, tetracycline, and trimethoprim.

<table border="1" style="border-collapse: collapse; width: 100%;">
<caption><strong>Table 5. Frequency of antimicrobial resistance of Salmonella isolates from chicken</strong></caption>
<thead>
<tr>
<th rowspan="2">Serovars</th>
<th colspan="9">Number (percentages) of antimicrobial resistance*</th>
</tr>
<tr>
<th>Amox + Clav</th>
<th>Ampicillin</th>
<th>Ciprofloxacin</th>
<th>Gentamicin</th>
<th>Nalidixic acid</th>
<th>Neomycin</th>
<th>Spectinomycin</th>
<th>Sulfamethoxazole</th>
<th>Tetracycline</th>
<th>Trimethoprim</th>
</tr>
</thead>
<tbody>
<tr>
<td>S. Bredeney (n=1)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>1 (100)</td>
</tr>
<tr>
<td>S. Derby (n=4)</td>
<td>3 (75)</td>
<td>3 (75)</td>
<td>4 (100)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>3 (75)</td>
<td>2 (50)</td>
<td>2 (50)</td>
</tr>
<tr>
<td>S. Haifa (n=2)</td>
<td>1 (50)</td>
<td>0 (0)</td>
<td>2 (100)</td>
<td>1 (50)</td>
<td>1 (50)</td>
<td>2 (100)</td>
<td>1 (50)</td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>2 (100)</td>
</tr>
<tr>
<td>S. Havana (n=1)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
</tr>
<tr>
<td>S. Muenster (n=5)</td>
<td>0 (0)</td>
<td>1 (20)</td>
<td>1 (20)</td>
<td>0 (0)</td>
<td>1 (20)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (20)</td>
<td>5 (100)</td>
<td>1 (20)</td>
</tr>
<tr>
<td>S. Mbandaka (n=6)</td>
<td>1 (17)</td>
<td>2 (33)</td>
<td>2 (33)</td>
<td>1 (17)</td>
<td>4 (67)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>5 (83)</td>
<td>6 (100)</td>
<td>5 (83)</td>
</tr>
<tr>
<td>S. Onireke (n=1)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>0 (0)</td>
<td>1 (100)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
<td>0 (0)</td>
</tr>
<tr>
<td>S. Virchow (n=50)</td>
<td>12 (24)</td>
<td>12 (24)</td>
<td>50 (100)</td>
<td>12 (24)</td>
<td>49 (98)</td>
<td>6 (12)</td>
<td>0 (0)</td>
<td>49 (98)</td>
<td>49 (98)</td>
<td>30 (60)</td>
</tr>
<tr>
<td><strong>Total (n=70)</strong></td>
<td><strong>18 (26)</strong></td>
<td><strong>19 (27)</strong></td>
<td><strong>61 (87)</strong></td>
<td><strong>14 (20)</strong></td>
<td><strong>57 (81)</strong></td>
<td><strong>8 (11)</strong></td>
<td><strong>1 (1)</strong></td>
<td><strong>61 (87)</strong></td>
<td><strong>65 (93)</strong></td>
<td><strong>53 (76)</strong></td>
</tr>
</tbody>
</table>
** All isolates were susceptible to: apramycin, ceftiofur, colistin, florfenicol and cefotaxime, chloramphenicol.*

### Detection of *qnr* genes

Four *S.* Derby isolates from chickens showed reduced susceptibility to ciprofloxacin and were susceptible to nalidixic acid. All four harbored the *qnrS1* gene. The *qnrS1* positive *S.* Derby were from farm K (n = 1), farm I (n = 2), and farm F (n = 1).

## Discussion

In this study, the overall positive isolation rate of *Salmonella* species from human blood samples is comparable with that from Ethiopia (4%) but higher than those from Kenya (1%) and Mozambique (2%). On the other hand, higher rates have been reported from Congo (7%) and Malawi (6% and 8%). The skewed distribution of 1:0.5 between males and females is difficult to explain, but may be caused by the fact that more males were examined or that the male population has a higher susceptibility to infections.

Our data reveals the relative importance of *Salmonella* associated with human bacteraemia predominated by *S.* Enteritidis and *S.* Typhimurium in Ibadan, Nigeria. Similar trends have been observed in African countries where *S.* Enteritidis and *S.* Typhimurium are the most common serovars causing human salmonellosis. It is interesting to note that the majority of the *S.* Enteritidis and *S.* Dublin were isolated from hospitals A, B, and C, whereas the remaining hospitals found very few. *S.* Typhimurium was mostly isolated from hospital B. Kariuki et al. found that the main serovars obtained from children with bacteraemia in Kenya were *S.* Enteritidis (55%) and *S.* Typhimurium (33%) compared to other non-typhoidal *Salmonella*. Similar trends have been observed elsewhere. Epidemic increase in prevalence of *S.* Typhimurium has recently been linked to circulation of a particular MLST clone, ST313, in sub-Saharan African countries; however, it has not been determined if the *S.* Typhimurium ST313 clone has spread to Ibadan, Nigeria. A future study will reveal if this is the case or if the isolates belong to the previously described clone of phage type U282. Similarly, the clonality of invasive *S.* Enteritidis also needs to be investigated to determine whether certain clones are responsible for causing bacteraemia compared to those causing mild gastroenteritis. It is also noteworthy that neither *S.* Enteritidis nor *S.* Typhimurium was isolated from chickens included in this study, indicating that intensively raised chickens are not reservoirs for invasive serovars in humans. It should be noted, however, that the samples obtained from chickens were collected in 2007, compared to the samples from humans, which were collected from 2004 to 2005. In 2004, *S.* Enteritidis and *S.* Typhimurium were isolated by Orji et al. from poultry droppings and beef.

*S.* Dublin was not reported as an important cause of invasive salmonellosis in some sub-Saharan African countries, but this serovar constitutes an important human pathogen in Upper Volta. *S.* Dublin is host-adapted to cattle and is highly invasive; it is transmitted to humans through beef, cheese, and raw milk. In many European countries, surveillance shows that *S.* Dublin is the most commonly isolated serovar from bovine meat, exceeding the level of *S.* Typhimurium. Only a few publications have described *S.* Dublin in animals from Nigeria and all of them also associate the serovar to cattle. Cattle farming is a thriving business in Nigeria because beef meat is highly consumed, either in soup dishes or roasted (suya); other products such as raw milk and cheese are heavily consumed in Nigeria without adequate control measures to prevent spread of zoonotic agents to human. It would be interesting to determine the occurrence of *S.* Dublin among cattle in Nigeria.

The study further reveals that *S.* Typhi still constitutes a significant public health importance in Ibadan, Nigeria; therefore, appropriate control measures, such as vaccination and food safety measurements, need to be put in place. Overall, the study supports the conclusions of other publications which report that invasive salmonellosis caused by NTS is of greater public health importance than the *S.* Typhi.

Rarely isolated *Salmonella* serovars such as *S.* Apapa, *S.* Mouschaui, *S.* Jukestown, and *S.* Oritamerin were observed causing invasive salmonellosis in humans. Severe human salmonellosis caused by those serovars has been previously linked to exposure to reptiles and amphibians. Several studies have shown that reptiles are reservoirs for *Salmonella* that are able to affect humans and recently constitute a significant public health problem in many countries due to captive pet reptiles; however, the majority of infections in Nigeria are not caused by captive pet reptiles but more likely by household wall lizards. Lizards are found around houses in Nigeria, most commonly in the rural areas and sometimes inside houses, thus constituting a potential health risk. Additionally, snake meat is a special delicacy in some parts of Nigeria, while snakes are revered in other places. It is interesting to note that all the rarely isolated serovars obtained, with the exception of *S.* Apapa, were isolated from samples at more rural sampling sites, thus suggesting that the sources of infections were associated with reptiles or the environment. Wall geckos have been previously described as a reservoir of *Salmonella* in Nigeria, constituting a public health problem that is difficult to control due to the nature of wild animals. Consequently, the potential risk of invasive salmonellosis due to the exposure to reptiles and amphibians should be disseminated to the people at risk in rural areas.

The majority of the human isolates were resistant to ampicillin, sulfamethoxazole, and trimethoprim; high resistance was also observed to chloramphenicol. Multidrug resistance to these old and commonly used antimicrobials is a public health problem and could facilitate usage of fluoroquinolones and third-generation cephalosporins for empiric treatment. *S.* Typhimurium was the most resistant of the serovars obtained from human blood samples and exhibited resistance to ampicillin, chloramphenicol, streptomycin, sulfamethoxazole, tetracycline, and trimethoprim. In many ways, the *S.* Typhimurium isolates in this study match the resistance profile of the multidrug-resistant invasive *S.* Typhimurium ST313 clone found in sub-Saharan Africa. This hypothesis needs to be further investigated by MLST typing before it is concluded. It is interesting that all the isolates of *S.* Typhi were susceptible to all the tested antibiotics, suggesting the efficacy of old antimicrobials for treatment of typhoid fever.

The overall prevalence rate (11%) of *Salmonella* spp. from chicken fecal samples is high and with prevalence rates of 26% and 15% in farms J and K, respectively. Similarly, high prevalence rates have also been reported from Maiduguri, northern Nigeria, and a high rate (37%) of *Salmonella* contamination of broiler farms has been reported from Algeria. Chicken is also a predominant source of *Salmonella* in Europe and has warranted initiation of targeted control measures, thus suggesting chickens as important reservoirs of *Salmonella* in Nigeria. The cause of the high prevalence rate in farms J and K is unknown due to limited data.

In this study, the predominant serovar in chicken was *S.* Virchow, which is one of the most common serovars in chicken. In Europe, *S.* Virchow was listed as the seventh most common serovar isolated from broiler meat in 2007. *S.* Virchow was observed in all farms expect G, with a high prevalence of 24% and 13% in farms J and K. Interestingly, *S.* Enteritidis was not isolated from chicken in this study. In contrast, *S.* Enteritidis is the most predominant serovar in *Gallus Gallus* flocks in the European Union, with a prevalence of 38%. This difference may be explained by differences in export patterns, production systems, climate, or feed.

Recently, *S.* Hiduddify, a rare serovar, was found to be the predominant serovar in chickens in Maiduguri, Northern Nigeria, but was also not isolated in the fecal samples of chickens in this study. Northern Nigeria is generally warmer than the region of this study; moreover, the chickens studied by Raufu et al. were largely free range in contrast to the intensively reared chickens we studied, which this may suggest the presence of *S.* Hiduddify in the environment where chickens likely pick up the serovar from foodstuffs as they scavenge for foods in the environment. This hypothesis needs to be confirmed in free-range chickens in the south or by investigation of the environment in Maiduguri, Nigeria. We also isolated *S.* Onireke, a rare serovar in chickens from Ibadan, Nigeria. We speculate that those rare isolates found in chicken also originate from reptiles or the environment like the rare serovars obtained from invasive *Salmonella* in humans, but further research is needed to explore this hypothesis.

Although none of the serovars obtained from chicken fecal samples was isolated from the analysed human blood samples, those serovars still constitute potential serious health risks for humans in Ibadan, Nigeria. In Europe, *S.* Virchow and *S.* Derby are among the ten most frequent serovars causing human salmonellosis and may be found in human stool samples from Ibadan, Nigeria, if investigated. A plausible explanation for why we could not link any of the serovars obtained from chickens to humans may be due to the analysis of human blood and chicken fecal samples in different years, suggesting time-based shifts in the prevalence of serovars. Analysis of blood samples from only humans could also be responsible because all serovars of NTS are not equally invasive.

The *Salmonella* isolates from chickens were more commonly resistant to tetracycline, sulfamethoxazole, nalidixic acid, and trimethoprim; this resistance may be attributed to indiscriminate use of antibiotics at recommended doses or at subtherapeutic doses as feed additives to promote growth, and as chemotherapeutic agents to control epizootics on the farms. The high level of resistance to nalidixic acid coupled with the high level of reduced susceptibility to ciprofloxacin is worrisome because fluoroquinolones are strategic in the treatment of invasive salmonellosis. The most resistant serovar from chicken was *S.* Virchow; this serovar is usually multiple resistant to antimicrobials. These observations call for regulation of antibiotic usage in Nigeria to stem the spread of resistance to antimicrobials. The finding of *qnrS1* in *S.* Derby is another public concern because *qnr* genes mediate resistance to fluoroquinolones, which can compromise treatment with fluoroquinolones; furthermore, *qnr* location on mobile genetic elements coupled with indiscriminate use of antibiotics in Nigeria will facilitate selection and potential spread to other serovars.

## Conclusion

This study reports the relative importance of *S.* Enteritidis, *S.* Typhimurium, and *S.* Dublin associated with human bacteraemia in Ibadan, Nigeria, and that *S.* Typhi still constitutes a significant public health risk. This is the first time that *S.* Dublin has been reported as an important cause of invasive salmonellosis in sub-Saharan African countries. We also observed rarely isolated *Salmonella* serovars previously associated with reptiles causing invasive salmonellosis in humans. As observed elsewhere in Nigeria, we can confirm the high prevalence rates of *Salmonella* spp. from chicken fecal samples, with *S.* Virchow as the predominant serovar. None of the serovars obtained from chicken fecal samples was isolated from the analyzed human blood samples; those serovars still constitute potential serious health risks for humans in Nigeria and should be controlled by targeted interventions. A high level of reduced susceptibility to ciprofloxacin and resistance to nalidixic acid was observed in all serovars isolated from chickens. Additionally, we detected the plasmid mediated quinolone resistance gene, *qnrS1*, in *S.* Derby.

## Acknowledgements

We are grateful to Mrs. Christina Aaby Svendsen and Miss Lisbeth Andersen (National Food Institute), and Mr. Adewale Bello and Miss Bimbo Adesuyi (Department of Microbiology, University of Ibadan) for their outstanding technical assistance. Additionally, we want to thank Dr. Antonio Vieira from the National Food Institute, Copenhagen, Denmark, for supervising the statistical analysis. This work was supported by the World Health Organization Global Foodborne Infections Network (WHO GFN).

## References

1. Grimont PAD, Weill FX (2007) Antigenic formulae of the Salmonella serovars, 9th edition. Paris: WHO Collaborating Center for Reference and Research on Salmonella at Institut Pasteur in France 150 p.
2. Uzzau S, Brown DJ, Wallis T, Rubino S, Leori G, Bernard S, Casadesus J, Platt DJ, Olsen JE (2000) Host adapted serotypes of Salmonella enterica. Epidemiol Infect 125: 229-255.
3. Mead PS, Slutsker L, Dietz V, McCaig LF, Bresee JS, Shapiro C, Griffin PM, Tauxe RV (1999) Food-related illness and death in the United States. Emerg Infect Dis 5: 607-625.
4. Adak GK, Long SM, O'Brien SJ (2002) Trends in indigenous foodborne disease and deaths, England and Wales: 1992 to 2000. Gut 51: 832-841.
5. Hald T, Lo Fo Wong DM, Aarestrup FM (2007) The attribution of human infections with antimicrobial resistant Salmonella bacteria in Denmark to sources of animal origin. Foodborne Pathog Dis 4: 313-326.
6. Braden CR (2006) Salmonella enterica serotype Enteritidis and eggs: a national epidemic in the United States. Clin Infect Dis 43: 512-517.
7. Hohmann EL (2001) Nontyphoidal salmonellosis. Clin Infect Dis 32: 263-269.
8. Jones TF, Ingram LA, Cieslak PR, Vugia DJ, Tobin-D'Angelo M, Hurd S, Medus C, Cronquist A, Angulo FJ (2008) Salmonellosis outcomes differ substantially by serotype. J Infect Dis 198: 109-114.
9. Morpeth SC, Ramadhani HO, Crump JA (2009) Invasive non-Typhi Salmonella disease in Africa. Clin Infect Dis 49: 606-611.
10. Hendriksen RS, Bangtrakulnonth A, Pulsrikarn C, Pornruangwong S, Noppornphan G, Emborg HD, Aarestrup FM (2009) Risk factors and epidemiology of the ten most common Salmonella serovars from patients in Thailand: 2002-2007. Foodborne Pathog Dis 6: 1009-1019.
11. Gordon MA (2008) Salmonella infections in immunocompromised adults. J Infect 56: 413-422.
12. Vandenberg O, Nyarukweba DZ, Ndeba PM, Hendriksen RS, Barzilay EJ, Schirvel C, Bisimwa BB, Collard JM, Aidara KA, Aarestrup FM (2010) Microbiologic and Clinical Features of Salmonella Species Isolated From Bacteremic Children in Eastern Democratic Republic of Congo. Pediatr Infect Dis J. 29: 504-10.
13. Kingsley RA, Msefula CL, Thomson NR, Kariuki S, Holt KE, Gordon MA, Harris D, Clarke L, Whitehead S, Sangal V, Marsh K, Achtman M, Molyneux ME, Cormican M, Parkhill J, Maclennan CA, Heyderman RS, Dougan G (2009) Epidemic multiple drug resistant Salmonella Typhimurium causing invasive disease in sub-Saharan Africa have a distinct genotype. Genome Res 19: 2279-2287.
14. Sire JM, Garin B, Macondo EA (2008) Low-level resistance to ciprofloxacin in non-Typhi Salmonella enterica isolated from human gastroenteritis in Dakar, Senegal (2004-2006). Int J Antimicrob Agents 31: 581-582.
15. Gordon MA, Graham SM, Walsh AL, Wilson L, Phiri A, Molyneux E, Zijlstra EE, Heyderman RS, Hart CA, Molyneux ME (2008) Epidemics of invasive Salmonella enterica serovar Enteritidis and S. enterica serovar Typhimurium infection associated with multidrug resistance among adults and children in Malawi. Clin Infect Dis 46: 963-969.
16. Usha G, Chunderika M, Prashini M, Willem SA, Yusuf ES (2008) Characterization of extended-spectrum beta-lactamases in Salmonella spp. at a tertiary hospital in Durban, South Africa. Diagn Microbiol Infect Dis 62: 86-91.
17. Alausa KO, Montefiore D, Sogbetun AO, Ashiru JO, Onile BA, Sobayo E (1977) Septicaemia in the tropics. A prospective epidemiological study of 146 patients with a high case fatality rate. Scand J Infect Dis 9: 181-185.
18. Hendriksen RS, Mikoleit M, Carlson VP, Karlsmose S, Vieira AR, Jensen AB, Seyfarth AM, DeLong SM, Weill FX, Wong DM, Angulo FJ, Wegener HC, Aarestrup FM (2009) WHO Global Salm-Surv external quality assurance system for serotyping of Salmonella isolates from 2000 to 2007. J Clin Microbiol 47: 2729-2736.
19. Ameh JA, Kwari HD, Abubakar YM (2001) Prevalence and antibiotic susceptibility of Salmonella Enteritidis in chicken meat sold in Maiduguri. Nigeria Res J Sci 33-37.
20. Raufu I, Hendriksen RS, Ameh JA, Aarestrup FM (2009) Occurrence and characterization of Salmonella Hiduddify from chickens and poultry meat in Nigeria. Foodborne Pathog Dis 6: 425-430.
21. McDermott PF (2006) Antimicrobial Resistance in notyphoidal Salmonellae. In: Antimicrobial Resistance in Bacteria of Animal Origin. Aaretrup FM Editor, Washington DC: ASM Press. 293-314.
22. Collignon P, Powers JH, Chiller TM, Aidara-Kane A, Aarestrup FM (2009) World Health Organization ranking of antimicrobials according to their importance in human medicine: A critical step for developing risk management strategies for the use of antimicrobials in food production animals. Clin Infect Dis 49: 132-141.
23. Hendriksen RS, Mikoleit M, Kornschober C, Rickert RL, Duyne SV, Kjelso C, Hasman H, Cormican M, Mevius D, Threlfall J, Angulo FJ, Aarestrup FM (2009) Emergence of Multidrug-Resistant Salmonella Concord Infections in Europe and the United States in Children Adopted From Ethiopia, 2003-2007. Pediatr Infect Dis J 28: 814-818.
24. Hoa NT, Diep TS, Wain J, Parry CM, Hien TT, Smith MD, Walsh AL, White NJ (1998) Community-acquired septicaemia in southern Viet Nam: the importance of multidrug-resistant Salmonella Typhi. Trans R Soc Trop Med Hyg 92: 503-508.
25. Cowan ST, Steel KJ (1974) Manual for the Identification of Medical Bacteria. Cambridge. University Press: 28-106.
26. Cruickshank R, Duguid JP, Marmion BP, Swainr HA (1975) The Practice of Medical Microbiology. Edinburgh. Churchill Livingstone: 171-191.
27. Hendriksen RS, Bangtrakulnonth A, Pulsrikarn C, Pornreongwong S, Hasman H, Song SW, Aarestrup FM (2008) Antimicrobial resistance and molecular epidemiology of Salmonella Rissen from animals, food products, and patients in Thailand and Denmark. Foodborne Pathog Dis 5: 605-619.
28. Cavaco LM, Hasman H, Xia S, Aarestrup FM (2009) qnrD, a novel gene conferring transferable quinolone resistance in Salmonella enterica serovar Kentucky and Bovismorbificans strains of human origin. Antimicrob Agents Chemother 53: 603-608.
29. Wang M, Guo Q, Xu X, Wang X, Ye X, Wu S, Hooper DC, Wang M (2009) New plasmid-mediated quinolone resistance gene, qnrC, found in a clinical isolate of Proteus mirabilis. Antimicrob Agents Chemother 53: 1892-1897.
30. Ghiorghis B, Geyid A, Haile M (1992) Bacteraemia in febrile out-patient children. East Afr Med J 69: 74-77.
31. Brent AJ, Oundo JO, Mwangi I, Ochola L, Lowe B, Berkley JA (2006) Salmonella bacteremia in Kenyan children. Pediatr Infect Dis J 25: 230-236.
32. Sigauque B, Roca A, Mandomando I, Morais L, Quinto L, Sacarlal J, Macete E, Nhamposa T, Machevo S, Aide P, Bassat Q, Bardaji A, Nhalungo D, Soriano-Gabarro M, Flannery B, Menendez C, Levine MM, Alonso PL (2009) Community-acquired bacteremia among children admitted to a rural hospital in Mozambique. Pediatr Infect Dis J 28: 108-113.
33. Bahwere P, Levy J, Hennart P, Donnen P, Lomoyo W, Dramaix-Wilmet M, Butzler JP, De MP (2001) Community-acquired bacteremia among hospitalized children in rural central Africa. Int J Infect Dis 5: 180-188.
34. Walsh AL, Phiri AJ, Graham SM, Molyneux EM, Molyneux ME (2000) Bacteremia in febrile Malawian children: clinical and microbiologic features. Pediatr Infect Dis J 19: 312-318.
35. Galanis E, Lo Fo Wong DM, Patrick ME, Binsztein N, Cieslik A, Chalermchikit T, Aidara-Kane A, Ellis A, Angulo FJ, Wegener HC (2006) Web-based surveillance and global Salmonella distribution, 2000-2002. Emerg Infect Dis 12: 381-388.
36. Brent AJ, Oundo JO, Mwangi I, Ochola L, Lowe B, Berkley JA (2006) Salmonella bacteremia in Kenyan children. Pediatr Infect Dis J 25: 230-236.
37. Graham SM, Molyneux EM, Walsh AL, Cheesbrough JS, Molyneux ME, Hart CA (2000) Nontyphoidal Salmonella infections of children in tropical Africa. Pediatr Infect Dis J 19: 1189-1196.
38. Kariuki S, Revathi G, Kariuki N, Kiiru J, Mwituria J, Muyodi J, Githinji JW, Kagendo D, Munyalo A, Hart CA (2006) Invasive multidrug-resistant non-typhoidal Salmonella infections in Africa: zoonotic or anthroponotic transmission? J Med Microbiol 55: 585-591.
39. Ojeniyi AA, Montefiore DG (1986) Food poisoning outbreak in Ibadan, Nigeria, due to a new phage type of Salmonella Typhimurium. Scand J Infect Dis 18: 551-552.
40. Oboegbulam SI, Oguike JU, Gugnani HC (1995) Microbiological studies on cases diagnosed as typhoid/enteric fever in south-east Nigeria. J Commun Dis 27: 97-100.
41. Orji MU, Onuigbo HC, Mbata TI (2005) Isolation of Salmonella from poultry droppings and other environmental sources in Awka, Nigeria. Int J Infect Dis 9: 86-89.
42. Ricosse JH, Saliou P, Albert JP, Menard M, Le MG, Schollhammer G, Trotobas J, Morvan D, Meyran M (1979) Current aspects of human salmonellosis in Upper Volta. Study carried out at Centre Muraz (Bobo-Dioulasso) from 1966 to 1977: isolation of 1,013 Salmonella strains. Med Trop Mars 39: 381-394.
43. Maguire H, Cowden J, Jacob M, Rowe B, Roberts D, Bruce J, Mitchell E (1992) An outbreak of Salmonella Dublin infection in England and Wales associated with a soft unpasteurized cows' milk cheese. Epidemiol Infect 109: 389-396.
44. Allerberger F, Liesegang A, Grif K, Prager R, Danzl J, Hock F, Ottl J, Dierich MP, Berghold C, Neckstaller I, Tschape H, Fisher I (2002) Occurrence of Salmonella enterica serovar Dublin in Austria. Euro Surveill 7: 65-70.
45. Vaillant V, Haeghebaert S, Desenclos JC, Bouvet P, Grimont F, Grimont PA, Burnens AP (1996) Outbreak of Salmonella Dublin infection in France, November December 1995. Euro Surveill 1: 9-10.
46. Anonymous (2009) The Community summary report on trends and sources of zoonotic agents in the European Union in 2007. The EFSA Journal 2009, Parma, Italy.
47. Collard P, Sen R (1956) Isolation of Salmonellae from cattle in Ibadan. West Afr Med J 5: 118-120.
48. Collard P, Sen R (1962) Salmonellosis at Ibadan, Nigeria. (5. Strains isolated during 1956-1960). West Afr Med J 11: 106-112.
49. Cooke FJ, De PE, Maguire C, Guha S, Pickard DJ, Farrington M, Threlfall EJ (2009) First report of human infection with Salmonella enterica serovar Apapa resulting from exposure to a pet lizard. J Clin Microbiol 47: 2672-2674.
50. Bertrand S, Rimhanen-Finne R, Weill FX, Rabsch W, Thornton L, Perevoscikovs J, van PW, Heck M (2008) Salmonella infections associated with reptiles: the current situation in Europe. Euro Surveill 13: 1-6.
51. Milstone AM, Agwu AG, Angulo FJ (2006) Alerting pregnant women to the risk of reptile-associated salmonellosis. Obstet Gynecol 107: 516-518.
52. Carlquist PR, Coates MS (1947) Salmonella Monschaui-A New Type. J Bacteriol 53: 249.
53. Collard P, Sen R (1960) Serotypes of Salmonella at Ibadan, Nigeria, with special note of the new serotypes isolated in Nigeria. J Infect Dis 106: 270-275.
54. Sharma VK, Rohde R, Garg DN, Kumar A (1977) Toads as natural reservoir of Salmonella. Zentralbl Bakteriol Orig A 239: 172-177.
55. Pedersen K, Lassen-Nielsen AM, Nordentoft S, Hammer AS (2009) Serovars of Salmonella from captive reptiles. Zoonoses Public Health 56: 238-242.
56. Mermin J, Hutwagner L, Vugia D, Shallow S, Daily P, Bender J, Koehler J, Marcus R, Angulo FJ (2004) Reptiles, amphibians, and human Salmonella infection: a population-based, case-control study. Clin Infect Dis 38: 253-261.
57. Bradley T, Angulo F, Mitchell M (2001) Public health education on Salmonella spp and reptiles. J Am Vet Med Assoc 219: 754-755.
58. Oboegbulem SI, Iseghohimhen AU (1985) Wall Geckos (Geckonidae) as reservoirs of Salmonellae in Nigeria: problems for epidemiology and public health. Int J Zoonoses 12: 228-232.
59. Mascher F, Reinthaler FF, Sixl W (1988) Serotyping of Salmonella strains in south-west Nigeria: epidemiological aspects. Geogr Med 18: 171-174.
60. Parry CM, Threlfall EJ (2008) Antimicrobial resistance in typhoidal and nontyphoidal salmonellae. Curr Opin Infect Dis 21: 531-538.
61. Elgroud R, Zerdoumi F, Benazzouz M, Bouzitouna-Bentchouala C, Granier SA, Fremy S, Brisabois A, Dufour B, Millemann Y (2009) Characteristics of Salmonella contamination of broilers and slaughterhouses in the region of Constantine (Algeria). Zoonoses Public Health 56: 84-93.
62. Anonymous (2009) Annual report on zoonoses in Denmark 2007. National Food Institute, Technical University of Denmark, Copenhagen, Denmark.
63. Bertrand S, Weill FX, Cloeckaert A, Vrints M, Mairiaux E, Praud K, Dierick K, Wildemauve C, Godard C, Butaye P, Imberechts H, Grimont PA, Collard JM (2006) Clonal emergence of extended-spectrum beta-lactamase (CTX-M-2)-producing Salmonella enterica serovar Virchow isolates with reduced susceptibilities to ciprofloxacin among poultry and humans in Belgium and France (2000 to 2003). J Clin Microbiol 44: 2897-2903.
64. Weinberger M, Solnik-Isaac H, Shachar D, Reisfeld A, Valinsky L, Andorn N, Agmon V, Yishai R, Bassal R, Fraser A, Yaron S, Cohen D (2006) Salmonella enterica serotype Virchow: epidemiology, resistance patterns and molecular characterisation of an invasive Salmonella serotype in Israel. Clin Microbiol Infect 12: 999-1005.
65. Weill FX, Lailler R, Praud K, Kerouanton A, Fabre L, Brisabois A, Grimont PA, Cloeckaert A (2004) Emergence of extended-spectrum-beta-lactamase (CTX-M-9)-producing multiresistant strains of Salmonella enterica serotype Virchow in poultry and humans in France. J Clin Microbiol 42: 5767-5773.
66. Ammari S, Laglaoui A, En-Nanei L, Bertrand S, Wildemauwe C, Barrijal S, Abid M (2009) Isolation, drug resistance and molecular characterization of Salmonella isolates in northern Morocco. J Infect Dev Ctries 3: 41-49.
67. Tibaijuka B, Molla B, Hildebrandt G, Kleer J (2003) Occurrence of Salmonellae in retail raw chicken products in Ethiopia. Berl Munch Tierarztl Wochenschr 116: 55-58.
68. Adetosoye AI (1980) Infective drug resistance among Escherichia coli isolated from clinically healthy domestic livestock. Vet Microbiol 333-342.
69. Lee HY, Su LH, Tsai MH, Kim SW, Chang HH, Jung SI, Park KH, Perera J, Carlos C, Tan BH, Kumarasinghe G, So T, Chongthaleong A, Hsueh PR, Liu JW, Song JH, Chiu CH (2009) High rate of reduced susceptibility to ciprofloxacin and ceftriaxone among nontyphoid Salmonella clinical isolates in Asia. Antimicrob Agents Chemother 53: 2696-2699.
70. Garcia-Fernandez A, Fortini D, Veldman K, Mevius D, Carattoli A (2009) Characterization of plasmids harbouring qnrS1, qnrB2 and qnrB19 genes in Salmonella. J Antimicrob Chemother 63: 274-281.