# Non-Typhoidal *Salmonella* in poultry meat and diarrhoeic patients: prevalence, antibiogram, virulotyping, molecular detection and sequencing of class I integrons in multidrug resistant strains

**Rasha M. Gharieb, Yasmine H. Tartor and Mariam H. E. Khedr**

**Gut Pathogens (2015) 7:34**
**DOI 10.1186/s13099-015-0081-1**

### Abstract

**Background:** The worldwide increase of food-borne infections with antibiotic resistant pathogens constitutes a major public health problem. Therefore, this study aimed to determine the prevalence, antibiogram, virulence genes profiles and integron characteristics of non-typhoidal *Salmonella* spp. isolated from poultry meat and diarrhoeic patients in Egypt. 

**Methods:** A total of 150 samples comprising (100 poultry meat and 50 diarrhoeic patients' stool) were examined for the presence of *Salmonella* spp. using culture methods followed by biochemical and serological identification of the isolates. All *Salmonella* strains were tested for their susceptibility to the antibiotics using disk diffusion method and screened for the presence of virulence genes and class I integrons using PCR. 

**Results:** The overall prevalence of *Salmonella* spp. in poultry meat samples was 10% compared to 4% in diarrhoeic patients. All the isolates were serologically identified into *Salmonella* Typhimurium (seven isolates), *S.* Derby, *S.* Kiel, *S.* Rubislaw (one isolate, each) and untypable strains (two isolates). Antibiotic susceptibility testing showed a higher resistance of the total isolates to erythromycin and tetracycline (100%, each), followed by amoxicillin-clavulanic acid (91.7%), trimethoprim-sulfamethoxazole (83.3 %), streptomycin, nalidixic acid, ampicillin-sulbactam (75%, each), gentamycin, ampicillin (66.7%, each), chloramphenicol (58.3%), ciprofloxacin (25%) and ceftriaxone (16.7%). Virulence genes profiles revealed the presence of *sopB* gene in five *Salmonella* strains isolated from poultry meat () and humans (). Moreover, *pefA* was only identified in three isolates from poultry meat. On the other hand, *S.* Kiel and *S.* Typhimurium (one isolate, each) were harboring *hilA* and *stn* genes, respectively. Class 1 integrons were detected in all *Salmonella* spp. with variable amplicon sizes ranged from 650-3000 bp. Sequencing of these amplicons revealed the presence of gene cassettes harboring *aac(3)-Id*, *aadA2*, *aadA4*, *aadA7*, *sat*, *dfrA15*, *lnuF* and *estX* resistance genes. Nucleotide sequence analysis showed point mutations in the *aac(3)-Id* of *S.* Derby, *aadA2*, *estX-sat* genes of *S.* Typhimurium. Meanwhile, frame shift mutation was observed in *aadA7* genes of *S.* Typhimurium. 

**Conclusions:** Increasing rate of antimicrobial resistance and class 1 integrons among multidrug resistant *Salmonella* spp. has prompted calls for the reduction of antimicrobial use in livestock to prevent future emergence of resistance. 

**Keywords:** *Salmonella* spp., Zoonoses, multidrug resistance, Virulence, Class I integrons 

---

### Background

Non-typhoidal *Salmonella* (NTS) is one of the major zoonotic food-borne pathogens representing an important public health problem worldwide. It can cause a variety of clinical manifestations ranging from mild gastroenteritis to bacteraemia and extra intestinal localized infections involving many organs. The global burden of NTS gastroenteritis has been estimated to be 93.8 million cases of gastroenteritis each year with 155,000 deaths annually [1]. Food of animal origin especially poultry meat is considered to be one of the major vehicles of *Salmonella* infections in humans and has been implicated in outbreaks of human salmonellosis [2]. 

Antimicrobial resistance and virulence of *Salmonella* strains play a vital role in systemic infections. *Salmonella* pathogenicity is dictated by an array of factors encoded by virulence genes that assist the organism to express its virulence in the host cells and ultimately manifest in the typical symptoms of salmonellosis. Some genes are known to be involved in adhesion and invasion including; plasmid encoded fimbriae (*pefA*) and hyper invasive locus (*hilA*) [3, 4]. *Salmonella* outer proteins (sop A-E) encoded by *sop* gene and (*stn*) codes for enterotoxin productions are associated with the actual manifestation of pathogenic processes [5]. In addition to virulence factors, the emergence of antimicrobial resistance among NTS has become a public health threat. Since food of animal origin is a major source of *Salmonella* spp., it has been suggested that abundant use of antimicrobials in food animals' production may contribute to the presence of antimicrobial resistance in these species and subsequently transferred to humans through food chain [6]. 

Factors contributed to the resistance and virulence of *Salmonella* may be located on chromosomes, plasmids, transposon and integrons. Integrons are DNA elements that can transfer antibiotic resistance genes among bacteria. Class I integrons are the most common type of integrons recognized among the multidrug resistant (MDR) *Salmonella* and have conserved regions (5'-CS and 3'-CS) which often contain gene cassettes [7]. These cassettes contain genes that confer resistance to antimicrobial agents including aminoglycosides, -lactams, chloramphenicol and trimethoprim as well as genes that confer resistance to antiseptics and disinfectants [8]. The present study was undertaken to (i) trace the prevalence and antimicrobial susceptibility patterns of NTS *enterica* serotypes isolated from poultry meat and diarrhoeic patients (ii) ascertain the presence of virulence genes and class I integrons using PCR (iii) sequencing of the amplified DNA fragments of class I integrons in order to identify the resistance genes located in integron gene cassettes. 

### Results and discussion

**Prevalence of *Salmonella* serovars in poultry meat and diarrhoeic patients**
In the present study, the overall prevalence of *Salmonella* spp. in the total examined poultry meat samples was 10% (Table 1). This result was compatible with previous studies in Nepal [9]. Moreover, our results regarding the prevalence of *Salmonella* spp. in broiler chicken meat (14%) substantiate the findings of others in Canada [10]. Serotyping of *Salmonella* spp. isolated from poultry meat in this study revealed the predominance of *S.* Typhimurium in chicken meat and this was in concordance with previous studies from India [11]. Additionally, other serovars such as *S.* Rubislaw, *S.* Kiel (chicken meat) and *S.* Derby (duck meat) were identified. 

It was obvious from Table 1 that 4% (2 out of 50) of stool samples from diarrhoeic patients were positive for *Salmonella* spp. and *S.* Typhimurium was the only serotype identified. Nearly similar prevalence in diarrhoeic patients was reported in India [12]. The isolation of *S.* Derby from poultry meat and humans has been previously recorded in Netherlands [13]. Moreover, *S.* Rubislaw was isolated from chicken meat in Senegal and diarrhoeic children in Gambia [14] and this confirms the zoonotic importance of the former serotypes. 

**Table 1 Frequency distribution of *Salmonella* serovars in poultry meat and diarrhoeic patients** 

<table>
<thead>
<tr>
<th rowspan="3">Source



Serotype</th>
<th colspan="3">Poultry meat</th>
<th rowspan="3">Diarrhoeic patients (n=50)



No. (%)</th>
<th rowspan="3">Total (150)



No. (%)</th>
</tr>
<tr>
<th>Chicken (n=50)</th>
<th>Duck (n=50)</th>
<th>Total (n=100)</th>
</tr>
<tr>
<th>No. (%)</th>
<th>No. (%)</th>
<th>No. (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td><em>S.</em> Typhimurium (O1,4,[5],12 H<sub>i</sub> iH: 1,2)</td>
<td>5 (10)</td>
<td>0 (0)</td>
<td>5 (5)</td>
<td>2 (4)</td>
<td>7 (4.7)</td>
</tr>
<tr>
<td><em>S.</em> Rubislaw (O11 H<sub>r</sub> H<sub>2</sub>: e,n,x)</td>
<td>1 (2)</td>
<td>0 (0)</td>
<td>1 (1)</td>
<td>0 (0)</td>
<td>1 (0.7)</td>
</tr>
<tr>
<td><em>S.</em> Kiel (O1,2,12 H, g, pH<sub>2</sub>)</td>
<td>1 (2)</td>
<td>0 (0)</td>
<td>1 (1)</td>
<td>0 (0)</td>
<td>1 (0.7)</td>
</tr>
<tr>
<td><em>S.</em> Derby (O1,4, [5],12 H,: fg H<sub>2</sub>: [1,2])</td>
<td>0 (0)</td>
<td>1 (2)</td>
<td>1 (1)</td>
<td>0 (0)</td>
<td>1 (0.7)</td>
</tr>
<tr>
<td>Untypable</td>
<td>0 (0)</td>
<td>2 (4)</td>
<td>2 (2)</td>
<td>0 (0)</td>
<td>2 (1.3)</td>
</tr>
<tr>
<td>Total</td>
<td>7 (14)</td>
<td>3 (6)</td>
<td>10 (10)</td>
<td>2 (4)</td>
<td>12 (8)</td>
</tr>
</tbody>
</table>

**Antibiotic susceptibility testing**
All *Salmonella* isolates were tested for their susceptibility towards antimicrobial drugs (Table 2). The results highlighted the higher resistance of the isolates to erythromycin and tetracycline (100%, each), followed by amoxicillin-clavulanic acid (91.7%), trimethoprim-sulfamethoxazole (83.3%), streptomycin, nalidixic acid, ampicillin-sulbactam (75%, each), gentamycin, ampicillin (66.7 %, each), chloramphenicol (58.3%), ciprofloxacin (25%) and ceftriaxone (16.7%). This isn't surprising because these antibiotics are cheap, easily affordable and commonly used in humans and poultry without prescription. 

In poultry, these drugs are used either for therapeutic purposes or as growth promoters added to the feed leading to the development of resistance in the enteric bacterial flora of poultry. Subsequently the pathogenic bacteria such as *Salmonella* may acquire resistance from this enteric flora and transfer this resistance to human's strains through food chain leading to the emergence of MDR *Salmonella* strains that constitute a public health risk and potentially affect the efficacy of drug treatment in humans. 

It was obvious from Table 2 that 100% of *Salmonella* spp. isolated from poultry meat were resistant to each of erythromycin and tetracycline, 9 (90%) were resistant to amoxicillin-clavulanic acid, 8 (80%) showing resistance to trimethoprim-sulfamethoxazole, 7 (70%) exhibiting resistance to each of streptomycin, ampicillin-sulbactam, nalidixic acid and gentamycin. Moreover, the lower resistance rates to chloramphenicol (50%), ciprofloxacin (30%) and ceftriaxone (10%) were observed. Nearly similar resistance rate to ampicillin, streptomycin, nalidixic acid, tetracycline and trimethoprim-sulfamethoxazole was recorded in *Salmonella* spp. isolated from chicken meat in Pakistan [15]. On the contrary, the lower resistance rate to ceftriaxone and ciprofloxacin corroborates with others in India [11]. 

Table 2 also showed that all *Salmonella* isolates of human origin were resistant to ampicillin, erythromycin, streptomycin, tetracycline, ampicillin-sulbactam, trimethoprim sulfamethoxazole, chloramphenicol, nalidixic acid and amoxicillin-clavulanic acid. Conversely, the resistance to ceftriaxone and gentamycin was found in 50% of the isolates. On the other hand, none of the isolates was resistant to ciprofloxacin. Consistent with our results, a higher resistance rate (100%) to ampicillin, chloramphenicol, streptomycin, trimethoprim-sulfamethoxazole and tetracyclines was recorded in *S.* Typhimurium strains isolated from diarrhoeic patients in Kenya [16]. Another worrisome situation in this study is that, 50% of *Salmonella* isolates from diarrhoeic patients were resistant to ceftriaxone and gentamycin. This is of particular concern because the extended spectrum cephalosporins such as ceftriaxone are the antibiotics of choice for treatment of invasive salmonellosis in children and the emergence of resistance toward these drugs could be attributed to inappropriate use of them in treatment of *Salmonella* infections in humans. The higher susceptibility of *S.* Typhimurium isolated from human source to ciprofloxacin (100%) in this study may be due to discriminate use of this drug because it is relatively expensive, not easily affordable to all people, not sold in private pharmacies without prescription. This result corroborates the findings of other investigators in Kenya [16]. Therefore, ciprofloxacin still to have a high potency against NTS strains in humans and continue to be the most successful drug used for treatment of septicaemic salmonellosis in adult humans. The increased resistance of *S.* Typhimurium isolated from humans to quinolones (nalidixic acid) in this study (100%) is a matter of concern and could be attributed to indiscriminate use of these antibiotics by human patients. Moreover, the presence of these drug residues in food of animal origin may results in increased resistance among human's isolates. 

Table 3 verified that all *Salmonella* spp. isolated from poultry and humans were MDR, exhibiting resistance to at least four or more antibiotics displayed by ten resistance patterns. 

**Table 2 Antibiogram of *Salmonella* serovars isolated from poultry meat and diarrhoeic patients** 

<table>
<thead>
<tr>
<th rowspan="2">Sources/serovars</th>
<th colspan="12">Antibiotic (concentration µg)</th>
</tr>
<tr>
<th>A (10)</th>
<th>E (15)</th>
<th>S (10)</th>
<th>Cx (5)</th>
<th>T (30)</th>
<th>Co (30)</th>
<th>As (20)</th>
<th>Sxt (25)</th>
<th>C (30)</th>
<th>Na (30)</th>
<th>G (10)</th>
<th>Amc (30)</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Chicken meat (7)</strong></td>
<td>4 (57.1)</td>
<td>7 (100)</td>
<td>5 (71.4)</td>
<td>2 (28.6)</td>
<td>7 (100)</td>
<td>1 (14.3)</td>
<td>5 (71.4)</td>
<td>6 (85.7)</td>
<td>4 (57.1)</td>
<td>5 (71.4)</td>
<td>6 (85.7)</td>
<td>6 (85.7)</td>
</tr>
<tr>
<td><em>S.</em> Typhimurium (5)</td>
<td>2 (20)</td>
<td>5 (100)</td>
<td>4 (80)</td>
<td>2 (40)</td>
<td>5 (100)</td>
<td>1 (20)</td>
<td>4 (80)</td>
<td>5 (100)</td>
<td>3 (60)</td>
<td>4 (80)</td>
<td>5 (100)</td>
<td>4 (80)</td>
</tr>
<tr>
<td><em>S.</em> Kiel (1)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>0</td>
<td>0</td>
<td>1 (100)</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>1 (100)</td>
</tr>
<tr>
<td><em>S.</em> Rubislaw (1)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>0</td>
<td>1 (100)</td>
<td>0</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>1 (100)</td>
<td>1 (100)</td>
</tr>
<tr>
<td><strong>Duck meat (3)</strong></td>
<td>2 (66.7)</td>
<td>3 (100)</td>
<td>2 (66.7)</td>
<td>1 (33.3)</td>
<td>3 (100)</td>
<td>0</td>
<td>2 (66.7)</td>
<td>2 (66.7)</td>
<td>1 (33.3)</td>
<td>2 (66.7)</td>
<td>1 (33.3)</td>
<td>3 (100)</td>
</tr>
<tr>
<td><em>S.</em> Derby(1)</td>
<td>0</td>
<td>1 (100)</td>
<td>0</td>
<td>0</td>
<td>1 (100)</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>1 (100)</td>
</tr>
<tr>
<td>Untypable(2)</td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>1 (50)</td>
<td>2 (100)</td>
<td>0</td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>1 (50)</td>
<td>2 (100)</td>
<td>1 (50)</td>
<td>2 (100)</td>
</tr>
<tr>
<td><strong>Total (10)</strong></td>
<td>6 (60)</td>
<td>10 (100)</td>
<td>7 (70)</td>
<td>3 (30)</td>
<td>10 (100)</td>
<td>1 (10)</td>
<td>7 (70)</td>
<td>8 (80)</td>
<td>5 (50)</td>
<td>7 (70)</td>
<td>7 (70)</td>
<td>9 (90)</td>
</tr>
<tr>
<td><strong>Human (2) <em>S.</em> Typhimurium</strong></td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>0</td>
<td>2 (100)</td>
<td>1 (50)</td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>2 (100)</td>
<td>1 (50)</td>
<td>2 (100)</td>
</tr>
<tr>
<td><strong>Total sensitive (%)</strong></td>
<td>2 (16.7)</td>
<td>0</td>
<td>2 (16.7)</td>
<td>4 (33.3)</td>
<td>0</td>
<td>4 (33.3)</td>
<td>1 (8.3)</td>
<td>2 (16.7)</td>
<td>2 (16.7)</td>
<td>0</td>
<td>2 (16.7)</td>
<td>0</td>
</tr>
<tr>
<td><strong>Total intermediate (%)</strong></td>
<td>2 (16.7)</td>
<td>0</td>
<td>1 (8.3)</td>
<td>5 (41.7)</td>
<td>0</td>
<td>6 (50)</td>
<td>2 (16.7)</td>
<td>0</td>
<td>3 (25)</td>
<td>3 (25)</td>
<td>2 (16.7)</td>
<td>1 (8.3)</td>
</tr>
<tr>
<td><strong>Total resistant (%)</strong></td>
<td>8 (66.7)</td>
<td>12 (100)</td>
<td>9 (75)</td>
<td>3 (25)</td>
<td>12 (100)</td>
<td>2 (16.7)</td>
<td>9 (75)</td>
<td>10 (83.3)</td>
<td>7 (58.3)</td>
<td>9 (75)</td>
<td>8 (66.7)</td>
<td>11 (91.7)</td>
</tr>
</tbody>
</table>


*A Ampicillin, E Erythromycin, S Streptomycin, Cx Ciprofloxacin, T Tetracycline, Co Ceftriaxone, As Ampicillin-Sulbactam, Sxt Trimethoprim-Sulfamethoxazole, C Chloramphenicol, Na Nalidixic acid, G Gentmycin, Amc Amoxicillin-Clavulanic acid* 

**Molecular detection of *Salmonella* virulence genes (virulotyping)**
Table 3 revealed the presence of *pefA* gene among three *Salmonella* strains isolated from poultry meat, belonged to *S.* Rubislaw, *S.* Typhimurium and untypable strain (one isolate, each) (Fig. 1). Moreover, *hilA* gene was only identified in one isolate of *S.* Kiel (Fig. 2). Furthermore, three *Salmonella* isolates from poultry meat comprising untypable strains (two isolates) and *S.* Typhimurium (one isolate) were harboring *sopB* gene (Fig. 3). The detection of *pefA* gene in *S.* Rubislaw in Gambia and Senegal [14] and *S.* Typhimurium in India [4] was previously reported. On the other hand, *hilA* gene was identified in *Salmonella* serotypes isolated from chickens in Iran [17]. 

It was obvious from Table 3 that all *S.* Typhimurium isolates from humans were harboring *sopB* compared to one isolate had *stn* gene (Figs. 3, 4). Consistent with our findings, *sopB* gene was previously identified in *S.* Typhimurium isolated from diarrhoeic patients and birds in India [18]. Meanwhile, *stn* gene was detected in *S.* Typhimurium isolated from humans in India [4]. 

**Table 3 Resistance pattern, virulence genes and class I integron profiles among multidrug resistant *Salmonella* serotypes in this study** 

<table>
<thead>
<tr>
<th>Strain no.</th>
<th>Serovar (source of isolate)</th>
<th>Resistance pattern</th>
<th colspan="4">Virulence genes</th>
<th>Integron amplicon size (bp)</th>
<th>Integron profile (IP)</th>
<th>Genes in the cassettes</th>
<th>Accession numbers</th>
</tr>
<tr>
<th></th>
<th></th>
<th></th>
<th>PefA</th>
<th>hilA</th>
<th>SopB</th>
<th>Stn</th>
<th></th>
<th></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td><em>S.</em> Derby (duck)</td>
<td>S T E Amc</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>937



1600</td>
<td>I</td>
<td>aac (3)-Id



aac (3)-Id, aadA7</td>
<td>KT581257



KT427378</td>
</tr>
<tr>
<td>2</td>
<td><em>S.</em> Kiel (chicken)</td>
<td>A T E Amc</td>
<td></td>
<td>+</td>
<td></td>
<td></td>
<td>937</td>
<td>II</td>
<td></td>
<td></td>
</tr>
<tr>
<td>3</td>
<td>Untypable (duck)</td>
<td>A C S Sxt T E Cx As Na G Amc</td>
<td>+</td>
<td></td>
<td>+</td>
<td></td>
<td>1500</td>
<td>III</td>
<td>aac (3)-Id, aadA7</td>
<td>KT581256</td>
</tr>
<tr>
<td>4</td>
<td><em>S.</em> Rubislaw (chicken)</td>
<td>A C S Sxt T E As Na G Amc</td>
<td>+</td>
<td></td>
<td></td>
<td></td>
<td>1500</td>
<td>IV</td>
<td>estX-sat</td>
<td>KT581255</td>
</tr>
<tr>
<td>5</td>
<td><em>S.</em> Typhimurium (chicken)</td>
<td>A C S Sxt T E As Na G Amc</td>
<td>+</td>
<td></td>
<td></td>
<td></td>
<td>1100



1500</td>
<td>V</td>
<td>sat</td>
<td>KT449570</td>
</tr>
<tr>
<td>6</td>
<td><em>S.</em> Typhimurium (chicken)</td>
<td>S Sxt T E Cx Co Na G</td>
<td></td>
<td></td>
<td>+</td>
<td></td>
<td>650



1550</td>
<td>VI</td>
<td>



aac (3)-Id, aadA7</td>
<td>



KT598359</td>
</tr>
<tr>
<td>7</td>
<td>Untypable (duck)</td>
<td>A Sxt T E As Na Amc</td>
<td></td>
<td></td>
<td>+</td>
<td></td>
<td>1600



3000</td>
<td>VII</td>
<td></td>
<td></td>
</tr>
<tr>
<td>8</td>
<td><em>S.</em> Typhimurium (human)</td>
<td>A C S Sxt T E Co As Na G Amc</td>
<td></td>
<td></td>
<td>+</td>
<td></td>
<td>1000</td>
<td>VIII</td>
<td></td>
<td></td>
</tr>
<tr>
<td>9</td>
<td><em>S.</em> Typhimurium (human)</td>
<td>A C S Sxt T E As Na Amc</td>
<td></td>
<td></td>
<td>+</td>
<td>+</td>
<td>650



750



1600</td>
<td>IX</td>
<td>aadA4





dfrA15</td>
<td>KT581253





KT449571</td>
</tr>
<tr>
<td>10</td>
<td><em>S.</em> Typhimurium (chicken)</td>
<td>S Sxt T E As G Amc</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>1900</td>
<td>X</td>
<td>aadA2, lnuF</td>
<td>KT449569</td>
</tr>
<tr>
<td>11</td>
<td><em>S.</em> Typhimurium (chicken)</td>
<td>A C S Sxt T E As Na G Amc</td>
<td></td>
<td></td>
<td>+</td>
<td></td>
<td>1900</td>
<td>X</td>
<td></td>
<td></td>
</tr>
<tr>
<td>12</td>
<td><em>S.</em> Typhimurium (chicken)</td>
<td>C Sxt T E Cx As Na G Amc</td>
<td></td>
<td></td>
<td>+</td>
<td></td>
<td>1550</td>
<td>VI</td>
<td>aac (3)-Id, aadA7</td>
<td>KT581254</td>
</tr>
</tbody>
</table>

**Molecular detection and sequencing of class 1 integrons**
A notable feature in this study is that all *Salmonella* strains isolated from poultry meat and humans were MDR and harboring class I integrons that are chromosomally located (Fig. 5). Ten different profiles of class I integrons with variable amplicon sizes ranged from 650-3000 bp were observed (Table 3). Similarily, class I integrons were detected in 97% of *S.* Typhimurium isolated from Norwegian diarrhoeic patients [19]. On the contrary, a lower detection rate (9%) was recorded in *Salmonella* spp. isolated from poultry meat [20]. 

**Sequencing**
Sequencing of the variable amplicons of class I integrons denoted by asterisks (Fig. 5) revealed the presence of gene cassettes containing aminoglycoside acetyltransferase [*aac(3)-Id* or *aacCA5*] gene which confers resistance against gentamycin, aminoglycoside adenyltransferase (*aadA2*, *aadA4*, *aadA7*) genes that confer resistance to streptomycin and spectinomycin, streptothricin acetyltransferase (*sat*) encoding resistance against streptothricin, dihydrofolate reductase type 15 (*dfrA15*) that confers resistance to trimethoprim, *lnuF* gene that codes for lincosamides resistance (lincomycin and clindamycin) and putative esterase (*estX*). Interestingly, the *aac (3)-Id* plus *aadA7* were the most predominant resistance genes identified in class I integron gene cassettes of 1600 bp (*S.* Derby, duck), 1500 bp (untypable strain, duck) and 1550 bp (*S.* Typhimurium, chicken). The same cassettes harboring *aac (3)-Id* and *aadA7* genes were previously identified in class I integron of *S.* Haifa (GenBank accession no. AY563051) [21], *S.* Newport (AY458224) [22] and *S.* Kentucky (AY463797) [23]. Despite the presence of these genes, some strains such as *S.* Derby was susceptible to gentamycin and *S.* Typhimurium strains (chicken) were sensitive to streptomycin and this could be attributed to point mutations in the *aac(3)-Id* gene of *S.* Derby [24] and frame shift mutation in *S.* Typhimurium [25, 26]. 

Nucleotide sequence analysis revealed point mutations in the *aac(3)-Id* gene of *S.* Derby leading to the amino acid changes at codon 36 (glutamine CAG$\to$proline CCG), codon 166, 172 (arginine AGA  threonine ACA), codon 187 (arginine CGA  CTA leucine) and codon 189 (cysteine TGC  arginine CGC). C-to-T transition at position 208 of the deduced polypeptide resulted in a leucine to phenylalanine substitution and A to C transversion at codon 214 resulted in asparagine to histidine substitution. Silent mutation was recorded only at nucleotide 138 and didn't result in amino acid substitution (CCA  CCG, both are proline). Moreover, sequence analysis of *aadA7* gene showed frame shift mutation due to additional cytosine at position 969 of the deposited sequence of *S.* Typhimurium from chicken. 

In the present study, only one isolate of *S.* Typhimurium from human was found to harbor class I integrons of 650, 750 and 1600 bp. Sequence analysis of 650 bp revealed the presence of gene cassette containing *aadA4* which showed 100% identity with *aadA4* previously identified in class 1 integron of *S.* Newport (CP006631) and plasmid of *S.* Indiana (ref NG_041636). In addition, the 1600 bp integron carried gene cassette containing *dfrA15* gene that showed 100% amino acid and nucleotide homology with *dfrA15* gene previously reported in *S. enterica* strain (KM823524). The *sat* gene detected in the gene cassette of class I integron (650 bp) of *S.* Typhimurium in this study was previously identified in *S.* Kedougou (DQ284538) [27] and plasmid of *S.* Choleraesuis (AY509004) [28]. 

Moreover, the 1500 bp amplicon identified in *S.* Typhimurium contained *estX-sat* gene cassette that is identical to *estX* gene previously identified in *S.* Typhimurium (EF051039), *sat* gene of *S.* Choleraesuis plasmid (ref NG_036624) and SC-B67 strain (AY509004). Point mutations were observed in *estX-sat* gene cassette in this study compared to the classical *estX* and *sat* genes (99% nucleotide identity); these genes displayed both missense and silent mutations. The nucleotide number 763 of *estX* was exchanged from T to C resulting in exchange of amino acid (phenylalanine) into leucine. Furthermore, at codon 288 (histidine TAC  tyrosine CAC) and 290 (threonine ACA  alanine GCA). Silent mutations were recorded at nucleotides 894 (GTC$\to$GTA, alanine), 903 (TAT  TAC, tyrosine) and 927 (GTC  GTA, both are valine). While in *sat* gene, silent mutations were detected at codon 418 (ACT  ACC, tyrosine), 428 (GTC  GTA, valine), 451 and 453 (GGC  GGT, GGA  GGG both are glycine). Amino acids replacement was recorded in *sat* gene at codon 459 (lysine AAA  glutamic acid GAA), 462 (proline CCG  leucine CTG) and 465 (iso-leucine ATC  threonine ACC). Nonsense mutation was predicted at nucleotide 1418 (serine TCA  stop codon TAA). Antimicrobial drug resistance can occur by point mutations in the bacterial genome [29] or through mobile genetic elements called integrons which are able to disseminate the antimicrobial resistance among the enterobacteriaceae by horizontal transfer [30]. 

In fact, this study reports for the first time the presence of *aadA2* plus *lnuF* genes within the same integron gene cassette of 1900 bp in *S. typhimurium* from chicken. The nucleotide sequence of *aadA2* and *lnuF* genes, respectively showed 99 and 100% identity with *aadA2* and *lnuF* genes previously reported in *S. enterica* serovar Stanley (EU118119). Moreover, the *aadA2* gene had 99% amino acid and nucleotide homology with *aadA2* previously identified in *S.* Typhimurium isolated from food-animals and humans in Mexico (FJ460233) [31]. On the other hand, the 1900 bp of class I integron was found to harbor *aadA2* plus *dfrA12* in *S. enterica* serovar Typhimurium isolated from food-animals, chicken and humans in Malaysia [32]. The nucleotide sequence of *aadA2* showed four base differences from other *aadA2* in GenBank reported cassettes and this caused point mutation which are responsible for an alteration in the codon 60 (glycine GGA  arginine CGA), codon 61 (isoleucine ATC  threonine ACC), codon 62 (asparagine AAC  aspartic acid GAC) and codon 105 (proline CCT  TCT serine) (Fig. 6). Similarly, point mutation in the *aadA2* gene of NTS *enterica* isolates was reported [29]. 

### Conclusions

The higher resistance rates displayed by *Salmonella* isolates in this study to the antibiotics and the detection of class 1 integron gene cassettes harboring resistance genes among MDR strains shows the potential of integrons to carry and spread resistance genes to other *S. enterica* isolates or to other bacteria. Therefore, a more prudent use of antibiotics in both humans and animals is required. 

### Methods

**Ethical approval**
This study was approved by Zagazig University, Egypt ethical board (Protocol No. 10122). Permission to collect stool samples was obtained from the administrators of Al-Ahrar General hospital. 

**Specimen collection**
The present work was conducted during the period from March, 2014 to February, 2015. A total of one hundred fresh poultry carcasses comprising broiler chicken and ducks (50, each) were randomly purchased from poultry retail outlets at Zagazig City, Egypt. In addition, fifty stool samples were collected from the patients attending the outpatients' clinic of Al-Ahrar General Hospital. The inclusion criteria were restricted to the patients who suffered from diarrhoea, fever and gave informed written consent to participate in the study. Moreover, patients received antibiotic treatments were excluded. A swab was obtained from each freshly passed stool sample, labeled, ice packed and transported to the Bacteriology Laboratory, Faculty of Veterinary Medicine, Zagazig University, Egypt within 24 h for bacteriological analysis. 

**Isolation and identification of *Salmonella* spp**
*Salmonella* isolation was carried out according to the standard methods recommended by ISO 6579 [33]. Twenty-five grams of poultry meat were excised from each sample and minced with 225 ml of buffered peptone water (BPW) enrichment broth. Also, each stool swab was inserted in sterile tube containing 9 ml BPW. All samples were incubated at  for 24 h. Aliquots (0.1 ml) of the pre-enriched culture was inoculated into 10 ml of Rappaport Vasiliadis (RV) enrichment broth and incubated at  for 24 h. A loopful of the enriched culture was streaked on Xylose-Lysine-Deoxycholate (XLD) agar plates (Oxoid, Basingstoke, UK) and incubated at  for 24 h. Suspected colonies with typical *Salmonella* morphology were confirmed biochemically by indole, citrate utilization, urease, triple sugar iron (TSI) and lysine iron agar (LIA) tests. All biochemically confirmed *Salmonella* isolates were serologically identified on the basis of somatic (O) and flagellar (H) antigens by slide agglutination using commercial antisera (SISIN, Berlin) following Kauffman-White scheme [34] in collaboration with Serology Unit, Animal Health Research Institute, Dokki, Egypt. 

**Antimicrobial susceptibility testing (antibiogram)**
Susceptibility of *Salmonella* isolates to various routine antimicrobial drugs was tested by the standard disc diffusion technique [35] using Mueller-Hinton agar and commercial antibiotic discs (Oxoid, UK). The antibiotics used were ampicillin (10 µg), erythromycin (15 µg), streptomycin (10 µg), ciprofloxacin (5 µg), tetracycline (30 µg), ceftriaxone (30 µg), ampicillin-sulbactam (20 µg), trimethoprim-sulfamethoxazole (25 µg), chloramphenicol (30 µg), nalidixic acid (30 µg), gentamicin (10 µg), amoxicillin-clavulanic acid (30 µg). The inhibition zones in mm were measured and scored as sensitive, intermediate and resistant categories in accordance with the critical breakpoints recommended by the Clinical and Laboratory Standards Institute [36]. 

**Molecular detection of *Salmonella* virulence genes using uniplex PCR**
A total of twelve *Salmonella* isolates were screened for the presence of virulence genes using PCR. These genes including; plasmid encoded fimbriae (*pefA*), hyper invasive locus (*hilA*), *Salmonella* outer protein (*sopB*) and *Salmonella* enterotoxin gene (*stn*). Briefly, DNA was extracted from the overnight bacterial culture using ABIO AB pure Genomic DNA extraction kit with modifications from the manufacturer's recommendation. DNA amplification was carried out using specific primers supplied from Metabion, Germany (Table 4). Primers were utilized in a 25 µl reaction containing 12.5 µl of Emerald Amp Max PCR Master Mix (Takara, Japan), one µl of each primer (20 pmol concentration), 4.5 µl of nuclease free water, and 6 µl of DNA template. The reaction was performed in a T3 Biometra thermal cycler. Positive and negative controls were included in each reaction. Aliquot of each amplicon along with a 100 bp molecular weight DNA ladder (Fermentas, USA) were separated by electrophoresis on 1.5% agarose gel (Applichem, Germany, GmbH) stained with  ethidium bromide (Sigma, USA) in 1 x TBE buffer on a mini slab horizontal electrophoresis unit (Bio-Rad, USA) at 100 V for 30 min. The gel was photographed by a gel documentation system (Alpha Innotech, Biometra) and the data was analyzed through computer software. 

**Table 4 Primers' sequences and amplification's conditions used for detection of *Salmonella* virulence genes** 

<table>
<thead>
<tr>
<th rowspan="2">Target gene</th>
<th rowspan="2">Primers' sequences </th>
<th rowspan="2">Amplified segment (bp)</th>
<th colspan="4">Amplification (35 cycles)</th>
<th rowspan="2">Final extension</th>
<th rowspan="2">References</th>
</tr>
<tr>
<th>Primary denaturation</th>
<th>Secondary denaturation</th>
<th>Annealing</th>
<th>Extension</th>
</tr>
</thead>
<tbody>
<tr>
<td><em>pefA</em></td>
<td>TGTTTCCGGGCTTGTGCT



CAGGGCATTTGCTGATTCTTCC</td>
<td>700</td>
<td> min</td>
<td></td>
<td></td>
<td></td>
<td> min</td>
<td>[4]</td>
</tr>
<tr>
<td><em>hilA</em></td>
<td>CGGAAGCTTATTTGCGCCATGCTGAGGTAG



GCATGGATCCCCGCCGGCGAGATTGTG</td>
<td>854</td>
<td> min</td>
<td></td>
<td></td>
<td></td>
<td> min</td>
<td>[37]</td>
</tr>
<tr>
<td><em>sopB</em></td>
<td>TCAGAAGRCGTCTAACCACTC



TACCGTCCTCATGCACACTC</td>
<td>517</td>
<td> min</td>
<td></td>
<td></td>
<td></td>
<td> min</td>
<td>[38]</td>
</tr>
<tr>
<td><em>stn</em></td>
<td>TTGTGTCGCTATCACTGGCAACC



ATTCGTAACCCGCTCTCGTCC</td>
<td>617</td>
<td> min</td>
<td></td>
<td></td>
<td></td>
<td> min</td>
<td>[4]</td>
</tr>
</tbody>
</table>

**Molecular detection and sequencing of class 1 integrons**
All MDR *Salmonella* isolates  were tested for the presence of conserved regions (5'-CS and 3'-CS) of class I integrons using specific primers with the following sequences: 5' CS-GGC ATC CAA GCA GCA AG and 3'CS-AAG CAG ACT TGA CCT GA [30]. Chromosomal DNA extraction, amplification and gel analysis were performed as mentioned before except to amplify the DNA in the thermal cycler we used a three-step profile (cycling conditions): 1 min of denaturation at  1 min of annealing at  and 5 min of extension at  for a total of 35 cycles. Five seconds were added to the extension time at each cycle. Amplified products with variable sizes were purified using QIA quick PCR Product extraction kit (Qiagen, Valencia). Each purified amplicon was sequenced in both forward and reverse directions using the amplification primers. DNA sequences were obtained by Applied Biosystems 3130 genetic analyzer (HITACHI, Japan). Nucleotide sequence analysis was compared with published GenBank DNA sequences using NCBI BLAST program ([http://www.ncbi.nlm.nih.gov/BLAST/](http://www.ncbi.nlm.nih.gov/BLAST/)) and alignment was performed usingMEGA6 program. 

**Nucleotide sequence accession numbers**
Nucleotide sequences from this study were deposited into the GenBank under accession numbers [KT427378, KT449569, KT449570, KT449571, KT581253, KT581254, KT581255, KT581256, KT581257 and KT598359].