# Occurrence of *Salmonella enterica* and *Escherichia coli* in raw chicken and beef meat in northern Egypt and dissemination of their antibiotic resistance markers

**Amira A. Moawad, Helmut Hotzel, Omnia Awad, Herbert Tomaso, Heinrich Neubauer, Hafez M. Hafez and Hosny El-Adawy**

**Journal:** Gut Pathogens (2017) 9:57
**DOI:** 10.1186/s13099-017-0206-9

---

### Abstract

**Background:** The global incidence of foodborne infections and antibiotic resistance is recently increased and considered of public health concern. Currently, scarcely information is available on foodborne infections and ESBL associated with poultry and beef meat in Egypt.

**Methods:** In total, 180 chicken and beef meat samples as well as internal organs were collected from different districts in northern Egypt. The samples were investigated for the prevalence and antibiotic resistance of *Salmonella enterica* serovars and *Escherichia coli*. All isolates were investigated for harbouring class 1 and class 2 integrons.

**Results:** Out of 180 investigated samples 15 *S. enterica* (8.3%) and 21 *E. coli* (11.7%) were isolated and identified. *S. enterica* isolates were typed as 9 *S.* Typhimurium (60.0%), 3 *S.* Paratyphi A (20.0%), 2 *S.* Enteritidis (13.3%) and 1 *S.* Kentucky (6.7%). Twenty-one *E. coli* isolates were serotyped into O1, O18, O20, O78, O103, O119, O126, O145, O146 and O158. The phenotypic antibiotic resistance profiles of *S. enterica* serovars to ampicillin, cefotaxime, cefpodoxime, trimethoprim/sulphamethoxazole and tetracycline were 86.7, 80.0, 60.0, 53.3 and 40.0%, respectively. Isolated *E. coli* were resistant to tetracycline (80.9%), ampicillin (71.4%), streptomycin, trimethoprim/sulphamethoxazole (61.9% for each) and cefotaxime (33.3%). The dissemination of genes coding for ESBL and AmpC -lactamase in *S. enterica* isolates included  (73.3%),  (73.3%) and  (13.3%). In *E. coli* isolates ,  and  were identified in 52.4, 42.9 and 14.3%, respectively. The plasmid-mediated quinolone resistance genes identified in *S. enterica* were *qnrA* (33.3%), *qnrB* (20.0%) and *qnrS* (6.7%) while *qnrA* and *qnrB* were detected in 33.3% of *E. coli* isolates. Class 1 integron was detected in 13.3% of *S. enterica* and in 14.3% of *E. coli* isolates. Class 2 integron as well as the colistin resistance gene *mcr-1* was not found in any of *E. coli* or *S. enterica* isolates.

**Conclusions:** This study showed high prevalence of *S. enterica* and *E. coli* as foodborne pathogens in raw chicken and beef meat in Nile Delta, Egypt. The emergence of antimicrobial resistance in *S. enterica* and *E. coli* isolates is of public health concern in Egypt. Molecular biological investigation elucidated the presence of genes associated with antibiotic resistance as well as class 1 integron in *S. enterica* and *E. coli*.

**Keywords:** Salmonella, Escherichia coli, Antibiotic resistance, Integron, Egypt

---

### Background

In spite of the improved technology and hygienic practices in developed countries at all stages of poultry and beef meat production, foodborne infections remain as a continuous threat to human and animal health. *Escherichia* (*E.*) *coli* and *S. enterica* serovars are the dominant members of Enterobacteriaceae causing foodborne infections. The expansion of antibiotic resistance in bacteria is also an emerging public health hazard due to the compromised efficacy in the treatment of infectious diseases.

The continuous exposure of bacterial strains by -lactams has led to dynamic and massive production and mutation of -lactamases. Extended-spectrum -lactamases (ESBLs) are derived from point mutations in the  and  -lactamase genes, which cannot hydrolyse cephamycins and are inhibited by clavulanic acid, while ampicillin class C -lactamase (AmpC) enzymes are active on cephamycins as well as oxyiminocephalosporins and monobactams. ESBLs are mostly located on mobile genetic elements (plasmids or integrons), which can facilitate their mobility from bacterial species to others by horizontal gene transfer.

In Egypt, *Salmonella* isolates from chicken meat and organs showed high resistance to different antibiotic classes where also, ESBL-producing *E. coli* were isolated from retail chicken meat and dairy products in Egypt. Quinolones considered as drugs of choice for treatment of human infections caused by Gram-negative bacteria. However, resistance to quinolones has been emerged over the time due to misuse and/or overdose of drugs in human and veterinary practice. Plasmid-mediated quinolone resistance (PMQR) represented by quinolone resistance (*qnr*) genes is widely distributed among bacteria.

Integrons are genetic elements able to capture individual antibiotic resistance genes including -lactamases encoding genes and stimulate their transcription and expression. The capture and spread of antibiotic resistance determinants by integrons stimulates the rapid evolution of multidrug resistances (MDR) among Gram-negative bacteria. Among five main integron classes, integrons 1 and 2 are most frequent in Gram-negative bacteria. More than 70 different antibiotic resistance genes have been characterized within integrons.

Colistin as antimicrobial substance was used against Gram-negative bacteria. Its usage has been limited due to its systemic toxicity but it was re-introduced as a last-line option in treatment of human infections. This study aimed to discuss the prevalence, serotyping, antibiotic resistance and characterization of resistance-associated genes in *S. enterica* serovars and *E. coli* isolated from beef and poultry meat products in the Nile Delta, Egypt.

### Methods

**Sampling, isolation and identification of bacteria**
One hundred and eighty samples were randomly collected aseptically (90 chicken and 90 beef meat and organs) from slaughterhouses and markets in four districts in northern Egypt (Dakahlia, Damietta, Kafr El-Sheikh and Gharbia governorates). Briefly, 30 samples of freshly slaughtered chickens and 30 samples of native frozen chickens from restaurants and supermarkets (each 15 from breast and from thigh muscles and skin) were taken. Thirty samples of organs (gizzard and liver) were collected from freshly slaughtered chickens. In addition, 30 beef meat samples of freshly slaughtered carcasses (neck, brisket, flank and rump muscles) were obtained from slaughterhouses. Thirty samples from locally frozen beef meat, collected from supermarkets and restaurants and 30 samples from organs (liver, spleen and heart) from freshly slaughtered animals were used in this study (Tables 1, 2).

Ten grams of each meat and organ sample were incised using sterile scalpel and forceps, transported immediately to the laboratory in ice bags, then transferred into sterile homogenizer flask containing 45 ml of nutrient broth (Oxoid, Manchester, UK). The mixture was allowed to stand for 15 min at room temperature. From each sample, 1 ml was added to 9 ml of buffered peptone water (Oxoid) for *E. coli* cultivation and 9 ml of Rappaport-Vassiliadis broth (Oxoid) for *S. enterica* isolation. After aerobic incubation at  overnight, a loopful from Rappaport-Vassiliadis broth was streaked on xylose lysine deoxycholate (XLD) agar (Oxoid) and from buffered peptone water on eosin methylene blue (EMB) agar (Oxoid). The inoculated plates were incubated aerobically at  for 18–24 h. Suspected colonies were identified biochemically using API 20E (bioMérieux, Marcy-l’Étoile, France).

All biochemically confirmed *Salmonella* isolates were serotyped on the basis of somatic (O) and flagellar (H) antigens by slide agglutination test using commercial antisera (SIFIN, Berlin, Germany) following Kauffman-White scheme. Serological identification of *E. coli* was carried out using slide agglutination method using commercial antisera (SIFIN). The serotyping was carried out at the Serology Unit, Animal Health Research Institute, Dokki, Egypt, and the Bacteriology Laboratory, Central Laboratories of Ministry of Health, Cairo, Egypt.

**Table 1: Sources, number of *S. enterica* isolates and results of their serovar identification**

<table>
<thead>
<tr>
<th rowspan="2">Serovar</th>
<th colspan="4">Poultry</th>
<th colspan="4">Beef</th>
<th rowspan="2">(Poultry + beef) Total n (%)</th>
</tr>
<tr>
<th>Fresh meat (n=30)</th>
<th>Frozen meat (n=30)</th>
<th>Fresh organs (n=30)</th>
<th>Total n (%)</th>
<th>Fresh meat (n=30)</th>
<th>Frozen meat (n=30)</th>
<th>Fresh organs (n=30)</th>
<th>Total n (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td><em>S.</em> Typhimurium</td>
<td>2</td>
<td>0</td>
<td>1</td>
<td>3 (3.3)</td>
<td>3</td>
<td>1</td>
<td>2</td>
<td>6 (6.7)</td>
<td>9 (60.0)</td>
</tr>
<tr>
<td><em>S.</em> Paratyphi A</td>
<td>0</td>
<td>1</td>
<td>1</td>
<td>2 (6.7)</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>1 (3.3)</td>
<td>3 (20.0)</td>
</tr>
<tr>
<td><em>S.</em> Enteritidis</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>1 (3.3)</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>1 (3.3)</td>
<td>2 (13.3)</td>
</tr>
<tr>
<td><em>S.</em> Kentucky</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>1 (3.3)</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0 (0)</td>
<td>1 (6.7)</td>
</tr>
<tr>
<td><strong>Total</strong></td>
<td><strong>4</strong></td>
<td><strong>1</strong></td>
<td><strong>2</strong></td>
<td><strong>7 (7.8)</strong></td>
<td><strong>4</strong></td>
<td><strong>1</strong></td>
<td><strong>3</strong></td>
<td><strong>8 (8.8)</strong></td>
<td><strong>15 (100)</strong></td>
</tr>
</tbody>
</table>

**Table 2: Sources, number of *E. coli* isolates and results of their serogroup identification**

<table>
<thead>
<tr>
<th>Serogroup</th>
<th>Fresh poultry meat (n=30)</th>
<th>Frozen poultry meat (n=30)</th>
<th>Fresh poultry organs (n=30)</th>
<th>Fresh beef meat (n=30)</th>
<th>Frozen beef meat (n=30)</th>
<th>Fresh beef organs (n=30)</th>
<th>Total number of isolates</th>
<th>% of E. coli isolates</th>
</tr>
</thead>
<tbody>
<tr>
<td>O1</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>2</td>
<td>9.5</td>
</tr>
<tr>
<td>O18</td>
<td>2</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>4</td>
<td>19</td>
</tr>
<tr>
<td>O20</td>
<td>1</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>2</td>
<td>9.5</td>
</tr>
<tr>
<td>O78</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>1</td>
<td>4</td>
<td>19</td>
</tr>
<tr>
<td>O103</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>1</td>
<td>4.8</td>
</tr>
<tr>
<td>O119</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>2</td>
<td>9.5</td>
</tr>
<tr>
<td>O126</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>2</td>
<td>9.5</td>
</tr>
<tr>
<td>O145</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>4.8</td>
</tr>
<tr>
<td>O146</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>4.8</td>
</tr>
<tr>
<td>O158</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>2</td>
<td>9.5</td>
</tr>
<tr>
<td><strong>Total</strong></td>
<td><strong>6</strong></td>
<td><strong>4</strong></td>
<td><strong>4</strong></td>
<td><strong>2</strong></td>
<td><strong>4</strong></td>
<td><strong>1</strong></td>
<td><strong>21</strong></td>
<td><strong>100</strong></td>
</tr>
</tbody>
</table>

**DNA extraction and purification**
The identified bacterial cultures were cultivated in Luria-Bertani (LB) broth (Oxoid) at  overnight. DNA was extracted and purified using DNeasy Blood and Tissue Kits (Qiagen, Manchester, UK) according to the manufacturer’s instructions.

**Molecular identification of *S. enterica* serovars and *E. coli***
The isolated strains were further identified as *S. enterica* and *E. coli* using PCR assays. The amplification targets and primers (Eurofins, Yokohama, Japan) are listed in Table 3. All phenotypically identified *S. enterica* serovars were tested for harbouring *ompC* gene. Isolated *E. coli* were confirmed by species-specific PCR targeting the 16S rRNA genes and resulting in a 585 bp amplicon. PCR assays used in this study resulted in *Salmonella*, *S.* Enteritidis and *S.* Typhimurium specific products of 204, 304 and 401 bp length, respectively.

The PCR protocol consisted of the following steps: (i) an initial denaturation step of 2 min at ; (ii) 30 cycles with 1 cycle consisting of 1 min at , 1 min at , and 2 min at ; and (iii) a final elongation step of 5 min at . Five-microliter aliquots of reaction mixture were run in electrophoreses using 1.5% agarose gels (Nippon Gene, Tokyo, Japan) and visualized under UV light after ethidium bromide staining.

**Table 3: PCR primers, their sequences and amplification targets used in this study**

<table>
<thead>
<tr>
<th>Primer</th>
<th>Sequence (5'-3')</th>
<th>Amplicon size (bp)</th>
<th>Amplification target</th>
<th>References</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="5"><strong><em>E. coli</em>, <em>Salmonella enterica</em> and <em>Salmonella</em> serovar identification</strong></td>
</tr>
<tr>
<td>ECO-F



ECO-R</td>
<td>GAC CTC GGT TTA GTT CAC AGA



CAC ACG CTG ACG CTG ACC A</td>
<td>585</td>
<td><em>E. coli</em></td>
<td>[67]</td>
</tr>
<tr>
<td>OMPCF



OMPCR</td>
<td>ATC GCT GAC TTA TGC AAT CG



CGG GTT GCG TTA TAG GTC TG</td>
<td>204</td>
<td><em>Salmonella</em> genus</td>
<td>[68]</td>
</tr>
<tr>
<td>ENTF



ENTR</td>
<td>TGT GTT TTA TCT GAT GCA AGA GG



TGA ACT ACG TTC GTT CTT CTG G</td>
<td>304</td>
<td><em>S.</em> Enteritidis</td>
<td>[68]</td>
</tr>
<tr>
<td>TYPHF



TYPHR</td>
<td>TTG TTC ACT TTT TAC CCC TGA A



CCC TGA CAG CCG TTA GAT ATT</td>
<td>401</td>
<td><em>S.</em> Typhimurium</td>
<td>[68]</td>
</tr>
<tr>
<td colspan="5"><strong>-Lactamases</strong></td>
</tr>
<tr>
<td>TEM-F



TEM-R</td>
<td>ATA AAA TTC TTG AAG ACG AAA



GAC AGT TAC CAA TGC TTA ATC</td>
<td>1080</td>
<td></td>
<td>[69]</td>
</tr>
<tr>
<td>OXA-F



OXA-R</td>
<td>TCA ACT TTC AAG ATC GCA



GTG TGT TTA GAA TGG TGA</td>
<td>591</td>
<td></td>
<td>[69]</td>
</tr>
<tr>
<td>OXA-F-2



OXA-R-2</td>
<td>ATT AAG CCC TTT ACC AAA CCA



AAG GGT TGG GCG ATT TTG CCA</td>
<td>501</td>
<td>Whole </td>
<td>[69]</td>
</tr>
<tr>
<td>OXA-23-F



OXA-23-R</td>
<td>GAT CGG ATT GGA GAA CCA GA



ATT TCT GAC CGC ATT TCC AT</td>
<td></td>
<td></td>
<td>[25]</td>
</tr>
<tr>
<td>CTX-M-F



CTX-M-R</td>
<td>CGC TTT GCG ATG TGC AG



ACC GCG ATA TCG TTG GT</td>
<td>550</td>
<td></td>
<td>[69]</td>
</tr>
<tr>
<td>CTX-M-F2



CTX-M-R2</td>
<td>CCA GAA TAA GGA ATC CCA TG



GCC GTC TAA GGC GAT AAA C</td>
<td></td>
<td>Whole </td>
<td>[69]</td>
</tr>
<tr>
<td>CMY-F



CMY-R</td>
<td>GAC AGC CTC TTT CTC CAC A



TGG AAC GAA GGC TAC GTA</td>
<td>1007</td>
<td></td>
<td>[69]</td>
</tr>
<tr>
<td>CMY-F2



CMY-R2</td>
<td>ACG GAA CTG ATT TCA TGA TG



GAA AGG AGG CCC AAT ATC CT</td>
<td></td>
<td>Whole </td>
<td>[69]</td>
</tr>
<tr>
<td>SHV-F



SHV-R</td>
<td>TTA TCT CCC TGT TAG CCA CC



GAT TTG CTG ATT TCG CTC GG</td>
<td>795</td>
<td></td>
<td>[69]</td>
</tr>
<tr>
<td colspan="5"><strong>Integrons</strong></td>
</tr>
<tr>
<td>5'-CS



3'-CS</td>
<td>GGC ATC CAA GCA GCA AG



AAG CAG ACT TGA CCT GA</td>
<td>152</td>
<td>Class 1 integron</td>
<td>[69]</td>
</tr>
<tr>
<td>hep74



hep51</td>
<td>CGG GAT CCC GGA CGG CAT GCA CGA TTT GTA



GAT GCC ATC GCA AGT ACG AG</td>
<td>491</td>
<td>Class 2 integron</td>
<td>[69]</td>
</tr>
<tr>
<td colspan="5"><strong>Plasmid-mediated quinolone resistance gene</strong></td>
</tr>
<tr>
<td>qnrA-F



qnrA-R</td>
<td>ATT TCT CAC GCC AGG ATT TG



GAT CGG CAA AGG TTA GGT CA</td>
<td>516</td>
<td></td>
<td>[69]</td>
</tr>
<tr>
<td>qnrB-F



qnrB-R</td>
<td>GAT CGT GAA AGC CAG AAA GG



ACG ATG CCT GGT AGT TGT CC</td>
<td>469</td>
<td></td>
<td>[69]</td>
</tr>
<tr>
<td>qnrS-F



qnrS-R</td>
<td>ACG ACA TTC GTC AAC TGC AA



TAA ATT GGC ACC CTG TAG GC</td>
<td>417</td>
<td></td>
<td>[69]</td>
</tr>
<tr>
<td colspan="5"><strong>Plasmid-mediated colistin resistance gene</strong></td>
</tr>
<tr>
<td>CLR5-F



CLR5-R</td>
<td>CGG TCA GTC CGT TTG TTC



CTT GGT CGG TCT GTA GGG</td>
<td>308</td>
<td></td>
<td>[24]</td>
</tr>
</tbody>
</table>

**Determination of antimicrobial susceptibility profiles**
The antimicrobial susceptibility was determined using the Kirby-Bauer disc diffusion test. Briefly, one colony picked up, streaked on Mueller-Hinton blood agar (Oxoid) and incubated at  overnight. Bacterial colonies were suspended in 0.9% NaCl to obtain a McFarland turbidity of 0.5 (Dr. Lange, photometer CADAS 30, Berlin, Germany) containing  colony-forming units (CFU)/ml of *E. coli* strain ATCC 25922. Three hundred l of the suspension were spread onto the surface of a Mueller-Hinton agar plate (Oxoid) using a sterile swab. The antimicrobial discs (Oxoid) of thirteen clinically used antibiotics, which are used in the Egyptian poultry and cattle farms (Tables 4, 5) were distributed onto the surface of the Mueller-Hinton agar plates using a multi-disc dispenser (Oxoid). The plates were incubated at  overnight. The diameters of inhibition zones were measured using sliding calipers and interpreted using standard break points according to Clinical and Laboratory Standards Institute. The antimicrobial susceptibility of colistin was determined by disc diffusion susceptibility testing using colistin discs (Oxoid) containing 10 g. The disc zone diameters was interpreted according to previous report.

**Table 4: Breakpoint values of antimicrobial agents according to CLSI, 2011 and phenotypic antimicrobial susceptibility profiles of 15 *S. enterica* isolates used in this study**

<table>
<thead>
<tr>
<th>Antibiotic class</th>
<th>Antimicrobial agent</th>
<th>Conc. (µg)</th>
<th>Zone diameter (mm) R / I / S</th>
<th>S. Typhimurium (9) R / S</th>
<th>S. Enteritidis (2) R / S</th>
<th>S. Kentucky (1) R / S</th>
<th>S. Paratyphi A (3) R / S</th>
<th>Total Salmonella (15) R (%) / S (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Penicillin</td>
<td>Amoxycillin-clavulanic acid</td>
<td>20</td>
<td> / 14-17 / </td>
<td>5 / 4</td>
<td>0 / 2</td>
<td>1 / 0</td>
<td>1 / 2</td>
<td>7 (47%) / 8 (53%)</td>
</tr>
<tr>
<td></td>
<td>Ampicillin</td>
<td>10</td>
<td> / 14-16 / </td>
<td>7 / 2</td>
<td>2 / 0</td>
<td>1 / 0</td>
<td>3 / 0</td>
<td>13 (87%) / 2 (13%)</td>
</tr>
<tr>
<td>Cephalosporin</td>
<td>Cefotaxime</td>
<td>30</td>
<td> / 23-25 / </td>
<td>7 / 2</td>
<td>2 / 0</td>
<td>0 / 1</td>
<td>3 / 0</td>
<td>12 (80%) / 3 (20%)</td>
</tr>
<tr>
<td></td>
<td>Cefpodoxime</td>
<td>20</td>
<td> / 18-20 / </td>
<td>7 / 2</td>
<td>1 / 1</td>
<td>0 / 1</td>
<td>1 / 2</td>
<td>9 (60%) / 6 (40%)</td>
</tr>
<tr>
<td></td>
<td>Ceftazidime</td>
<td>30</td>
<td> / 18-20 / </td>
<td>5 / 3</td>
<td>2 / 0</td>
<td>0 / 1</td>
<td>3 / 0</td>
<td>10 (67%) / 4 (26%)</td>
</tr>
<tr>
<td></td>
<td>Ceftriaxone</td>
<td>30</td>
<td> / 20-22 / </td>
<td>6 / 3</td>
<td>2 / 0</td>
<td>0 / 1</td>
<td>2 / 1</td>
<td>10 (67%) / 5 (33%)</td>
</tr>
<tr>
<td>Miscellaneous</td>
<td>Chloramphenicol</td>
<td>30</td>
<td> / 13-17 / </td>
<td>0 / 9</td>
<td>0 / 2</td>
<td>0 / 1</td>
<td>0 / 3</td>
<td>0 (0%) / 15 (100%)</td>
</tr>
<tr>
<td></td>
<td>Colistin</td>
<td>10</td>
<td> / 12-13 / </td>
<td>0 / 9</td>
<td>0 / 2</td>
<td>0 / 1</td>
<td>0 / 3</td>
<td>0 (0%) / 15 (100%)</td>
</tr>
<tr>
<td>Fluoroquinolone</td>
<td>Ciprofloxacin</td>
<td>5</td>
<td> / 16-20 / </td>
<td>0 / 9</td>
<td>0 / 2</td>
<td>0 / 1</td>
<td>0 / 3</td>
<td>0 (0%) / 15 (100%)</td>
</tr>
<tr>
<td></td>
<td>Enrofloxacin</td>
<td>5</td>
<td> / 18-20 / </td>
<td>1 / 7</td>
<td>0 / 1</td>
<td>0 / 1</td>
<td>0 / 2</td>
<td>1 (7%) / 11 (73%)</td>
</tr>
<tr>
<td></td>
<td>Nalidixic acid</td>
<td>30</td>
<td> / 14-18 / </td>
<td>3 / 6</td>
<td>1 / 1</td>
<td>1 / 0</td>
<td>0 / 3</td>
<td>5 (33%) / 10 (67%)</td>
</tr>
<tr>
<td>Aminoglycoside</td>
<td>Streptomycin</td>
<td>10</td>
<td> / 12-14 / </td>
<td>2 / 7</td>
<td>1 / 1</td>
<td>1 / 0</td>
<td>0 / 2</td>
<td>4 (26%) / 10 (67%)</td>
</tr>
<tr>
<td>Tetracycline</td>
<td>Tetracycline</td>
<td>30</td>
<td> / 12-14 / </td>
<td>3 / 6</td>
<td>2 / 0</td>
<td>1 / 0</td>
<td>0 / 3</td>
<td>6 (40%) / 9 (60%)</td>
</tr>
<tr>
<td>Sulphonamide</td>
<td>Trimethoprim/sulphamethoxazole</td>
<td>25</td>
<td> / 11-15 / </td>
<td>4 / 2</td>
<td>1 / 1</td>
<td>1 / 0</td>
<td>2 / 1</td>
<td>8 (53%) / 4 (26%)</td>
</tr>
</tbody>
</table>
S sensitive, I intermediate, R resistant

**Table 5: Breakpoint values of antimicrobial agents according to CLSI, 2011 and phenotypic antimicrobial susceptibility profiles of *E. coli* isolates used in this study**

<table>
<thead>
<tr>
<th rowspan="2">Antibiotic class</th>
<th rowspan="2">Antimicrobial agent</th>
<th rowspan="2">Conc. (µg)</th>
<th rowspan="2">Zone diameter (R / I / S)</th>
<th colspan="3">E. coli (21) S / I / R</th>
<th colspan="3">Resistance rate %</th>
</tr>
<tr>
<th>Poultry (15)</th>
<th>Beef (6)</th>
<th>Total (21)</th>
<th>Poultry</th>
<th>Beef</th>
<th>Total</th>
</tr>
</thead>
<tbody>
<tr>
<td>Penicillin</td>
<td>Amoxycillin-clavulanic acid</td>
<td>20</td>
<td> / 14-17 / </td>
<td>4 / 1 / 10</td>
<td>3 / 0 / 3</td>
<td>7 / 1 / 13</td>
<td>66.7</td>
<td>50.0</td>
<td>61.9</td>
</tr>
<tr>
<td></td>
<td>Ampicillin</td>
<td>10</td>
<td> / 14-16 / </td>
<td>2 / 1 / 12</td>
<td>2 / 1 / 3</td>
<td>4 / 2 / 15</td>
<td>80.0</td>
<td>50.0</td>
<td>71.4</td>
</tr>
<tr>
<td>Cephalosporin</td>
<td>Cefotaxime</td>
<td>30</td>
<td> / 23-25 / </td>
<td>7 / 2 / 6</td>
<td>4 / 1 / 1</td>
<td>11 / 3 / 7</td>
<td>40.0</td>
<td>16.7</td>
<td>33.3</td>
</tr>
<tr>
<td></td>
<td>Cefpodoxime</td>
<td>20</td>
<td> / 18-20 / </td>
<td>12 / 0 / 3</td>
<td>2 / 2 / 2</td>
<td>14 / 2 / 5</td>
<td>20.0</td>
<td>33.3</td>
<td>23.8</td>
</tr>
<tr>
<td></td>
<td>Ceftazidime</td>
<td>30</td>
<td> / 18-20 / </td>
<td>9 / 1 / 5</td>
<td>6 / 0 / 0</td>
<td>15 / 1 / 5</td>
<td>33.3</td>
<td>0</td>
<td>23.8</td>
</tr>
<tr>
<td></td>
<td>Ceftriaxone</td>
<td>30</td>
<td> / 20-22 / </td>
<td>7 / 5 / 3</td>
<td>4 / 1 / 1</td>
<td>11 / 6 / 4</td>
<td>20.0</td>
<td>16.7</td>
<td>19.0</td>
</tr>
<tr>
<td>Miscellaneous</td>
<td>Chloramphenicol</td>
<td>30</td>
<td> / 13-17 / </td>
<td>9 / 3 / 3</td>
<td>4 / 1 / 1</td>
<td>13 / 4 / 4</td>
<td>20.0</td>
<td>16.7</td>
<td>19.0</td>
</tr>
<tr>
<td></td>
<td>Colistin</td>
<td>10</td>
<td> / 12-13 / </td>
<td>15 / 0 / 0</td>
<td>6 / 0 / 0</td>
<td>21 / 0 / 0</td>
<td>0</td>
<td>0</td>
<td>0</td>
</tr>
<tr>
<td>Fluoroquinolone</td>
<td>Ciprofloxacin</td>
<td>5</td>
<td> / 16-20 / </td>
<td>8 / 3 / 4</td>
<td>2 / 2 / 2</td>
<td>10 / 5 / 6</td>
<td>26.7</td>
<td>33.3</td>
<td>28.6</td>
</tr>
<tr>
<td></td>
<td>Enrofloxacin</td>
<td>5</td>
<td> / 18-20 / </td>
<td>8 / 5 / 2</td>
<td>4 / 1 / 1</td>
<td>12 / 6 / 3</td>
<td>13.3</td>
<td>16.7</td>
<td>14.3</td>
</tr>
<tr>
<td></td>
<td>Nalidixic acid</td>
<td>30</td>
<td> / 14-18 / </td>
<td>10 / 0 / 5</td>
<td>4 / 1 / 1</td>
<td>14 / 1 / 6</td>
<td>33.3</td>
<td>16.7</td>
<td>28.6</td>
</tr>
<tr>
<td>Aminoglycoside</td>
<td>Streptomycin</td>
<td>10</td>
<td> / 12-14 / </td>
<td>5 / 1 / 9</td>
<td>2 / 0 / 4</td>
<td>7 / 1 / 13</td>
<td>60.0</td>
<td>66.7</td>
<td>61.9</td>
</tr>
<tr>
<td>Tetracycline</td>
<td>Tetracycline</td>
<td>30</td>
<td> / 12-14 / </td>
<td>3 / 0 / 12</td>
<td>1 / 0 / 5</td>
<td>4 / 0 / 17</td>
<td>80.0</td>
<td>83.3</td>
<td>81.0</td>
</tr>
<tr>
<td>Sulphonamide</td>
<td>Trimethoprim/sulphamethoxazole</td>
<td>25</td>
<td> / 11-15 / </td>
<td>3 / 2 / 10</td>
<td>2 / 1 / 3</td>
<td>5 / 3 / 13</td>
<td>66.7</td>
<td>50.0</td>
<td>61.9</td>
</tr>
</tbody>
</table>
S sensitive, I intermediate, R resistant.

**Molecular detection of antimicrobial resistance associated genes**
*Escherichia coli* and *S. enterica* isolates were tested for -lactamase-encoding genes , , ,  and  by PCR using universal primers for the corresponding gene families (Table 3) as described previously. PCR amplification was also used for screening of plasmid-mediated quinolone resistance genes *qnrA*, *qnrB* and *qnrS*. For detection of variants of , , ,  and  amplified PCR fragments were purified from agarose gels using Nucleospin Gel Extraction Kit (Macherey-Nagel, Düren, Germany). The products were sequenced and the sequencing results were analyzed using BLAST. Colistin resistance associated gene *mcr-1* was identified using a PCR assay (Table 3) as described by Liu et al. PCR for detection of carbapenemases  gene was performed using OXA-23-F and OXA-23-R (Table 3). PCR reaction and conditions were performed according to Braun et al.

**Detection and sequencing of class 1 and class 2 integrons**
PCR assays for detection of class 1 and class 2 integrons were performed using primers given in Table 3 and yielded PCR fragments that were purified from agarose gels using Nucleospin Gel Extraction Kit (Macherey-Nagel, Düren, Germany). The products were sequenced in Genome Centre, Gifu University, Japan. The sequencing results were analyzed using BLAST.

### Results

Out of 180 examined samples 15 *S. enterica* (8.3%) (7 from poultry and 8 from beef) (Table 1) as well as 21 *E. coli* (11.7%) (15 from poultry and 6 from beef) (Table 2) were isolated and identified using bacteriological methods and biochemical characterization.

**Serological characterization of *S. enterica* and *E. coli***
Fifteen isolated *S. enterica* were typed as 9 *S. enterica* serovar Typhimurium (60.0%; 3 from chicken and 6 from beef), 3 *S. enterica* Paratyphi A (20.0%; 2 from chickens and 1 from beef), 2 *S. enterica* serovar Enteritidis (13.3%; 1 from chicken and 1 from beef) and 1 *S. enterica* serovar Kentucky (6.7%) from chicken sample (Table 1). Twenty-one *E. coli* isolates were characterized as 4 O18 (19.0%) and 4 O78 (19.0%) serotypes. Other serotypes are given in Table 2.

**Molecular identification of *Salmonella* serovars**
All 15 recovered *S. enterica* serovars harboured *ompC* gene, which confirms *Salmonella* genus. Nine *S. enterica* serovar Typhimurium (60.0%) and two *S. enterica* serovar Enteritidis (13.3%) were identified by means of serovar-specific bands using PCR (Table 3). Four other isolates could not be typed with PCR method (26.7%).

**Determination of antimicrobial susceptibility profiles**
The results of determination of antimicrobial resistance of *S. enterica* and *E. coli* isolates to thirteen antibiotics are given in Tables 4 and 5. Most of the *S. enterica* isolates showed resistance to ampicillin (87.0%) and cefotaxime (80.0%) and all were susceptible to chloramphenicol, colistin and ciprofloxacin. Resistance to other antibiotics was shown in Table 4. Seven isolates of *S. enterica* serovar Typhimurium were resistant to ampicillin, cefotaxime and cefpodoxime (77.8%), while 6 isolates (66.7%) were highly sensitive to ciprofloxacin. Resistance to other antibiotics was weaker developed. Both *S. enterica* serovar Enteritidis isolates were resistant to ampicillin, cefotaxime and tetracycline. Additionally, one isolate showed resistance to streptomycin, nalidixic acid, trimethoprim/sulphamethoxazole and streptomycin. In addition to the entire resistance to ampicillin and cefotaxime, majority of the *S. enterica* serovar Paratyphi A isolates were resistant to amoxicillin-clavulanic acid, ceftriaxone and trimethoprim/sulphamethoxazole. *Salmonella enterica* serovar Kentucky isolate showed no pronounced susceptibility to any of the tested antibiotics and was resistant to six of them (Table 4). Seven out of 15 *S. enterica* serovars (46.7%) revealed phenotypic multidrug resistance exhibiting by resistance to three or more classes of antibiotics (Table 4).

*Escherichia coli* isolates showed resistance to tetracycline, ampicillin, streptomycin, trimethoprim/sulphamethoxazole and amoxicillin-clavulanic acid with 80.9, 71.4, 61.9, 61.9 and 61.9%, respectively. Ten (47.6%), 9 (42.8%) and 9 (42.8%) isolates were susceptible to ciprofloxacin, enrofloxacin and ceftriaxone, respectively. All tested isolates were susceptible to colistin. Thirteen *E. coli* isolates (61.9%) were characterized as multidrug resistant (Table 5). The multidrug resistance of isolated *Salmonella* and *E. coli* was shown in Tables 6 and 7, respectively.

**Table 6: Phenotypic resistance and resistance determinants found in *S. enterica* isolates in this study**

<table>
<thead>
<tr>
<th>Source</th>
<th>Serovar</th>
<th>Resistance phenotype</th>
<th>Resistance genes/class 1 integrons</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="4"><strong>Poultry</strong></td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>Typhimurium</td>
<td>CPD, CTX, NAL</td>
<td>, , , </td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>Typhimurium</td>
<td>AMC, AMP, CPD</td>
<td></td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>Enteritidis</td>
<td>AMP, CTX, TET</td>
<td>, </td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>Kentucky</td>
<td>AMC, AMP, NAL, STR, TET, T/S</td>
<td>, </td>
</tr>
<tr>
<td>Fresh poultry organs (liver)</td>
<td>Typhimurium</td>
<td>AMC, AMP, CPD, CRO, CTX, NAL, T/S</td>
<td>, , , , </td>
</tr>
<tr>
<td>Fresh poultry organs (liver)</td>
<td>Paratyphi A</td>
<td>AMC, AMP, CAZ, CRO, CTX, T/S</td>
<td>, </td>
</tr>
<tr>
<td>Frozen poultry meat</td>
<td>Paratyphi A</td>
<td>AMC, AMP, CRO, CTX, T/S</td>
<td>, </td>
</tr>
<tr>
<td colspan="4"><strong>Beef</strong></td>
</tr>
<tr>
<td>Fresh beef meat</td>
<td>Typhimurium</td>
<td>AMP, CAZ, CTX</td>
<td></td>
</tr>
<tr>
<td>Fresh beef meat</td>
<td>Typhimurium</td>
<td>AMP, CAZ, CRO, CTX</td>
<td></td>
</tr>
<tr>
<td>Fresh beef meat</td>
<td>Typhimurium</td>
<td>AMC, CPD, CTX, TET, T/S</td>
<td>, </td>
</tr>
<tr>
<td>Fresh beef meat</td>
<td>Enteritidis</td>
<td>AMP, CPD, CTX, NAL, STR, TET, T/S</td>
<td>, , , , </td>
</tr>
<tr>
<td>Fresh beef organs (spleen)</td>
<td>Typhimurium</td>
<td>AMP, CPD, CTX, STR, TET, T/S</td>
<td>, </td>
</tr>
<tr>
<td>Fresh beef organs (liver)</td>
<td>Typhimurium</td>
<td>AMC, AMP, CPD</td>
<td>-</td>
</tr>
<tr>
<td>Fresh beef organs (liver)</td>
<td>Paratyphi A</td>
<td>AMP, CPD, CTX</td>
<td></td>
</tr>
<tr>
<td>Frozen beef meat</td>
<td>Typhimurium</td>
<td>AMC, AMP, CAZ, CPD, CRO, CTX, ENR, NAL, STR, TET, T/S</td>
<td>, , , , , </td>
</tr>
</tbody>
</table>
AMC amoxycillin-clavulanic acid, AMP ampicillin, CAZ ceftazidime, CPD cefpodoxime, CRO chloramphenicol, CTX cefotaxime, ENR enrofloxacin, NAL nalidixic acid, STR streptomycin, TET tetracycline, T/S trimethoprim/sulphamethoxazole

**Table 7: Phenotypic resistance and resistance determinants found in *E. coli* isolates in this study**

<table>
<thead>
<tr>
<th>Source</th>
<th>E. coli serotype</th>
<th>Resistance phenotype</th>
<th>Resistance genes/class 1 integron</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="4"><strong>Poultry</strong></td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>O1</td>
<td>STR, TET</td>
<td>-</td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>O18</td>
<td>AMP, CIP, CPD, NAL, STR, TET, T/S</td>
<td>, </td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>O18</td>
<td>AMC, AMP, CHL, CIP, CRO, CTX, NAL, STR, TET, T/S</td>
<td>, , , </td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>O78</td>
<td>AMC, AMP, CAZ, TET, T/S</td>
<td>, </td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>O119</td>
<td>AMC, AMP, CAZ, CTX, STR, TET, T/S</td>
<td>, </td>
</tr>
<tr>
<td>Fresh poultry meat</td>
<td>O20</td>
<td>AMC, AMP, CPD, TET, T/S</td>
<td>, , </td>
</tr>
<tr>
<td>Fresh poultry organs (liver)</td>
<td>O18</td>
<td>AMC, AMP, CAZ, CHL, CIP, CRO, CTX, ENR, NAL, STR, TET, T/S</td>
<td>, , , , </td>
</tr>
<tr>
<td>Fresh poultry organs (liver)</td>
<td>O126</td>
<td>AMP, CTX, STR</td>
<td></td>
</tr>
<tr>
<td>Fresh poultry organs (gizzard)</td>
<td>O20</td>
<td>AMC, AMP, STR, T/S</td>
<td></td>
</tr>
<tr>
<td>Fresh poultry organs (liver)</td>
<td>O145</td>
<td>CAZ, TET</td>
<td>-</td>
</tr>
<tr>
<td>Frozen poultry meat</td>
<td>O1</td>
<td>CTX, NAL, TET</td>
<td>, </td>
</tr>
<tr>
<td>Frozen poultry meat</td>
<td>O18</td>
<td>AMC, AMP, STR, TET, T/S</td>
<td>, </td>
</tr>
<tr>
<td>Frozen poultry</td>
<td>O78</td>
<td>AMC, AMP, CPD</td>
<td>-</td>
</tr>
<tr>
<td>Frozen poultry meat</td>
<td>O146</td>
<td>AMC, AMP, TET, T/S</td>
<td></td>
</tr>
<tr>
<td>Frozen organ poultry</td>
<td>O78</td>
<td>AMC, AMP, CAZ, CHL, CIP, CRO, CTX, ENR, NAL, STR, TET, T/S</td>
<td>, , , , </td>
</tr>
<tr>
<td colspan="4"><strong>Beef</strong></td>
</tr>
<tr>
<td>Fresh beef meat</td>
<td>O119</td>
<td>CPD</td>
<td>-</td>
</tr>
<tr>
<td>Fresh beef meat</td>
<td>O158</td>
<td>CPD, TET</td>
<td>-</td>
</tr>
<tr>
<td>Fresh beef organs (liver)</td>
<td>O78</td>
<td>AMC, AMP, CHL, CIP, CRO, CTX, ENR, NAL, STR, TET, T/S</td>
<td>, , , , , </td>
</tr>
<tr>
<td>Frozen beef meat</td>
<td>O103</td>
<td>STR, TET</td>
<td></td>
</tr>
<tr>
<td>Frozen beef meat</td>
<td>O126</td>
<td>AMC, AMP, CIP, STR, TET, T/S</td>
<td>, , </td>
</tr>
<tr>
<td>Frozen beef meat</td>
<td>O158</td>
<td>AMC, AMP, STR, TET, T/S</td>
<td></td>
</tr>
</tbody>
</table>
AMC amoxycillin-clavulanic acid, AMP ampicillin, CAZ ceftazidime, CPD cefpodoxime, CRO chloramphenicol, CTX cefotaxime, ENR enrofloxacin, NAL nalidixic acid, STR streptomycin, TET tetracycline, T/S trimethoprim/sulphamethoxazole

**Molecular detection of resistance-associated genes**
Six of 8 screened resistance-associated genes were detected in *S. enterica* serovars by PCR (Table 6). Eleven (73.3%) isolates harboured  gene (1, 3 and 14) associated with cefotaxime resistance and 11 (73.3%) isolates harboured  gene associated with penicillin and narrow spectrum -lactamase resistance. Five (33.3%) isolates harboured quinolone resistance gene A (), while 3 (20.0%), 2 (13.3%) and 1 (6.7%) of *S. enterica* serovars possessed ,  and  genes, respectively.  associated with resistance to ampicillin and cephalothin and  associated with plasmid-mediated ampicillin resistance were not found in any isolate.

In *E. coli* isolates, 5 of 8 screened resistance-associated genes were detected by PCR (Table 7). Eleven (52.4%) out of 21 isolates harboured  (1 and 104), while  (1 and 14) was detected in 9 isolates (42.9%). Seven isolates (33.3%) carried  and  and in three isolates (14.3%)  was detected. Genes ,  and  were not detected in any of the *E. coli* isolates. Plasmid-mediated colistin resistance gene *mcr-1* and carbapenemase resistance gene  were not identified neither in *S. enterica* nor in *E. coli* isolates using PCR assay.

**Integron gene cassettes and DNA sequencing**
Class 1 integron was detected in two isolates of *S.* Typhimurium using PCR (Table 6). The inserted gene cassettes identified three types of antimicrobial resistance genes associated with class 1 integron: dihydrofolate reductase types (*dfrA15* and *dfrA17*) which confer resistance to trimethoprim, and aminoglycoside adenyltransferase type *aadA2* that confers resistance to streptomycin and spectinomycin. Three *E. coli* isolates of serotypes O18, O78 and O78 were harbouring class 1 integron (Table 7). The inserted gene cassettes identified three types of class 1 integron. The identified antimicrobial resistance genes were dihydrofolate reductase types; *dfrA1*, *dfrA15* and *dfrA1-orf* which confer resistance to trimethoprim. All *S. enterica* and *E. coli* isolates were negative for class 2 integrons.

### Discussion

Salmonellosis and *Salmonella* infections considered as critical threats to human and animal health. In this study, the prevalence of *S. enterica* serovars in chicken and beef meat was 8.3%, which was considerably lower than incidence rates that reported in Ethiopia (12.0% in raw meat), Canary islands (16.5% in chicken meat), Northwestern Spain (17.9% in chicken), Ethiopia (17.9% in chicken and giblets), and Egypt (10.0% in poultry meat). On the other hand, it was higher than reported previously in meat products in Egypt (6.6%) and in ground beef in the United States (4.2%). In total, *S. enterica* serovar Typhimurium was the dominant serovar. These results were in accordance with previous study from chicken products in Cambodia and Thailand. Controversially, *S. enterica* serovar Enteritidis was the dominant serotype in imported frozen poultry samples from Brazil to Canary Islands and in chicken carcasses in Spain. *Salmonella enterica* serovar Kentucky was frequently detected in samples coming from US to Canary Islands.

Prevalence of *E. coli* in chicken meat and organs was 16.7% which was lower than in Nigeria (43.4%) in frozen poultry meat but higher than in Korea (4.9% in poultry meat). Prevalence of *E. coli* in beef meat was 6.7%, while in Korea and Iran were found 4.1 and 29.0%, respectively. In this study, 19.0% of *E. coli* isolates typed as O78 was mainly from poultry products, while in China, O78 was identified in 60.0% of *E. coli* isolated from chicken and ducks. *E. coli* type O158 identified in 9.5% only in beef meat while O158 detected in 22.7% of food isolates in Chile.

Bacterial antimicrobial resistance is a global emerging problem of public health concern. In the current study, a high percentage of *S. enterica* serovars were resistant to ampicillin and cefotaxime. The resistance to other antimicrobial agents was variable while all isolates were sensitive to chloramphenicol and ciprofloxacin. The results were in a partial accordance with results of previous reports stating that all *S. enterica* isolated from chicken meat and beef were sensitive to ciprofloxacin. 72.4% of *Salmonella* isolates in Thai meat products were resistant to ampicillin while 71.0% of isolates in Cambodian meat products were resistant to sulfamethoxazole. Controversially, *S. enterica* isolated from animals and food of animal origin in Italy were highly resistant to ampicillin, chloramphenicol, streptomycin and tetracycline and all *Salmonella* isolated from beef in Tunisia were susceptible to amoxicillin and clavulanic acid. In total, 46.7% of *S. enterica* serovars showed multidrug resistance, which was higher than resistance of *Salmonella* isolated from raw chicken (31.0%) in Pennsylvania during 2010. Less multidrug resistant *S. enterica* isolates (6.0%) were identified in cattle hides and carcasses in the US.

The results of this study showed that the antimicrobial resistance of *E. coli* isolated from poultry was higher than from beef to the most of tested antibiotics. The high resistance to tetracycline, ampicillin, amoxycillin-clavulanic acid, trimethoprim/sulphamethoxazole and streptomycin in this study was in agreement with previous reports conducted in Egypt and Algeria. While in Spain, most of *E. coli* isolated from diarrhoeic and healthy lambs were highly resistant to tetracycline and streptomycin but show lower resistance to ampicillin. Most of *E. coli* isolates in this study were sensitive to enrofloxacin, chloramphenicol and ceftriaxone which is in agreement with previous results in Spain. In this study, 61.9% of *E. coli* isolates were multidrug resistant. Similar results reported previously in Ghana and US.

Most of the phenotypically antibiotic resistant *S. enterica* serovars carried antibiotic resistance marker genes associated with -lactams and quinolones. The -lactamase encoding gene  conferring resistance to penicillins and first-generation cephalosporins was detected in 73.3% of *S. enterica* serovars and in 76.9% of ampicillin-resistant isolates which is significantly high in comparison to 57.3% () in *S. enterica* isolated from retail chickens in China. Another -lactamase encoding gene , an AmpC beta-lactamase gene that confers resistance to a wide variety of -lactam antibiotics detected in 13.3% of isolates. In contrast, in 4.7% of *S. enterica* serovars Typhimurium and Enteritidis originated from diseased broilers in Egypt  was identified and all resistant *S. enterica* serovar Typhimurium isolated from retail meat in US were harbouring . The same gene could not be detected previously in any *S. enterica* serovar Typhimurium isolated from chicken meat in Egypt.  could be identified in 73.3% of isolated *S. enterica* serovars. In 24.0 and 18.8% of different *Salmonella* serovars isolated from retail chicken carcasses in China,  and  were detected, respectively.

The  is responsible for the plasmid-mediated ampicillin resistance and -lactamase encoding genes  conferring resistance to ampicillin and cephalothin not detected in any of *S. enterica* serovars in this study. This result was in accordance with a result for *S. enterica* serovar Typhimurium isolated from chicken meat in Egypt. In contrast, 30.2% of ESBL-producing *Salmonella* isolated from raw chicken carcasses in China were harbouring . Although none of *S. enterica* serovars were phenotypically resistant to ciprofloxacin, *qnrA*, *qnrB* and *qnrS* genes were detected in 33.3, 20.0 and 6.7%, respectively in all *S. enterica* serovars and in 100, 60.0 and 20.0%, respectively in *S. enterica* resistant to nalidixic acid. A different result showed that 1.16% of nalidixic acid-resistant *S. enterica* serovars isolated from animal products in Tunisia carried *qnr* gene. In other studies in China *qnrA*, *qnrB* and *qnrS* genes detected with low percentage in *S. enterica* serovar Enteritidis isolated from retail raw poultry.

The  gene was detected in 52.3% of all isolated *E. coli* and in 73.3% of ampicillin-resistant isolates. Other studies detected  in 97.1 and 75.0% of *E. coli* isolates from lambs in Spain and meat products in Cambodia, respectively. In China,  was identified in 7.8% of ESBL-producing *E. coli* recovered from meat products. The -lactamase encoding gene  was not found in any of the isolated *E. coli*. In contrast,  was detected in 89.0% *E. coli* isolated from poultry meat in Denmark and in 12.5% of ESBL-producing *E. coli* isolated from meat products in Cambodia. The  detected in 42.9% of *E. coli* isolated in this study. In Ghana 50.0% of ESBL-producing *E. coli* isolates from meat harboured  while these -lactamase encoding genes  and  were detected rarely (1.6%) in *E. coli* isolates from meat products in China. The  gene was not found in any of the *E. coli* isolates. Controversially, previous studies detected  in 9.4, 5.3 and 2.0% in meat products in China, broiler chickens and chicken carcasses in Iran, respectively.  was detected in 14.3% of the *E. coli* isolates which is in contrast with previous report conducted in retail meat in US where the gene could not be identified. The most common carbapenemase types were  and  accounting for 47% of all identified carbapenemase genes. In this study,  was not identified in all isolated strains.

In isolated *E. coli*, *qnrA* and *qnrB* detected in 33.3% of isolates, all nalidixic acid and/or ciprofloxacin resistant isolates harboured both genes. While *qnrB* and *qnrS* were identified in 10.4% of nalidixic acid resistant *E. coli* isolated from Algerian retail chicken meat and in 10.0% of *E. coli* isolates obtained from bovine carcasses in Mexico. In contrast, *qnrA*, *qnrB* and *qnrS* genes could not be identified in enrofloxacin-resistant *E. coli* strains from poultry in Mexico.

The presence of genetic elements such as integrons and transposons are often associated with multi-resistant phenotypes among *Salmonella* isolates. In this study, class 1 integron detected in 13.3% of *S. enterica* serovars. The identified gene cassettes were *dfrA15*, *dfrA17* and *aadA2*. In previous studies, class 1 integron identified in 90.0% of multi-drug resistant *Salmonella* isolates from retail chicken meat in Japan and the identified genes cassettes were *dfrA1*, *dfrA7*, *aadA1*, *aadB*, and *catB3*. The gene cassettes of class 1 integron which detected in *Salmonella* spp. isolated from poultry meat in Egypt were harbouring *aac (3)-Id*, *aadA2*, *aadA4*, *aadA7*, *sat*, *dfrA15*, *lnuF* and *estX* resistance genes. In a study conducted in Portugal, 75.0% of *S. enterica* isolated from different sources including food products had one or two class 1 integrons.

The identified gene cassettes of class 1 integron *dfrA1*, *dfrA15* and *dfrA1-orf* which confer resistance to trimethoprim were identified in 14.3% of *E. coli* isolated in this study. This is in agreement with *E. coli* isolates from retail chickens in Japan and in 1.9 and 11.4% of isolates from Thai and Cambodian meat samples obtained from slaughterhouses and fresh markets and the most common gene cassette was *dfrA12-aadA2*. Class 2 integrons were not detected in all isolates of this study which is in contrast to 5.6% positive samples in Egypt.

In this study, *mcr-1* gene associated with colistin resistance was neither detected in *S. enterica* nor *E. coli* isolates. This result was in agreement with the result reported by Doumith et al. who investigated 24,000 isolates of Enterobacteriaceae from food and humans including *S. enterica* and *E. coli* and found only 15 *mcr-1* positive isolates. Recently, Quesada et al. identified the gene *mcr-1* in nine *S. enterica* and *E. coli* isolates from poultry and swine for the first time in Spain. Jayol et al. found the *mcr-1* in 13% of *E. coli* and *S. enterica*. Considering the frequent use of colistin in animal production and the importance of this antimicrobial for the control of multi-resistant Gram-negative nosocomial infections in humans, monitoring the dissemination of resistance to colistin is mandatory.

In conclusion, the results of this study showed high prevalence of *S. enterica* and *E. coli* as foodborne pathogens isolated from poultry and beef meat in Egypt. The emergence of antimicrobial resistance of *S. enterica* and *E. coli* isolates is of public concerns in Egypt. Significant resistance was detected to penicillin, cephalosporins, tetracycline and sulphonamides. Dissemination of ESBL and AmpC--lactamase resistance-associated genes in *S. enterica* and *E. coli* was determined. Presence of class 1 integron in *S. enterica* and *E. coli* and genes associated with antibiotic resistance was also confirmed. Class 2 integron was not detected in any isolate. Further work should be performed to characterize *S. enterica* and *E. coli* isolates of animal and human origin from the same region sharing the same resistance markers in order to highlight potential horizontal gene transfer by these zoonotic organisms.