# Antimicrobial resistance of Campylobacter isolates from small scale and backyard chicken in Kenya

**Nguyen et al. Gut Pathogens (2016) 8:39**
**DOI:** 10.1186/s13099-016-0121-5

**Authors:** Tuan Ngoc Minh Nguyen, Helmut Hotzel, John Njeru, Joyce Mwituria, Hosny El-Adawy, Herbert Tomaso, Heinrich Neubauer and Hafez M. Hafez.

## Abstract

**Background:** Thermophilic *Campylobacter* species are a major cause of bacterial foodborne diarrhoea in humans worldwide. Poultry and their products are the predominant source for human campylobacteriosis. Resistance of *Campylobacter* to antibiotics is increasing worldwide, but little is known about the antibiotic resistance in *Campylobacter* isolated from chicken in Kenya. In this study, 35 suspected *Campylobacter* strains isolated from faeces and cloacal swabs of chicken were tested for their susceptibility to seven antibiotics using a broth microdilution assay and molecular biological investigations.

**Results:** Overall, DNA of thermophilic *Campylobacter* was identified in 53 samples by PCR (34 *C. jejuni*, 18 *C. coli* and one mix of both species) but only 35 *Campylobacter* isolates (31 *C. jejuni* and 4 *C. coli*) could be re-cultivated after transportation to Germany. Isolates were tested for their susceptibility to antibiotics using a broth microdilution assay. Additionally, molecular biological detection of antibiotic resistance genes was carried out. *C. jejuni* isolates showed a high rate of resistance to nalidixic acid, tetracycline and ciprofloxacin of 77.4, 71.0 and 71.0%, respectively. Low resistance (25.8%) was detected for gentamicin and chloramphenicol. Multidrug resistance in *C. jejuni* could be detected in 19 (61.3%) isolates. Resistance pattern of *C. coli* isolates was comparable. Resistance to ciprofloxacin was confirmed by MAMA-PCR and PCR-RFLP in all phenotypically resistant isolates. The *tet(O)* gene was detected only in 54.5% of tetracycline resistant *C. jejuni* isolates. The *tet(A)* gene, which is also responsible for tetracycline resistance, was found in 90.3% of *C. jejuni* and in all *C. coli* isolates. Thirteen phenotypically erythromycin-resistant isolates could not be characterised by using PCR-RFLP and MAMA-PCR.

**Conclusions:** To the best of our knowledge, this study is the first report about resistance to antibiotics in thermophilic *Campylobacter* originating from chicken in Kenya. *Campylobacter* spp. show a high level of resistance to ciprofloxacin, nalidixic acid and tetracycline but also a remarkable one to chloramphenicol and gentamicin and they are multidrug resistant. Resistance to antibiotics is a global public health concern. In Kenya, resistance surveillance needs further attention in the future. Efforts to establish at least a National Laboratory with facilities for performing phenotypic and genotypic characterization of thermophilic *Campylobacter* is highly recommended.

**Keywords:** Campylobacter, Antibiotic resistance, Microdilution, Chicken, Kenya 

---

## Background

Thermophilic *Campylobacter* (C.) species have become the most frequent cause of bacterial gastroenteritis in humans worldwide. Campylobacteriosis exceed the total number of those caused by *Salmonella*, *Shigella*, and *Escherichia coli* O157:H7 in humans, recently. *Campylobacter* infections are normally self-limiting in adults but can cause diarrhoea or even mortality in children in developing as well as in developed countries. A study from western Kenya showed that 20% of patients with diarrhoea were infected by *Campylobacter* but in the group of children below 5 years *Campylobacter* was represented with 42%.

Commercial poultry and free-living birds are natural reservoirs of thermophilic campylobacters. The organism has been isolated from numerous bird species, including Columbiformes and domestic and free-living Galliformes and Anseriformes. *C. jejuni* has been found in all areas of commercial poultry production. Prevalence rates in poultry, especially in slaughter-age broiler flocks, could reach 100% on some farms. Although, *Campylobacter* is insignificant for poultry health, it is a predominant cause of foodborne gastroenteritis in humans worldwide, and contaminated poultry meat is recognized as the main source of human infections. In general, the knowledge about *Campylobacter* in Kenya is limited. Most of the published reports describe prevalence and antibiotic resistance in *Campylobacter* of human origin. Other reports gave information about *Campylobacter* as cause of foodborne diseases and contamination of raw chicken and beef from butcheries and markets in Nairobi. Information on thermophilic *Campylobacter* of animal origin from Kenya is lacking.

Resistance against antibiotics in bacteria is of public health concern. Most commonly used drugs in treatment of campylobacteriosis in humans are erythromycin, fluoroquinolones or tetracycline. Although, this antimicrobial treatment is usually not necessary, however the misuse of antibiotics is widespread in Kenya. Attention on resistance of *Campylobacter* is raising and warning has been launched not to misuse antibiotics such as macrolides, fluoroquinolones or alternative drugs. Kenyan *Campylobacter* isolates from humans showed a high resistance rate against erythromycin (52%), but only low resistance to ciprofloxacin, tetracycline and nalidixic acid with 6, 18 and 26% in the past, respectively.

Clinical breakpoints of *Campylobacter* susceptibility based on epidemiological cut-off (ECOFF) values were recommended. EUCAST MIC distributions incorporate human and veterinary clinical data from several sources worldwide. The method of choice for testing antibiotic susceptibility and determination of minimum inhibitory concentration (MIC) values of *Campylobacter* isolates is the broth microdilution assay.

In addition to phenotypical determination of antibiotic resistance, genetic analysis of resistance determinants in *Campylobacter* can be carried out. A replacement of threonine by isoleucine at amino acid 86 in the *gyrA* gene and a mutation at position 2074 and 2075 on the 23S rRNA gene are the main mechanisms for fluoroquinolone and erythromycin resistance, respectively. Presence of *tet(O)* and/or *tet(A)* genes is responsible for tetracycline resistance. A mismatch amplification mutation assay (MAMA-PCR) can be used for detection of the mutations in *gyrA* and 23S rRNA genes in *C. coli* and *C. jejuni* responsible for ciprofloxacin and erythromycin resistance, respectively. PCR-restriction fragment length polymorphism (PCR-RFLP) technique is available for detection of erythromycin resistance as well as specific PCR assays for *tet(O)* and *tet(A)* genes. These methods allow the investigation of antibiotic resistance of *Campylobacter* even in samples from which no *Campylobacter* could be isolated.

To the best of our knowledge there is no report available about antibiotic resistance of thermophilic *Campylobacter* species isolated from chicken in Kenya. MICs and results of molecular assays on the resistance of recent Kenyan *C. coli* and *C. jejuni* are presented.

## Methods

### Sample collection and *Campylobacter* isolation

In total, 35 geographically different native breed layer flocks were sampled. The chickens were housed in backyards and homesteads of small scale farmers from the outskirts of Thika, a town 40 km northeastern of Nairobi, Kenya. Farmers kept between 10 and 1000 layers. The birds were fed on commercially formulated ration from different sources and sometimes supplied with the leftover and residual food. All the manufactures used antibiotics as part of the ingredients in the feed. During the rearing of these chickens, antibiotics were used for prevention and treatment of diseases without any instructions.

Ten to 30 cloacal swabs and faecal samples were collected from each flock according to flock size. *Campylobacter* were isolated in Kenya Medical Research Institute, Nairobi according to the guidelines of ISO 10272-1. The isolates were preserved in 1.5 ml Eppendorf tubes filled with skimmed milk medium for 1-week transportation from Kenya to Friedrich-Loeffler-Institut, Jena, Germany for further laboratory analysis. *Campylobacter* strains were re-cultivated on both Mueller-Hinton agar and CCDA (Oxoid GmbH, Wesel, Germany) under microaerophilic conditions (5% , 10% , and 85% ) at  for 48–72 h.

### DNA extraction

DNA from viable bacteria was extracted using the High Pure PCR Template Preparation Kit™ (Roche Diagnostics, Mannheim, Germany) according to the manufacturer’s instructions. Skimmed milk samples of *Campylobacter* that could not be re-cultivated were treated with phenol-chloroform to extract DNA. Briefly, 500 µl of skimmed milk medium was boiled for 5 min. After cooling the liquid was mixed with 500 µl buffer-saturated phenol (Carl Roth GmbH, Karlsruhe, Germany) and centrifuged for 5 min. at 13,400 rpm (miniSpin, Eppendorf, Hamburg, Germany). 500 µl chloroform/isoamyl alcohol (24:1 vol/vol) was added to the aqueous phase, mixed and centrifuged for 5 min at 13,400 rpm. DNA from the aqueous phase was precipitated by mixing with 0.6 volume of isopropanol at room temperature. After centrifugation, the supernatant was discarded and the DNA was air dried and finally dissolved in 50 µl 10 mM Tris (Carl Roth GmbH).

### Multiplex PCR for identification of *Campylobacter* species

A mPCR assay was used to identify thermophilic *Campylobacter* species (*C. jejuni*, *C. coli*, and *C. lari*) as described by El-Adawy et al.. Briefly, the PCR was performed in a 50-µl reaction mixture containing 5.0 µl of 10x Taq reaction buffer complete (Jena Bioscience GmbH, Jena, Germany), 2.0 µl of dNTP mix (2 mM each; Carl Roth GmbH), 2.0 µl of each primer (Jena Bioscience GmbH), and 0.2 µl of Taq Pol thermostable DNA polymerase (Jena Bioscience GmbH). Amplification reactions were carried out in a TRIO Thermoblock cycler (Biometra, Göttingen, Germany) using the following programme: one cycle of 1 min at  was followed by 35 cycles each consisting of 60 s at  of denaturation, 90 s at  of annealing, and 60 s at  of elongation. The PCR was terminated after a final extension step of 5 min. at . Amplification generated 857, 589, 522, and 462 base pair DNA fragments specific for the genus *Campylobacter* and the species *C. jejuni*, *C. lari*, and *C. coli*, respectively.

### Antimicrobial susceptibility testing and determination of MICs

The antimicrobial susceptibility of *C. jejuni* and *C. coli* isolates was tested against seven antibiotic agents (chloramphenicol, erythromycin, ciprofloxacin, nalidixic acid, gentamicin, streptomycin, and tetracycline) using the Sensititre™ Campylobacter plates-EUCAMP (Trek Diagnostic Systems Ltd., East Grinstead, UK). The MIC values were detected using different concentration ranges as previously described. Briefly, *Campylobacter* isolates grown on Mueller-Hinton agar (Oxoid GmbH) supplemented with 10% bovine blood under microaerophilic conditions were suspended in NaCl solution (0.9%) to obtain a turbidity corresponding to a McFarland standard of 0.5 (Dr. Lange, CADAS photometer 30, Berlin, Germany). One-hundred and fifty milliliters of the above suspension were diluted with 10 ml Mueller-Hinton broth (Oxoid GmbH) resulting in a concentration of approximately  colony forming units . One hundred milliliters of the inoculum was filled in each well of the plate; the plates were sealed and incubated at  for 24 h under microaerophilic conditions. Results were read either visually or photometrically (Tecan Deutschland GmbH, Crailsheim, Germany) using the computer program easyWIN fitting (version V6.1, 2000). *C. jejuni* DSM 4688 and *C. coli* DSM 4689 (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH) were included in each batch of broth microdilution assay for quality control. The lowest concentration of antibiotics that prevents visible growth of the microorganism is defined as the MIC.

### Molecular biological detection of antibiotic resistance determinants

Extracted *Campylobacter* DNA from all samples and strains was used for molecular biological determination of selected antibiotic resistance determinants by PCR.

#### Erythromycin resistance

Detection of mutations at positions 2074 and 2075 in domain V of the 23S rRNA gene, which mediates resistance to erythromycin, was carried out by MAMA-PCR and PCR-RFLP as described previously. Genes responsible for resistance of erythromycin and ciprofloxacin were tested at two loci using MAMA-PCR and PCR-RFLP. Primers and their sequences are given in Table 1.

#### Ciprofloxacin resistance

A single point mutation (Thr-86-Ile) in the quinolone resistance-determining region (QRDR) of *gyrA* gene was defined as source of high-level resistance to fluoroquinolones. MAMA-PCR for *C. jejuni* isolates was carried out as described previously , for *C. coli* a procedure according to Zirnstein et al. was used. Primer details are given in Table 1.

#### Tetracycline resistance

Primers DMT1 and DMT2 (Jena Bioscience GmbH) were used for the detection of the *tet(O)* gene which is strongly associated with tetracycline resistance in *C. jejuni* and *C. coli* as described previously. As a second gene locus associated with tetracycline resistance the presence of *tet(A)* was examined by a previously described PCR assay. An alternative, in-house validated PCR assay was created based on *tet(A)* sequences (GenBank acc. no. JX891463 and JX891464). Briefly, primers tet-A-1 and tet-A-2 (Table 1; Jena Bioscience GmbH) were used with the following PCR programme: An initial denaturation at  for 60 s was followed by 35 cycles of denaturation ( for 15 s), annealing ( for 60 s) and extension ( for 30 s). PCR was terminated by final extension at  for 60 s. The PCR resulted in a 486 bp product. All PCR products were analyzed by electrophoresis on 1.5% agarose gels, staining with ethidium bromide and visualization under UV light.

**Table 1: List of primers and primer sequences used for detection of antimicrobial resistance genes**

<table>
<thead>
<tr>
<th>Antibiotic</th>
<th>Method</th>
<th>Primer</th>
<th>Sequence (5'-3')</th>
<th>Amplicon length (bp)</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">Erythromycin</td>
<td>MAMA-PCR</td>
<td>23SRNA-F



ERY2075-R



ERY2074-R</td>
<td>TTA GCT AAT GTT GCC CGT ACC G



TAG TAA AGG TCC ACG GGG TCG C



AGT AAA GGT CCA CGG GGT CTG G</td>
<td>485</td>
<td>[17]</td>
</tr>
<tr>
<td>PCR-RFLP</td>
<td>F2-campy-235



R2-campy-235</td>
<td>AAT TGA TGG GGT TAG CAT TAG C



CAA CAA TGG CTC ATA TAC AAC TGG</td>
<td>316</td>
<td>[20]</td>
</tr>
<tr>
<td rowspan="2">Ciprofloxacin</td>
<td>MAMA-PCR</td>
<td>CampyMAMAgyrA1



CampyMAMAgyrA5</td>
<td>TTT TTA GCA AAG ATT CTG AT



CAA AGC ATC ATA AAC TGC AA</td>
<td>265</td>
<td>[19]</td>
</tr>
<tr>
<td>MAMA-PCR</td>
<td>GZgyrACcoli3F



CampyMAMAgyrA8</td>
<td>TAT GAG CGA TAT TAT CGG TC



TAA GGC ATC GTA AAC AGC CA</td>
<td>192</td>
<td>[24]</td>
</tr>
<tr>
<td rowspan="3">Tetracycline</td>
<td>tet(O) PCR</td>
<td>DMT 1



DMT 2</td>
<td>GGC GTT TTG TTT ATG TGC G



ATG GAC AAC CCG ACA GAA GC</td>
<td>559</td>
<td>[25]</td>
</tr>
<tr>
<td>tet(A) PCR</td>
<td>Tet(A)-F



Tet(A)-R</td>
<td>GTG AAA CCC AAC ATA CCC C



GAA GGC AAG CAG GAT GTA G</td>
<td>888</td>
<td>[18]</td>
</tr>
<tr>
<td>tet(A) PCR</td>
<td>tet-A-1



tet-A-2</td>
<td>GCT CAC GTT GAC GCA GGA AAG



ATC GTC ATT GTC CGT TAC</td>
<td>486</td>
<td>This study</td>
</tr>
</tbody>
</table>

### DNA sequencing

PCR products obtained by *tet(A)* PCRs were sequenced by cycle sequencing with BigDye Terminator v1.1 Cycle Sequencing Kit (Applied Biosystems, Darmstadt, Germany) according to the instructions of the manufacturer. In addition to the amplification primers of the Tet(A)-F/R fragment tet-A-A (5'-AAT TTT CTT CAA ATA AGG-3') and tet-A-B (5'-GTC ATT CTT ATA TTA AGT GG-3') were used as sequencing primers. Sequencing products were analyzed with an ABI PRISM 3130 genetic analyzer.

### MALDI-TOF mass spectrometry

Cultured bacteria were suspended in 300 µl of bi-distilled water and mixed with 900 ml of ethanol (Carl Roth GmbH). Further treatment of samples and measurement were described by El-Ashker et al..

## Results

### Identification of bacteria

In total, 58 isolates suspected as *Campylobacter* were recovered from faeces and cloacal swabs of chicken flocks in Kenya. After storage in skimmed milk medium and transportation to Germany only 40 of these isolates could be re-cultivated. Four *C. coli* and 31 *C. jejuni* were identified by mPCR (Table 2). Five other isolates were identified by MALDI-TOF mass spectrometry as members of genera *Bacillus*, *Staphylococcus*, *Ochrobactrum* as well as two *Bordetella* isolates. Eighteen skimmed milk tubes contained *Campylobacter* DNA [14 *C. coli*, 3 *C. jejuni* and one sample harboured both *C. coli* and *C. jejuni* (Table 2)].

**Table 2: Results of cultivation and multiplex PCR identification of Campylobacter isolates**

<table>
<thead>
<tr>
<th>Cultivation</th>
<th colspan="3">mPCR identification of Campylobacter</th>
<th>Total</th>
</tr>
<tr>
<th></th>
<th>C. jejuni</th>
<th>C. coli</th>
<th>Not identified / C. coli/C. jejuni</th>
<th>n (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Positive (n)</td>
<td>31</td>
<td>4</td>
<td>5 (Not identified)</td>
<td>40 (69.0)</td>
</tr>
<tr>
<td>Negative (n)</td>
<td>3</td>
<td>14</td>
<td>1 (C. coli/C. jejuni)</td>
<td>18 (31.0)</td>
</tr>
</tbody>
</table>

### Antimicrobial susceptibility profiles and multidrug resistance

The results of antimicrobial susceptibility testing of *C. jejuni* and *C. coli* isolates and the rate of resistance to seven antimicrobial agents are given in Tables 3 and 4, respectively. The *C. jejuni* isolates showed a high rate of resistance to nalidixic acid, tetracycline and ciprofloxacin with 77.4, 71.0 and 71.0%, respectively. Low resistance rates were detected for gentamicin and chloramphenicol, both with 25.8% of the isolates. For the low number of *C. coli* isolates  a similar pattern was observed. Only two isolates were susceptible to all tested antimicrobial agents, one isolate was resistant to all tested antibiotics. The multidrug resistance profiles of 31 *C. jejuni* isolates are shown in Table 5. Multidrug resistance to three or more classes of antibiotics was found in 19 isolates (61.3%) and was observed in a range between 5.3 and 26.3%.

**Table 3: Results of MIC determination and resistance rates of Kenyan *Campylobacter jejuni* isolates**

<table>
<thead>
<tr>
<th>Class</th>
<th>Antibiotic</th>
<th colspan="11">Antibiotic susceptibility of Campylobacter jejuni (n = 31)



MIC (µg/ml)</th>
<th>R (%)</th>
</tr>
<tr>
<th></th>
<th></th>
<th>0.06</th>
<th>0.12</th>
<th>0.25</th>
<th>0.5</th>
<th>1</th>
<th>2</th>
<th>4</th>
<th>8</th>
<th>16</th>
<th>32</th>
<th>64</th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td>Chloramphenicol</td>
<td></td>
<td>6</td>
<td>3</td>
<td>5</td>
<td></td>
<td></td>
<td></td>
<td>8</td>
<td></td>
<td></td>
<td></td>
<td>25.8</td>
</tr>
<tr>
<td>Macrolides</td>
<td>Erythromycin</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>14</td>
<td>12</td>
<td>2</td>
<td></td>
<td>12</td>
<td></td>
<td></td>
<td>51.6</td>
</tr>
<tr>
<td>Fluoroquinolones</td>
<td>Ciprofloxacin</td>
<td></td>
<td>5</td>
<td>2</td>
<td>2</td>
<td>21</td>
<td>1</td>
<td>9</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>71.0</td>
</tr>
<tr>
<td>Quinolones</td>
<td>Nalidixic acid</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>1</td>
<td>4</td>
<td>2</td>
<td></td>
<td>2</td>
<td></td>
<td>22</td>
<td>77.4</td>
</tr>
<tr>
<td>Aminoglycosides</td>
<td>Gentamicin</td>
<td></td>
<td>1</td>
<td>2</td>
<td>5</td>
<td>11</td>
<td>4</td>
<td>2</td>
<td>1</td>
<td>5</td>
<td></td>
<td></td>
<td>25.8</td>
</tr>
<tr>
<td>Aminoglycosides</td>
<td>Streptomycin</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>12</td>
<td>4</td>
<td>12</td>
<td>1</td>
<td>2</td>
<td></td>
<td></td>
<td>41.9</td>
</tr>
<tr>
<td>Tetracyclines</td>
<td>Tetracycline</td>
<td></td>
<td>6</td>
<td>1</td>
<td>2</td>
<td>20</td>
<td></td>
<td>1</td>
<td></td>
<td></td>
<td></td>
<td>1</td>
<td>71.0</td>
</tr>
</tbody>
</table>

**Table 4: Results of MIC determination and resistance rates of Kenyan *Campylobacter coli* isolates**

<table>
<thead>
<tr>
<th>Class</th>
<th>Antibiotic</th>
<th colspan="11">Antibiotic susceptibility of Campylobacter coli (n = 4)



MIC (µg/ml)</th>
<th>R (%)</th>
</tr>
<tr>
<th></th>
<th></th>
<th>0.06</th>
<th>0.12</th>
<th>0.25</th>
<th>0.5</th>
<th>1</th>
<th>2</th>
<th>4</th>
<th>8</th>
<th>16</th>
<th>32</th>
<th>64</th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td>Chloramphenicol</td>
<td></td>
<td></td>
<td>2</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>2</td>
<td></td>
<td></td>
<td></td>
<td>50.0</td>
</tr>
<tr>
<td>Macrolides</td>
<td>Erythromycin</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>4</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>0</td>
</tr>
<tr>
<td>Fluoroquinolones</td>
<td>Ciprofloxacin</td>
<td>1</td>
<td></td>
<td></td>
<td></td>
<td>3</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>75.0</td>
</tr>
<tr>
<td>Quinolones</td>
<td>Nalidixic acid</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>1</td>
<td></td>
<td></td>
<td>3</td>
<td></td>
<td></td>
<td>75.0</td>
</tr>
<tr>
<td>Aminoglycosides</td>
<td>Gentamicin</td>
<td></td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>25.0</td>
</tr>
<tr>
<td>Aminoglycosides</td>
<td>Streptomycin</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>2</td>
<td>2</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>0</td>
</tr>
<tr>
<td>Tetracyclines</td>
<td>Tetracycline</td>
<td></td>
<td>1</td>
<td>1</td>
<td>1</td>
<td></td>
<td>1</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>25.0</td>
</tr>
</tbody>
</table>

**Table 5: Multidrug resistance profiles of 19 Campylobacter jejuni isolates**

<table>
<thead>
<tr>
<th>Antibiotic resistance profile</th>
<th>No. of resistant isolates (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td>TET, CIP, NAL, ERY, GEN, STR, CHL</td>
<td>1 (5.3)</td>
</tr>
<tr>
<td>TET, CIP, NAL, ERY, STR, CHL</td>
<td>1 (5.3)</td>
</tr>
<tr>
<td>TET, CIP, NAL, ERY, GEN, STR</td>
<td>3 (15.8)</td>
</tr>
<tr>
<td>TET, CIP, NAL, GEN, STR, CHL</td>
<td>1 (5.3)</td>
</tr>
<tr>
<td>TET, CIP, NAL, ERY, STR</td>
<td>2 (10.5)</td>
</tr>
<tr>
<td>TET, CIP, NAL, ERY, CHL</td>
<td>1 (5.3)</td>
</tr>
<tr>
<td>TET, CIP, NAL, ERY</td>
<td>2 (10.5)</td>
</tr>
<tr>
<td>TET, CIP, NAL, STR</td>
<td>1 (5.3)</td>
</tr>
<tr>
<td>TET, CIP, NAL</td>
<td>5 (26.3)</td>
</tr>
<tr>
<td>CIP, NAL, ERY, STR, CHL</td>
<td>1 (5.3)</td>
</tr>
<tr>
<td>CIP, NAL, STR</td>
<td>1 (5.3)</td>
</tr>
</tbody>
</table>


*TET tetracycline, CIP ciprofloxacin, NAL nalidixic acid, ERY erythromycin, GEN gentamicin, STR streptomycin, CHL chloramphenicol* 

### Molecular biological detection of antibiotic resistance determinants

DNA of 35 viable *Campylobacter* isolates and of 18 non-growing samples was investigated by PCR to detect antibiotic resistance. Mismatch amplification mutation assay (MAMA-PCR) was used to characterize a *gyrA* gene mutation associated with ciprofloxacin resistance as well as mutations in 23S rRNA genes as cause of erythromycin resistance. The molecular biological detection of resistance to ciprofloxacin in both *C. coli* and *C. jejuni* was also confirmed by change of amino acid 86 from threonine to isoleucine in the *gyrA* gene. Additional to the ciprofloxacin resistant *Campylobacter* isolates (Tables 3, 4), three *Campylobacter* DNAs were detected harbouring the *gyrA* gene mutation. The results were confirmed by PCR-RFLP according to Vacher et al.. Mutations at positions 2074 and 2075 of the 23S rRNA genes in 13 phenotypically erythromycin-resistant isolates could neither be detected by using PCR-RFLP nor MAMA-PCR.

The *tet(O)* gene which is mainly responsible for tetracycline-resistance was detected by PCR in 12 out of 22 resistant *C. jejuni* isolates (54.5%) and in all tetracycline resistant *C. coli* isolates. *tet(O)* gene was not detected in DNA extracted from the non-growing samples. Additionally, a newly developed PCR assay was used for the detection of the *tet(A)* gene. *tet(A)* was detected in 28 out of 31 *C. jejuni* (90.3%) and in all 4 *C. coli* isolates. In 3 out of 14 non-growing samples which harboured *C. coli* DNA, *tet(A)* gene could be found as well as in the one sample where both *C. jejuni* and *C. coli* were detected.

## Discussion

The antimicrobial susceptibility patterns among *Campylobacter* isolates originating from chicken in Kenya were analyzed according to the guidelines of CLSI for Enterobacteriaceae which had been guided by previous reports. Clinical breakpoints for interpretation of MIC values of *C. jejuni* and *C. coli* from chicken are available. In this study a commercially available broth microdilution assay was used for the determination of MIC values for seven antibiotics. The assay already proved to be suitable in previous investigations.

In this study, only 40 out of 58 suspected *Campylobacter* samples could be re-cultivated. The storage conditions (temperature, microaerophilic atmosphere) using skimmed milk medium were possibly not ideal. However, it had been demonstrated that *C. jejuni* can survive up to 14 days at  or 2.5 days at  in sterile skimmed milk. Alternative storage of *Campylobacter* cultures using transport medium (for example Amies medium) or cryovials is recommended for future investigations.

In 53 out of 58 collected samples, *Campylobacter* DNA was identified by mPCR assay. The majority of the cultures proved to be *C. jejuni* (88.6%) which is in agreement with previous studies in chicken. In 18 DNA extracts of non-viable samples, 3 *C. jejuni* (16.7%), one mixed population of *C. jejuni/C. coli* and 14 *C. coli* were identified. These findings are in agreement with those of a previous study that found a longer viability of *C. jejuni* in comparison to *C. coli* in biological milieu. It may be possible that some *C. coli* isolates had been lost during the shipment period. In summary, *C. jejuni* was identified much more often than *C. coli* (64.2%) by mPCR investigation. In agreement with other studies, the findings highlighted the usefulness of mPCR as a reliable, sensitive, time and cost saving method for identification of thermophilic *Campylobacter*.

The antibiotic susceptibility of 35 *Campylobacter* isolates from Kenyan chicken was investigated using European Committee on Antimicrobial Susceptibility Testing and epidemiological cut-off values (EUCAST-ECOFFS). A broth microdilution assay was used as a standardized, easy, and reliable method for the determination of MIC of seven antibiotics. High resistance rates were obtained for ciprofloxacin, nalidixic acid and tetracycline with more than 70% which is in agreement with a recent European Food Safety Authority (EFSA) report. These results are in contrast to those of Brooks et al. who reported resistance rates for *Campylobacter* recovered from humans with diarrhoea in Western Kenya for ciprofloxacin, nalidixic acid and tetracycline with 6, 26 and 18% in 2006, respectively. The general high rates of resistance in the chicken isolates may be caused by availability and uncontrolled use of antibiotics by small farmers. Resistance to chloramphenicol is remarkable with 25.8% in this investigation. Use of chloramphenicol is banned in animal breeding in Europe for more than 20 years, but still it is often used in many third world countries. It is easy to obtain antibiotics over-the-counter and other unregulated venues and injudicious use promotes the development of resistance to antimicrobial agents. Resistance to gentamicin in the isolates obtained from chicken was low in this study (25.8%), but *Campylobacter* isolated from broilers and turkeys were totally susceptible to gentamicin. Erythromycin resistance rates found in this study correspond to those of similar studies elsewhere.

Multidrug resistance was detected in 61.3% of the *Campylobacter* isolates. Eleven different combinations were found (Table 5). Frequent, resistance to ciprofloxacin, nalidixic acid and tetracycline was identified (17 out of 19 multidrug resistant isolates) which is in agreement with previous investigation using Vietnamese *Campylobacter* isolates. However, EFSA reported low level of multidrug resistance in *C. jejuni* from broilers of the member states of the EU.

The emerging of antibiotic resistance has been attributed to the overuse and misuse of antimicrobial agents in both the developed and developing world. Antibiotics are widely used as growth supplements in livestock and to prevent infections. The emerging of multidrug resistance may reflect acquisition of different resistance determinants on the same DNA molecule or single determinants, such as multidrug pumps, that specify efflux activity against different antimicrobial agents. The mechanisms of genetic resistance might be chromosomal or plasmid-borne, and represent a combination of endogenous and acquired genes. In general, mechanisms of antibiotic resistance as modification of the antibiotic by aminoglycoside-modifying enzymes (AphA, AadE, Sat), enzymatic inactivation of the antibiotic by -lactamase and modification of the DNA gyrase target, mutations in 23S rRNA genes were included for aminoglycosides, beta-lactams, fluoroquinolones, macrolides and tetracyclines, respectively. The multidrug efflux pump CmeABC has been involved in the resistance mechanisms of *C. jejuni* and *C. coli* to tetracyclines, fluoroquinolones, macrolides and beta-lactams.

Molecular biological methods were used for detection of antibiotic resistance determinants either using DNA isolated from cultures or that of non-cultured bacteria. All isolates of this study which were resistant to ciprofloxacin carried a mutation of the amino acid 86 of the *gyrA* resulting in a change from threonine to isoleucine. This mutation was detected also in 3 DNA samples extracted from skimmed milk. The MAMA-PCR protocol allowed the detection of the *gyrA* mutation and PCR-RFLP was confirming the mutation from (ACA to ATA) of amino acid 86. This result was in agreement with previous reports showing that both methods are simple, reliable, rapid tools that can be used as screening methods. In *Campylobacter*, resistance to erythromycin is chromosomally encoded by an alteration of the 23S rRNA gene. High level resistance to erythromycin is caused by mutations at position 2074 and/or 2075 of the domain V of this gene. In this study the mutations were neither detected by MAMA-PCR nor by PCR-RFLP.

The *tet(O)* gene is known to be responsible for tetracycline resistance in *Campylobacter* isolates. In this study, only 54.5% of the tetracycline resistant isolates harboured the *tet(O)* gene. The *tet(A)* gene also plays role in resistance to tetracycline. The efflux gene *tet(A)* is coding for an approximately 46 kDa membrane-bound efflux protein for membrane-associated proteins and is involved in the export of tetracycline from the cell. In this study, using the recommended primers for *tet(A)* amplification PCR products of 696 bp instead of 888 bp were obtained. DNA sequencing of amplicons and database search resulted in 99.0% homology to a partial putative integral membrane protein and a putative periplasmic protein. Hence, a new PCR assay based on *tet(A)* gene sequences for *C. jejuni* and *C. coli* was developed. Parameters such as limit of detection, limit of quantification, PCR efficiency and specificity were considered during an in-house validation process. Amplicon length was 486 and the amplicons were sequenced to confirm the identity. The *tet(A)* gene was much more frequently identified in the Kenyan *Campylobacter* isolates than *tet(O)* (35 vs 13).

To the best of our knowledge this is the first report on the status of antibiotic susceptibility of thermophilic *Campylobacter* from chicken in Kenya. High level of resistance to ciprofloxacin, erythromycin and nalidixic acid as well as multidrug resistance was detected previously in Kenya. In Kenya, this problem is reported to be caused by the increasing rate of unregulated over-the-counter sale without prescriptions of these antibiotics, mainly to humans self-treatment of suspected infections and to a lesser extent for use in animals. These findings also demonstrate the potential for antibiotic-resistant bacteria to spread through the food chain from animals treated with antibiotics for humans. Such misuse and overuse may have resulted in the selection of resistant mutants or acquisition of antibiotic resistance genes from other organisms through the process of genetic exchange. It is recommendable that a long-term local surveillance programme is adopted for monitoring changes in resistance among *Campylobacter* isolates. Efforts to establish at least a National Laboratory with facilities for performing phenotyping and genotyping methods is highly recommended. Emphasis should be given on educational advertising to reduce the input of antibiotics in animal breeding to minimize the potential hazard for humans.

## Conclusions

To the best of our knowledge, this study is the first report about resistance to antibiotics in thermophilic *Campylobacter* originating from chicken in Kenya. *Campylobacter* spp. show a high level of resistance to ciprofloxacin, nalidixic acid and tetracycline but also a remarkable one to chloramphenicol and gentamicin and they are multidrug resistant. Resistance to antibiotics is a global public health concern. In Kenya, resistance surveillance needs further attention in the future. Efforts to establish at least a National Laboratory with facilities for performing phenotypic and genotypic characterization of thermophilic *Campylobacter* is highly recommended.

## References

1. Man SM. The clinical importance of emerging Campylobacter species. Nat Rev Gastroenterol Hepatol. 2011;8:669–85. 


2. European Food Safety Authority (EFSA) Manual for reporting of food-borne outbreaks in accordance with Directive 2003/99/EC from the year 2010/2011. European Food Safety Authority (EFSA), Parma. 2011; 138. 


3. Coker AO, Isokpehi RD, Thomas BN, Amisu KO, Obi CL. Human campylobacteriosis in developing countries. Emerg Infect Dis. 2002;8:237–44. 


4. Saidi SM, Iijima Y, Sang WK, Mwangudza AK, Oundo JO, Taga K, et al. Epidemiological study on infectious diarrheal diseases in children in a coastal rural area of Kenya. Microbiol Immunol. 1997;41:773–8. 


5. Brooks JT, Ochieng JB, Kumar L, Okoth G, Shapiro RL, Wells JG, et al. Surveillance for bacterial diarrhea and antimicrobial resistance in rural Western Kenya, 1997–2003. Clin Infect Dis. 2006;43:393–401. 


6. Agunos A, Waddell L, Léger D, Taboada E. A systematic review characterizing on-farm sources of Campylobacter spp. for broiler chickens. PLOS ONE. 2014;9(8):e104905. 


7. Sahin O, Kassem II, Shen Z, Lin J, Rajashekara G, Zhang Q. Campylobacter in poultry: ecology and potential interventions. Avian Dis. 2015;59:185–200. 


8. Hermans D, Pasmans F, Messens W, Martel A, Van Immerseel F, Rasschaert G, et al. Poultry as a host for the zoonotic pathogen Campylobacter jejuni. Vector Borne Zoonotic Dis. 2012;12:89–98 


9. Joyce T, McGuigan KG, Elmore-Meegan M, Conroy RM. Prevalence of enteropathogens in stools of rural Maasai children under five years of age in the Maasailand region of the Kenyan Rift Valley. East Afr Med J. 1996;73:59–62. 


10. Shapiro RL, Kumar L, Phillips-Howard P, Wells JG, Adcock P, Brooks J, et al. Antimicrobial-resistant bacterial diarrhea in rural western Kenya. J Infect Dis. 2001;183:1701–4. 


11. Swierczewski BE, Odundo EA, Koech MC, Ndonye JN, Kirera RK, Odhiambo CP, et al. Surveillance for enteric pathogens in a case-control study of acute diarrhea in Western Kenya. Trans R Soc Trop Med Hyg. 2013;107:83–90. 


12. Ombui JN, Kagiko MM, Arimi SM. Foodborne diseases in Kenya. East Afr Med J. 2001;78:40–4. 


13. Osano O, Arimi SM. Retail poultry and beef as sources of Campylobacter jejuni. East Afr Med J. 1999;76:141–3. 


14. Blaser B, Engberg J. Clinical aspects of Campylobacter jejuni and Campylobacter coli infections. In: Nachamkin I, Szymanski CM, Blaser MJ, editors. Campylobacter. Washington DC: American Society for Microbiology; 2008. p. 99–121. 


15. EFSA: Summary Report on antimicrobial resistance in zoonotic and indicator bacteria from humans, animals and food. EFSA J. 2015;13(2):4036. doi:10.2903/j.efsa.2015.4036. 


16. EUCAST. Breakpoint tables for interpretation of MICs and zone diameters. [http://www.eucast.org](http://www.eucast.org). Version 3.1. 2013. 


17. Ge B, Wang F, Sjölund-Karlsson M, McDermott PF. Antimicrobial resistance in Campylobacter: susceptibility testing methods and resistance trends. J Microbiol Methods. 2013;95:57–67. 


18. Luber P, Bartelt E, Genschow E, Wagner J, Hahn H. Comparison of broth microdilution, E test, and agar dilution methods for antibiotic susceptibility testing of Campylobacter jejuni and Campylobacter coli. J Clin Microbiol. 2003;41:1062–8. 


19. Wang Y, Huang WM, Taylor DE. Cloning and nucleotide sequence of the Campylobacter jejuni gyrA gene and characterization of quinolone resistance mutations. Antimicrob Agents Chemother. 1993;37:457–63. 


20. Alonso R, Mateo E, Girbau C, Churruca E, Martinez I, Fernandez-Astorga A. PCR-restriction fragment length polymorphism assay for detection of gyrA mutations associated with fluoroquinolone resistance in Campylobacter coli. Antimicrob Agents Chemother. 2004;48:4886–8. 


21. Alonso R, Mateo E, Churruca E, Martinez I, Girbau C, Fernandez-Astorga A. MAMA-PCR assay for the detection of point mutations associated with high-level erythromycin resistance in Campylobacter jejuni and Campylobacter coli strains. J Microbiol Methods. 2005;63:99–103. 


22. Abdi-Hachesoo B, Khoshbakht R, Sharifiyazdi H, Tabatabaei M, Hosseinzadeh S, Asasi K. Tetracycline resistance genes in Campylobacter jejuni and C. coli isolated from poultry carcasses. Jundishapur J Microbiol. 2014;7:e12129. 


23. Zirnstein G, Li Y, Swaminathan B, Angulo F. Ciprofloxacin resistance in Campylobacter jejuni isolates: detection of gyrA resistance mutations by mismatch amplification mutation assay PCR and DNA sequence analysis. J Clin Microbiol. 1999;37:3276–80. 


24. Vacher S, Menard A, Bernard E, Megraud F. PCR-restriction fragment length polymorphism analysis for detection of point mutations associated with macrolide resistance in Campylobacter spp. Antimicrob Agents Chemother. 2003;47:1125–8. 


25. International Standards Organization. [ISO] 10272-1. Microbiology of food and animal feeding stuffs—horizontal method for detection and enumeration of Campylobacter spp. Part 1: detection method. 2006. 


26. El-Adawy H, Hotzel H, Tomaso H, Neubauer H, Hafez HM. Elucidation of colonization time and prevalence of thermophilic Campylobacter species during turkey rearing using multiplex polymerase chain reaction. Poult Sci. 2012;91:454–9. 


27. El-Adawy H, Hotzel H, Düpre S, Tomaso H, Neubauer H, Hafez HM. Determination of antimicrobial sensitivities of Campylobacter jejuni isolated from commercial turkey farms in Germany. Avian Dis. 2012;56:685–92. 


28. Zirnstein G, Helsel L, Li Y, Swaminathan B, Besser J. Characterization of gyrA mutations associated with fluoroquinolone resistance in Campylobacter coli by DNA sequence analysis and MAMA-PCR. FEMS Microbiol Lett. 2000;190:1–7. 


29. Mazi W, Senok AC, Al-Mahmeed A, Arzese A, Bindayna K, Botta G. Trends in antibiotic sensitivity pattern and molecular detection of tet(O)-mediated tetracycline resistance in Campylobacter jejuni isolates from human and poultry sources. Jpn J Infect Dis. 2008;61:82–4. 


30. El-Ashker M, Gwida M, Tomaso H, Monecke S, Ehricht R, El-Gohary F, et al. Staphylococci in cattle and buffaloes with mastitis in Dakahlia Governorate. Egypt J Dairy Sci. 2015;98:7450–9. 


31. Clinical and Laboratory Standards Institute (CLSI). Performance standards for antimicrobial susceptibility testing; 20th informal supplement M100-S20. 2010; 30(1). 


32. Hariharan H, Sharma S, Chikweto A, Matthew V, DeAllie C. Antimicrobial drug resistance as determined by the E-test in Campylobacter jejuni, C. coli, and C. lari isolates from the ceca of broiler and layer chickens in Grenada. Comp Immunol Microbiol Infect Dis. 2009;32:21–8. 


33. Luangtongkum T, Morishita TY, El-Tayeb AB, Ison AJ, Zhang Q. Comparison of antimicrobial susceptibility testing of Campylobacter spp. by the agar dilution and the agar disk diffusion methods. J Clin Microbiol. 2007;45:590–4. 


34. Luber P, Wagner J, Hahn H, Bartelt E. Antimicrobial resistance in Campylobacter jejuni and Campylobacter coli strains isolated in 1991 and 2001–2002 from poultry and humans in Berlin, Germany. Antimicrob Agents Chemother. 2003;47:3825–30. 


35. The European Committee on Antimicrobial Susceptibility Testing (EUCAST). Breakpoint tables for interpretation of MICs and zone diameters. Version 4.0. 2014. 


36. Clinical and Laboratory Standards Institute (CLSI). Performance standards for antimicrobial disk and dilution susceptibility tests for bacterial isolates from animals; approved standards—3rd ed. M31-A3 2008; 28(8). 


37. El-Adawy H, Ahmed MFE, Hotzel H, Tomaso H, Tenhagen BA, Hartung J, et al. Antimicrobial susceptibilities of Campylobacter jejuni and Campylobacter coli recovered from organic turkey farms in Germany. Poult Sci. 2015;94:2831–7. 


38. Christopher FM, Smith GC, Vanderzant C. Effect of temperature and pH on the survival of Campylobacter fetus. J Food Prot. 1982;45:253–9. 


39. Doyle MP, Roman DJ. Prevalence and survival of Campylobacter jejuni in unpasteurized milk. Appl Environ Microbiol. 1982;44:1154–8. 


40. Blaser MJ, Hardesty HL, Powers B, Wang WL. Survival of Campylobacter fetus subsp. jejuni in biological milieus. J Clin Microbiol. 1980;11:309–13. 


41. European Food Safety Authority (EFSA)/European Centre for Disease Prevention and Control (ECDC). EU summary report on antimicrobial resistance in zoonotic and indicator bacteria from humans, animals and food in 2013. EFSA J. 2015; 13:4036. 


42. Center for Disease Dynamics, Economics & Policy, 2015. State of the World’s antibiotics. 2015. CDDEP: Washington DC, 38–48. 


43. European Food Safety Authority (EFSA). Scientific opinion on chloramphenicol in food and feed. EFSA J. 2014; 12:3907. 


44. Luangtongkum T, Morishita TY, Ison AJ, Huang S, McDermott PF, Zhang Q. Effect of conventional and organic production practices on the prevalence and antimicrobial resistance of Campylobacter spp. in poultry. Appl Environ Microbiol. 2006;72:3600–7. 


45. Nguyen TNM, Hotzel H, El-Adawy H, Tran HT, Le MTH, Tomaso H, et al. Genotyping and antibiotic resistance of thermophilic Campylobacter isolated from chicken and pig meat in Vietnam. Gut Pathogens. 2016;8:19. 


46. Ventola CL. The antibiotic resistance crisis: part 1: causes and threats. P T. 2015;40:277–83. 


47. Levy SB. Factors impacting on the problem of antibiotic resistance. J Antimicrob Chemother. 2002;49:25–30. 


48. Zhang Q, Plummer P. Mechanisms of antibiotic resistance in Campylobacter. In: Nachamkin I, Szymanski C, Blaser M, editors. Campylobacter, 3rd ed. Washington DC: ASM Press; 2008. p. 263–76. 


49. Iovine NM. Resistance mechanisms in Campylobacter jejuni. Virulence. 2013;4:230–40. 


50. Chopra I, Roberts M. Tetracycline antibiotics: mode of action, applications, molecular biology, and epidemiology of bacterial resistance. Microbiol Mol Rev. 2001;65:232–60. 


51. Kariuki S, Revathi G, Kariuki N, Kiiru J, Mwiruria J, Hart CA. Characterisation of community acquired non-typhoidal Salmonella from bacteraemia and diarrhoeal infection in children admitted to hospital in Nairobi, Kenya. BMC Microbiol. 2006;6:101.