# Prevalence, Virulence, and Antibiotic Susceptibility of *Salmonella* spp. Strains Isolated from Beef in Greater Tunis (Tunisia)

**Walid Oueslati, Mohamed Ridha Rjeibi, Moez Mhadhbi, Mounir Jbeli, Samia Zrelli, Abdelfettah Ettriqui** *Laboratory of Food Microbiology, University Manouba, National School of Veterinary Medicine of Sidi Thabet, Tunisia* 

---

## Abstract

The aim of this work was to investigate the presence of *Salmonella* spp. in 300 beef meat samples collected from cattle carcasses of different categories (young bulls, culled heifers, and culled cows). The detection of *Salmonella* spp. was performed by the alternative VIDAS Easy Salmonella technique and confirmed by PCR using *Salmonella* specific primers. *Salmonella* serotypes were determined by slide agglutination tests. The resistance to 12 antibiotics was determined by the diffusion method on Mueller-Hinton agar antibiotic discs. The overall contamination rate of beef by *Salmonella* spp. was 5.7% (). This rate varied from naught () in bulls' meat to 14% () in culled cows' meat (). The prevalence of *Salmonella* spp. was higher in summer and in cattle with digestive disorders: chronic gastroenteritis (), traumatic peritonitis (), and intestinal obstruction () (). Of the 17 *Salmonella* isolates, 6 serotypes were identified, namely *Salmonella* Montevideo (), *Salmonella* Anatum (), *Salmonella* Minnesota (), *Salmonella* Amsterdam (), *Salmonella* Kentucky (), and *Salmonella* Brandenburg () (). Unlike other serotypes, *S.* Montevideo was present during the whole year except winter. Almost all of the strains () were resistant to at least one of the 12 tested antibiotics. Multidrug-resistance concerned  of the strains, including Amoxicillin (), Tetracycline (), Streptomycin (), and Nalidixic acid (). All the strains were sensitive to the association (Amoxicillin + Clavulanic acid), Cefoxitin, and Ceftazidime. In addition, our study showed that all *Salmonella* strains (17) were positive for invasion gene *invA* and negative for the virulence gene *spvC*. Only one isolate (*S.* Kentucky) harbored the *h-li* virulence gene.

**Keywords:** Salmonella, Beef, Serotype, Antibiotic resistance, *invA*, *spvC*.

---

## 1. Introduction

Salmonellosis is a bacterial disease with a rising prevalence in the cattle industry. It is frequent in calves of one to ten weeks of age, but also reported in dairy and beef cattle. Typical clinical signs of acute *Salmonella* infection include fever associated with diarrhea. Infected animals can be asymptomatic and shed up to one billion *Salmonella* a day in their feces. Salmonellosis is a major food-borne disease worldwide.

The pathogenicity of *Salmonella* depends essentially on its chromosomal and/or plasmatic virulence genes. The chromosomal *invA* gene enables the invasion of epithelial cells. The *h-li* gene is involved in the control of phase change and motility of *Salmonella* and is required for optimal transcription of several genes of invasion. The virulence plasmid *spv* genes play a role in the multiplication of *Salmonella* in its host cell and increase the severity of enteritis. It allows both infection and persistence in extra-intestinal sites and under hostile conditions.

Ciprofloxacin and Cefotaxime are the most commonly used antibiotics for the treatment of invasive *Salmonella* infections in humans. However, multidrug-resistant *Salmonella* strains are becoming a real worldwide threat. Since 2002 in Europe, an emergence of Ciprofloxacin-resistant *Salmonella* spp. isolates has been reported in travelers returning from Northeast and Eastern Africa. In African countries, multidrug-resistant *Salmonella* spp. strains exhibiting resistance to Ciprofloxacin have been reported in several studies. For these reasons, food-borne diseases caused by *Salmonella* spp. are a significant public health concern around the world.

Non-typhoid serovars are increasing in importance as significant pathogens of both humans and animals. Indeed, there are about 17 million cases of acute gastroenteritis or diarrhea annually due to non-typhoid salmonellosis, causing 3 million deaths. Over 1 million cases of salmonellosis are attributed to consumption of contaminated foods (meat, eggs...) each year in the United States of America. The most common serotypes associated with human diseases in the U.S.A. are *S.* Typhimurium, Enteritidis, Newport, Heidelberg, and Javiana. *Salmonella* infections lead to approximately 19,000 cases of hospitalizations and nearly 400 deaths each year in the U.S.A., with an economic burden due to lost work, increased medical costs, and deaths. Undercooked beef is the most involved food in gastroenteritis, after eggs and egg products. In France, 35% of food-borne *Salmonella* infections were attributable to the consumption of undercooked beef mainly in children under fifteen.

In Tunisia, *Salmonella* food-borne infections constitute an emerging public health problem. The most frequently isolated serotypes in 2010 were *S.* Enteritidis (45%) and *S.* Typhimurium (27%). In 2011, two episodes of *Salmonella* food-borne infections occurred in South Tunisia following the consumption of beef; it caused one death and more than 117 clinical cases.

The deep contamination of meat can be related to either clinical or asymptomatic infections. This phenomenon is increased by the slaughtering stress, especially in the absence of respect for water diet and the animals' rest. However, surface meat contamination occurs also during slaughtering from intestinal contents. This fecal contamination is due to poor hygiene during animal slaughtering. There is a high risk of beef meat deep contamination by *Salmonella* spp., especially in sanitary culled animals.

The present work aimed to on the one hand estimate the prevalence of *Salmonella* meat deep contamination in both healthy bulls and culled females (heifers and cows) and on the other hand study the isolated *Salmonella* strains.

---

## 2. Materials and Methods

### 2.1. Samples

The present study was carried out during the four seasons from September 2013 to August 2014; it concerned a total number of 300 beef meat samples designated for human consumption collected in Greater Tunis (Tunisia) from carcasses. During each season, 75 animals (25 bulls, 25 culled heifers, and 25 culled cows) were sampled. Information about age, sex, disease history, and cause of culling of the animals was collected. But, there is no information about the health care provided to the animals in their respective farms. Bulls were healthy on the day of slaughtering but culled females were affected by at least one disease (chronic gastroenteritis, traumatic peritonitis, intestinal obstruction, chronic feet diseases, chronic mastitis...). Disposable sterile bags were used to collect approximately 100 g of meat from the inner parts of the iliopsoas muscle. The meat surface was sterilized and the sample was transferred to a sterile stomacher bag. As recommended by Codex Alimentarius (1987), samples were stored at  until used.

### 2.2. Salmonella spp. VIDAS Detection

The detection of *Salmonella* spp. by alternative VIDAS Easy Salmonella (bioMérieux, SA Marcy l'Étoile, France) technique, was performed in four steps: primary enrichment, selective enrichment, VIDAS, and confirmation. Twenty-five grams of meat sample were homogenized in 225 ml of Buffered Peptone Water. The mixture was incubated at  for 22 h. A volume of 100 µl of media was transferred to 10 ml Salmonella Xpress broth 2 (SX2) and incubated at  for 24 h. One milliliter of the culture was heated for 15 min at  then cooled at room temperature. A volume of 0.5 ml SX2 broth was transferred into a SLM array and deposited in the VIDAS. Confirmation of positive samples was performed by isolation of *Salmonella* from unheated SX2 broth on selective agar (XLD and SS) and identification by urease test followed by API 20E system test according to ISO 6579, 2002 reference technique.

### 2.3. Serotypes of Salmonella Strains

Serotypes of *Salmonella* isolates were determined by slide agglutination tests with specific immune sera against O, H, and Vi *Salmonella* antigens at the National Centre for *Salmonella*, *Shigella* and *Vibrio*, Pasteur Institute of Tunis (Tunisia). The interpretation of the results was performed according to the White-Kauffmann-Le Minor scheme.

### 2.4. Molecular Study

InnuPREP DNA Mini Kit was used for the extraction of *Salmonella* genomic DNA from the selective enrichment SX2 broth. One milliliter of SX2 broth was centrifuged for 10 min at 7500 rpm, the pellet was mixed with a washing solution and proteinase K (). The mixture was heated at  for 10 min then centrifuged at 12,000 rpm for 2 min. The pellet was washed twice and DNA was eluted by centrifugation at 8000 rpm for 1 min then stored at  until used.

The molecular confirmation of *Salmonella* was carried out by performing a PCR detecting a 1 kb DNA fragment, using specific primers: F-5'ACCACGCTCTTTCGTCTGG3' and R-5GAACTGACTACGTAGACGCTC3. The invasion gene *invA* and virulence genes *spvC* and *h-li* respectively have sizes of 275, 669, and 173 bp. All *Salmonella* isolated strains were tested for three invasion and virulence genes (*invA*, *h-li*, and *spvC*) using three specific primer sets:

* *invA*: F-5'TATCGCCACGTTCGGGCAA3' and R-5'TCGCACCGTCAAAGGAACC3'
* *spvC*: F-5'AGCCTCGGCTACTGGTCTTG3' and R-5'CCGCAGCAAGAGTCACCTCA3'
* 
*h-li*: F-5'CGGAAATACCATCAAATA3' and R-5'CCCAAACCCATACTTACTCTG3'.



The PCR mixture consisted of 2.5 µl of 10x PCR buffer, 3 mM , 0.4 mM of each dNTP, 25 µM of each primer, 1.25 U Taq Polymerase, sterile distilled water, and 2.5 µl of DNA template. The DNA amplification was performed using the following program: 1 min denaturation at  followed by 35 cycles ( for 20 s,  for 20 s, and  for 2 min) and a final extension at  for 4 min. Amplification was performed in a thermocycler. PCR products were electrophorized in 1.5% agarose gel and visualized with ultraviolet transillumination after ethidium bromide staining.

### 2.5. Antibiotic Resistance

Resistance test to 12 antibiotics was screened by the diffusion method on Mueller-Hinton agar antibiotic discs. Interpretation of the results was done according to the Clinical and Laboratory of Standards Institute criteria.

### 2.6. Statistical Analyses

The infection prevalence percentages were compared using Epi Info 6 with Mantel Haenszel Chi-square test. Observed differences were considered significant when the p value was lower than 0.05.

---

## 3. Results

### 3.1. Prevalence of Salmonella spp. in Beef

The deep meat contamination prevalence by *Salmonella* spp. was 5.7% (). The 17 isolates were positive to PCR using *Salmonella* specific primers (Fig. 1). This rate varies from naught () for bulls' meat to 14% () for culled cows' meat (). *Salmonella* spp. beef meat prevalence was higher during the summer season (13.3%; ) compared to winter (1.3%; 1/75) ().

A total number of 6 serotypes were identified, namely *S.* Montevideo (), *S.* Anatum (), *S.* Minnesota (), *S.* Amsterdam (), *S.* Kentucky (), and *S.* Brandenburg () () (Table 1). Unlike other serotypes, *S.* Montevideo was present during the whole year except winter. It showed a prevalence peak during the summer season (6.7%; ) () (Fig. 1).

Digestive disorders (chronic gastroenteritis (), traumatic peritonitis (), and intestinal obstruction ()) were significantly related to *Salmonella* spp. meat contamination () (Table 1). Moreover, chronic feet diseases () and chronic mastitis () were factors for deep meat *Salmonella* spp. contamination. Prevalence of *Salmonella* contamination was significantly higher in culled cows' meat with digestive disorders (23.5%; ) compared to cows with chronic feet diseases (12.1%; ) and cows with chronic mastitis (6.1%; ) ().

<table border="1">
<caption><strong>Table 1</strong>



Characteristics of Salmonella spp. positive animals and corresponding virulence and antibiotic resistance profile.</caption>
<thead>
<tr>
<th rowspan="2">Serotype</th>
<th rowspan="2">Animals' ages (years)</th>
<th rowspan="2">Culling motif</th>
<th colspan="3">Virulence gene</th>
<th colspan="12">Antibiotic resistance profile</th>
</tr>
<tr>
<th>invA</th>
<th>spvC</th>
<th>h-li</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
<th>9</th>
<th>10</th>
<th>11</th>
<th>12</th>
</tr>
</thead>
<tbody>
<tr>
<td>Salmonella Montevideo</td>
<td>5</td>
<td>Chronic gastroenteritis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>6</td>
<td>Chronic gastroenteritis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>6</td>
<td>Chronic gastroenteritis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>3</td>
<td>Digestive occlusive syndrome</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td></td>
<td>6</td>
<td>Chronic lameness</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>7</td>
<td>Chronic lameness</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>7</td>
<td>Chronic lameness</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>10</td>
<td>Chronic mastitis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td>Salmonella Anatum</td>
<td>3</td>
<td>Chronic gastroenteritis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>6</td>
<td>Traumatic peritonitis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>6</td>
<td>Chronic lameness</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>R</td>
</tr>
<tr>
<td>Salmonella Minnesota</td>
<td>6</td>
<td>Chronic gastroenteritis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td></td>
<td>6</td>
<td>Traumatic peritonitis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>Salmonella Amsterdam</td>
<td>3</td>
<td>Traumatic peritonitis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>-</td>
<td>R</td>
</tr>
<tr>
<td></td>
<td>6</td>
<td>Digestive occlusive syndrome</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>R</td>
<td>-</td>
<td>R</td>
</tr>
<tr>
<td>Salmonella Kentucky</td>
<td>5</td>
<td>Chronic gastroenteritis</td>
<td>+</td>
<td>-</td>
<td>+</td>
<td>R</td>
<td>S</td>
<td>R</td>
<td>R</td>
<td>S</td>
<td>S</td>
<td>R</td>
<td>R</td>
<td>R</td>
<td>R</td>
<td>R</td>
<td>-</td>
</tr>
<tr>
<td>Salmonella Brandenburg</td>
<td>9</td>
<td>Chronic mastitis</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>S</td>
<td>S</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
</tbody>
</table>
<small>S: Sensitive; R: Resistant. 




1: Amoxicillin; 2: Amoxicillin + Clavulanic acid; 3: Chloramphenicol; 4: Cephalothin; 5: Cefoxitin; 6: Ceftazidime; 7: Ciprofloxacin; 8: Gentamicin; 9: Kanamycin; 10: Nalidixic acid; 11: Streptomycin; 12: Tetracycline. </small>

### 3.2. Prevalence of Pathogenic Genes and Antibiotic Resistance

*Salmonella* strains (17) were positive for invasion gene *invA* and negative for the virulence gene *spvC*. Only one isolate (*S.* Kentucky) serotype was positive for *h-li* gene (Table 1, Fig. 3). Almost all () of the strains were resistant to at least one of the antibiotics. Multi-resistance concerned  of the strains, including Amoxicillin (), Tetracycline (), Streptomycin (), and Nalidixic acid (). Furthermore, all strains were sensitive to the association (Amoxicillin + Clavulanic acid), Cefoxitin, and Ceftazidime (Table 1).

---

## 4. Discussion

The results of our study showed that *Salmonella* spp. beef contamination rate was 5.7% (). It varies from naught () in bulls' meat to 14% () in culled cows' meat (). Our results were comparable to those of Ben Jaafar et al. (2002) who found a prevalence of 4.2% in Tunisian beef meat catering. This rate was lower to the results of Derouiche (2001) in Tunisia (9.3%). This difference could be explained by the fact that Derouiche collected samples from culled cattle.

Our study revealed that the contamination prevalence of the culled cows' meat (14%) was significantly higher than that of culled heifers and bulls' meat (3% and 0% respectively) (). This is likely due to the high prevalence of latent carriage in culled cows, immunodepression in some animals, the absence of water diet, and fatigue before slaughtering.

Seasonal fluctuation of *Salmonella* spp. beef meat contamination showed a significant difference between the prevalence of *Salmonella* spp. during the summer season (13.3%; ) and winter (1.3%; ) (). Our results were comparable to those of Ayaz et al. (2010) who found a higher prevalence of *Salmonella* spp. in Turkish beef meat in summer than in other seasons. In Nepal, Maharjan et al. (2006) indicated that the seasonal prevalence of *Salmonella* spp. in beef meat was highest in the months of April and May (summer season).

Of the 17 *Salmonella* isolates, 6 serotypes were identified, namely *S.* Montevideo, *S.* Anatum, *S.* Minnesota, *S.* Amsterdam, *S.* Kentucky, and *S.* Brandenburg. Seasonal distribution of *Salmonella* spp. serotypes indicated that unlike other serotypes, *S.* Montevideo was present during the whole year except winter and peaking during the summer season (6.7%; ) (). *S.* Montevideo is frequently isolated from diarrhea cattle in France. However, our results do not agree with those of Derouiche (2001) and Ben Jaafar et al. (2002) who revealed the predominance of *S.* Anatum (40%) followed by *S.* Corvallis. Our study did not reveal the presence of the two most frequent serotypes involved in collective food poisoning namely *S.* Enteritidis and *S.* Typhimurium.

Digestive disorders were the main cattle disease history. This is explained by the fact that digestive tract disturbances increase *Salmonella* absorption from the intestinal lumen. Our study showed that all studied *Salmonella* strains () were positive for invasion gene *invA* and negative for *spvC* virulence gene. Only one isolate (*S.* Kentucky) was positive for *h-li* gene. Our results were similar to those of Karraouan et al. (2010) who indicated that all strains () of *Salmonella* were positive for the invasion gene *invA*. The results of our study about the presence of the gene *spvC* are not consistent with those of Abouzeed et al. (2000) . Our results were similar to those of Turki et al. (2012) who indicated that all strains () of *S.* Kentucky isolated from different sources were negative for the *spvC* gene.

As it was reported in several countries, we noticed a high multidrug-resistance rate in *Salmonella* isolated from beef. Our study indicated that almost all () of the strains were resistant to at least one of the 12 tested antibiotics. Multidrug-resistant *Salmonella* concerned  of the strains. In Tunisian human medicine practice, an antibacterial treatment is not indicated for uncomplicated forms of *Salmonella* gastroenteritis. Quinolone therapy is reserved to clinically severe cases. Our results are similar to those from the study of Karraouan et al. (2010) which studied antibiotic resistance of 39 *Salmonella* isolates from turkey meat in Morocco. The rates of resistance against Amoxicillin and Tetracycline were high in Tunisian hospitals (54.5 and 40% respectively). *S.* Kentucky was the most resistant serotype ().

---

## 5. Conclusion

Our study revealed a low prevalence of *Salmonella* spp. meat contamination. This rate varied from naught for bulls meat to 14% for culled cows' meat. In addition, the prevalence of *Salmonella* spp. was higher in summer and in cattle with digestive disorders. Six serotypes of *Salmonella* were identified; *S.* Montevideo was the most frequent, followed by *S.* Anatum, *S.* Minnesota, *S.* Amsterdam, *S.* Kentucky, and *S.* Brandenburg. Unlike other serotypes, *S.* Montevideo was present during the whole year except winter. All *Salmonella* strains were positive for invasion gene *invA* and negative for *spvC* virulence gene. Only one (*S.* Kentucky) was positive for *h-li* gene. Almost all of the strains were resistant to at least one antibiotic. Multidrug-resistant *Salmonella* concerned many serotypes, including Amoxicillin, Tetracycline, Streptomycin, and Nalidixic acid. *S.* Kentucky was the most resistant serotype. The association of multidrug-resistance and invasion gene in *Salmonella* isolates justifies the need of surveillance system for both human and animal health sectors.