Moawad _et al. Gut Pathog (2017) 9:57_ Page 6 of 13


**Table 5 Breakpoint values of antimicrobial agents according to CLSI, 2011 and phenotypic antimicrobial susceptibility**
**profiles of** _**E. coli**_ **isolates used in this study**


**Antibiotic class** **Antimicrobial agent** **Conc. (μg)** **Zone diameter (9)** _**E. coli**_ **(21)** **Resistance rate**



**Poultry**
**(15)**



**Beef (6)** **Total (21)** **Poultry** **Beef** **Total**



**R** **I** **S** **R** **I** **S** **R** **I** **S** **R** **I** **S** **%** **%** **%**



Penicillin Amoxycillin–clavulanic acid 20 ≤ 13 14–17 ≥ 18 10 4 1 3 3 - 13 7 1 66.7 50.0 61.9

Ampicillin 10 ≤ 13 14–16 ≥ 17 12 2 1 3 3           - 15 5 1 80.0 50.0 71.4

Cephalosporin Cefotaxime 30 ≤ 22 23–25 ≥ 26 6 7 2 1 3 2 7 10 4 40.0 16.7 33.3



Ampicillin 10 ≤ 13 14–16 ≥ 17 12 2 1 3 3 - 15 5 1 80.0 50.0 71.4

Cefotaxime 30 ≤ 22 23–25 ≥ 26 6 7 2 1 3 2 7 10 4 40.0 16.7 33.3

Cefpodoxime 20 ≤ 17 18–20 ≥ 21 3 12 - 2 4 - 5 16 - 20.0 33.3 23.8

Ceftazidime 30 ≤ 17 18–20 ≥ 21 5 9 1 - 6 - 5 15 1 33.3 0 23.8

Ceftriaxone 30 ≤ 19 20–22 ≥ 23 3 5 7 1 3 2 4 8 9 20.0 16.7 19.0



Miscellaneous Chloramphenicol 30 ≤ 12 13–17 ≥ 18 3 9 3 1 1 4 4 10 7 20.0 16.7 19.0

Colistin 10 ≤ 11 12–13 ≥ 14           - 1 14           -           - 6           - 1 20           -           -           
Fluoroquinolone Ciprofloxacin 5 ≤ 15 16–20 ≥ 21 4 3 8 2 2 2 6 5 10 26.7 33.3 28.6



Colistin 10 ≤ 11 12–13 ≥ 14 - 1 14 - - 6 - 1 20 - - 


Enrofloxacin 5 ≤ 17 18–20 ≥ 21 2 5 8 1 4 1 3 9 9 13.3 16.7 14.3

Nalidixic acid 30 ≤ 13 14–18 ≥ 19 5 12 3 1 2 3 6 14 6 33.3 16.7 28.6



Aminoglycoside Streptomycin 10 ≤ 11 12–14 ≥ 15 9 5 1 4 2 - 13 7 1 60.0 66.7 61.9

Tetracycline Tetracycline 30 ≤ 11 12–14 ≥ 15 12 3 - 5 1 - 17 4 - 80.0 83.3 81.0

Sulphonamide Trimethoprim/sulphamethoxazole 25 ≤ 4 3 ≥ 2 10 2 3 3 - 3 17 2 6 66.7 50.0 81.0



_S_ sensitive, _I_ intermediate, _R_ resistant


break points according to Clinical and Laboratory Standards Institute [21].
The antimicrobial susceptibility of colistin was determined by disc diffusion susceptibility testing using
colistin discs (Oxoid) containing 10 μg. The disc zone
diameters was interpreted according to previous report

[22].


**Molecular detection of antimicrobial resistance associated**
**genes**
_Escherichia coli_ and _S. enterica_ isolates were tested for
β-lactamase-encoding genes _bla_ TEM, _bla_ SHV, _bla_ CTX-M,
_bla_ OXA and _bla_ CMY by PCR using universal primers for
the corresponding gene families (Table 3) as described
previously [23]. PCR amplification was also used for
screening of plasmid-mediated quinolone resistance
genes _qnr_ A, _qnr_ B and _qnr_ S.
For detection of variants of _bla_ TEM, _bla_ SHV, _bla_ CTX-M,
_bla_ OXA and _bla_ CMY amplified PCR fragments were purified from agarose gels using Nucleospin Gel Extraction
Kit (Macherey–Nagel, Düren, Germany). The products
were sequenced and the sequencing results were analyzed
[using BLAST (http://blast.ncbi.nlm.nih.gov/blast.cgi).](http://blast.ncbi.nlm.nih.gov/blast.cgi)
Colistin resistance associated gene _mcr_ -1was identified using a PCR assay (Table 3) as described by Liu et al.

[24]. PCR for detection of carbapenemases _bla_ OXA-23
gene was performed using OXA-23-F and OXA-23-R



(Table 3). PCR reaction and conditions were performed
according to Braun et al. [25].


**Detection and sequencing of class 1 and class 2 integrons**
PCR assays for detection of class 1 and class 2 integrons
were performed using primers given in Table 3 and
yielded PCR fragments that were purified from agarose
gels using Nucleospin Gel Extraction Kit (Macherey–
Nagel, Düren, Germany). The products were sequenced
in Genome Centre, Gifu University, Japan. The sequenc[ing results were analyzed using BLAST (http://blast.ncbi.](http://blast.ncbi.nlm.nih.gov/blast.cgi)
[nlm.nih.gov/blast.cgi).](http://blast.ncbi.nlm.nih.gov/blast.cgi)


**Results**
Out of 180 examined samples 15 _S. enterica_ (8.3%) (7
from poultry and 8 from beef) (Table 1) as well as 21 _E._
_coli_ (11.7%) (15 from poultry and 6 from beef) (Table 2)
were isolated and identified using bacteriological methods and biochemical characterization.


**Serological characterization of** _**S. enterica**_ **and** _**E. coli**_
Fifteen isolated _S. enterica_ were typed as 9 _S. enterica_
serovar Typhimurium (60.0%; 3 from chicken and 6 from
beef), 3 _S. enterica_ Paratyphi A (20.0%; 2 from chickens
and 1 from beef), 2 _S. enterica_ serovar Enteritidis (13.3%;
1 from chicken and 1 from beef) and 1 _S. enterica_ serovar
Kentucky (6.7%) from chicken sample (Table 1).


