the prevalence of Salmonella all over the world, but most common are Salmonella Enteritidis, Salmonella Typhimurium, and Salmonella Pullorum (Batista et al., 2015). For example, S. Senftenberg has been proved to be the dominant serovar in cooked meat products in Chinese Henan province (Yu, Jiang, Zhou, Wu, & Wu, 2014). The various serotypes have different virulence characteristics. Serotype Enteritidis is mainly transmitted through contaminated meat and egg products, which is the primary cause of human salmonellosis (Foley et al., 2011). This serotype was also the most widespread in our study (Figure 1). Serotype Typhi is defined as often being resistant to some first-line recommended antibiotics (Hardjo Lugito & Cucunawangsih, 2017). There are 11 serovars found in our Salmonella isolates. These serovars have also been frequently isolated from different sources in China (Gong et al., 2014). S. Pullorum, which formed a large proportion of the isolates, are similar to strains isolated from gastroenteritis (Luo, Yi, Yao, Zhu, & Qin, 2017). As Salmonella serovars are closely related to pathogenicity, it is necessary to monitor changes in their distribution.

Molecular subtyping is widely used to analyze genetic diversity. MLST divided the Salmonella isolates into three clonal groups from different sources, which differed in all seven alleles profiled. We found that diverse STs mean that the isolates originated from more than one common ancestor and possess several distantly related STs. In the database, our STs were first isolated from diverse sources. For example, ST31, ST39, and ST40 were from pig farms, pig slaughterhouses, retail markets, and humans in China (Zheng et al., 2017). ST533, ST813, and ST640 were from clinics in Ethiopia and France. Our MLST results were used for analyzing correlations among the Salmonella strains. ERIC-PCR fingerprinting has also been used to confirm epidemiological relationships between various Salmonella isolates (Li, Liu, et al., 2017). In Korea, researchers claimed that ERIC-PCR combined with virulence profiling offered a rapid approach to characterize antimicrobial-resistant Salmonella (Jin & Lee, 2017). We did not find a significant association between ERIC-PCR clusters, source, and other characteristics. However, isolates in the last two MLST and ERIC-PCR were similar. In general, MLST and ERIC-PCR are useful as phylogenetic tools for investigations outbreaks of this pathogen.

## 5 | CONCLUSION

Salmonella enterica food poisoning due to egg consumption is a longstanding problem. Our study is the first to comprehensively analyze the prevalence, antibiotic resistance, serotype, and molecular subtype of strains isolated from Guangdong eggs, which are an important local food of native. The results showed contamination ratio of Salmonella was 5.40%. Antibiotic resistance was widespread, especially for  $ \beta $ -lactam antibiotics. The isolates belonged to multiple serovars. MLST and ERIC-PCR typing showed genetic similarity and diversity within those isolates. These results provide useful information to improve strategies to control and treat Salmonella infections.

## ACKNOWLEDGMENTS

Our work was supported by Guangdong Forestry Science and Technology Innovation Project (2016KJCX037) and Quality Engineering Training Base Project of Guangdong Eco-engineering Polytechnic (2017ZLGC-XJ-GZS03).

## CONFLICT OF INTEREST

The authors declare that they do not have any conflict of interest.

## AUTHOR CONTRIBUTIONS

Tengfei Xie, Gang Wu are the common first authors, finished this article experiment and write the article together. Jing Zhao (Corresponding Author) give the idea and experiments support. Xujun He, Zengzhe Lai, Huatong Zhang help to finish the experiment on article.

## ETHICAL STATEMENT

Ethical Review: This study does not involve any human or animal testing.

## I NFORMED CONSENT

Written informed consent was obtained from all study participants.

## ORCID

Jing Zhao https://orcid.org/0000-0003-2032-4371

## REFERENCES

Abbott, S. L., Ni, F. C., & Janda, J. M. (2012). Increase in extraintestinal infections caused by Salmonella enterica subspecies II–IV. Emerging Infectious Diseases, 18, 637–639.

AI, M. E. P. E. (2004). Salmonella Enteritidis infections, United States, 1985–1999 – Volume 10, Number 1–January 2004 – Emerging Infectious Disease journal – CDC. Emerging Infectious Diseases, 10, 1–7.

Andrews, J. R., & Ryan, E. T. (2015). Diagnostics for invasive Salmonella infections: Current challenges and future directions. Vaccine, 33, C8–C15. https://doi.org/10.1016/j.vaccine.2015.02.030

Badouei, M. A., Ghalejooghi, B. M., & Madadgar, O. (2012). Study on Salmonella contamination of traditionally produced edible poultry eggs. Comparative Clinical Pathology, 21, 1093–1097. https://doi.org/10.1007/s00580-011-1238-z

Batista, D. F. A., Freitas Neto, O. C., Barrow, P. A., de Oliveira, M. T., Almeida, A. M., Ferraudo, A. S., & Berchieri Jr., A. (2015). Identification and characterization of regions of difference between the Salmonella Gallinarum biovar Gallinarum and the Salmonella Gallinarum biovar Pullorum genomes. Infection Genetics and Evolution, 30, 74–81. https://doi.org/10.1016/j.meegid.2014.12.007

Ben, S. R., Abbassi, M. S., García, V., Garcíafierro, R., Fernández, J., Kilani, H., … Rodicio, M. R. (2017). Antimicrobial drug resistance and genetic properties of Salmonella enterica serotype Enteritidis circulating