Gharieb _et al. Gut Pathog (2015) 7:34_ Page 10 of 11


**Table 4 Primers’ sequences and amplification’s conditions used for detection of** _**Salmonella**_ **virulence genes**



**Target** **Primers’ sequences 5′–3′** **Amplified**
**gene** **segment**
**(bp)**



**Primary**
**denaturation**



**Amplification (35 cycles)** **Final** **References**
**extension**

**Secondary** **Annealing Extension**
**denaturation**



_pef_ A TGTTTCCGGGCTTGTGCT 700 94 °C 10 min 94 °C 45 s 55 °C 45 s 72 °C 45 s 72 °C 10 min [4]

CAGGGCATTTGCTGATTCTTCC

_hil_ A CGGAAGCTTATTTGCGCCATGCT GAGGTAG 854 94 °C 10 min 94 °C 45 s 60 °C 45 s 72 °C 45 s 72 °C 10 min [37]

GCATGGATCCCCGCCGGCGAGAT TGTG

_sop_ B TCAGAAGRCGTCTAACCACTC 517 94 °C 5 min 94 °C 30 s 58 °C 30 s 72 °C 30 s 72 °C 7 min [38]

TACCGTCCTCATGCACACTC

_stn_ TTGTGTCGCTATCACTGGCAACC 617 94 °C 10 min 94 °C 45 s 59 °C 45 s 72 °C 45 s 72 °C 10 min [4]

ATTCGTAACCCGCTCTCGTCC



3′CS- AAG CAG ACT TGA CCT GA [30]. Chromosomal
DNA extraction, amplification and gel analysis were performed as mentioned before except to amplify the DNA
in the thermal cycler we used a three-step profile (cycling
conditions): 1 min of denaturation at 94 °C, 1 min of
annealing at 55 °C, and 5 min of extension at 72 °C for a
total of 35 cycles. Five seconds were added to the extension time at each cycle. Amplified products with variable
sizes were purified using QIA quick PCR Product extraction kit (Qiagen, Valencia). Each purified amplicon was
sequenced in both forward and reverse directions using
the amplification primers. DNA sequences were obtained
by Applied Biosystems 3130 genetic analyzer (HITACHI,
Japan).Nucleotide sequence analysis was compared with
published GenBank DNA sequences using NCBI BLAST
program [(http://www.ncbi.nlm.nih.gov/BLAST/)](http://www.ncbi.nlm.nih.gov/BLAST/) and
alignment was performed usingMEGA6 program.


**Nucleotide sequence accession numbers**
Nucleotide sequences from this study were deposited
into the GenBank under accession numbers [KT427378,
KT449569, KT449570, KT449571, KT581253, KT581254,
KT581255, KT581256, KT581257 and KT598359].


**Authors’ contributions**
RMG, YHT and MHE participated in the design of the study, collected and analyzed the samples. YHT participated in the sequence alignment. RMG and YHT
drafted the manuscript. All authors read and approved the final manuscript.


**Author details**
1 Depatment of Zoonoses, Faculty of Veterinary Medicine, Zagazig University,
44511 Zagazig, Egypt. [2] Department of Bacteriology, Mycology and Immunology, Faculty of Veterinary Medicine, Zagazig University, 44511 Zagazig, Egypt.
3 Department of Veterinary Public Health, Faculty of Veterinary Medicine,
Zagazig University, 44511 Zagazig, Egypt.


**Aknowledgements**
We are grateful to thank all administrators and clinicians in Al-Ahrar General
Hospital, Zagazig, Egypt for their assistance in collecting stool samples.


**Competing interests**
The authors declare that they have no competing interests.



Received: 17 September 2015  Accepted: 10 December 2015


**References**
1. Majowicz SE, Musto J, Scallan E, Angulo FJ, Kirk M, O’Brien SJ, et al. The
global burden of non-typhoidal _Salmonella_ gastroenteritis. Clin Infect Dis.
2010;50:882–9.
2. EFSA (European Food Safety Authority). The community summary report
on trends and sources of zoonoses and zoonotic agents in the European
[Union in 2007. EFSA J. 2009;223:312. doi:10.2903/j.efsa.2009.223r.](http://dx.doi.org/10.2903/j.efsa.2009.223r)
3. Lostroh CP, Bajaj V, Lee CA. The cis requirements for transcriptional activation by HilA, a virulence determinant encoded on SPI-1. Mol Microbiol.
2000;37:300–15.
4. Murugkar HV, Rahman H, Dutta PK. Distribution of virulence genes in
_Salmonella_ serovars isolated from man and animals. Indian J Med Res.
2003;117:66–70.
5. Van Asten AJ, van Dijk JE. Distribution of classic virulence factors among
_Salmonella_ spp. FEMS Immunol Med Microbiol. 2005;44:251–9.
6. Threlfall EJ. Antimicrobial drug resistance in _Salmonella_ : problems and
perspectives in food- and water-borne infections. FEMS Microbiol Rev.
2002;26:141–8.
7. Chu C, Chiu CH. Evolution of the virulence plasmids of non-typhoidal
_Salmonella_ and its association with antimicrobial resistance. Microbes
Infect. 2006;8(7):1931–6.
8. Rowe-Magnus DA, Mazel D. The role of integrons in antibiotic resistance
gene capture. Int J Med Microbiol. 2002;292(2):115–25.
9. Maharjan M, Joshi V, Joshi DD, Manandhar P. Prevalence of _Salmo-_
_nella_ species in various raw meat samples of a local market in Kathmandu. Trends in the study of disease agents. Ann N Y Acad Sci.
2006;1081:249–56.
10. Arsenault J, Letellier A, Quessy S, Boulianne M. Prevalence and risk factors
for _Salmonella_ and _Campylobacter_ spp. carcass contamination in broiler
chickens slaughtered in Quebec, Canada. J Food Prot. 2007;70(8):1820–8.
11. Kaushik P, Anjay Kumari S, Bharti SK, Dayal S. Isolation and prevalence
of _Salmonella_ from chicken meat and cattle milk collected from local
markets of Patna, India. Vet World. 2014;7(2):62–5.
12. Murugkar HV, Rahman H, Kumar A, Bhattacharyya D. Isolation, phage typing & antibiogram of _Salmonella_ from man and animals in northeastern
India. Indian J Med Res. 2005;122:237–42.
13. Vo ATT, Duijkeren EV, Fluit AC, Wannet WJB, Verbruggen AJ, Maas HME,
et al. Antibiotic resistance, integrons and _Salmonella_ genomic island
1 among non-typhoidal _Salmonella_ serovars in the Netherlands. Int J
Antimicrob Agents. 2006;28(3):172–9.
14. Dione MM, Ikumapayi U, Saha D, Mohammed NI, Adegbola RA, Geerts
S, et al. Antimicrobial resistance and virulence genes of non-typhoidal
_Salmonella_ isolates in the Gambia and Senegal. J Infect Dev Ctries.
2011;5(11):765–75.


