228 Eur J Clin Microbiol Infect Dis (2016) 35:227–234



increase of rebellious human infections due to resistant bacteria. The association between deficiency in new marketed antibiotics and accelerated evolution of bacterial resistance in
both animals and humans represents a real Bemergency^, and
is more considered as an ecosystem problem, especially for
commensal microfloras [5].
Many studies have demonstrated that some APEC strains
could belong to the same clones as human extraintestinal pathogenic ones (ExPEC), and may act as potential pathogens for
humans as they often possess similar virulence genes [6–9].
However, few studies have been investigating the relationship
between avian non-clinical bacteria and human pathogenic
ones.
The present study was designed to assess in detail rates,
patterns, and mechanisms of antibiotic resistance in chicken
commensal, human commensal, and pathogenic E. coli strains
in the Constantine region (North-East of Algeria), and to evaluate the connection between them.


Material and methods


Chicken and chicken farmers’ commensal isolates


Fourteen independent farms keeping chicken flocks (A to N,
aged 35 days or more) from various regions of Constantine
were randomly enrolled from March 2011 through December
2012. On each farm, a mixture of 20 fresh poultry droppings
was collected as well as a stool sample from a consenting
chicken farmer. At the laboratory, samples were diluted and
cultured on MacConkey agar (BioMerieux, France).
Suspected colonies were identified using the API 20E system
(BioMérieux) and from each sample five E. coli colonies were
arbitrarily selected for antibiotic susceptibility testing.
The following information was also recorded: flock size,
age, health status and antibiotics used for birds, other animals
kept at the farm, recent hospital stay and antibiotics taken by
poultry farmers, their family members, or their companion
animals during the last 3 months.


Human pathogenic isolates


We selected 94 (consecutive non-repetitive) human pathogenic E. coli isolates obtained from various clinical samples and
collected from different wards of the Military Hospital of
Constantine from March 2011 to December 2012.
Epidemiological and clinical data were collected for each
patient.


Antimicrobial susceptibility testing and ESBL detection


All isolates were subjected to susceptibility testing against 31
selected antibiotics important in clinical practice by the disc



diffusion method. Results were interpreted according to the
[SFM/EUCAST recommendations [http://www.sfm-](http://www.sfm-microbiologie.org/)
[microbiologie.org]. ESBL-production was screened by the](http://www.sfm-microbiologie.org/)
double-disk synergy test and fluoroquinolones-resistant isolates were selected for further molecular analyses at the
INSERM U1047 (France).


Characterization of β-lactamase-encoding and PMQR
genes


Plasmid or chromosomal DNAwas extracted from the isolates
using the EZ1 DNA Tissue kit on the BioRobot EZ1 extraction platform (QIAGEN, France). Genes encoding for the
most clinically prevalent carbapenemases (blaKPC, blaOXA
48-like [,][ bla] VIM [,][ bla] IMP [, and][ bla] NDM [) and ESBLs (][bla] TEM [,]
blaSHV, and blaCTX-M) were detected by PCR and confirmed
by sequencing the PCR products [10–12]. Similarly, plasmidmediated quinolone resistance (PMQR) genes (qnrA, qnrB,
qnrS, qepA, aac(6’)-Ib-cr, and oqxAB) were screened and confirmed (by PCR and sequencing) in all the quinolone-resistant
isolates [13–16].


Phylotyping and O25b subgrouping


Phylogenetic groups of the E. coli isolates and the O25b subgroup of the ESBL-producers were determined by PCR as
previously described [17, 18].


Clonality analysis


The genetic relationship of all the human and avian
ciprofloxacin-resistant isolates was studied using Rep-PCR
(DiversiLab system, BioMérieux). Isolates with identical patterns were considered indistinguishable if their similarity percentage was≥95 %.


MLST


Multilocus sequence typing (MLST) was carried out by PCR
amplification and sequencing of eight housekeeping genes:
dinB, icdA, pabB, polB, putP, trpA, trpB, and uidA. Allelic
profile and sequence type determinations were assigned ac[cording to the Institut Pasteur’s MLSTweb site scheme (www.](http://www.pasteur.fr/mlst)
[pasteur.fr/mlst).](http://www.pasteur.fr/mlst)


Statistical analysis


Comparisons of antimicrobial resistance rates distribution and
the resistance genes characteristics of the different E. coli isolated from poultry (P), their farmers (F) and clinical samples
(C) were performed by calculating Fisher’s exact test (at 95 %
CI and p <0.05 %) using SAS/ETS® version 9.1 (SAS
Institute Inc., Cary, NC, USA, 2004).


