|Col1|Col2|Col3|Col4|Col5|
|---|---|---|---|---|
||||||



the United States. Furthermore, some serovars are also more dan
gerous. For example, _S. enterica_ serotype Typhimurium, which can

cause various symptoms include diarrheal disease (Andrews & Ryan,

2015; El‐Tayeb, Ibrahim, Al‐Salamah, Almaary, & Elbadawi, 2017).

Molecular subtyping of _Salmonella_ is effective for epidemiological

investigations of infections and outbreaks. Multilocus sequence

typing (MLST) is a good method based on sequence analysis of

some housekeeping genes. Given its high repeatability in globally

dispersed laboratories, MLST is becoming an important method in

the investigation of various pathogens including _Salmonella_ (Yang

et al., 2015). Researchers have found that many bacteria contain

conserved repetitive intergenic consensus sequences, which have

proven to be useful for subtyping pathogens. As a PCR‐based finger
printing technique, ERIC‐PCR is easy, fast, and relatively cheap (Li,

Liu, Li, Xu, & Zheng, 2017).

As eggs are a popular food and Guangdong is an economic center

in south China with more than 110 million people, it is essential for

us to understand the prevalence of _S. enterica_ in this region. So far,

little is known of the distribution of strains among eggs. The aim of

our study was to determine the antimicrobial resistance, serotype,

and genetic diversity of _S. enterica_ in eggs from Guangdong, China.

The information generated in this study will aid in evaluating the

prevalence and population of _S. enterica_ to ensure egg safety.


**2** | **MATERIALS AND METHODS**


**2.1** | **Sample collection, culture of** _**Salmonella**_


From June 2017 to June 2018, we collected 1,000 egg samples from

farms and markets in Guangdong, China.

Isolation and identification of strains were carried out accord
ing to previous reports (Yang et al., 2015). Briefly, each egg was

disinfected with 75% alcohol, the shell was carefully removed, and

the egg yolks and whites were mixed. Then, 25 g of the sample was

added to 225 ml of buffered peptone broth, and 1 ml of the solu
tion was incubated in 10 ml of selenite cystine broth (SC) at 37°C

and 10 ml of tetrathionate brilliant green broth (TTB) at 42°C for

24 hr. The SC and TTB cultures were streaked onto xylose–lysine–

tergitol 4 (XLT4) selective agar plates and chromogenic _Salmonella_

agar plates (37°C, 24 hr). Presumptive colonies were stabbed into

a triple sugar iron slant and incubated at 37°C for 24 hr. Typical

_Salmonella_ phenotypes were further confirmed with API 20E test

strips (BioMerieux French).


**2.2** | **Antimicrobial susceptibility**


As per the guidelines of the Clinical and Laboratory Standards

Institute (CLSI, 2018), the susceptibility of the _Salmonella_ isolates to

antibiotic was examined by disk diffusion. Briefly, Mueller–Hinton

agar and a panel of 14 antibiotic disks were selected for the re
sistance tests. These antibiotics are commonly used in agriculture

and life. Them can cover among five classes: macrolides (azithro
mycin, AZM, 15 μg; erythromycin, ERY, 15 μg), aminoglycosides



(gentamicin, GEN, 10 μg; kanamycin, KAN, 30 μg; streptomycin, SM,

10 μg), quinolones (ciprofloxacin, CIP, 5 μg; nalidixic acid, NA, 30 μg),

β‐lactams (amoxicillin, AMC, 10 μg; ampicillin, AMP, 10 μg; cepha
zolin, CEP, 30 μg; penicillin, PEN, 10 μg; piperacillin, PIP, 10 μg), and

tetracyclines (minocycline, MIN, 30 μg; tetracycline, TET, 30 μg). The

results were expressed as sensitive (S), intermediate (I), and resistant

(R), following the methodology of the CLSI.


**2.3** | **Serotyping of** _**Salmonella**_ **isolates**


All the _Salmonella_ isolates were serotyped by agglutination tests

on the basis of somatic‐O and H antisera in accordance with the

Kauffmanne–White scheme (Boonkhot et al., 2015).


**2.4** | **Molecular subtyping**


The strains were analyzed by MLST using seven housekeeping genes:

_aroC_, _dnaN_, _hemD_, _hisD_, _purE_, _sucA_, and _thrA_ . The PCR conditions were

taken from the _Salmonella_ [MLST website and database (http://mlst.](http://mlst.warwick.ac.uk/mlst/dbs/Senterica/)

[warwi​ck.ac.uk/mlst/dbs/Sente​rica/; Jolley, Chan, & Maiden, 2004).](http://mlst.warwick.ac.uk/mlst/dbs/Senterica/)

Each reaction mixture included the following (total volume, 25 ml): 2

*PCR Mix (Qiagen), 12.5 μl; 1 μl each primer, dd H2O, 9.5 ml; and DNA

template, 1 μl. The PCR conditions were as follows: 96°C for 5 min;

35 cycles for 96°C for 1 min; 55°C for 1 min; 72°C for 1 min; and final

extension at 72°C for 10 min. The products were sequenced on BGI

instrument. The seven gene sequences were uploaded to the MLST

database for comparison to get allele numbers and define STs. The

MLST evolution tree was built using the Kimura‐2‐parameter in Mega

6.0 (Tamura, Stecher, Peterson, Filipski, & Kumar, 2013).

We used the ERIC‐PCR universal primers (ERIC1: 5‐ATGTAAGCT

CCTGGGGATTCAC‐3 and ERIC2: 5‐AAGTAAGTGACTGGGGTGA

GCG‐3). The PCR reaction consisted of 12.5 μl 2* TaKaRa Taq Mix;

1 μM primers; and 100 ng template DNA. The PCR program was

as follows: denaturation at 95°C, 5 min; 35 cycles of 94°C for 45 s,

52°C for 1 min, and 72°C for 3 min; and then 72°C for 10 min. The

products were separated by electrophoresis (2.0% agarose gels) for

40 min at 90 V. Cluster result analysis used a numerical taxonomy

and multivariate analysis software package, based on Dice's similar
ity coefficient ( _SD_ ) (James Rohlf, 2000).


**3** | **RESULTS**


**3.1** | _**Salmonella**_ **in eggs**


We isolated _Salmonella_ from 54 (5.40%) out of 1,000 eggs. Of these,

38 strains were from farms and 16 were from markets. The strains

were stored at −80°C in trypticase soy broth (20% glycerol). All posi
tive samples and strains information are shown in Table S1.


**3.2** | **Antibiotic susceptibilities**


The _Salmonella_ isolates were classified as sensitive, intermedi
ate, or resistant based on the diameter of inhibition zones around


