Table 2 Sources, number of E. coli isolates and results of their serogroup identification



<table><tr><td>Serogroup</td><td>Fresh poultry meat (n = 30)</td><td>Frozen poultry meat (n = 30)</td><td>Fresh poultry organs (n = 30)</td><td>Fresh beef meat (n = 30)</td><td>Frozen beef meat (n = 30)</td><td>Fresh beef organs (n = 30)</td><td>Total number of isolates</td><td>% of E. coli isolates</td></tr><tr><td>O1</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2</td><td>9.5</td></tr><tr><td>O18</td><td>2</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td><td>4</td><td>19</td></tr><tr><td>O20</td><td>1</td><td>-</td><td>1</td><td>-</td><td>-</td><td>-</td><td>2</td><td>9.5</td></tr><tr><td>O78</td><td>1</td><td>0</td><td>0</td><td>1</td><td>1</td><td>1</td><td>4</td><td>19</td></tr><tr><td>O103</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>1</td><td>4.8</td></tr><tr><td>O119</td><td>1</td><td>-</td><td>-</td><td>1</td><td>-</td><td>-</td><td>2</td><td>9.5</td></tr><tr><td>O126</td><td>0</td><td>0</td><td>1</td><td>0</td><td>1</td><td>0</td><td>2</td><td>9.5</td></tr><tr><td>O145</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>1</td><td>4.8</td></tr><tr><td>O146</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>4.8</td></tr><tr><td>O158</td><td>0</td><td>0</td><td>0</td><td>1</td><td>1</td><td>0</td><td>2</td><td>9.5</td></tr><tr><td>Total</td><td>6</td><td>3</td><td>4</td><td>3</td><td>4</td><td>1</td><td>21</td><td>100</td></tr></table>

allowed to stand for 15 min at room temperature. From each sample, 1 ml was added to 9 ml of buffered peptone water (Oxoid) for E. coli cultivation and 9 ml of Rappaport–Vassiliadis broth (Oxoid) for S. enterica isolation. After aerobic incubation at 37 °C overnight, a loopful from Rappaport–Vassiliadis broth was streaked on xylose lysine deoxycholate (XLD) agar (Oxoid) and from buffered peptone water on eosin methylene blue (EMB) agar (Oxoid). The inoculated plates were incubated aerobically at 37 °C for 18–24 h. Suspected colonies were identified biochemically using API 20E (bioMérieux, Marcy-l'Étoile, France).

All biochemically confirmed Salmonella isolates were serotyped on the basis of somatic (O) and flagellar (H) antigens by slide agglutination test using commercial antisera (SIFIN, Berlin, Germany) following Kauffman–White scheme  $ [19] $ . Serological identification of E. coli was carried out using slide agglutination method using commercial antisera (SIFIN). The serotyping was carried out at the Serology Unit, Animal Health Research Institute, Dokki, Egypt, and the Bacteriology Laboratory, Central Laboratories of Ministry of Health, Cairo, Egypt.

## DNA extraction and purification

The identified bacterial cultures were cultivated in Luria–Bertani (LB) broth (Oxoid) at  $ 37\ °C $  overnight. DNA was extracted and purified using DNeasy Blood and Tissue Kits (Qiagen, Manchester, UK) according to the manufacturer's instructions.

Molecular identification of S. enterica serovars and E. coli

The isolated strains were further identified as S. enterica and E. coli using PCR assays. The amplification targets and primers (Eurofins, Yokohama, Japan) are listed in Table 3. All phenotypically identified S. enterica serovars were tested for harbouring ompC gene. Isolated E. coli were confirmed by species-specific PCR targeting the 16S rRNA genes and resulting in a 585 bp amplicon. PCR assays used in this study resulted in Salmonella, S. Enteritidis and S. Typhimurium specific products of 204, 304 and 401 bp length, respectively.



The PCR protocol consisted of the following steps: (i) an initial denaturation step of 2 min at 95 °C; (ii) 30 cycles with 1 cycle consisting of 1 min at 95 °C, 1 min at 57 °C, and 2 min at 72 °C; and (iii) a final elongation step of 5 min at 72 °C.

Five-microliter aliquots of reaction mixture were run in electrophoreses using 1.5% agarose gels (Nippon Gene, Tokyo, Japan) and visualized under UV light after ethidium bromide staining.

## Determination of antimicrobial susceptibility profiles

The antimicrobial susceptibility was determined using the Kirby–Bauer disc diffusion test [20]. Briefly, one colony picked up, streaked on Mueller–Hinton blood agar (Oxoid) and incubated at  $ 37\ °C $  overnight. Bacterial colonies were suspended in 0.9% NaCl to obtain a McFarland turbidity of 0.5 (Dr. Lange, photometer CADAS 30, Berlin, Germany) containing  $ 1–2 \times 10^{8} $  colony-forming units (CFU)/ml of E. coli strain ATCC 25922. Three hundred  $ \mu $ l of the suspension were spread onto the surface of a Mueller–Hinton agar plate (Oxoid) using a sterile swab. The antimicrobial discs (Oxoid) of thirteen clinically used antibiotics, which are used in the Egyptian poultry and cattle farms (Tables 4, 5) were distributed onto the surface of the Mueller–Hinton agar plates using a multi-disc dispenser (Oxoid). The plates were incubated at  $ 37\ °C $  overnight. The diameters of inhibition zones were measured using sliding calipers and interpreted using standard