antimicrobial agents, such as fluoroquinolones and third generation cephalosporins, is also an emerging problem (Lunguya et al., 2013). In developed countries, many surveys have been conducted at the molecular level to monitor the incidence of MDR Salmonella enterica in meat and dairy products (Ahmed et al., 2009a; Barlow and Gobius, 2006; Clemente et al., 2013; Li et al., 2013; Mohamed et al., 2014; Thong and Modarressi, 2011; Van Keesel et al., 2013; Wong and Chen, 2013; Zhao et al., 2009). However, the extent of antibiotic resistance to S. enterica in many developing countries and the molecular mechanisms underlying this resistance remain unclear. The misuse of antibiotics in humans and animals in these countries is not uncommon. Therefore, the aim of this study was to perform a large scale survey to determine the incidence of MDR strains of S. enterica isolated from meat and dairy products in Egypt and to characterize the molecular mechanisms of antimicrobial resistance.

### 2. Materials and methods

#### 2.1. Bacterial isolates

A total of 69 isolates of S. enterica (28 isolates of S. enterica serovar Typhimurium, 22 isolates of S. enterica serovar Enteritidis, 16 isolates of S. enterica serovar Infantis and 3 isolates of non-typable serovars) were used in this study. They were isolated in Egypt from 800 meat products (480 beef and 320 chickens) and 800 dairy products (480 milk, 240 cheese and 80 yogurt) as previously described (Ahmed and Shimamoto, 2014).

#### 2.2. Antimicrobial susceptibility testing

The antimicrobial sensitivity phenotypes of bacterial isolates were determined using a Kirby–Bauer disk diffusion assay according to the standards and interpretive criteria described by the Clinical and Laboratory Standards Institute (CLSI, 2011). The following antibiotics were used: ampicillin (AMP), 10  $ \mu $ g; amoxicillin–clavulanic acid (AMC), 20/10  $ \mu $ g; cefoxitin (FOX), 30  $ \mu $ g; cefotetan (CTT), 30  $ \mu $ g; cefotaxime (CTX), 30  $ \mu $ g; cefpodoxime (CPD), 10  $ \mu $ g; ceftriaxone (CRO), 30  $ \mu $ g; aztreonam (ATM), 30  $ \mu $ g; nalidixic acid (NAL), 30  $ \mu $ g; ciprofloxacin (CIP), 5  $ \mu $ g; chloramphenicol (CHL), 30  $ \mu $ g; gentamicin (GEN), 10  $ \mu $ g; kanamycin (KAN), 30  $ \mu $ g; oxacillin (OXA) 30  $ \mu $ g; streptomycin (STR), 10  $ \mu $ g; spectinomycin (SPX), 10  $ \mu $ g; sulfamethoxazole/trimethoprim (SXT), 23.75/1.25  $ \mu $ g and tetracycline (TET), 30  $ \mu $ g. The disks were purchased from Oxoid, UK, and the results were recorded based on CLSI guidelines (CLSI, 2011). The reference strain, Escherichia coli ATCC 25922, was included as a quality control.

#### 2.3. Bacterial DNA preparation

DNA was prepared using boiled lysates, as previously described (Ahmed et al., 2009a). All selected single colonies were subcultured in LB broth. An overnight bacterial culture (200  $ \mu $ l) was mixed with 800  $ \mu $ l of distilled water and boiled for 10 min. The resulting solution was centrifuged and the supernatant used as the DNA template. DNA template was stored at  $ -20\ ^{\circ}C $  until use.

#### 2.4. PCR screening for integrons and antimicrobial resistance genes

Conserved primers were used for the detection and identification of class 1 and class 2 integrons as previously described (Ahmed et al., 2013). PCR screening for TEM, SHV, CTX-M, OXA and CMY  $ \beta $ -lactamase-encoding genes was performed using universal primers for the TEM, SHV, OXA, CTX-M and CMY families, as described previously (Ahmed et al., 2013). Furthermore, PCR amplification was used to screen for plasmid-mediated quinolone resistance genes, qnrA, qnrB, qnrS and  $ aac(6')-Ib-cr $ , using previously described primers (Ahmed et al., 2013). Finally, the florfenicol resistance gene, floR, was detected using StCM-L and StCM-R primers, as described previously (Ahmed et al., 2013). The PCR reaction products were subjected to electrophoresis in a 1.0% agarose gel, stained with ethidium bromide and visualized under UV light. PCR fragments were then purified from the agarose gel using a QIAquick Gel Extraction Kit (Qiagen, Japan). Both DNA strands of the PCR product were sequenced using an ABI automatic DNA sequencer (Model 373; Perkin-Elmer).



##### 2.5. Identification of S. enterica serovar Typhimurium DT104 by PCR

The PCR amplification of an internal segment of the 16S-to-23S spacer region of bacterial rRNA genes (size = 162 bp) was used to identify S. enterica serovar Typhimurium DT104 as previously described (Pritchett et al., 2000).

#### 2.6. Transconjugation experiments and plasmid incompatibility grouping

Transferability of plasmids was determined by mating-out assay using S. enterica isolates as donors and a rifampicin-resistant mutant of E. coli HB101 as a recipient as described previously (Ahmed et al., 2013). Some S. enterica isolates which have no potent resistance gene marker were not tested. Transconjugants were selected on agar supplemented with 250 mg/l rifampicin and 100 mg/l ampicillin. Plasmid DNA was extracted from both S. enterica isolates and E. coli transconjugants using the Kado and Liu method (Kado and Liu, 1981). Plasmid incompatibility grouping was determined by PCR-based replicon typing as previously described (Carattoli et al., 2005). The transfer of integrons and resistance genes was confirmed by PCR assays on the transconjugants as described previously (Ahmed et al., 2013).

#### 2.7. Computer analysis of the sequence data

A similarity search was carried out using the BLAST program available at the NCBI BLAST homepage (http://blast.ncbi.nlm.nih.gov/Blast.cgi).

### 3. Results and discussion

##### 3.1. Incidence of MDR S. enterica in meat and dairy products

The emergence and spread of antimicrobial resistance in bacteria constitute a threat to human health and present a major financial burden. Food is generally considered to be the most important vector for the spread of resistance between humans and animals (WHO, 2007). In this study 47 out of 69 isolates (68.1%) showed multidrug resistance phenotypes to at least three classes of antimicrobials (Table 1). The incidence of MDR isolates was higher in meat products (69.8%) than in dairy products (62.5%). The MDR S. enterica serovars were: S. enterica serovar Typhimurium (34.8%), S. enterica serovar Enteritidis (21.8%), S. enterica serovar Infantis (10.1%) and S. enterica non-typable serovar (1.4%) (Table 1). The highest resistance was to ampicillin (95.7%), then to kanamycin (93.6%), spectinomycin (93.6%), streptomycin (91.5%) and sulfamethoxazole/trimethoprim (91.5%) (Table 1). Interestingly, the results of multidrug resistance phenotypes in this study are quite similar to that we previously reported in S. enterica isolated from diarrheic calves in Egypt as 66.7% of S. enterica showed multidrug resistance phenotypes to ampicillin, amoxicillin, streptomycin, spectinomycin, sulfamethoxazole/trimethoprim and kanamycin (Ahmed et al., 2009b). Our results confirm the high level of resistance to the traditional antibiotics used widely in Egypt. Unfortunately, in developing countries, these antibiotics are widely used to treat diarrhea because of their low cost and availability (Van et al., 2012).