present, this is not extensively used for disinfection. It is thus highly likely that phages survive under routine conditions of disinfection not only in the food industry.

Our results show for the first time that generalized transduction without the need for homologous recombination is a neglected but nevertheless efficient way to transfer antimicrobial resistance to  $ E.\ coli $  in different environments, including in food of animal origin. This conclusion has important consequences for hygiene, sanitation, and public health when it is important to combat antimicrobial resistance.

## ACKNOWLEDGMENTS

Amira Shousha was granted a Vetmeduni Success Scholarship from the University of Veterinary Medicine, Vienna, Austria, to perform this study. Nattakarn Awaiwanont was funded by the Technologiestipendien Südostasien in the framework of ASEA-UNINET, granted by the OeAD (Austrian Agency for International Cooperation in Education & Research), financed by the Austrian Federal Ministry for Science and Research.

We thank Stefan Hertwig for discussions and Graham Tebb for critical reading of and suggestions on the manuscript.

## REFERENCES

1. Laube H, Friese A, von Salviati C, Guerra B, Käsbohrer A, Kreienbrock L, Roesler U. 2013. Longitudinal monitoring of extended-spectrum-beta-lactamase/AmpC-producing Escherichia coli at German broiler chicken fattening farms. Appl Environ Microbiol 79:4815–4820. http://dx.doi.org/10.1128/AEM.00856-13.

2. Bronzwaer S, Lönnroth A, Haigh R. 2004. The European Community strategy against antimicrobial resistance. Euro Surveill 9(1):pii=441. http://www.eurosurveillance.org/ViewArticle.aspx?ArticleId=441.

3. Vogt D, Overesch G, Endimiani A, Collaud A, Thomann A, Perreten V. 2014. Occurrence and genetic characteristics of third-generation cephalosporin-resistant Escherichia coli in Swiss retail meat. Microb Drug Resist 20:485–494. http://dx.doi.org/10.1089/mdr.2013.0210.

4. European Food Safety Authority, European Centre for Disease Prevention and Control. 2015. EU summary report on antimicrobial resistance in zoonotic and indicator bacteria from humans, animals and food in 2013. EFSA J 13:4036. http://dx.doi.org/10.2903/j.efsa.2015.4036.

5. Ibekwe AM, Murinda SE, Graves AK. 2011. Genetic diversity and antimicrobial resistance of Escherichia coli from human and animal sources uncovers multiple resistances from human sources. PLoS One 6:e20819. http://dx.doi.org/10.1371/journal.pone.0020819.

6. Kruse H, Sørum H. 1994. Transfer of multiple drug resistance plasmids between bacteria of diverse origins in natural microenvironments. Appl Environ Microbiol 60:4015–4021.

7. Mayrhofer S, Paulsen P, Smulders FJM, Hilbert F. 2006. Antimicrobial resistance in commensal Escherichia coli isolated from muscle foods as related to the veterinary use of antimicrobial agents in food-producing animals in Austria. Microb Drug Resist 12:278–283. http://dx.doi.org/10.1089/mdr.2006.12.278.

8. Gunell M, Webber MA, Kotilainen P, Lilly AJ, Caddick JM, Jalava J, Huovinen P, Siitonen A, Hakanen AJ, Piddock LJV. 2009. Mechanisms of resistance in nontyphoidal Salmonella enterica strains exhibiting a nonclassical quinolone resistance phenotype. Antimicrob Agents Chemother 53:3832–3836. http://dx.doi.org/10.1128/AAC.00121-09.

9. Woodford N, Ellington MJ. 2007. The emergence of antibiotic resistance by mutation. Clin Microbiol Infect 13:5–18. http://dx.doi.org/10.1111/j.1469-0691.2006.01492.x.

10. Mempel M, Feucht H, Ziebuhr W, Endres M, Laufs R, Grüter L. 1994. Lack of mecA transcription in slime-negative phase variants of methicillin-resistant Staphylococcus epidermidis. Antimicrob Agents Chemother 38:1251–1255. http://dx.doi.org/10.1128/AAC.38.6.1251.

11. Mah TF. 2012. Biofilm-specific antibiotic resistance. Future Microbiol 7:1061–1072. http://dx.doi.org/10.2217/fmb.12.76.

12. Boerlin P, Reid-Smith RJ. 2008. Antimicrobial resistance: its emergence and transmission. Anim Health Res Rev 9:115–126. http://dx.doi.org/10.1017/S146625230800159X.

13. Martinez JL, Fajardo A, Garmendia L, Hernandez A, Linares JF, Mar-

tínez-Solano L, Sánchez MB. 2009. A global view of antibiotic resistance. FEMS Microbiol Rev 33:44–65. http://dx.doi.org/10.1111/j.1574-6976.2008.00142.x.

14. Wiedenbeck J, Cohan FM. 2011. Origins of bacterial diversity through horizontal genetic transfer and adaptation to new ecological niches. FEMS Microbiol Rev 35:957–976. http://dx.doi.org/10.1111/j.1574-6976.2011.00292.x.

15. Normark BH, Normark S. 2002. Evolution and spread of antibiotic resistance. J Intern Med 252:91–106. http://dx.doi.org/10.1046/j.1365-2796.2002.01026.x.

16. Volkova VV, Lu Z, Besser T, Gröhn YT. 2014. Modeling the infection dynamics of bacteriophages in enteric Escherichia coli: estimating the contribution of transduction to antimicrobial gene spread. Appl Environ Microbiol 80:4350–4362. http://dx.doi.org/10.1128/AEM.00446-14.

17. Fineran PC, Petty NK, Salmond GPC. 2009. Transduction: host DNA transfer by bacteriophages, p 666–679. In Schaechter M (ed), The desk encyclopedia of microbiology. Elsevier Academic Press, San Diego, CA.

18. Hendrix RW, Smith MCM, Burns RN, Ford ME, Hatfull GF. 1999. Evolutionary relationships among diverse bacteriophages and prophages: all the world's a phage. Proc Natl Acad Sci U S A 96:2192–2197. http://dx.doi.org/10.1073/pnas.96.5.2192.

19. Schwarzer D, Buettner FFR, Browning C, Nazarov S, Rabsch W, Bethe A, Oberbeck A, Bowman VD, Stummeyer K, Mühlenhoff M, Leiman PG, Gerardy-Schahn R. 2012. A multivalent adsorption apparatus explains the broad host range of phage phi92: a comprehensive genomic and structural analysis. J Virol 86:10384–10398. http://dx.doi.org/10.1128/JVI.00801-12.

20. Evans TJ, Crow MA, Williamson NR, Orme W, Thomson NR, Komitopoulou E, Salmond GPC. 2010. Characterization of a broad-host-range flagellum-dependent phage that mediates high-efficiency generalized transduction in, and between, Serratia and Pantoea. Microbiology 156:240–247. http://dx.doi.org/10.1099/mic.0.032797-0.

21. Weitz JS, Poisot T, Meyer JR, Flores CO, Valverde S, Sullivan MB, Hochberg ME. 2013. Phage-bacteria infection networks. Trends Microbiol 21:82–91. http://dx.doi.org/10.1016/j.tim.2012.11.003.

22. Zhang Z, Kottadiel VI, Vafabakhsh R, Dai L, Chemla YR, Ha T, Rao VB. 2011. A promiscuous DNA packaging machine from bacteriophage T4. PLoS Biol 9:e1000592. http://dx.doi.org/10.1371/journal.pbio.1000592.

23. Schmieder R, Edwards R. 2012. Insights into antibiotic resistance through metagenomic approaches. Future Microbiol 7:73–89. http://dx.doi.org/10.2217/fmb.11.135.

24. EUCAST. 2010. Antimicrobial susceptibility testing. http://www.eucast.org/ast_of_bacteria/.

25. International Organization for Standardization. 2000. Water quality—detection and enumeration of bacteriophages. Part 2. Enumeration of somatic coliphages. ISO 10705-2:2000. International Organization for Standardization, Geneva, Switzerland.

26. Groisman EA. 1991. In vivo genetic engineering with bacteriophage Mu. Methods Enzymol 204:180–212. http://dx.doi.org/10.1016/0076-6879(91)04010-L.

27. Sacher A. 2008. Ph.D. thesis. Veterinärmedizinischen Universität Wien, Vienna, Austria.

28. Schmieger H, Schicklmaier P. 1999. Transduction of multiple drug resistance of Salmonella enterica serovar typhimurium DT104. FEMS Microbiol Lett 170:251–256. http://dx.doi.org/10.1111/j.1574-6968.1999.tb13381.x.

29. Milić N, Milanović M, Grujić Letić N, Sekulić MT, Radonić J, Mihajlović I, Miloradov MV. 2013. Occurrence of antibiotics as emerging contaminant substances in aquatic environment. Int J Environ Health Res 23:296–310. http://dx.doi.org/10.1080/09603123.2012.733934.

30. Zhang J, Chang VWC, Giannis A, Wang J-Y. 2013. Removal of cytostatic drugs from aquatic environment: a review. Sci Total Environ 445-446:281–298. http://dx.doi.org/10.1016/j.scitotenv.2012.12.061.

31. Balcazar JL. 2014. Bacteriophages as vehicles for antibiotic resistance genes in the environment. PLoS Pathog 10:e1004219. http://dx.doi.org/10.1371/journal.ppat.1004219.

32. Colomer-Lluch M, Imamovic L, Jofre J, Muniesa M. 2011. Bacteriophages carrying antibiotic resistance genes in fecal waste from cattle, pigs, and poultry. Antimicrob Agents Chemother 55:4908–4911. http://dx.doi.org/10.1128/AAC.00535-11.

33. Colomer-Lluch M, Jofre J, Muniesa M. 2011. Antibiotic resistance genes