# _Original Article_ **Antibiotic resistance and resistance genes in E scherichia coli from poultry** **farms, southwest Nigeria**

Olawale O Adelowo [1,2], Obasola E Fagade [2], Yvonne Agersø [3]

_1 Department of Pure and Applied Biology, Ladoke Akintola University of Technology, Ogbomoso, Nigeria_
_2 Environmental Microbiology and Biotechnology Laboratory, Department of Botany and Microbiology, University of_
_Ibadan, Nigeria_
_3 Department for Microbiology and Risk Assessment, National Food Institute Technical University of Denmark,_
_Lyngby, Denmark_

**Abstract**
Introduction: This study investigated the mechanisms of resistance in 36 _E. coli_ isolated from waste, litter, soil and water samples collected
from poultry farms in Southwestern Nigeria.
Methodology: Minimum inhibitory concentration (MIC) distributions of the isolates were determined using the methods of the Clinical and
Laboratory Standard Institute and resistance genes detected by PCR.
Results: A total of 30 isolates (94%) showed resistance to more than one antimicrobial. Percentage resistance was: tetracycline 81%,
sulphamethoxazole 67%, streptomycin 56%, trimethoprim 47 %, ciprofloxacin 42%, ampicillin 36%, spectinomycin 28%, nalidixic acid
25%, chloramphenicol 22%, neomycin 14%, gentamicin 8%, amoxicillin-clavulanate, ceftiofur, cefotaxime, colistin, florfenicol and
apramycin 0%. Resistance genes found among the isolates include _bla-TEM_ (85%), _sul2_ (67%), _sul3_ (17%), _aadA_ (65%), _strA_ (70%), _strB_
(61%), _catA1_ (25%), _cmlA1_ (13%), _tetA_ (21%) and _tetB_ (17%). Class 1 and 2 integrons were found in five (14%) and six (17%) isolates,
respectively, while one isolate was positive for both classes of integrons. Seven out of eight isolates with resistance to ciprofloxacin and MIC
≤ 32 mg/L to nalidixic acid contained _qnrS_ genes.
Conclusions: Our findings provided additional evidence that the poultry production environment in Nigeria represents an important reservoir
of antibiotic resistance genes such as _qnrS_ that may spread from livestock production farms to human populations via manure and water.

**Key words:** Farm waste; livestock production environment; _qnrS._

_J Infect Dev Ctries_ 2014; 8(9):1103-1112 _._ doi:10.3855/jidc.4222

(Received 10 September 2013 – Accepted 14 May 2014)

Copyright © 2014 Adelowo _et al_ . This is an open-access article distributed under the Creative Commons Attribution License, which permits unrestricted use,
distribution, and reproduction in any medium, provided the original work is properly cited.

**Introduction** poultry waste [7,8,6,9], commercial poultry production



Antimicrobial drug use in livestock production has
been implicated as a risk factor in the development
and dissemination of drug resistance from livestock
production farms [1,2]. Food animals and their
production environments are reservoirs of both
resistant bacteria and resistance genes that could be
transferred to humans either by direct contact between
animals and humans or indirectly via the food
production chain [3,4]; or as a result of the spread of
animal waste on land [5,2]. In Nigeria, antimicrobial
agents are routinely used in livestock production
especially as additives to feed and water [6]. This may
result in a selective advantage and a consequent
increase in the abundance of resistant bacteria in
animals, their wastes and surrounding environment

[5]. Antibiotic resistant bacteria have been detected in



poultry waste [7,8,6,9], commercial poultry production
environments [10,11] and poultry products [12,13].

Most food animal production farms in Nigeria
have no waste treatment facility. Wastes generated on
these farms are either dumped in heaps on farmlands
or at remote locations, often inside or close to water
bodies. Wastes are also used as organic fertilizer or as
feed supplements, especially in fish ponds. This raises
the fear that human beings resident in the vicinity of
the farm and/or animal waste dump may be exposed to
antibiotic resistant bacteria in the waste. In addition,
close interaction between human and animal
ecosystems is a common occurrence in Nigeria with
livestock kept in close proximity to or inside human
residences. This practice has been shown to favour
high rates of transmission of zoonotic _E. coli_ between
livestock and humans in rural Uganda [14]. Although
_E. coli_ is a normal inhabitant of the intestinal tract of




Adelowo _et al. –_ Resistant _E. coli_ from Nigeria _J Infect Dev Ctries_ 2014; 8(9):1103-1112.



warm blooded animals, commensal _E. coli_ from
humans and animals can cause extra intestinal
infections and are a potential reservoir of antimicrobial
resistance genes [15]. These bacterial strains,
especially when they carry mobile genetic elements
such as plasmids, transposons and integrons, can play
important roles in the dissemination of transmissible
resistance genes [16].

Little is known about the factors contributing to
the problem of antibiotic resistance in Nigeria. This is
partly because there is a dearth of information on the
types, quantity and frequency of antibiotics used in
farm animal production system in Nigeria. Very little
is also known about the antibiotic resistance profile of
bacteria (such as _E. coli_ ) commonly associated with


**Table 1.** Description of study farms



animal production systems in Nigeria despite the
growing global concern about the transmission of
antibiotic resistant bacteria through the food animal
production chain. The few studies that have
investigated antibiotic resistance in _E. coli_ from the
poultry production system in Nigeria stopped at
phenotypic level without a corresponding investigation
of the molecular mechanisms of resistance [17]. Yet
such studies are warranted because of the potentials of
_E. coli_ as reservoirs of transferable antibiotic
resistance genes that could be disseminated into
human populations as a result of the contact with
animal wastes. This study therefore investigated
antimicrobial susceptibility and molecular mechanisms
of resistance in _E. coli_ isolated from eight poultry



**Animal**
**Farm/City:** **Age (yrs):** **Description:** **Antimicrobial use:**
**population:**
**Ogbomosho:**
OG1 15 Teaching and research farm, poultry, Furazolidone, nitrofurantoin, ampicillin, 2000 birds
swine and fish operation. Swine and neomycin, streptomycin, sulphonamides,
pond not operating at time of study. tetracyclines, chloramphenicol,



Furazolidone, nitrofurantoin, ampicillin,
neomycin, streptomycin, sulphonamides,
tetracyclines, chloramphenicol,
erythromycin.



2000 birds



OG2 6 Research and demonstration farm,
poultry operation.


OG3 9 Poultry operation in a medium density
residential area. Farm separated from
adjoining building by a brickwall.


OG4 unknown Integrated poultry, swine and
aquaculture operation in a rural
agricultural community. Uses poultry
waste as feed supplement in pond. Pond
operation suspended about 5 months
before sample collection.

**Ibadan:**
IB1 >20 Teaching and research farm, poultry
operation


IB2 17 Integrated poultry, swine and
aquaculture operation in a rural
agricultural community. Swine and fish
pond not operating at time of study. This
is the only farm in this study that
mentioned using enrofloxacin and
tylosin.

**Osogbo:**
OS1 5 Poultry and swine operation **,** swine
operation relocated due to protests from
surrounding residents

OS2 unknown Poultry operation in a rural agricultural
community. People living around this
farm use water from a well located in
the farm for domestic purposes



Furazolidone, nitrofurantoin, ampicillin,
neomycin, streptomycin, sulphonamides
tetracyclines, chloramphenicol,
erythromycin.

Furazolidone, nitrofurantoin, ampicillin,
neomycin, streptomycin, sulphonamides
tetracyclines, chloramphenicol,
erythromycin.

Furazolidone, nitrofurantoin, ampicillin,
streptomycin sulphonamides, doxycycline,
chloramphenicol.


Furazolidone, nitrofurantoin, ampicillin,
neomycin, streptomycin, sulphonamides
tetracyclines, chloramphenicol,
erythromycin.

Furazolidone, nitrofurantoin, ampicillin,
neomycin, streptomycin, sulphonamides
tetracyclines, chloramphenicol,
enrofloxacin, tylosin.


Furaltadone, nitrofurantoin, streptomycin,
sulphonamides, tetracyclines,
chloramphenicol, erythromycin.

Furazolidone, nitrofurantoin, ampicillin
streptomycin, sulphonamides,
tetracyclines doxycycline,
chloramphenicol.



750 birds


2500 birds


1200 birds, 120
pigs


2500 birds


2500 birds


1000 birds


750 birds



-a brief profile, including types of antibiotics used in each of the farms sampled in this study. OG1- OG4, represent the four farms sampled in Ogbomoso, IB1

 - IB2, represent the two farms sampled in Ibadan, OS1-OS2, represent the two farms sampled in Osogbo


1104




Adelowo _et al. –_ Resistant _E. coli_ from Nigeria _J Infect Dev Ctries_ 2014; 8(9):1103-1112.



production farms in Ogbomoso, Ibadan and Osogbo in
Southwestern Nigeria. Four of the farms combined
swine and/or fish ponds with poultry operations. The
farms with aquaculture operations use the wastes
generated from the poultry pens as feed supplement in
the ponds. Our primary aim is to assess the potential of
the poultry production environment in Nigeria as a
reservoir of antibiotic resistance genes that can be
mobilized into human populations.


**Methodology**
_Bacterial strains_

Thirty six _E. coli_ isolated from samples of faecal
materials, litter, farm soil, water and fish pond
sediment collected from the study farms in May and
June 2008 were included in the study. The water
samples were collected as previously described [18]
while samples of soil; litter and faecal materials were
collected in sterile polythene sample bags as described

[6].The eight farms included in the study were selected
from Ogbomoso and Ibadan in Oyo State (8 [o] 0′ N 4 [o] 0′
E) and Osogbo in Osun State (7 [ o] 46′ N 4 [ o] 34′ E) of
Southwestern Nigeria. The farms are those who
voluntarily agreed to participate in the study and are
described in Table 1. Among the farms, only one has a
comprehensive record of drug usage, it was also the
only farm that admitted using enrofloxacin. The
remaining farms admitted using one or more of
furazolidone, furaltadone, nitrofurantoin, ampicillin,
neomycin, streptomycin, sulphonamides, tetracyclines,
doxycycline, chloramphenicol and erythromycin
regularly on the farms for animal health management.
The drugs are usually part of formulations used as feed
supplements, commonly referred to as “premix” which
often contained mixtures of antimicrobials. The farm
with combined aquaculture operation suspended
aquaculture operations about 5 months before the
period of sediment collection from the fish pond.

The organisms were isolated from the samples
within 24 hours on MacConkey agar as previously
described [6] and were identified at the Unit of
Antimicrobial Resistance and Molecular
Epidemiology, National Food Institute, DTU,
Copenhagen, Denmark by streaking on Blood agar,
MacConkey agar and CHROMagar orientation
Medium (Bencton Dickinson GmBH, Heigelberg,
Germany). Thirty-six non duplicated isolates
confirmed as _E. coli_ were randomly selected from the
samples and stored at -80 [o] C prior to further analysis
for Minimum inhibitory concentration (MIC)
determination and molecular characterization.



_Susceptibility testing_

The susceptibility of the isolates to antimicrobial
agents was tested at the National Food Institute, DTU,
Denmark using commercial dehydrated Sensititre
panels (Trek Diagnostic Ltd, East Grinstead, UK)
containing the antibiotics listed below at the stated test
concentrations in mg/L. Amoxicillin (2-32 mg/L) –
clavulanate (1-16 mg/L), ampicillin (1-32 mg/L),
apramycin (4-32 mg/L), cefotaxime (0.125-4 mg/L),
ceftiofur (0.5-8 mg/L), chloramphenicol (2-64 mg/L),
ciprofloxacin (0.015-4 mg/L), colistin (1-16 mg/L),
florfenicol (2-64 mg/L), gentamicin (0.5-16 mg/L),
nalidixic acid (4-64 mg/L), neomycin (2-32 mg/L),
spectinomycin (16-256 mg/L), streptomycin (8-128
mg/L), sulphamethoxazole (64-1024 mg/L),
tetracycline (2-32 mg/L) and trimethoprim (1-32
mg/L). Panels were incubated aerobically at 37 [o] C for
18-20 hours. MIC distributions were determined
according to the Clinical and Laboratory Standards
Institute (CLSI) guidelines [19]. The MIC results were
interpreted using the European Committee for
Antimicrobial Susceptibility Testing (EUCAST)
epidemiological cut-off values and _E. coli_ ATCC
25922 was used as a positive control strain.


_Molecular mechanisms of resistance_

Polymerase chain reaction (PCR) amplification of
class 1 and 2 integrons and antimicrobial resistance
genes were performed as described previously [20].
One to four colonies of the tested isolates on blood
agar were suspended in 100µl of TE buffer and lysed
as previously described [20]. The lysates were used as
template in PCR to detect _strA, strB_ and _aadA_ in
streptomycin resistant isolates, _bla-TEM_ in ampicillin
resistant isolates, _sul2_ and _sul3_ in sulphamethoxazole
resistant isolates, _tet_ (A), _tet_ (B), _tet_ (C) _, tet_ (D) _, tet_ (E),
_tet_ (G) and _tet_ (39) in tetracycline resistant isolates,
_cmlA,_ _catA1_ and _floR_ (primers for _floR:_ Flor-1, 5’ATGGCAGGCGATATTCATTA-3’; flor-2: 5’AAACGGGTTGTCACGATCAT-3’) in
chloramphenicol resistant isolates, acc _(3)-IV_ in
gentamicin resistant isolates, _qnrA_, _qnrB_, _qnrC_, _qnrD,_
_qnrS_, _aac(6’)Ib_ and _qepA_ in isolates resistant to
ciprofloxacin (≥ 0.125mg/L) but sensitive to nalidxic
acid with MIC ≤ 32 mg/L [20, 22-25]. The variable
regions of Class 1 and Class 2 integrons were
amplified and representatives of the different sized
amplicons sequenced to characterise their gene
contents as described by Peirano _et al_ [21]. The
following were used as positive control strains in PCR:
_E. coli_ ( _tet_ (A)), _Salmonella Typhimurium_ DT104
( _tet_ (G)), _E. coli_ CSH50::Tn10 ( _tet_ (B)), _E. coli_


1105




Adelowo _et al. –_ Resistant _E. coli_ from Nigeria _J Infect Dev Ctries_ 2014; 8(9):1103-1112.



DO7pBR322 ( _tet_ (C)), _E. coli_ C600psl106 ( _tet_ (D)), _E._
_coli_ HB 101 psl 1604 _tet_ (E)), _E. coli_ YA5605
pUC19::tet(39) ( _tet(39)_ ), _E. coli_ K13 (acc _(3)-IV_ ),
_Salmonella_ Weltevreden TA428/97 ( _catA1_ ), _S._
Typhimurium P502212 DT104 ( _floR_ ) _,_ Paratyphi
B63.48 ( _bla-TEM_ ), _S._ Typhimurium DT104 ( _strA, strB,_
_aadA1_ ), _Salmonella_ B ( _sul2_ ), _Salmonella_ C ( _sul3_ ), _S._
Typhimurium DS611 ( _cmlA_, _int1, sul1_ ), _Salmonella_
(int2).


**Results**
_Antimicrobial susceptibility_

Tables 2 and 3 showed the distribution of MICs
and phenotypic patterns of resistance among the
isolates. Resistance to tetracycline (81%),
sulphamethoxazole (67%), streptomycin (56%),
trimethoprim (47%), ciprofloxacin (42%) and
ampicillin (36%) was the most commonly detected
among the isolates tested while phenotypic resistance
to spectinomycin (28%), nalidixic acid (25%),
chloramphenicol (22%), neomycin (14%) and
gentamicin (8%) was less commonly detected. In
contrast, all the isolates were sensitive to amoxicillinclavulanate, ceftiofur, cefotaxime, colistin, florfenicol
and apramycin. Among the 15 ciprofloxacin resistant
isolates, eight isolates had MICs to nalidixic acid ≤ 32
mg/L. While this study was not designed to investigate
the relationship between antibiotic use and resistance
on the study farms due primarily to the nonavailability of antibiotic-use information as mentioned
earlier, it is important to report that tetracycline,
streptomycin, sulphonamides, and ampicillin are
among the antimicrobials used on all the study farms.

Among isolates of the present study, resistance to
two or more antibiotics was detected in 30 of 36 (83%)
isolates. When looking at isolates from poultry
droppings/litter, swine waste, soil (including fish pond
sediment) and well water, 93% (14/15), 63% (5/8),
89% (8/9) and 75% (3/4) of the isolates respectively
showed resistance to two or more antimicrobials. Four
isolates from pig manure (EC137, EC146), farm soil
(EC162) and well water (EC116) were fully
susceptible to all antimicrobials tested (Table 3).


_Mechanisms of resistance_

Table 3 showed the phenotypic patterns of
resistance and resistance genes profiles of the isolates.
Among the 36 isolates tested, six (17%) carried class 2
integrons, five (17%) of 24 sulphamethoxazole
resistant isolates carried class 1 integrons, while only
one isolate (3%) (EC3) carried both classes of
integrons. The variable region of the class 1 integrons



of one representative isolate (EC117) selected from
among the most common sized amplicon (1500 bp
found in four isolates) was partially sequenced (935
bp). The partial sequence showed 99% sequence
identity with position 695 to 1629 containing _aadA5_ in
_E. coli_ GL1 (GenBank accession number FJ807902).
As expected the sulphonamide resistance gene _sul1_
was detected in all class 1 integron-positive isolates as
part of the integron. The variable regions of the class 2
integrons detected among the _int2_ positive isolates
showed three different sized PCR products. The
product of one representative isolate (EC3, EC111 and
EC113) for each type was sequenced. The partial
sequences showed 88% identity with position 1057 to
436 of _E. coli_ encoding a _sat1_ (GenBank accession
number X56815) for isolate EC111, 100% identity
with position 2073 to 2608 of _E. coli_ encoding a
_dfrA1_ - _sat2_ (partial) (GenBank accession number
HM439239) for EC113 and 100% identity with
position 1995 to 2668 of _E. coli_ encoding a _sat2_
(GenBank accession number AY183453) for isolate
EC3.

In addition to _sul1_ carried as part of _int1, sul2_ and
_sul3_ were detected in 16 (67%) and 4 (17%) of 24
sulphamethoxazole – resistant isolates. All the isolates
positive for _sul3_ were from poultry waste or farm soil
from Ogbomoso. Three of the _int1_ / _sul1_ positive
isolates were also positive for _sul2._ However,
_sul2/sul3_ and _sul1/sul3_ combinations could not be
detected in any of the isolates. Of 23 strains with
phenotypic resistance to streptomycin, spectinomycin,
or a combination of both antibiotics, _aadA, strA_ and
_strB_ encoding resistance to spectinomycin /
streptomycin were detected in 15 (65%) (including
two isolates, EC123 and EC226 with intermediate
resistance to streptomycin and spectinomycin), 16
(70%) and 14 (61%) isolates respectively. Five of
these isolates were positive for the _aadA/strA/strB_
combination while _strA_ was found to occur together
with _strB_ in all 14 isolates positive for _strB_ . Further,
12 (86%) of 14 isolates carrying the _strA/strB_
combination were also positive for _sul2._

The only tested ampicillin resistance gene _bla-TEM_
was found in 11 (85%) of 13 ampicillin-resistant
isolates in the present study. The highest prevalence (7
of 8 ampicillin resistant isolates) of this gene was
found among isolates from Ogbomoso. _catA1_ and
_cmlA1_ was detected in 2 (25%) and 1 (13%) of eight
chloramphenicol resistant isolates respectively.


1106




**Table 2.** Distribution of MICs and occurrence of resistance among the _E. coli_ isolates from each source

**Compound** **% Resistance** **Distribution of MICs**


|0.015 0.03 0.06 0.125 0.25 0.5 1 2 4 8 16|Col2|Col3|Col4|32 64 128 256 512 1024 >1024 mg/L|Col6|
|---|---|---|---|---|---|
|**Chloramphenicol**<br>**POa=27**<br>**5 **<br>**6 **<br> <br>**SWb=0**<br>**3 **<br>**4 **<br>**1 **<br>**SOc=33**<br>**1 **<br>**4 **<br>**1 **<br> <br>**WAd=25**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br> <br>**1 **|**Chloramphenicol**<br>**POa=27**<br>**5 **<br>**6 **<br> <br>**SWb=0**<br>**3 **<br>**4 **<br>**1 **<br>**SOc=33**<br>**1 **<br>**4 **<br>**1 **<br> <br>**WAd=25**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br> <br>**1 **|**Chloramphenicol**<br>**POa=27**<br>**5 **<br>**6 **<br> <br>**SWb=0**<br>**3 **<br>**4 **<br>**1 **<br>**SOc=33**<br>**1 **<br>**4 **<br>**1 **<br> <br>**WAd=25**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br> <br>**1 **|**Chloramphenicol**<br>**POa=27**<br>**5 **<br>**6 **<br> <br>**SWb=0**<br>**3 **<br>**4 **<br>**1 **<br>**SOc=33**<br>**1 **<br>**4 **<br>**1 **<br> <br>**WAd=25**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br> <br>**1 **|**2 **<br>**2 **<br>**1 **<br>**1 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>|**2 **<br>**2 **<br>**1 **<br>**1 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>|
|**Florfenicol**<br>**PO=0**<br>**5 **<br>**8 **<br>**2 **<br>**SW=0**<br>**3 **<br>**5 **<br> <br>**SO=0**<br>**2 **<br>**5 **<br>**2 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**2 **<br>|**Florfenicol**<br>**PO=0**<br>**5 **<br>**8 **<br>**2 **<br>**SW=0**<br>**3 **<br>**5 **<br> <br>**SO=0**<br>**2 **<br>**5 **<br>**2 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**2 **<br>|**Florfenicol**<br>**PO=0**<br>**5 **<br>**8 **<br>**2 **<br>**SW=0**<br>**3 **<br>**5 **<br> <br>**SO=0**<br>**2 **<br>**5 **<br>**2 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**2 **<br>|**Florfenicol**<br>**PO=0**<br>**5 **<br>**8 **<br>**2 **<br>**SW=0**<br>**3 **<br>**5 **<br> <br>**SO=0**<br>**2 **<br>**5 **<br>**2 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**2 **<br>|<br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br>|
|**Trimethoprim**<br>**PO=53**<br>**7 **<br> <br>**SW=25**<br>**7 **<br> <br>**SO=56**<br>**4 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Trimethoprim**<br>**PO=53**<br>**7 **<br> <br>**SW=25**<br>**7 **<br> <br>**SO=56**<br>**4 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br>**1 **<br>|**8 **<br>**1 **<br>**1 **<br>**4 **<br> <br> <br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|**8 **<br>**1 **<br>**1 **<br>**4 **<br> <br> <br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|**8 **<br>**1 **<br>**1 **<br>**4 **<br> <br> <br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|**8 **<br>**1 **<br>**1 **<br>**4 **<br> <br> <br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|
|**Amoxicillin + Clavulanate**<br>**PO=0**<br>**3 **<br>**5 **<br>**5 **<br>**2 **<br>**SW=0**<br>**5 **<br>**3 **<br> <br>**SO=0**<br>**1 **<br>**6 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>**1 **|**Amoxicillin + Clavulanate**<br>**PO=0**<br>**3 **<br>**5 **<br>**5 **<br>**2 **<br>**SW=0**<br>**5 **<br>**3 **<br> <br>**SO=0**<br>**1 **<br>**6 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>**1 **|**Amoxicillin + Clavulanate**<br>**PO=0**<br>**3 **<br>**5 **<br>**5 **<br>**2 **<br>**SW=0**<br>**5 **<br>**3 **<br> <br>**SO=0**<br>**1 **<br>**6 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>**1 **|**Amoxicillin + Clavulanate**<br>**PO=0**<br>**3 **<br>**5 **<br>**5 **<br>**2 **<br>**SW=0**<br>**5 **<br>**3 **<br> <br>**SO=0**<br>**1 **<br>**6 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>**1 **|<br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br>|
|**Ceftiofur**<br>**PO=0**<br>**14**<br>**1 **<br>**SW=0**<br>**8 **<br> <br>**SO=0**<br>**9 **<br> <br> <br>**WA=0**<br> <br> <br> <br> <br> <br>**4 **<br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|
|**Nalidixic Acid**<br>**PO=33**<br>**5 **<br>**2 **<br>**3 **<br>**SW=13**<br>**7 **<br> <br>**SO=11**<br>**5 **<br>**2 **<br>**1 **<br> <br>**WA=50**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br> <br>|**Nalidixic Acid**<br>**PO=33**<br>**5 **<br>**2 **<br>**3 **<br>**SW=13**<br>**7 **<br> <br>**SO=11**<br>**5 **<br>**2 **<br>**1 **<br> <br>**WA=50**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br> <br>|**Nalidixic Acid**<br>**PO=33**<br>**5 **<br>**2 **<br>**3 **<br>**SW=13**<br>**7 **<br> <br>**SO=11**<br>**5 **<br>**2 **<br>**1 **<br> <br>**WA=50**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br> <br>|**Nalidixic Acid**<br>**PO=33**<br>**5 **<br>**2 **<br>**3 **<br>**SW=13**<br>**7 **<br> <br>**SO=11**<br>**5 **<br>**2 **<br>**1 **<br> <br>**WA=50**<br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br> <br>|**1 **<br>**4 **<br>**1 **<br>**1 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br>|**1 **<br>**4 **<br>**1 **<br>**1 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br>|
|**Neomycin**<br>**PO=13**<br>**12**<br>**1 **<br> <br>**SW=0**<br>**8 **<br> <br>**SO=22**<br>**5 **<br>**2 **<br> <br> <br>**WA=25**<br> <br> <br> <br> <br> <br> <br> <br>**3 **<br> <br>|**Neomycin**<br>**PO=13**<br>**12**<br>**1 **<br> <br>**SW=0**<br>**8 **<br> <br>**SO=22**<br>**5 **<br>**2 **<br> <br> <br>**WA=25**<br> <br> <br> <br> <br> <br> <br> <br>**3 **<br> <br>|**Neomycin**<br>**PO=13**<br>**12**<br>**1 **<br> <br>**SW=0**<br>**8 **<br> <br>**SO=22**<br>**5 **<br>**2 **<br> <br> <br>**WA=25**<br> <br> <br> <br> <br> <br> <br> <br>**3 **<br> <br>|**1 **<br>**1 **<br>**2 **<br> <br> <br>**1 **<br> <br> <br> <br> <br> <br>|**1 **<br>**1 **<br>**2 **<br> <br> <br>**1 **<br> <br> <br> <br> <br> <br>|**1 **<br>**1 **<br>**2 **<br> <br> <br>**1 **<br> <br> <br> <br> <br> <br>|
|**Spectinomycin**<br>**PO=47**<br>**8 **<br> <br>**SW=13**<br>**6 **<br>**2 **<br> <br>**SO=44**<br>**2 **<br>**2 **<br>**1 **<br> <br>**WA=25**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>|**Spectinomycin**<br>**PO=47**<br>**8 **<br> <br>**SW=13**<br>**6 **<br>**2 **<br> <br>**SO=44**<br>**2 **<br>**2 **<br>**1 **<br> <br>**WA=25**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>|**Spectinomycin**<br>**PO=47**<br>**8 **<br> <br>**SW=13**<br>**6 **<br>**2 **<br> <br>**SO=44**<br>**2 **<br>**2 **<br>**1 **<br> <br>**WA=25**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>|**Spectinomycin**<br>**PO=47**<br>**8 **<br> <br>**SW=13**<br>**6 **<br>**2 **<br> <br>**SO=44**<br>**2 **<br>**2 **<br>**1 **<br> <br>**WA=25**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>|**Spectinomycin**<br>**PO=47**<br>**8 **<br> <br>**SW=13**<br>**6 **<br>**2 **<br> <br>**SO=44**<br>**2 **<br>**2 **<br>**1 **<br> <br>**WA=25**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**2 **<br>**1 **<br>|**2 **<br>**3 **<br>**2 **<br>**2 **<br>**3 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br>|




**Table 2.** _**(continued)**_ Distribution of MICs and occurrence of resistance among the _E. coli_ isolates from each source

|Streptomycin PO=67 4 1<br>SW=50 3 1<br>SO=44 4 1<br>WA=50 1 1|Col2|Col3|Col4|Col5|Col6|1 1 8<br>1 2 1<br>1 3<br>2|Col8|
|---|---|---|---|---|---|---|---|
|**Tetracycline**<br>**PO=87**<br>**1 **<br>**1 **<br> <br>**SW=75**<br>**2 **<br> <br>**SO=89**<br>**2 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Tetracycline**<br>**PO=87**<br>**1 **<br>**1 **<br> <br>**SW=75**<br>**2 **<br> <br>**SO=89**<br>**2 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Tetracycline**<br>**PO=87**<br>**1 **<br>**1 **<br> <br>**SW=75**<br>**2 **<br> <br>**SO=89**<br>**2 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Tetracycline**<br>**PO=87**<br>**1 **<br>**1 **<br> <br>**SW=75**<br>**2 **<br> <br>**SO=89**<br>**2 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Tetracycline**<br>**PO=87**<br>**1 **<br>**1 **<br> <br>**SW=75**<br>**2 **<br> <br>**SO=89**<br>**2 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**13**<br>**6 **<br>**7 **<br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|**13**<br>**6 **<br>**7 **<br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|**13**<br>**6 **<br>**7 **<br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|
|**Sulphamethoxazole**<br>**PO=73**<br>**4 **<br> <br>**SW=25**<br>**6 **<br> <br>**SO=89**<br>**1 **<br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br> <br>|**Sulphamethoxazole**<br>**PO=73**<br>**4 **<br> <br>**SW=25**<br>**6 **<br> <br>**SO=89**<br>**1 **<br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br> <br>|**Sulphamethoxazole**<br>**PO=73**<br>**4 **<br> <br>**SW=25**<br>**6 **<br> <br>**SO=89**<br>**1 **<br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br> <br>|**Sulphamethoxazole**<br>**PO=73**<br>**4 **<br> <br>**SW=25**<br>**6 **<br> <br>**SO=89**<br>**1 **<br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br> <br>|**Sulphamethoxazole**<br>**PO=73**<br>**4 **<br> <br>**SW=25**<br>**6 **<br> <br>**SO=89**<br>**1 **<br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br> <br>|**Sulphamethoxazole**<br>**PO=73**<br>**4 **<br> <br>**SW=25**<br>**6 **<br> <br>**SO=89**<br>**1 **<br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br> <br>|**Sulphamethoxazole**<br>**PO=73**<br>**4 **<br> <br>**SW=25**<br>**6 **<br> <br>**SO=89**<br>**1 **<br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br> <br>|**11**<br>**2 **<br>**8 **<br> <br> <br>**3 **<br>|
|**Colistin**<br>**PO=0**<br>**15**<br> <br>**SW=0**<br>**8 **<br> <br>**SO=22**<br>**9 **<br> <br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br>**4 **<br>|**Colistin**<br>**PO=0**<br>**15**<br> <br>**SW=0**<br>**8 **<br> <br>**SO=22**<br>**9 **<br> <br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br>**4 **<br>|**Colistin**<br>**PO=0**<br>**15**<br> <br>**SW=0**<br>**8 **<br> <br>**SO=22**<br>**9 **<br> <br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br>**4 **<br>|**Colistin**<br>**PO=0**<br>**15**<br> <br>**SW=0**<br>**8 **<br> <br>**SO=22**<br>**9 **<br> <br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br>**4 **<br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|
|**Ciprofloxacin**<br>**PO=60**<br>**6 **<br> <br>**SW=13**<br>**7 **<br> <br>**SO=33**<br>**5 **<br>**1 **<br> <br>**WA=50**<br>**1 **<br>**1 **|**3 **<br>**2 **<br>**2 **<br>**2 **<br>**1 **<br>**2 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>**1 **<br> <br> <br> <br> <br> <br> <br> <br> <br>|**3 **<br>**2 **<br>**2 **<br>**2 **<br>**1 **<br>**2 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>**1 **<br> <br> <br> <br> <br> <br> <br> <br> <br>|**3 **<br>**2 **<br>**2 **<br>**2 **<br>**1 **<br>**2 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>**1 **<br> <br> <br> <br> <br> <br> <br> <br> <br>|**3 **<br>**2 **<br>**2 **<br>**2 **<br>**1 **<br>**2 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>**1 **<br> <br> <br> <br> <br> <br> <br> <br> <br>|**3 **<br>**2 **<br>**2 **<br>**2 **<br>**1 **<br>**2 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>**1 **<br> <br> <br> <br> <br> <br> <br> <br> <br>|**3 **<br>**2 **<br>**2 **<br>**2 **<br>**1 **<br>**2 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>**1 **<br> <br> <br> <br> <br> <br> <br> <br> <br>|**3 **<br>**2 **<br>**2 **<br>**2 **<br>**1 **<br>**2 **<br>**1 **<br> <br>**1 **<br> <br> <br> <br> <br> <br>**1 **<br> <br> <br> <br> <br> <br> <br> <br> <br>|
|**Ampicillin**<br>**PO=40**<br>**4 **<br>**4 **<br>**1 **<br>**SW=25**<br>**1 **<br>**2 **<br>**3 **<br> <br>**SO=22**<br>**2 **<br>**4 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Ampicillin**<br>**PO=40**<br>**4 **<br>**4 **<br>**1 **<br>**SW=25**<br>**1 **<br>**2 **<br>**3 **<br> <br>**SO=22**<br>**2 **<br>**4 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Ampicillin**<br>**PO=40**<br>**4 **<br>**4 **<br>**1 **<br>**SW=25**<br>**1 **<br>**2 **<br>**3 **<br> <br>**SO=22**<br>**2 **<br>**4 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Ampicillin**<br>**PO=40**<br>**4 **<br>**4 **<br>**1 **<br>**SW=25**<br>**1 **<br>**2 **<br>**3 **<br> <br>**SO=22**<br>**2 **<br>**4 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**Ampicillin**<br>**PO=40**<br>**4 **<br>**4 **<br>**1 **<br>**SW=25**<br>**1 **<br>**2 **<br>**3 **<br> <br>**SO=22**<br>**2 **<br>**4 **<br> <br> <br>**WA=75**<br> <br> <br> <br> <br> <br> <br> <br> <br>**1 **<br>|**6 **<br>**2 **<br>**1 **<br> <br>**2 **<br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|**6 **<br>**2 **<br>**1 **<br> <br>**2 **<br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|**6 **<br>**2 **<br>**1 **<br> <br>**2 **<br> <br> <br>**3 **<br> <br> <br> <br> <br> <br>|
|**Gentamicin**<br>**PO=7**<br>**14**<br> <br>**SW=0**<br>**7 **<br>**1 **<br>**SO=22**<br>**4 **<br>**2 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br>**4 **<br>|**Gentamicin**<br>**PO=7**<br>**14**<br> <br>**SW=0**<br>**7 **<br>**1 **<br>**SO=22**<br>**4 **<br>**2 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br>**4 **<br>|**Gentamicin**<br>**PO=7**<br>**14**<br> <br>**SW=0**<br>**7 **<br>**1 **<br>**SO=22**<br>**4 **<br>**2 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br>**4 **<br>|**1 **<br>**1 **<br>**2 **<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|**1 **<br>**1 **<br>**2 **<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|**1 **<br>**1 **<br>**2 **<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|**1 **<br>**1 **<br>**2 **<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|**1 **<br>**1 **<br>**2 **<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|
|**Cefotaxime**<br>**PO=0**<br>**14**<br>**1 **<br>**SW=0**<br>**7 **<br>**1 **<br>**SO=0**<br>**9 **<br> <br> <br>**WA=0**<br> <br> <br> <br>**4 **<br>|**Cefotaxime**<br>**PO=0**<br>**14**<br>**1 **<br>**SW=0**<br>**7 **<br>**1 **<br>**SO=0**<br>**9 **<br> <br> <br>**WA=0**<br> <br> <br> <br>**4 **<br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>|
|**Apramycin**<br>**PO=0**<br>**11**<br>**4 **<br> <br>**SW=0**<br>**7 **<br>**1 **<br> <br>**SO=0**<br>**7 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**4 **<br> <br>|**Apramycin**<br>**PO=0**<br>**11**<br>**4 **<br> <br>**SW=0**<br>**7 **<br>**1 **<br> <br>**SO=0**<br>**7 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**4 **<br> <br>|**Apramycin**<br>**PO=0**<br>**11**<br>**4 **<br> <br>**SW=0**<br>**7 **<br>**1 **<br> <br>**SO=0**<br>**7 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**4 **<br> <br>|**Apramycin**<br>**PO=0**<br>**11**<br>**4 **<br> <br>**SW=0**<br>**7 **<br>**1 **<br> <br>**SO=0**<br>**7 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**4 **<br> <br>|**Apramycin**<br>**PO=0**<br>**11**<br>**4 **<br> <br>**SW=0**<br>**7 **<br>**1 **<br> <br>**SO=0**<br>**7 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**4 **<br> <br>|**Apramycin**<br>**PO=0**<br>**11**<br>**4 **<br> <br>**SW=0**<br>**7 **<br>**1 **<br> <br>**SO=0**<br>**7 **<br>**1 **<br>**1 **<br> <br>**WA=0**<br> <br> <br> <br> <br> <br> <br> <br> <br>**4 **<br> <br>|<br> <br> <br> <br> <br> <br> <br>|<br> <br> <br> <br> <br> <br> <br>|



-the frequency of resistance to each concentration of antibiotics tested in this study. Solid vertical lines indicate EUCAST epidemiological cut off values. Where only a solid line is shown, the cut off value
and breakpoint are the same. Footnotes a: PO = Poultry waste (n = 15), b: SW = Swine waste (n = 8), c: SO = Farm Soil and fish pond sediments (n = 9), d: WA = well water (n = 4).




**Table 3.** Phenotypic pattern of resistance and resistance genes profiles of _E. coli_ isolates included in this study

**Isolates** **Farm** ~~**[a ]**~~ **Source** **Phenotypic resistance** **Resistance genes profile**
**IBADAN**
EC3 IB2 Poultry droppings chl, tmp, nal, neo, spe, str, smx, cip, amp _int1/sul1_ ~~_[b]_~~ _, int2, sul2, bla-TEM, aadA, strA, strB_
EC40 IB2 Poultry droppings nal, str, tet, smx, cip, amp _sul2, bla-TEM, strA, strB_
EC42 IB2 Poultry droppings chl, tet, cip _qnrS_
EC54 IB1 Poultry droppings tmp, str, tet, smx, cip, amp _qnrS, sul2, bla-TEM, strA, strB_
**OGBOMOSO**
EC70 OG1 Poultry litter tmp, spe, str, tet, smx, cip, amp _qnrS, int2, sul2, bla-TEM, aadA, strA, strB, tet(A)_
EC75 OG1 Poultry litter tet _tet(A)_
EC87 OG1 Poultry litter chl, tmp, nal, spe, str, tet, smx, cip, amp _sul3, bla-TEM, aadA_
EC78 OG4 Poultry droppings tmp, nal, tet, smx, cip _sul2, tet(A)_
EC111 OG4 Poultry droppings spe, str, tet _int2, aadA_
EC114 OG2 Poultry droppings tmp, neo, str, tet, smx, amp _bla-TEM_, _strA, strB, tet(A)_
EC157 OG3 Poultry droppings tmp, spe, str, tet, smx _int2, sul2, aadA, strA, strB, tet(A)_
EC199 OG4 Poultry droppings nal, str, tet, smx, cip _qnrS, sul2, strA_
EC232 OG1 Poultry droppings chl, tet, smx, gen _sul3, aadA, cmlA_
EC146 OG4 Swine waste Susceptible to all Antibiotics tested
EC160 OG4 Swine waste str, tet, smx, amp _sul2, bla-TEM, strA, strB_
EC170 OG4 Swine waste str, tet _strA, strB, tet(B)_
EC96 OG4 Fish pond sediment nal, str, tet, smx, cip _sul2, strA, strB, tet(A)_
EC233 OG4 Fish pond sediment str, tet, smx, cip _qnrS, sul2, strA, strB_
EC69 OG4 Manured soil chl, spe, tet, smx, gen _sul3, aadA_
EC113 OG2 Farm soil chl, tmp, neo, spe, str, tet, smx, cip, amp _qnrS, int2, sul2, bla-TEM, catA1, aadA,strA, strB_
EC117 OG3 Farm soil tmp, neo, spe, str, tet, smx, amp _int1/sul1, sul2, aadA, strA, strB, tet(B)_
EC121 OG3 Farm soil tmp, spe, tet, smx _int2, sul2, aadA, tet(B)_
EC162 OG3 Farm soil Susceptible to all Antibiotics tested
EC1801 OG3 Farm soil tmp, smx _sul2_
EC226 OG4 Farm soil chl, tet, smx, gen _sul3, aadA_
EC116 OG3 Well water 2 Susceptible to all Antibiotics tested
EC142 OG3 Well water 2 tmp, nal, str, tet, smx, cip, amp _sul2, bla-TEM, strA, strB, tet(B)_
EC164 OG3 Well water 5 chl, tmp, nal, neo, str, tet, smx, cip, amp _int1/sul1, sul2, bla-TEM, catA1, strA, tet(B)_
**OSOGBO**
EC143 OS1 Poultry droppings tmp, tet, cip _qnrS_
EC149 OS1 Poultry droppings str, smx _sul2, strA, strB_
EC122 OS1 Swine waste tet No resistance genes detected
EC123 OS1 Swine waste tmp, nal, tet, smx, cip _int1/sul1, aadA_
EC137 OS1 Swine waste Susceptible to all Antibiotics tested
EC138 OS1 Swine waste tmp, str, tet, amp _aadA_
EC236 OS1 Swine waste spe, str, tet _int2, aadA_
EC197 OS2 Well water tmp, spe, tet, smx, amp _int1/sul1, bla-TEM, aadA_
chl; chloramphenicol, tmp; trimethoprim, nal; naldixic acid, neo; neomycin, str; streptomycin, spe; spectinomycin, smx; sulphamethoxazole, cip; ciprofloxacin, amp; ampicillin, gen; gentamicin, tet;
tetracycline.
Footnote a: See Table 1 for farm codes.
Footnote b: _sul1_ gene was detected as part of class 1 integrons in all _int1_ positive isolates.




Adelowo _et al. –_ Resistant _E. coli_ from Nigeria _J Infect Dev Ctries_ 2014; 8(9):1103-1112.



None of the eight chloramphenicol resistant isolates,
including those that were negative for _catA1_ and _cmlA_,
were positive for _floR_ . Majority of the tetracycline
resistant isolates did not give positive amplification
signals for most of the tetracycline resistant genes we
tested for.

Even though all positive controls gave positive
amplification signals, _tet_ (C), _tet_ (D) _, tet_ (E), _tet_ (G) or
_tet_ (39) could not be detected among the 29 tetracycline
resistant isolates. However, _tet_ (A) and _tet_ (B) were
found in six (21%) and five (17%) isolates
respectively. Compared to the present study, other
studies reported higher frequencies of _tet_ (A) and
_tet_ (B). For example, in a study of 317 _E. coli_ isolated
from pigs, cattle and poultry in Germany, Guerra _et al._

[15] found _tet_ (A) and _tet_ (B) in 66% and 42% of the
isolates respectively. Similarly, Enne _et al_ [26] found
56% of 103 tetracycline resistant _E. coli_ from farm
animals at slaughter to contain _tet_ (B). _acc(3)-IV,_ the
only gentamicin resistance gene tested for in this study
was not detected in any of the four gentamicin
resistant isolates.

Seven out of eight isolates that were resistant to
ciprofloxacin and had MIC values to nalidixic acid of
≤32 mg/L contained _qnrS._ Of these isolates, the PCR
products of three isolates (EC54, EC143, EC199) were
sequenced and the sequences showed 100% identity to
_qnrS1_ from _E. coli_ (GenBank accession number
GQ214053).


**Discussion**

Development of resistance to antimicrobial agents
is a problem of global proportion which is increasingly
frustrating efforts to treat infectious diseases. This has
been attributed to human and agricultural use of
antimicrobials. Thus, our study investigated the
occurrence and molecular mechanisms of antibiotic
resistance among commensal _E. coli_ isolated from
selected poultry farms in southwestern Nigeria, an area
of the world where little is known on antibiotic use
and its contribution to the development and
dissemination of resistance. A serious limitation to this
study was the difficulty encountered in sample
collection which limited the number of isolates
included in the study. It was difficult convincing farm
owners to participate in the study and most of the
volunteer farms restricted access to specified areas of
the farms, and none allowed sample collection more
than twice. Nonetheless, our findings demonstrated
that the commensal _E. coli_ included in this study were
commonly resistant to antibiotics used in human and
veterinary medicine. However, because our study did



not include isolates from farms without antimicrobial
use, it was difficult to draw a definite conclusion on
the association between drug use on the study farms
and the occurrence of resistance in the isolates.

The observed levels of resistance to ciprofloxacin
(42%) and trimethoprim (47%) were however
unexpected since only one of the eight farms admitted
using the fluoroquinolone enrofloxacin and no farm
used trimethoprim. Thus, it is quite possible that use of
trimethoprim, a common component of poultry premixes used widely in the farms of the present study
was underreported. More importantly, our observation
about the levels of resistance to ciprofloxacin and
trimethoprim suggested that factors other than
antibiotic use may be contributing to the selection of
resistance among the present isolates.

Similar to the observed pattern of phenotypic
resistance, genes conferring resistance to
streptomycin/spectinomycin, sulphamethoxazole and
ampicillin were also found to be common among the
isolates. However, it seemed that local and
geographical factors played a role in the occurrence of
some resistance genes; all isolates positive for _sul3_ and
tetracycline resistance genes are from Ogbomoso. In a
sharp contrast to this, other genes seem to have a
widespread occurrence within the study farms. For
example, isolates with _qnrS_ originated from six
different farms (including the farm admitting usage of
fluoroquinolones) and from four different sources:
poultry droppings, fish pond sediment and farm soil.
The only isolate that did not contain any of the
quinolone resistance genes screened for was from well
water (EC142). The widespread occurrence of _qnrS_
observed in this study may be a result of their coselection and transfer with other resistance
determinants on mobile genetic elements. _qnr_, a
plasmid-mediated horizontally transferable gene
encoding quinolone resistance was discovered for the
first time in 1998 and has been in circulation for at
least 20 years [27]. Qnr proteins are capable of
protecting DNA gyrase from the action of quinolones
and _qnr_ are increasingly detected in isolates of clinical
importance [28]. Our findings indicated that _qnrS_ was
widely spread among poultry and their production
environment in Nigeria. _qnrS1_ was recently found
among _Salmonella_ and _E. coli_ from humans and
animal sources in Nigeria [29,13].

Our study provided additional evidence that food
animals production environments may be important
reservoirs of clinically important antibiotic resistance
genes such as plasmid-mediated quinolone resistance
(PMQR) genes in this part of the world. This is of


1110




Adelowo _et al. –_ Resistant _E. coli_ from Nigeria _J Infect Dev Ctries_ 2014; 8(9):1103-1112.



concern as fluoroquinolones were listed by the World
Health Organization as critically important
antimicrobials for human health [30]. Therefore, it
seems very important to pay more attention to the
spread of resistance through such production systems
in the tropical climate, and regulation of antimicrobial
usage in production systems with close interaction
between human and animal ecosystems such as this,
prevalent in the developing world, seems crucial.

Even though we tested for the resistance genes
most commonly found in _E. coli_, resistance in a
number of isolates appeared to be conferred by
mechanisms other than those tested in this study. Thus
resistance genes investigated in this study could not be
detected in two ampicillin-resistant, one
sulphamethoxazole-resistant, five chloramphenicolresistant and eighteen tetracycline-resistant isolates.
These isolates may contain known mechanisms not
tested for in this study or unknown mechanisms. The
large fraction of tetracycline resistant isolates without
any of the tested tetracycline resistance genes is
unexpected and may be due to geographical
differences between this and previous studies on
tetracycline resistant _E. coli._ The results of this study
underscore the importance of routine screening for
antibiotic resistance in commensal bacteria from food
animal production farms using both phenotypic and
genotypic methods. This is very crucial, especially in
countries such as Nigeria with limited studies on
occurrence of resistance genes in food animal
production systems.

In conclusion, data from the present study
suggested that commensal _E. coli_ from the study farms
may act as reservoirs of antimicrobial drug resistance
genes such as _qnrS_ which may be mobilized into
human populations. To the best of our knowledge, this
is the first comprehensive study of molecular
mechanisms of resistance in commensal _E. coli_
associated with livestock farms in Nigeria. Results
should however be interpreted with caution as a
limited number of isolates were investigated and a
comprehensive history of antibiotic use necessary to
make informed decision on the contribution of
antibiotic use to the observed resistance is not
available. It however pointed attention to the critical
need for the regulation of antimicrobial drug usage in
livestock production and continuous monitoring of
antibiotic resistance in developing countries for public
health safety.



**Acknowledgements**
We would like to thank Jacob Dyring Jensen and Hanne
Mordhorst for excellent technical assistance and the
International Union of Microbiological Societies (IUMS),
Society for General Microbiology (SGM), UK and
UNESCO for a Short term Fellowship to the first author
under the UNESCO-IUMS-SGM Short term fellowship.

**References**
1. Bauer – Garland J, Frye JG, Gray JT, Berrang ME, Harrison
MA, Fedorka–Cray PJ (2006) Transmission of _Salmonella_
_entrica_ serotype Typhimurium in Poultry with or without
Antimicrobial Selective Pressure. J Appl Microbiol _,_ 100:
1301-1308.
2. Gosh S, LaPara, TM (2007) The effects of subtherapeutic
antibiotic use in farm animals on the proliferation and
persistence of antibiotic resistance among soil bacteria. ISME
J 1: 191-203.
3. Marshall BM, Levy SB (2011) Food animals and
antimicrobials: Impacts on human health. Clin Microbiol Rev
24: 718-733.
4. WHO (2011) Tackling antibiotic resistance from a food safety
perspective in Europe. Available:
http://www.euro.who.int/__data/assets/pdf_file/0005/136454/
e94889.pdf. Accessed 12 December 2012.
5. Heuer H, Smalla, K (2007) Manure and sulfadiazine
synergistically increased bacterial antibiotic resistance in soil
over at least two months. Environ Microbiol 9: 657-666.
6. Adelowo OO, Ojo FA, Fagade OE (2009) Prevalence of
multiple antibiotic resistance among bacteria isolates from
selected poultry wastedumps in Southwestern Nigeria. World
J Microbiol Biotechnol 25: 713-719.
7. Kelley TR, Pancorbo OC, Merka WC, Barnhart HM (1998)
Antibiotic resistance of bacterial litter isolates. Poult Sci 77:
243–247.
8. Khan AA, Nawaz MS, Summage WC, Khan SA, Lin J (2005)
Isolation and molecular characterization of fluoroquinoloneresistant _Escherichia coli_ from poultry litter. Poult Sci 84:
61–66.
9. Furtula V, Farrell EG, Diarrassouba F, Rempel H, Pritchard J,
Diarra MS (2010) Veterinary pharmaceuticals and antibiotic
resistance of _Escherichia coli_ isolates in poultry litter from
commercial farms and controlled feeding trials. Poult Sci 89:
180–188
10. Hayes JR, English LL, Carr LE, Wagner DD, Joseph SW

(2004) Multiple-antibiotic resistance of _Enterococcus_ spp.
Isolated from commercial poultry production environments.
Appl Environ Microbiol 70: 6005–6011.
11. Nilsson O, Greko C, Bengtsson B (2009) Environmental

contamination by vancomycin resistant enterococci (VRE) in
Swedish broiler production. Acta Veterinaria Scandinavica
doi:10.1186/1751-0147-51-49 _._
12. Girlich D, Poirel L, Carattoli A, Kempf I, Lartigue MF,

Bertini A, Nordman P, (2007) Extended-spectrum βlactamase CTX-M-1 in _Escherichia coli_ isolates from healthy
poultry in France. Appl Environ Microbiol 73: 4681–4685.
13. Fortini D, Fashae K, Garcia-Fernandez A, Villa L, Carattoli A

(2011) Plasmid mediated quinolone resistance and βlactamases in _E. coli_ from healthy animals from Nigeria. J
Antimicrob Chemother 66: 1269-1272.


1111




Adelowo _et al. –_ Resistant _E. coli_ from Nigeria _J Infect Dev Ctries_ 2014; 8(9):1103-1112.



14. Rwego IB, Gillespie TR, Isabirye-Basuta G, Goldberg TL

(2008) High Rates of _Escherichia coli_ Transmission between
Livestock and Humans in Rural Uganda. J Clin Microbiol 46:
3187-3191.
15. Guerra B, Junker E, Schroeter A, Malorny B, Lehmann S,

Helmuth R (2003) Phenotypic and genotypic characterization
of antimicrobial resistance in German Escherichia coli
isolates from cattle, swine and poultry. J Antimicrob
Chemother 52: 489-492.
16. Agersø Y, Sandvang D (2005) Class I integrons and

tetracycline resistance genes in _Alcaligenes, Arthrobacter_ and
_Pseudomonas_ spp. isolated from pigsties and manured soil.
Appl Environ Microbiol 46: 7941-7947.
17. Ogunleye AO, Oyekunle MA, Sonibare AO (2008) Multidrug

resistant _Escherichia coli_ isolates of poultry origin in
Abeokuta, South Western Nigeria. Vet. arhiv 78:501-509.
18. Adelowo OO, Akinlabi IA, Fagade OE (2012) Environmental

impact assessment of Attenda abattoir, Ogbomoso
southwestern Nigeria on surface and groundwater quality
using geo-electrical imaging and microbiological analysis.
Environ Monit Assess 184:4565-4574.
19. [CLSI]Clinical and Laboratory Standards Institute (2005)

Performance Standards for Antimicrobial Susceptibility
Testing: Fifteenth Informational Supplement M100-S15 _._
CLSI, Wayne, PA, USA.
20. Peirano G, Agersø Y, Aarestrup FM, Falavina dos Reis FM,

Rodirgues DP (2006) Occurrence of integrons and
antimicrobial resistance genes among _Salmonella enterica_
from Brazil. J Antimicrob Chemother 58: 305-309.
21. Peirano G, Agersø Y, Aarestrup FM, Rodirgues DP (2005)

Occurrence of integrons and resistance genes among
sulphonamide-resistant _Shigella_ spp. from Brazil. J
Antimicrob Chemother 55: 301-305.
22. Agersø Y, Guardabassi L (2005) Identification of Tet 39, a

novel class of tetracycline resistance determinant in
_Acinetobacter_ spp. of environmental and clinical origin. J
Antimicrob Chemother 55: 566-569.
23. Agersø Y, Bruun MS, Dalsgaard I, Larsen JL (2007) The

tetracycline resistance gene _tet_ (E) is frequently occurring and
present on large horizontally transferable plasmids in
_Aeromonas_ spp. from fish farms. Aquaculture 266: 47-52
24. Cavaco LM, Hasman H, Xia S, Aarestrup FM (2009) _qnrD_, a

novel gene conferring transferable quinolone resistance in



_Salmonella enterica_ serovar Kentucky and Bovismorbificans
strains of human origin. Antimicrob Agents Chemother 53:
603-608.
25. Wang M, Guo Q, Xu X, Wang X, Ye X, Wu S, Hooper DC,

Wang M (2009) New plasmid-mediated quinolone resistance
gene, qnrC, found in a clinical isolate of _Proteus mirabilis._
Antimicrob Agents Chemother 53: 1892-1897.
26. Enne VI, Cassar C, Sprigings K, Woodward MJ, Bennett PM

(2008) A high prevalence of antimicrobial resistant
_Escherichia coli_ isolated from pigs and a low prevalence of
antimicrobial resistant _E. coli_ from cattle and sheep in Great
Britain at slaughter. FEMS Microbiol Lett 278: 193-199.
27. Martinez-Martinez L, Pascual A, Jacoby GA (1998)

Quinolone resistance from a transferable plasmid. Lancet 351:
797–799.
28. Strahilevitz J, Jacoby GA, Hooper DC, Robicsek A (2009)

Plasmid-Mediated Quinolone Resistance: a Multifaceted
Threat. Clin Microbial Rev 22: 664–689.
29. Fashae K, Ogunsola F, Aarestrup FM, Hendricksen RS (2010)

Antimicrobial susceptibility and serovars of _Salmonella_ from
chickens and humans in Ibadan, Nigeria. J Infect Dev Ctries
4: 484-494. doi:10.3855/jidc.909.
30. Collignon P, Powers JH, Chiller TM, Aidara-Kane A,

Aarestrup FM (2009) World Health Organization ranking of
antimicrobials according to their importance in human
medicine: A critical step for developing risk management
strategies for the use of antimicrobials in food production
animals. Clin Infect Dis 49: 132-141.

**Corresponding author**
Yvonne Agersø
Department for Microbiology and Risk Assessment, National Food
Institute,
Technical University of Denmark, Kemitorvet building 204st
DK-2800 Lyngby, Denmark
Phone +4535886273
Fax: +4535886341
Email: yvoa@food.dtu.dk

**Conflict of interests:** No conflict of interests is declared.


1112


