Moawad _et al. Gut Pathog (2017) 9:57_
DOI 10.1186/s13099-017-0206-9

### **RESEARCH**


## Gut Pathogens

### **Open Access**


# Occurrence of Salmonella enterica and Escherichia coli in raw chicken and beef meat in northern Egypt and dissemination of their antibiotic resistance markers

Amira A. Moawad [1,2], Helmut Hotzel [1], Omnia Awad [4], Herbert Tomaso [1], Heinrich Neubauer [1], Hafez M. Hafez [5]
and Hosny El‑Adawy [1,3*]


**Abstract**
**Background:** The global incidence of foodborne infections and antibiotic resistance is recently increased and con‑
sidered of public health concern. Currently, scarcely information is available on foodborne infections and ESBL associ‑
ated with poultry and beef meat in Egypt.
**Methods:** In total, 180 chicken and beef meat samples as well as internal organs were collected from different
districts in northern Egypt. The samples were investigated for the prevalence and antibiotic resistance of _Salmonella_
_enterica_ serovars and _Escherichia coli_ . All isolates were investigated for harbouring class 1 and class 2 integrons.
**Results:** Out of 180 investigated samples 15 _S. enterica_ (8.3%) and 21 _E. coli_ (11.7%) were isolated and identified.
_S. enterica_ isolates were typed as 9 _S._ Typhimurium (60.0%), 3 _S._ Paratyphi A (20.0%), 2 _S_ . Enteritidis (13.3%) and 1 _S._
Kentucky (6.7%). Twenty-one _E. coli_ isolates were serotyped into O1, O18, O20, O78, O103, O119, O126, O145, O146
and O158. The phenotypic antibiotic resistance profiles of _S. enterica_ serovars to ampicillin, cefotaxime, cefpodoxime,
trimethoprim/sulphamethoxazole and tetracycline were 86.7, 80.0, 60.0, 53.3 and 40.0%, respectively. Isolated _E. coli_
were resistant to tetracycline (80.9%), ampicillin (71.4%), streptomycin, trimethoprim/sulphamethoxazole (61.9%
for each) and cefotaxime (33.3%). The dissemination of genes coding for ESBL and AmpC β-lactamase in _S. enterica_
isolates included _bla_ CTX-M (73.3%), _bla_ TEM (73.3%) and _bla_ CMY (13.3%). In _E. coli_ isolates _bla_ TEM, _bla_ CTX-M and _bla_ OXA were
identified in 52.4, 42.9 and 14.3%, respectively. The plasmid-mediated quinolone resistance genes identified in _S._
_enterica_ were _qnr_ A (33.3%), _qnr_ B (20.0%) and _qnr_ S (6.7%) while _qnr_ A and _qnr_ B were detected in 33.3% of _E. coli_ isolates.
Class 1 integron was detected in 13.3% of _S. enterica_ and in 14.3% of _E. coli_ isolates. Class 2 integron as well as the
colistin resistance gene _mcr_ -1 was not found in any of _E. coli_ or _S. enterica_ isolates.
**Conclusions:** This study showed high prevalence of _S. enterica_ and _E. coli_ as foodborne pathogens in raw chicken and
beef meat in Nile Delta, Egypt. The emergence of antimicrobial resistance in _S. enterica_ and _E. coli_ isolates is of public
health concern in Egypt. Molecular biological investigation elucidated the presence of genes associated with antibi‑
otic resistance as well as class 1 integron in _S. enterica_ and _E. coli_ .
**Keywords:** _Salmonella_, _Escherichia coli_, Antibiotic resistance, Integron, Egypt


*Correspondence: hosny.eladawy@fli.de
1 Friedrich-Loeffler-Institut, Institute of Bacterial Infections and Zoonoses,
Naumburger Str. 96a, 07743 Jena, Germany
Full list of author information is available at the end of the article


© The Author(s) 2017. This article is distributed under the terms of the Creative Commons Attribution 4.0 International License
[(http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium,](http://creativecommons.org/licenses/by/4.0/)
provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license,
[and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (http://creativecommons.org/](http://creativecommons.org/publicdomain/zero/1.0/)
[publicdomain/zero/1.0/) applies to the data made available in this article, unless otherwise stated.](http://creativecommons.org/publicdomain/zero/1.0/)




Moawad _et al. Gut Pathog (2017) 9:57_ Page 2 of 13



**Background**
In spite of the improved technology and hygienic practices in developed countries at all stages of poultry and
beef meat production, foodborne infections remain as a
continuous threat to human and animal health. _Escheri-_
_chia_ ( _E_ .) _coli_ and _S. enterica_ serovars are the dominant
members of _Enterobacteriaceae_ causing foodborne infections. The expansion of antibiotic resistance in bacteria is
also an emerging public health hazard due to the compromised efficacy in the treatment of infectious diseases

[1].
The continuous exposure of bacterial strains by
β-lactams has led to dynamic and massive production
and mutation of β-lactamases [2]. Extended-spectrum
β-lactamases (ESBLs) are derived from point mutations
in the _bla_ TEM-1 and _bla_ SHV-1 _β_ -lactamase genes, which
cannot hydrolyse cephamycins and are inhibited by clavulanic acid [3], while ampicillin class C _β_ -lactamase
( _Amp_ C) enzymes are active on cephamycins as well as
oxyiminocephalosporins and monobactams [4]. ESBLs
are mostly located on mobile genetic elements (plasmids
or integrons), which can facilitate their mobility from
bacterial species to others by horizontal gene transfer [5].
In Egypt, _Salmonella_ isolates from chicken meat and
organs showed high resistance to different antibiotic
classes [6–8] where also, ESBL-producing _E_ . _coli_ were
isolated from retail chicken meat and dairy products in
Egypt [9, 10].
Quinolones considered as drugs of choice for treatment
of human infections caused by Gram-negative bacteria.
However, resistance to quinolones has been emerged
over the time due to misuse and/or overdose of drugs in
human and veterinary practice [11]. Plasmid-mediated
quinolone resistance (PMQR) represented by quinolone
resistance ( _qnr_ ) genes is widely distributed among bacteria [12–14].
Integrons are genetic elements able to capture individual antibiotic resistance genes including β-lactamases
encoding genes and stimulate their transcription and
expression [15]. The capture and spread of antibiotic



resistance determinants by integrons stimulates the rapid
evolution of multidrug resistances (MDR) among Gramnegative bacteria [16]. Among five main integron classes,
integrons 1 and 2 are most frequent in Gram-negative
bacteria. More than 70 different antibiotic resistance
genes have been characterized within integrons [16].
Colistin as antimicrobial substance was used against
Gram-negative bacteria. Its usage has been limited due to
its systemic toxicity but it was re-introduced as a last-line
option in treatment of human infections [17, 18].
This study aimed to discuss the prevalence, serotyping, antibiotic resistance and characterization of resistance-associated genes in _S. enterica_ serovars and _E. coli_
isolated from beef and poultry meat products in the Nile
Delta, Egypt.


**Methods**
**Sampling, isolation and identification of bacteria**
One hundred and eighty samples were randomly collected aseptically (90 chicken and 90 beef meat and
organs) from slaughterhouses and markets in four districts in northern Egypt (Dakahlia, Damietta, Kafr ElSheikh and Gharbia governorates). Briefly, 30 samples
of freshly slaughtered chickens and 30 samples of native
frozen chickens from restaurants and supermarkets (each
15 from breast and from thigh muscles and skin) were
taken. Thirty samples of organs (gizzard and liver) were
collected from freshly slaughtered chickens. In addition,
30 beef meat samples of freshly slaughtered carcasses
(neck, brisket, flank and rump muscles) were obtained
from slaughterhouses. Thirty samples from locally frozen
beef meat, collected from supermarkets and restaurants
and 30 samples from organs (liver, spleen and heart)
from freshly slaughtered animals were used in this study
(Tables 1, 2).
Ten grams of each meat and organ sample were incised
using sterile scalpel and forceps, transported immediately to the laboratory in ice bags, then transferred into
sterile homogenizer flask containing 45 ml of nutrient broth (Oxoid, Manchester, UK). The mixture was



**Table 1 Sources, number of** _**S. enterica**_ **isolates and results of their serovar identification**

**Serovar** **Poultry** **Beef** **(Poultry + beef)**
**n (%)**
**Fresh meat** **Frozen meat** **Fresh organs** **Total** **Fresh meat** **Frozen meat** **Fresh organs** **Total**
**(n = 30)** **(n = 30)** **(n = 30)** **n (%)** **(n = 30)** **(n = 30)** **(n = 30)** **n (%)**

_S._ Typhimurium 2 0 1 3 (3.3) 3 1 2 6 (6.7) 9 (60.0)

_S_ . Paratyphi A 0 1 1 2 (6.7) 0 0 1 1 (3.3) 3 (20.0)

_S._ Enteritidis 1 0 0 1 (3.3) 1 0 0 1 (3.3) 2 (13.3)

_S._ Kentucky 1 0 0 1 (3.3) 0 0 0 0 (0) 1 (6.7)

Total 4 1 2 7 (7.8) 4 1 3 8 (8.8) 15 (100)




Moawad _et al. Gut Pathog (2017) 9:57_ Page 3 of 13


**Table 2 Sources, number of** _**E. coli**_ **isolates and results of their serogroup identification**



**Serogroup** **Fresh poultry**
**meat**
**(n = 30)**



**Frozen poultry**
**meat**
**(n = 30)**



**Fresh poultry**
**organs**
**(n = 30)**



**Fresh beef**
**meat**
**(n = 30)**



**Frozen beef**
**meat**
**(n = 30)**



**Fresh beef**
**organs**
**(n = 30)**



**Total number** **% of** _**E. coli**_
**of isolates** **isolates**



O1 1 1 0 0 0 0 2 9.5

O18 2 1 1 0 0 0 4 19

O20 1 - 1 - - - 2 9.5

O78 1 0 0 1 1 1 4 19

O103 0 0 0 0 1 0 1 4.8

O119 1 - - 1 - - 2 9.5

O126 0 0 1 0 1 0 2 9.5

O145 0 0 1 0 0 0 1 4.8

O146 0 1 0 0 0 0 1 4.8

O158 0 0 0 1 1 0 2 9.5

Total 6 3 4 3 4 1 21 100



allowed to stand for 15 min at room temperature. From
each sample, 1 ml was added to 9 ml of buffered peptone
water (Oxoid) for _E. coli_ cultivation and 9 ml of Rappaport–Vassiliadis broth (Oxoid) for _S. enterica_ isolation.
After aerobic incubation at 37 °C overnight, a loopful from Rappaport–Vassiliadis broth was streaked on
xylose lysine deoxycholate (XLD) agar (Oxoid) and from
buffered peptone water on eosin methylene blue (EMB)
agar (Oxoid). The inoculated plates were incubated aerobically at 37 °C for 18–24 h. Suspected colonies were
identified biochemically using API 20E (bioMérieux,
Marcy-l’Étoile, France).
All biochemically confirmed _Salmonella_ isolates were
serotyped on the basis of somatic (O) and flagellar (H)
antigens by slide agglutination test using commercial
antisera (SIFIN, Berlin, Germany) following Kauffman–
White scheme [19]. Serological identification of _E. coli_
was carried out using slide agglutination method using
commercial antisera (SIFIN). The serotyping was carried out at the Serology Unit, Animal Health Research
Institute, Dokki, Egypt, and the Bacteriology Laboratory, Central Laboratories of Ministry of Health, Cairo,
Egypt.


**DNA extraction and purification**
The identified bacterial cultures were cultivated in Luria–
Bertani (LB) broth (Oxoid) at 37 °C overnight. DNA was
extracted and purified using DNeasy Blood and Tissue
Kits (Qiagen, Manchester, UK) according to the manufacturer’s instructions.


**Molecular identification of** _**S. enterica**_ **serovars and** _**E. coli**_
The isolated strains were further identified as _S. enterica_
and _E. coli_ using PCR assays. The amplification targets
and primers (Eurofins, Yokohama, Japan) are listed in



Table 3. All phenotypically identified _S. enterica_ serovars were tested for harbouring _omp_ C gene. Isolated _E._
_coli_ were confirmed by species-specific PCR targeting the
16S _r_ RNA genes and resulting in a 585 bp amplicon. PCR
assays used in this study resulted in _Salmonella_, _S_ . Enteritidis and _S_ . Typhimurium specific products of 204, 304
and 401 bp length, respectively.
The PCR protocol consisted of the following steps: (i)
an initial denaturation step of 2 min at 95 °C; (ii) 30 cycles
with 1 cycle consisting of 1 min at 95 °C, 1 min at 57 °C,
and 2 min at 72 °C; and (iii) a final elongation step of
5 min at 72 °C.
Five-microliter aliquots of reaction mixture were run in
electrophoreses using 1.5% agarose gels (Nippon Gene,
Tokyo, Japan) and visualized under UV light after ethidium bromide staining.


**Determination of antimicrobial susceptibility profiles**
The antimicrobial susceptibility was determined using
the Kirby–Bauer disc diffusion test [20]. Briefly, one colony picked up, streaked on Mueller–Hinton blood agar
(Oxoid) and incubated at 37 °C overnight. Bacterial colonies were suspended in 0.9% NaCl to obtain a McFarland turbidity of 0.5 (Dr. Lange, photometer CADAS 30,
Berlin, Germany) containing 1–2 × 10 [8] colony-forming
units (CFU)/ml of _E. coli_ strain ATCC 25922. Three hundred μl of the suspension were spread onto the surface of
a Mueller–Hinton agar plate (Oxoid) using a sterile swab.
The antimicrobial discs (Oxoid) of thirteen clinically used
antibiotics, which are used in the Egyptian poultry and
cattle farms (Tables 4, 5) were distributed onto the surface of the Mueller–Hinton agar plates using a multi-disc
dispenser (Oxoid). The plates were incubated at 37 °C
overnight. The diameters of inhibition zones were measured using sliding calipers and interpreted using standard




Moawad _et al. Gut Pathog (2017) 9:57_ Page 4 of 13


**Table 3 PCR primers, their sequences and amplification targets used in this study**


**Primer** **Sequence (5′–3′)** **Amplicon size (bp)** **Amplification target** **References**


_E. coli_, _Salmonella enterica_ and _Salmonella_ serovar identification

ECO-f GAC CTC GGT TTA GTT CAC AGA 585 _E. coli_ [67]

ECO-r CAC ACG CTG ACG CTG ACC A

OMPCF ATC GCT GAC TTA TGC AAT CG 204 _Salmonella_ genus [68]

OMPCR CGG GTT GCG TTA TAG GTC TG

ENTF TGT GTT TTA TCT GAT GCA AGA GG 304 _S._ Enteritidis [68]

ENTR TGA ACT ACG TTC GTT CTT CTG G

TYPHF TTG TTC ACT TTT TAC CCC TGA A 401 _S_ . Typhimurium [68]

TYPHR CCC TGA CAG CCG TTA GAT ATT

β-Lactamases

TEM-F ATA AAA TTC TTG AAG ACG AAA 1080 _bla_ TEM [69]

TEM-R GAC AGT TAC CAA TGC TTA ATC

OXA-F TCA ACT TTC AAG ATC GCA 591 _bla_ OXA [69]

OXA-R GTG TGT TTA GAA TGG TGA

OXA-F-2 ATT AAG CCC TTT ACC AAA CCA Whole _bla_ OXA [69]

OXA-R-2 AAG GGT TGG GCG ATT TTG CCA

OXA-23-F GAT CGG ATT GGA GAA CCA GA 501 _bla_ OXA-23 [25]

OXA-23-R ATT TCT GAC CGC ATT TCC AT

CTX-M-F CGC TTT GCG ATG TGC AG 550 _bla_ CTX-M [69]

CTX-M-R ACC GCG ATA TCG TTG GT

CTX-M-F2 CCA GAA TAA GGA ATC CCA TG Whole _bla_ CTX-M [69]

CTX-M-R2 GCC GTC TAA GGC GAT AAA C

CMY-F GAC AGC CTC TTT CTC CAC A 1007 _bla_ CMY [69]

CMY-R TGG AAC GAA GGC TAC GTA

CMY-F2 ACG GAA CTG ATT TCA TGA TG Whole _bla_ CMY [69]

CMY-R2 GAA AGG AGG CCC AAT ATC CT

SHV-F TTA TCT CCC TGT TAG CCA CC 795 _bla_ SHV [69]

SHV-R GAT TTG CTG ATT TCG CTC GG

Integrons

5′-CS GGC ATC CAA GCA GCA AG 152 Class 1 integron [69]

3′-CS AAG CAG ACT TGA CCT GA

hep74 CGG GAT CCC GGA CGG CAT GCA CGA TTT GTA 491 Class 2 integron [69]

hep51 GAT GCC ATC GCA AGT ACG AG

Plasmid-mediated quinolone resistance gene

qnrA-F ATT TCT CAC GCC AGG ATT TG 516 _qnr_ A [69]

qnrA-R GAT CGG CAA AGG TTA GGT CA

qnrB-F GAT CGT GAA AGC CAG AAA GG 469 _qnr_ B [69]

qnrB-R ACG ATG CCT GGT AGT TGT CC

qnrS-F ACG ACA TTC GTC AAC TGC AA 417 _qnr_ S [69]

qnrS-R TAA ATT GGC ACC CTG TAG GC

Plasmid-mediated colistin resistance gene

CLR5-F CGG TCA GTC CGT TTG TTC 308 _mcr_ -1 [24]

CLR5-R CTT GGT CGG TCT GTA GGG




**Table 4 Breakpoint values of antimicrobial agents according to CLSI, 2011 and phenotypic antimicrobial susceptibility profiles of 15** _**S. enterica**_ **isolates used**
**in this study**



**Antibiotic class** **Antimicrobial agent** **Conc. (μg)** **Zone diameter (mm)** _**S.**_ **Typhimu‑**
**rium (9)**



_**S.**_ **Enteritidis**
**(2)**



_**S.**_ **Kentucky**
**(1)**



_**S.**_ **Paratyphi**
**A (3)**



**Total** _**Salmonella**_ **(15)**



**R** **I** **S** **R** **I** **S** **R** **I** **S** **R** **I** **S** **R** **I** **S** **R** **I** **S**



Penicillin Amoxycillin–clavulanic acid 20 ≤ 13 14–17 ≥ 18 5 4 - - 2 - 1 - - 2 1 - 8 (53%) 7 (47%) 


Ampicillin 10 ≤ 13 14–16 ≥ 17 7 2 - 2 - - 1 - - 3 - - 13 (87%) 2 (13%) 
Cefotaxime 30 ≤ 22 23–25 ≥ 26 7 1 1 2 - - - 1 - 3 - - 12 (80%) 2 (13%)

Cefpodoxime 20 ≤ 17 18–20 ≥ 21 7 2 - 1 1 - - 1 - 1 2 - 9 (60%) 6 (40%) 


Cephalosporin Cefotaxime 30 ≤ 22 23–25 ≥ 26 7 1 1 2 - - - 1 - 3 - - 12 (80%) 2 (13%) 1 (7%)



Ceftazidime 30 ≤ 17 18–20 ≥ 21 3 5 1 - 2 - - 1 - 1 2 - 4 (26%) 10 (67%) 1 (7%)



Ceftriaxone 30 ≤ 19 20–22 ≥ 23 3 6 - - 2 - - 1 - 2 1 - 5 (33%) 10 (67%) 


Miscellaneous Chloramphenicol 30 ≤ 12 13–17 ≥ 18 - 5 4 - - 2 - 1 - - 1 2 - 7 (47%) 8 (53%)



Colistin 10 ≤ 11 12–13 ≥ 14 - - 9 - - 2 - - 1 - - 3 - - 15 (100%)



Fluoroquinolone Ciprofloxacin 5 ≤ 15 16–20 ≥ 21 - 3 6 - 1 1 - 1 - - - 3 - 5 (33%) 10 (67%)



Enrofloxacin 5 ≤ 17 18–20 ≥ 21 1 7 1 - 1 1 - 1 - - 2 1 1 (7%) 11 (73%) 3 (20%)



Nalidixic acid 30 ≤ 13 14–18 ≥ 19 3 6 - 1 1 - 1 - - - 3 - 5 (33%) 10 (67%) 


Aminoglycoside Streptomycin 10 ≤ 11 12–14 ≥ 15 2 7 - 1 1 - 1 - - - 2 1 4 (26%) 10 (67%) 1 (7%)



Tetracycline Tetracycline 30 ≤ 11 12–14 ≥ 15 3 6 - 2 - - 1 - - - 3 - 6 (40%) 9 (60%) 


Sulphonamide Trimethoprim/sulphamethoxazole 25 ≤ 4 3 ≥ 2 4 2 3 1 1 - 1 - - 2 1 - 8 (53%) 4 (26%) 3 (20%)



_S_ sensitive, _I_ intermediate, _R_ resistant




Moawad _et al. Gut Pathog (2017) 9:57_ Page 6 of 13


**Table 5 Breakpoint values of antimicrobial agents according to CLSI, 2011 and phenotypic antimicrobial susceptibility**
**profiles of** _**E. coli**_ **isolates used in this study**


**Antibiotic class** **Antimicrobial agent** **Conc. (μg)** **Zone diameter (9)** _**E. coli**_ **(21)** **Resistance rate**



**Poultry**
**(15)**



**Beef (6)** **Total (21)** **Poultry** **Beef** **Total**



**R** **I** **S** **R** **I** **S** **R** **I** **S** **R** **I** **S** **%** **%** **%**



Penicillin Amoxycillin–clavulanic acid 20 ≤ 13 14–17 ≥ 18 10 4 1 3 3 - 13 7 1 66.7 50.0 61.9

Ampicillin 10 ≤ 13 14–16 ≥ 17 12 2 1 3 3           - 15 5 1 80.0 50.0 71.4

Cephalosporin Cefotaxime 30 ≤ 22 23–25 ≥ 26 6 7 2 1 3 2 7 10 4 40.0 16.7 33.3



Ampicillin 10 ≤ 13 14–16 ≥ 17 12 2 1 3 3 - 15 5 1 80.0 50.0 71.4

Cefotaxime 30 ≤ 22 23–25 ≥ 26 6 7 2 1 3 2 7 10 4 40.0 16.7 33.3

Cefpodoxime 20 ≤ 17 18–20 ≥ 21 3 12 - 2 4 - 5 16 - 20.0 33.3 23.8

Ceftazidime 30 ≤ 17 18–20 ≥ 21 5 9 1 - 6 - 5 15 1 33.3 0 23.8

Ceftriaxone 30 ≤ 19 20–22 ≥ 23 3 5 7 1 3 2 4 8 9 20.0 16.7 19.0



Miscellaneous Chloramphenicol 30 ≤ 12 13–17 ≥ 18 3 9 3 1 1 4 4 10 7 20.0 16.7 19.0

Colistin 10 ≤ 11 12–13 ≥ 14           - 1 14           -           - 6           - 1 20           -           -           
Fluoroquinolone Ciprofloxacin 5 ≤ 15 16–20 ≥ 21 4 3 8 2 2 2 6 5 10 26.7 33.3 28.6



Colistin 10 ≤ 11 12–13 ≥ 14 - 1 14 - - 6 - 1 20 - - 


Enrofloxacin 5 ≤ 17 18–20 ≥ 21 2 5 8 1 4 1 3 9 9 13.3 16.7 14.3

Nalidixic acid 30 ≤ 13 14–18 ≥ 19 5 12 3 1 2 3 6 14 6 33.3 16.7 28.6



Aminoglycoside Streptomycin 10 ≤ 11 12–14 ≥ 15 9 5 1 4 2 - 13 7 1 60.0 66.7 61.9

Tetracycline Tetracycline 30 ≤ 11 12–14 ≥ 15 12 3 - 5 1 - 17 4 - 80.0 83.3 81.0

Sulphonamide Trimethoprim/sulphamethoxazole 25 ≤ 4 3 ≥ 2 10 2 3 3 - 3 17 2 6 66.7 50.0 81.0



_S_ sensitive, _I_ intermediate, _R_ resistant


break points according to Clinical and Laboratory Standards Institute [21].
The antimicrobial susceptibility of colistin was determined by disc diffusion susceptibility testing using
colistin discs (Oxoid) containing 10 μg. The disc zone
diameters was interpreted according to previous report

[22].


**Molecular detection of antimicrobial resistance associated**
**genes**
_Escherichia coli_ and _S. enterica_ isolates were tested for
β-lactamase-encoding genes _bla_ TEM, _bla_ SHV, _bla_ CTX-M,
_bla_ OXA and _bla_ CMY by PCR using universal primers for
the corresponding gene families (Table 3) as described
previously [23]. PCR amplification was also used for
screening of plasmid-mediated quinolone resistance
genes _qnr_ A, _qnr_ B and _qnr_ S.
For detection of variants of _bla_ TEM, _bla_ SHV, _bla_ CTX-M,
_bla_ OXA and _bla_ CMY amplified PCR fragments were purified from agarose gels using Nucleospin Gel Extraction
Kit (Macherey–Nagel, Düren, Germany). The products
were sequenced and the sequencing results were analyzed
[using BLAST (http://blast.ncbi.nlm.nih.gov/blast.cgi).](http://blast.ncbi.nlm.nih.gov/blast.cgi)
Colistin resistance associated gene _mcr_ -1was identified using a PCR assay (Table 3) as described by Liu et al.

[24]. PCR for detection of carbapenemases _bla_ OXA-23
gene was performed using OXA-23-F and OXA-23-R



(Table 3). PCR reaction and conditions were performed
according to Braun et al. [25].


**Detection and sequencing of class 1 and class 2 integrons**
PCR assays for detection of class 1 and class 2 integrons
were performed using primers given in Table 3 and
yielded PCR fragments that were purified from agarose
gels using Nucleospin Gel Extraction Kit (Macherey–
Nagel, Düren, Germany). The products were sequenced
in Genome Centre, Gifu University, Japan. The sequenc[ing results were analyzed using BLAST (http://blast.ncbi.](http://blast.ncbi.nlm.nih.gov/blast.cgi)
[nlm.nih.gov/blast.cgi).](http://blast.ncbi.nlm.nih.gov/blast.cgi)


**Results**
Out of 180 examined samples 15 _S. enterica_ (8.3%) (7
from poultry and 8 from beef) (Table 1) as well as 21 _E._
_coli_ (11.7%) (15 from poultry and 6 from beef) (Table 2)
were isolated and identified using bacteriological methods and biochemical characterization.


**Serological characterization of** _**S. enterica**_ **and** _**E. coli**_
Fifteen isolated _S. enterica_ were typed as 9 _S. enterica_
serovar Typhimurium (60.0%; 3 from chicken and 6 from
beef), 3 _S. enterica_ Paratyphi A (20.0%; 2 from chickens
and 1 from beef), 2 _S. enterica_ serovar Enteritidis (13.3%;
1 from chicken and 1 from beef) and 1 _S. enterica_ serovar
Kentucky (6.7%) from chicken sample (Table 1).




Moawad _et al. Gut Pathog (2017) 9:57_ Page 7 of 13



Twenty-one _E. coli_ isolates were characterized as 4 O18
(19.0%) and 4 O78 (19.0%) serotypes. Other serotypes are
given in Table 2.


**Molecular identification of** _**Salmonella**_ **serovars**
All 15 recovered _S. enterica_ serovars harboured _omp_ C
gene, which confirms _Salmonella_ genus. Nine _S. enterica_
serovar Typhimurium (60.0%) and two _S. enterica_ serovar
Enteritidis (13.3%) were identified by means of serovarspecific bands using PCR (Table 3). Four other isolates
could not be typed with PCR method (26.7%).


**Determination of antimicrobial susceptibility profiles**
The results of determination of antimicrobial resistance
of _S. enterica_ and _E. coli_ isolates to thirteen antibiotics
are given in Tables 4 and 5. Most of the _S. enterica_ isolates showed resistance to ampicillin (87.0%) and cefotaxime (80.0%) and all were susceptible to chloramphenicol,
colistin and ciprofloxacin. Resistance to other antibiotics
was shown in Table 4.
Seven isolates of _S. enterica_ serovar Typhimurium
were resistant to ampicillin, cefotaxime and cefpodoxime
(77.8%), while 6 isolates (66.7%) were highly sensitive to
ciprofloxacin. Resistance to other antibiotics was weaker
developed.
Both _S. enterica_ serovar Enteritidis isolates were resistant to ampicillin, cefotaxime and tetracycline. Additionally, one isolate showed resistance to streptomycin,
nalidixic acid, trimethoprim/sulphamethoxazole and
streptomycin.



In addition to the entire resistance to ampicillin and
cefotaxime, majority of the _S. enterica_ serovar Paratyphi
A isolates were resistant to amoxicillin–clavulanic acid,
ceftriaxone and trimethoprim/sulphamethoxazole.
_Salmonella enterica_ serovar Kentucky isolate showed
no pronounced susceptibility to any of the tested antibiotics and was resistant to six of them (Table 4). Seven
out of 15 _S. enterica_ serovars (46.7%) revealed phenotypic
multidrug resistance exhibiting by resistance to three or
more classes of antibiotics (Table 4).
_Escherichia coli_ isolates showed resistance to tetracycline, ampicillin, streptomycin, trimethoprim/sulphamethoxazole and amoxicillin–clavulanic acid with
80.9, 71.4, 61.9, 61.9 and 61.9%, respectively. Ten (47.6%),
9 (42.8%) and 9 (42.8%) isolates were susceptible to ciprofloxacin, enrofloxacin and ceftriaxone, respectively.
All tested isolates were susceptible to colistin. Thirteen
_E. coli_ isolates (61.9%) were characterized as multidrug
resistant (Table 5).
The multidrug resistance of isolated _Salmonella_ and _E._
_coli_ was shown in Tables 6 and 7, respectively.


**Molecular detection of resistance‑associated genes**
Six of 8 screened resistance-associated genes were
detected in _S. enterica_ serovars by PCR (Table 6). Eleven
(73.3%) isolates harboured _bla_ CTX-M gene (1, 3 and 14)
associated with cefotaxime resistance and 11 (73.3%)
isolates harboured _bla_ TEM-1 gene associated with penicillin and narrow spectrum β-lactamase resistance. Five
(33.3%) isolates harboured quinolone resistance gene



**Table 6 Phenotypic resistance and resistance determinants found in** _**S. enterica**_ **isolates in this study**


**Source** **Serovar** **Resistance phenotype** **Resistance genes/class 1 integrons**


Poultry

Fresh poultry meat Typhimurium CPD, CTX, NAL _bla_ CTX-M-1, _qnr_ A, _qnr_ B, _qnr_ S

Fresh poultry meat Typhimurium AMC, AMP, CPD _bla_ TEM-1
Fresh poultry meat Enteritidis AMP, CTX, TET _bla_ CTX-M-1, _bla_ TEM-1
Fresh poultry meat Kentucky AMC, AMP, NAL, STR, TET, T/S _bla_ TEM-1, _qnr_ A

Fresh poultry organs (liver) Typhimurium AMC, AMP, CPD, CRO, CTX, NAL, T/S _bla_ CTX-M-1, _bla_ TEM-1, _qnr_ A, _dfrA15_, _dfrA17_

Fresh poultry organs (liver) Paratyphi A AMC, AMP, CAZ CRO, CTX, T/S _bla_ CTX-M-14, _bla_ TEM-1
Frozen poultry meat Paratyphi A AMC, AMP, CRO, CTX, T/S _bla_ CTX-M-14, _bla_ TEM-1
Beef

Fresh beef meat Typhimurium AMP, CAZ, CTX _bla_ CTX-M-1
Fresh beef meat Typhimurium AMP, CAZ, CRO, CTX _bla_ CTX-M-1
Fresh beef meat Typhimurium AMC, CPD, CTX, TET, T/S _bla_ CTX-M-3, _bla_ TEM-1
Fresh beef meat Enteritidis AMP, CPD, CTX, NAL, STR, TET, T/S _bla_ CMY-2, _bla_ CTX-M-1, _bla_ TEM-1, _qnr_ A, _qnr_ B

Fresh beef organs (spleen) Typhimurium AMP, CPD, CTX, STR,TET, T/S _bla_ CTX-M-14, _bla_ TEM-1
Fresh beef organs (liver) Typhimurium AMC, AMP, CPD 
Fresh beef organs (liver) Paratyphi A AMP, CPD, CTX _bla_ TEM1
Frozen beef meat Typhimurium AMC, AMP, CAZ, CPD, CRO, CTX, ENR, NAL, STR, TET, T/S _bla_ CMY-2, _bla_ CTX-M-1, _bla_ TEM1, _qnr_ A, _qnr_ B, _aadA2_


_AMC_ amoxycillin–clavulanic acid, _AMP_ ampicillin, _CAZ_ ceftazidime, _CPD_ cefpodoxime, _CRO_ chloramphenicol, _CTX_ cefotaxime, _ENR_ enrofloxacin, _NAL_ nalidixic acid, _STR_
streptomycin, _TET_ tetracycline, _T/S_ trimethoprim/sulphamethoxazole




Moawad _et al. Gut Pathog (2017) 9:57_ Page 8 of 13


**Table 7 Phenotypic resistance and resistance determinants found in** _**E. coli**_ **isolates in this study**


**Source** _**E. coli**_ **serotype** **Resistance phenotype** **Resistance genes/class 1 integron**


Poultry

Fresh poultry meat O1 STR, TET 
Fresh poultry meat O18 AMP, CIP, CPD, NAL, STR, TET, T/S _qnr_ A, _qnr_ B

Fresh poultry meat O18 AMC, AMP, CHL, CIP, CRO, CTX, NAL, STR, TET, T/S _bla_ CTX-M-1, _bla_ OXA-1, _qnr_ A, _qnr_ B

Fresh poultry meat O78 AMC, AMP, CAZ, TET, T/S _bla_ CTX-M-1, _bla_ TEM-1
Fresh poultry meat O119 AMC, AMP, CAZ, CTX, STR, TET, T/S _bla_ CTX-M-14, _bla_ TEM-104
Fresh poultry meat O20 AMC, AMP, CPD, TET, T/S _bla_ CTX-M1, _bla_ OXA-1, _bla_ TEM-1
Fresh poultry organs (liver) O18 AMC, AMP, CAZ, CHL, CIP, CRO, CTX, ENR, NAL, STR, _bla_ CTX-M-1, _bla_ TEM-104, _qnr_ A, _qnr_ B, _dfr_ A1-orf
TET, T/S

Fresh poultry organs (liver) O126 AMP, CTX, STR _bla_ CTX-M-1
Fresh poultry organs O20 AMC, AMP, STR, T/S _bla_ TEM-1
(gizzard)

Fresh poultry organs (liver) O145 CAZ, TET 
Frozen poultry meat O1 CTX, NAL, TET _qnr_ A, _qnr_ B

Frozen poultry meat O18 AMC, AMP, STR, TET, T/S _bla_ OXA-1, _bla_ TEM-1
Frozen poultry 078 AMC, AMP, CPD

Frozen poultry meat O146 AMC, AMP, TET, T/S _bla_ TEM-1
Frozen organ poultry 078 AMC, AMP, CAZ, CHL, CIP, CRO, CTX, ENR, NAL, STR, _bla_ CTX-M-14, _bla_ TEM-1, _qnr_ A, _qnr_ B, _dfr_ A15
TET, T/S

Beef

Fresh beef meat O119 CPD 
Fresh beef meat O158 CPD, TET 
Fresh beef organs (liver) O78 AMC, AMP, CHL, CIP, CRO, CTX, ENR, NAL, STR, TET, _bla_ CTX-M-1, _bla_ TEM-104, _qnr_ A _, qnr_ B, _dfr_ A1, _dfr_ A1-orf
T/S

Frozen beef meat O103 STR, TET _bla_ CTX-M-1
Frozen beef meat O126 AMC, AMP, CIP, STR, TET, T/S _bla_ TEM-1, _qnr_ A, _qnr_ B

Frozen beef meat O158 AMC, AMP, STR, TET, T/S _bla_ TEM-1

_AMC_ amoxycillin–clavulanic acid, _AMP_ ampicillin, _CAZ_ ceftazidime, _CPD_ cefpodoxime, _CRO_ chloramphenicol, _CTX_ cefotaxime, _ENR_ enrofloxacin, _NAL_ nalidixic acid, _STR_
streptomycin, _TET_ tetracycline, _T/S_ trimethoprim/sulphamethoxazole



A ( _qnr_ A), while 3 (20.0%), 2 (13.3%) and 1 (6.7%) of _S._
_enterica_ serovars possessed _qnr_ B, _bla_ CMY and _qnr_ S genes,
respectively. _bla_ OXA associated with resistance to ampicillin and cephalothin and _bla_ SHV associated with plasmid-mediated ampicillin resistance were not found in
any isolate.
In _E. coli_ isolates, 5 of 8 screened resistance-associated
genes were detected by PCR (Table 7). Eleven (52.4%) out
of 21 isolates harboured _bla_ TEM (1 and 104), while _bla_ CTX
M [ (1 and 14) was detected in 9 isolates (42.9%). Seven iso-]
lates (33.3%) carried _qnr_ A and _qnr_ B and in three isolates
(14.3%) _bla_ OXA-1 was detected. Genes _bla_ CMY, _bla_ SHV and
_qnr_ S were not detected in any of the _E. coli_ isolates.
Plasmid-mediated colistin resistance gene _mcr_ -1 and
carbapenemase resistance gene _bla_ OXA-23 were not
identified neither in _S. enterica_ nor in _E. coli_ isolates
using PCR assay.


**Integron gene cassettes and DNA sequencing**
Class 1 integron was detected in two isolates of _S._ Typhimurium using PCR (Table 6). The inserted gene cassettes



identified three types of antimicrobial resistance genes
associated with class 1 integron: dihydrofolate reductase types ( _dfr_ A15 and _dfr_ A17) which confer resistance
to trimethoprim, and aminoglycoside adenyltransferase
type _aad_ A2 that confers resistance to streptomycin and
spectinomycin.
Three _E. coli_ isolates of serotypes O18, O78 and O78
were harbouring class 1 integron (Table 7). The inserted
gene cassettes identified three types of class 1 integron.
The identified antimicrobial resistance genes were dihydrofolate reductase types; _dfr_ A1, _dfr_ A15 and _dfr_ A1-orf
which confer resistance to trimethoprim.
All _S. enterica_ and _E. coli_ isolates were negative for class
2 integrons.


**Discussion**
Salmonellosis and _Salmonella_ infections considered as
critical threats to human and animal health. In this study,
the prevalence of _S. enterica_ serovars in chicken and
beef meat was 8.3%, which was considerably lower than
incidence rates that reported in Ethiopia (12.0% in raw




Moawad _et al. Gut Pathog (2017) 9:57_ Page 9 of 13



meat) [26], Canary islands (16.5% in chicken meat) [27],
Northwestern Spain (17.9% in chicken) [28], Ethiopia
(17.9% in chicken and giblets) [29], and Egypt (10.0% in
poultry meat) [7]. On the other hand, it was higher than
reported previously in meat products in Egypt (6.6%) and
in ground beef in the United States (4.2%) [30, 31].
In total, _S. enterica_ serovar Typhimurium was the
dominant serovar. These results were in accordance with
previous study from chicken products in Cambodia and
Thailand [32]. Controversially, _S_ . _enterica_ serovar Enteritidis was the dominant serotype in imported frozen
poultry samples from Brazil to Canary Islands and in
chicken carcasses in Spain [27, 28]. _Salmonella enterica_
serovar Kentucky was frequently detected in samples
coming from US to Canary Islands [27].
Prevalence of _E. coli_ in chicken meat and organs was
16.7% which was lower than in Nigeria (43.4%) in frozen poultry meat [33] but higher than in Korea (4.9% in
poultry meat) [34]. Prevalence of _E. coli_ in beef meat was
6.7%, while in Korea and Iran were found 4.1 and 29.0%,
respectively [34, 35].
In this study, 19.0% of _E. coli_ isolates typed as O78 was
mainly from poultry products, while in China, O78 was
identified in 60.0% of _E. coli_ isolated from chicken and
ducks [36]. _E. coli_ type O158 identified in 9.5% only in
beef meat while O158 detected in 22.7% of food isolates
in Chile [37].
Bacterial antimicrobial resistance is a global emerging
problem of public health concern.
In the current study, a high percentage of _S. enterica_
serovars were resistant to ampicillin and cefotaxime.
The resistance to other antimicrobial agents was variable
while all isolates were sensitive to chloramphenicol and
ciprofloxacin. The results were in a partial accordance
with results of previous reports stating that all _S. enterica_
isolated from chicken meat and beef were sensitive to
ciprofloxacin [38]. 72.4% of _Salmonella_ isolates in Thai
meat products were resistant to ampicillin while 71.0%
of isolates in Cambodian meat products were resistant
to sulfamethoxazole [32]. Controversially, _S. enterica_ isolated from animals and food of animal origin in Italy were
highly resistant to ampicillin, chloramphenicol, streptomycin and tetracycline [39] and all _Salmonella_ isolated
from beef in Tunisia were susceptible to amoxicillin and
clavulanic acid [40].
In total, 46.7% of _S. enterica_ serovars showed multidrug
resistance, which was higher than resistance of _Salmo-_
_nella_ isolated from raw chicken (31.0%) in Pennsylvania
during [41]. Less multidrug resistant _S. enterica_ isolates
(6.0%) were identified in cattle hides and carcasses in the
US [42].
The results of this study showed that the antimicrobial resistance of _E. coli_ isolated from poultry was higher



than from beef to the most of tested antibiotics. The
high resistance to tetracycline, ampicillin, amoxycillin–
clavulanic acid, trimethoprim/sulphamethoxazole and
streptomycin in this study was in agreement with previous reports conducted in Egypt [10, 43] and Algeria [44].
While in Spain, most of _E. coli_ isolated from diarrhoeic
and healthy lambs were highly resistant to tetracycline
and streptomycin but show lower resistance to ampicillin

[45]. Most of _E. coli_ isolates in this study were sensitive
to enrofloxacin, chloramphenicol and ceftriaxone which
is in agreement with previous results in Spain [45].
In this study, 61.9% of _E. coli_ isolates were multidrug
resistant. Similar results reported previously in Ghana
and US [23, 46].
Most of the phenotypically antibiotic resistant _S. enter-_
_ica_ serovars carried antibiotic resistance marker genes
associated with β-lactams and quinolones.
The β-lactamase encoding gene _bla_ TEM conferring
resistance to penicillins and first-generation cephalosporins was detected in 73.3% of _S. enterica_ serovars and
in 76.9% of ampicillin-resistant isolates which is significantly high in comparison to 57.3% ( _bla_ TEM-1) in _S. enter-_
_ica_ isolated from retail chickens in China [47]. Another
β-lactamase encoding gene _bla_ CMY-2, an AmpC betalactamase gene that confers resistance to a wide variety
of β-lactam antibiotics detected in 13.3% of isolates. In
contrast, in 4.7% of _S. enterica_ serovars Typhimurium
and Enteritidis originated from diseased broilers in Egypt
_bla_ CMY-2 was identified [48] and all resistant _S. enterica_
serovar Typhimurium isolated from retail meat in US
were harbouring _bla_ CMY [49]. The same gene could not
be detected previously in any _S. enterica_ serovar Typhimurium isolated from chicken meat in Egypt [8].
_bla_ CTX-M could be identified in 73.3% of isolated _S._
enterica serovars. In 24.0 and 18.8% of different _Salmo-_
_nella_ serovars isolated from retail chicken carcasses in
China, _bla_ CTX-M-15 and _bla_ CTX-M-3 were detected, respectively [47].
The _bla_ SHV is responsible for the plasmid-mediated
ampicillin resistance and β-lactamase encoding genes
_bla_ OXA conferring resistance to ampicillin and cephalothin not detected in any of _S. enterica_ serovars in this
study. This result was in accordance with a result for _S._
_enterica_ serovar Typhimurium isolated from chicken
meat in Egypt [8]. In contrast, 30.2% of ESBL-producing
_Salmonella_ isolated from raw chicken carcasses in China
were harbouring _bla_ OXA-1 [47].
Although none of _S. enterica_ serovars were phenotypically resistant to ciprofloxacin _, qnr_ A, _qnr_ B and _qnr_ S
genes were detected in 33.3, 20.0 and 6.7%, respectively in
all _S. enterica_ serovars and in 100, 60.0 and 20.0%, respectively in _S. enterica_ resistant to nalidixic acid. A different
result showed that 1.16% of nalidixic acid-resistant _S._




Moawad _et al. Gut Pathog (2017) 9:57_ Page 10 of 13



_enterica_ serovars isolated from animal products in Tunisia carried _qnr_ gene [50]. In other studies in China _qnr_ A,
_qnr_ B and _qnr_ S genes detected with low percentage in _S._
_enterica_ serovar Enteritidis isolated from retail raw poultry [51, 52].
The _bla_ TEM gene was detected in 52.3% of all isolated
_E. coli_ and in 73.3% of ampicillin-resistant isolates. Other
studies detected _bla_ TEM in 97.1 and 75.0% of _E. coli_ isolates from lambs in Spain and meat products in Cambodia, respectively [32, 45]. In China, _bla_ TEM was identified
in 7.8% of ESBL-producing _E. coli_ recovered from meat
products [53].
The β-lactamase encoding gene _bla_ CMY was not found
in any of the isolated _E. coli_ . In contrast, _bla_ CMY-2 was
detected in 89.0% _E. coli_ isolated from poultry meat in
Denmark [54] and in 12.5% of ESBL-producing _E. coli_
isolated from meat products in Cambodia [32].
The _bla_ CTX-M detected in 42.9% of _E. coli_ isolated
in this study. In Ghana 50.0% of ESBL-producing _E._
_coli_ isolates from meat harboured _bla_ CTX-M [46] while
these β-lactamase encoding genes _bla_ CTX-M-15 and _bla_ CTX
M-9 [ were detected rarely (1.6%) in ] _[E. coli]_ [ isolates from ]
meat products in China [53].
The _bla_ SHV gene was not found in any of the _E. coli_ isolates. Controversially, previous studies detected _bla_ SHV in
9.4, 5.3 and 2.0% in meat products in China, broiler chickens and chicken carcasses in Iran, respectively [53, 55, 56].
_bla_ OXA-1 was detected in 14.3% of the _E. coli_ isolates
which is in contrast with previous report conducted in
retail meat in US where the gene could not be identified

[23].
The most common carbapenemase types were
_bla_ OXA-23 and _bla_ OXA-48 accounting for 47% of all
identified carbapenemase genes [25, 57]. In this study,
_bla_ OXA-23 was not identified in all isolated strains.
In isolated _E. coli_, _qnr_ A and _qnr_ B detected in 33.3% of
isolates, all nalidixic acid and/or ciprofloxacin resistant
isolates harboured both genes. While _qnr_ B and _qnr_ S were
identified in 10.4% of nalidixic acid resistant _E. coli_ isolated from Algerian retail chicken meat [44] and in 10.0%
of _E. coli_ isolates obtained from bovine carcasses in Mexico [58]. In contrast, _qnr_ A, _qrn_ B and _qrn_ S genes could
not be identified in enrofloxacin-resistant _E. coli_ strains
from poultry in Mexico [59].
The presence of genetic elements such as integrons and
transposons are often associated with multi-resistant
phenotypes among _Salmonella_ isolates [60]. In this study,
class 1 integron detected in 13.3% of _S. enterica_ serovars.
The identified gene cassettes were _dfr_ A15, _dfr_ A17 and
_aad_ A2. In previous studies, class 1 integron identified in
90.0% of multi-drug resistant _Salmonella_ isolates from
retail chicken meat in Japan and the identified genes cassettes were _dfr_ A1, _dfr_ A7, _aad_ A1, _aad_ B, and _cat_ B3 [61,



62]. The gene cassettes of class 1 integron which detected
in _Salmonella_ spp. isolated from poultry meat in Egypt
were harbouring _aac_ (3)-Id, _aad_ A2, _aad_ A4, _aad_ A7, _sat_,
_dfr_ A15, _lnu_ F and _est_ X resistance genes [7]. In a study
conducted in Portugal, 75.0% of _S. enterica_ isolated from
different sources including food products had one or two
class 1 integrons [63].
The identified gene cassettes of class 1 integron _dfr_ A1,
_dfr_ A15 and _dfr_ A1-orf which confer resistance to trimethoprim were identified in 14.3% of _E. coli_ isolated in this
study. This is in agreement with _E. coli_ isolates from retail
chickens in Japan [62] and in 1.9 and 11.4% of isolates
from Thai and Cambodian meat samples obtained from
slaughterhouses and fresh markets and the most common gene cassette was _dfr_ A12- _aad_ A2 [32].
Class 2 integrons were not detected in all isolates of
this study which is in contrast to 5.6% positive samples in
Egypt [10].
In this study, _mcr_ -1 gene associated with colistin
resistance was neither detected in _S. enterica_ nor _E. coli_
isolates. This result was in agreement with the result
reported by Doumith et al. [64] who investigated 24,000
isolates of _Enterobacteriaceae_ from food and humans
including _S. enterica_ and _E. coli_ and found only 15 _mcr_ -1
positive isolates. Recently, Quesada et al. identified the
gene _mcr_ - _1_ in nine _S. enterica_ and _E. coli_ isolates from
poultry and swine for the first time in Spain [65]. Jayol
et al. found the _mcr_ -1 in 13% of _E. coli_ and _S. enterica_

[66]. Considering the frequent use of colistin in animal
production and the importance of this antimicrobial for
the control of multi-resistant Gram-negative nosocomial
infections in humans, monitoring the dissemination of
resistance to colistin is mandatory.
In conclusion, the results of this study showed high
prevalence of _S. enterica_ and _E. coli_ as foodborne pathogens isolated from poultry and beef meat in Egypt. The
emergence of antimicrobial resistance of _S. enterica_ and
_E. coli_ isolates is of public concerns in Egypt. Significant
resistance was detected to penicillin, cephalosporins,
tetracycline and sulphonamides. Dissemination of ESBL
and AmpC-β-lactamase resistance-associated genes in
_S. enterica_ and _E. coli_ was determined. Presence of class
1 integron in _S. enterica_ and _E. coli_ and genes associated with antibiotic resistance was also confirmed. Class
2 integron was not detected in any isolate. Further work
should be performed to characterize _S. enterica_ and _E._
_coli_ isolates of animal and human origin from the same
region sharing the same resistance markers in order to
highlight potential horizontal gene transfer by these
zoonotic organisms.


**Authors’ contributions**
AAM participated in the conception and design of the study. AAM performed
the farm and laboratory work. AAM, HH, HE, HT, HN and HMH analyzed the




Moawad _et al. Gut Pathog (2017) 9:57_ Page 11 of 13



data and wrote the manuscript and contributed to manuscript discussion. All
authors read and approved the final manuscript.


**Author details**
1 Friedrich-Loeffler-Institut, Institute of Bacterial Infections and Zoonoses,
Naumburger Str. 96a, 07743 Jena, Germany. [2] Bacteriology department, Animal
Health Research Institute (AHRI), Mansoura branch, Mansoura, 35516, Egypt.
3 Faculty of Veterinary Medicine, Kafrelsheikh University, Kafr El‑Sheikh 33516,
Egypt. [4] Faculty of Veterinary Medicine, Mansoura University, Mansoura, Egypt.
5 Institute of Poultry Diseases, Free University Berlin, Berlin, Germany.


**Acknowledgements**
The authors thank technical assistants at Kafrelsheikh University for their excel‑
lent technical assistance. The authors thank Animal Health Research Institute,
Mansoura Laboratory for its great effort in sample collection and preparation.


**Competing interests**
The authors declare that they have no competing interests.


**Availability of data**
All the data supporting the results presented in the main manuscript.


**Consent for publication**
Not applicable.


**Ethics approval and consent for participate**
This study was carried out in strict accordance with the recommendations of
the, Egyptian Network of Research Ethics Committees (ENREC) which com‑
plies with the international laws and regulation regarding ethical considera‑
tions in research.


**Funding**
This study was done without any specific funding.


**Publisher’s Note**
Springer Nature remains neutral with regard to jurisdictional claims in pub‑
lished maps and institutional affiliations.


Received: 23 June 2017  Accepted: 10 October 2017


**References**
1. Helmy Y, El-Adawy H, Abdelwhab E. A comprehensive review of common
bacterial, parasitic and viral zoonoses at the human–animal interface in
Egypt. Pathogens. 2017;6:33.
2. Pitout J, Laupland K. Extended-spectrum beta-lactamase-producing
_Enterobacteriaceae_ : an emerging public-health concern. Lancet Infect Dis.
2008;8:159–66.
3. Bush K, Jacoby GA. Updated functional classification of β-lactamases.
Antimicrob Agents Chemother. 2010;54:969–76.
4. Jacoby GA. AmpC β-lactamases. Clin Microbiol Rev. 2009;22:161–82.
5. Pfeifer Y, Cullik A, Witte W. Resistance to cephalosporins and carbap‑
enems in Gram-negative bacterial pathogens. Int J Med Microbiol.
2010;300:371–9.
6. Abd-Elghany SM, Sallam KI, Abd-Elkhalek A, Tamura T. Occurrence,
genetic characterization and antimicrobial resistance of _Salmo-_
_nella_ isolated from chicken meat and giblets. Epidemiol Infect.
2015;143:997–1003.
7. Gharieb RM, Tartor YH, Khedr MH. Non-typhoidal _Salmonella_ in poultry
meat and diarrhoeic patients: prevalence, antibiogram, virulotyping,
molecular detection and sequencing of class I integrons in multidrug
resistant strains. Gut Pathog. 2015;7:15–81.
8. Ahmed HA, El-Hofy FI, Shafik SM, Abdelrahman MA, Elsaid GA. Char‑
acterization of virulence-associated genes, antimicrobial resistance
genes, and class 1 integrons in _Salmonella enterica_ serovar Typhimurium
isolates from chicken meat and humans in Egypt. Foodborne Pathog Dis.
2016;13:281–8.



9. Abdallah HM, Reuland EA, Wintermans BB, Al Naiemi N, Koek A, Abdelwa‑
hab AM, et al. Extended-spectrum β-lactamases and/or carbapenemasesproducing _Enterobacteriaceae_ isolated from retail chicken meat in
Zagazig, Egypt. PLoS ONE. 2015;10:e0136052.
10. Ahmed AM, Shimamoto T. Molecular analysis of multidrug resistance in
Shiga toxin-producing _Escherichia coli_ O157:H7 isolated from meat and
dairy products. Int J Food Microbiol. 2015;193:68–73.
11. Andriole VT. The quinolones: past, present, and future. Clin Infect Dis.
2005113-19.
12. Flach CF, Boulund F, Kristiansson E, Larsson DJ. Functional verification of
computationally predicted _qnr_ genes. Ann Clin Microbiol Antimicrob.
2013;12:34.
13. Guan X, Xue X, Liu Y, Wang J, Wang Y, Wang K, et al. Plasmid-mediated
quinolone resistance—current knowledge and future perspectives. J Int
Med Res. 2013;41:20–30.
14. Robicsek A, Jacoby GA, Hooper DC. The worldwide emergence of
plasmid-mediated quinolone resistance. Lancet Infect Dis. 2006;6:629–40.
15. Hanau-Bercot B, Podglajen I, Casin I, Collatz E. An intrinsic control
element for translational initiation in class 1 integrons. Mol Microbiol.
2002;44:119–30.
16. Rowe-Magnus DA, Guerout AM, Mazel D. Bacterial resistance evolu‑
tion by recruitment of super-integron gene cassettes. Mol Microbiol.
2002;43:1657–69.
17. Tzouvelekis L, Markogiannakis A, Piperaki E, Souli M, Daikos G. Treating
infections caused by carbapenemase-producing _Enterobacteriaceae_ . Clin
Microbiol Infect Dis. 2014;20:862–72.
18. Sun S, Negrea A, Rhen M, Andersson D. Genetic analysis of colistin resist‑
ance in _Salmonella enterica_ serovar Typhimurium. Antimicrob Agents
Chemother. 2009;53:2298–305.
19. Popoff MY, Bockemuhl J, Gheesling LL. Supplement 2002 (no. 46) to the
Kauffmann–White scheme. Res Microbiol. 2004;155:568–70.
20. Bauer AW, Kirby WM, Sherris JC, Turck M. Antibiotic susceptibility testing
by a standardized single disk method. Am J Clin Pathol. 1966;45:493–6.
21. CLSI. Performance standards for antimicrobial susceptibility testing;
twenty-first informational supplement: CLSI document M100-S21. CLSI.
2011;31:42–51.
22. Maalej S, Meziou M, Rhimi F, Hammami A. Comparison of disc diffusion,
Etest and agar dilution for susceptibility testing of colistin against _Entero-_
_bacteriaceae_ . Lett Appl Microbiol. 2011;53:546–51.
23. Zhao S, Blickenstaff K, Bodeis-Jones S, Gaines SA, Tong E, McDermott PF.
Comparison of the prevalences and antimicrobial resistances of _Escheri-_
_chia coli_ isolates from different retail meats in the United States, 2002 to
2008. Appl Environ Microbiol. 2012;78:1701–7.
24. Liu Y, Wang Y, Walsh T, Yi LX, Zhang R, Spencer J, et al. Emergence of
plasmid-mediated colistin resistance mechanism MCR-1 in animals and
human beings in China: a microbiological and molecular biological study.
Lancet Infect Dis. 2016;16:161–8.
25. Braun S, Monecke S, Thurmer A, Ruppelt A, Makarewicz O, Pletz M,
et al. Rapid identification of carbapenemase genes in gram-negative
bacteria with an oligonucleotide microarray-based assay. PLoS ONE.
2014;9:e102232.
26. Ejo M, Garedew L, Alebachew Z, Worku W. Prevalence and antimicro‑
bial resistance of _Salmonella_ isolated from animal-origin food items in
Gondar, Ethiopia. Biomed Res Int. 2016;4290506:15–20.
27. Hernandez T, Sierra A, Rodriguez-Alvarez C, Torres A, Arevalo MP, Calvo M,
et al. _Salmonella enterica_ serotypes isolated from imported frozen chicken
meat in the Canary islands. J Food Prot. 2005;68:2702–6.
28. Capita R, Alonso-Calleja C, Prieto M. Prevalence of _Salmonella enterica_
serovars and genovars from chicken carcasses in slaughterhouses in
Spain. J Appl Microbiol. 2007;103:1366–75.
29. Tibaijuka B, Molla B, Hildebrandt G, Kleer J. Occurrence of Salmonellae in
retail raw chicken products in Ethiopia. Berl Munch Tierarztl Wochenschr.
2003;116:55–8.
30. Ahmed AM, Shimamoto T. Isolation and molecular characteriza‑
tion of _Salmonella enterica_, _Escherichia coli_ O157:H7 and _Shigella_
spp. from meat and dairy products in Egypt. Int J Food Microbiol.
2014;169:M405–10.
31. Bosilevac JM, Guerini MN, Kalchayanand N, Koohmaraie M. Prevalence
and characterization of salmonellae in commercial ground beef in the
United States. Appl Environ Microbiol. 2009;75:1892–900.




Moawad _et al. Gut Pathog (2017) 9:57_ Page 12 of 13



32. Trongjit S, Angkititrakul S, Tuttle RE, Poungseree J, Padungtod P, Chuan‑
chuen R. Prevalence and antimicrobial resistance in _Salmonella enterica_
isolated from broiler chickens, pigs and meat products in Thailand-Cam‑
bodia border provinces. Microbiol Immunol. 2017;61:23–33.
33. Adeyanju GT, Ishola O. _Salmonella_ and _Escherichia coli_ contamination of
poultry meat from a processing plant and retail markets in Ibadan, Oyo
State, Nigeria. Springerplus. 2014;3:139.
34. Lee GY, Jang HI, Hwang IG, Rhee MS. Prevalence and classification of
pathogenic _Escherichia coli_ isolated from fresh beef, poultry, and pork in
Korea. Int J Food Microbiol. 2009;134:196–200.
35. Momtaz H, Dehkordi FS, Hosseini MJ, Sarshar M, Heidari M. Serogroups,
virulence genes and antibiotic resistance in Shiga toxin-producing
_Escherichia coli_ isolated from diarrheic and non-diarrheic pediatric
patients in Iran. Gut Pathog. 2013;5:1757–4749.
36. Wang S, Meng Q, Dai J, Han X, Han Y, Ding C, et al. Development of an
allele-specific PCR assay for simultaneous sero-typing of avian patho‑
genic _Escherichia coli_ predominant O1, O2, O18 and O78 strains. PLoS
ONE. 2014;9:e96904.
37. Alexandre M, Prado V, Ulloa MT, Arellano C, Rios M. Detection of entero‑
hemorrhagic _Escherichia coli_ in meat foods using DNA probes, enzymelinked immunosorbent assay and polymerase chain reaction. J Vet Med B
Infect Dis Vet Public Health. 2001;48:321–30.
38. Larkin C, Poppe C, McNab B, McEwen B, Mahdi A, Odumeru J. Antibiotic
resistance of _Salmonella_ isolated from hog, beef, and chicken carcass
samples from provincially inspected abattoirs in Ontario. J Food Prot.
2004;67:448–55.
39. Perugini AG, Carullo MR, Esposito A, Caligiuri V, Capuano F, Galiero G, et al.
Characterization of antimicrobial resistant _Salmonella enterica_ serovars
Enteritidis and Typhimurium isolates from animal and food in Southern
Italy. Vet Res Commun. 2010;34:387–92.
40. Oueslati W, Rjeibi MR, Mhadhbi M, Jbeli M, Zrelli S, Ettriqui A. Prevalence,
virulence and antibiotic susceptibility of _Salmonella_ spp. strains, isolated
from beef in Greater Tunis (Tunisia). Meat Sci. 2016;119:154–9.
41. M’Ikanatha NM, Sandt CH, Localio AR, Tewari D, Rankin SC, Whichard
JM, et al. Multidrug-resistant _Salmonella_ isolates from retail chicken
meat compared with human clinical isolates. Foodborne Pathog Dis.
2010;7:929–34.
42. Brichta-Harhay DM, Arthur TM, Bosilevac JM, Kalchayanand N, Shackelford
SD, Wheeler TL, et al. Diversity of multidrug-resistant _Salmonella enterica_
strains associated with cattle at harvest in the United States. Appl Environ
Microbiol. 2011;77:1783–96.
43. Ahmed AM, Shimamoto T, Shimamoto T. Molecular characterization
of multidrug-resistant avian pathogenic _Escherichia coli_ isolated from
septicemic broilers. Int J Med Microbiol. 2013;303:475–83.
44. Laarem M, Barguigua A, Nayme K, Akila A, Zerouali K, El Mdaghri N, et al.
Occurrence of plasmid-mediated quinolone resistance and virulence
genes in avian _Escherichia coli_ isolates from Algeria. J Infect Dev Ctries.
2017;11:143–51.
45. Medina A, Horcajo P, Jurado S, De la Fuente R, Ruiz-Santa-Quiteria JA,
Dominguez-Bernal G, et al. Phenotypic and genotypic characterization of
antimicrobial resistance in enterohemorrhagic _Escherichia coli_ and atypi‑
cal enteropathogenic _E. coli_ strains from ruminants. J Vet Diagn Investig.
2011;23:91–5.
46. Rasmussen MM, Opintan JA, Frimodt-Moller N, Styrishave B. Beta
lactamase producing _Escherichia coli_ isolates in imported and locally
produced chicken meat from Ghana. PLoS ONE. 2015;10:e0139706.
47. Qiao J, Zhang Q, Alali WQ, Wang J, Meng L, Xiao Y, et al. Characterization
of extended-spectrum β-lactamases (ESBLs)-producing _Salmonella_ in
retail raw chicken carcasses. Int J Food Microbiol. 2017;248:72–81.
48. Ahmed AM, Shimamoto T. Genetic analysis of multiple antimicrobial
resistance in _Salmonella_ isolated from diseased broilers in Egypt. Micro‑
biol Immunol. 2012;56:254–61.
49. Folster JP, Tolar B, Pecic G, Sheehan D, Rickert R, Hise K, et al. Characteriza‑
tion of blaCMY plasmids and their possible role in source attribution of
_Salmonella enterica_ serotype Typhimurium infections. Foodborne Pathog
Dis. 2014;11:301–6.
50. Al-Gallas N, Abbassi MS, Gharbi B, Manai M, Ben Fayala MN, Bichihi R, et al.
Occurrence of plasmid-mediated quinolone resistance determinants and
_rmt_ B gene in _Salmonella enterica_ serovar Enteritidis and Typhimurium



isolated from food-animal products in Tunisia. Foodborne Pathog Dis.
2013;10:813–9.
51. Wang Y, Yang B, Wu Y, Zhang Z, Meng X, Xi M, et al. Molecular charac‑
terization of _Salmonella enterica_ serovar Enteritidis on retail raw poultry
in six provinces and two National cities in China. Food Microbiol.
2015;46:74–80.
52. Zhang Z, Meng X, Wang Y, Xia X, Wang X, Xi M, et al. Presence of _qnr_,
_aac_ (6′)-Ib, _qep_ A, _oqx_ AB, and mutations in gyrase and topoisomerase in
nalidixic acid-resistant _Salmonella_ isolates recovered from retail chicken
carcasses. Foodborne Pathog Dis. 2014;11:698–705.
53. Li L, Ye L, Kromann S, Meng H. Occurrence of extended-spectrum
B-lactamases, plasmid-mediated quinolone resistance, and disinfect‑
ant resistance genes in _Escherichia coli_ isolated from ready-to-eat meat
products. Foodborne Pathog Dis. 2017;14:109–15.
54. Hansen KH, Bortolaia V, Nielsen CA, Nielsen JB, Schonning K, Agerso Y,
et al. Host-specific patterns of genetic diversity among IncI1-Igamma and
IncK plasmids encoding CMY-2 B-lactamase in _Escherichia coli_ isolates
from humans, poultry meat, poultry, and dogs in Denmark. Appl Environ
Microbiol. 2016;82:4705–14.
55. Jafari RA, Motamedi H, Maleki E, Ghanbarpour R, Mayahi M. Phylogenetic
typing and detection of extended-spectrum B-lactamases in _Escheri-_
_chia coli_ isolates from broiler chickens in Ahvaz, Iran. Vet Res Forum.
2016;7:227–33.
56. Bagheri M, Ghanbarpour R, Alizade H. Shiga toxin and beta-lactamases
genes in _Escherichia coli_ phylotypes isolated from carcasses of broiler
chickens slaughtered in Iran. Int J Food Microbiol. 2014;177:16–20.
57. Kaase M. Carbapenemases in gram-negative bacteria. Current data
and trends of resistance resulting from the work of national reference
centres. Bundesgesundheitsbl Gesundheitsforsch Gesundheitsschutz.
2012;55:1401–4.
58. de Oca SAM, Talavera-Rojas M, Soriano-Vargas E, Barba-Leon J, Vazquez
Navarrete J. Determination of extended spectrum β-lactamases/AmpC
B-lactamases and plasmid-mediated quinolone resistance in _Escherichia_
_coli_ isolates obtained from bovine carcasses in Mexico. Trop Anim Health
Prod. 2015;47:975–81.
59. Del Rio-Avila C, Rosario C, Arroyo-Escalante S, Carrillo-Casas EM, Diaz
Aparicio E, Suarez-Guemes F, et al. Characterisation of quinolone-resistant
_Escherichia coli_ of 1997 and 2005 isolates from poultry in Mexico. Br Poult
Sci. 2016;57:494–500.
60. Guerra B, Soto S, Helmuth R, Mendoza MC. Characterization of a self
transferable plasmid from _Salmonella enterica_ serotype Typhimurium
clinical isolates carrying two integron-borne gene cassettes together
with virulence and drug resistance genes. Antimicrob Agents Chemother.
2002;46:2977–81.
61. Ahmed AM, Ishida Y, Shimamoto T. Molecular characterization of anti‑
microbial resistance in _Salmonella_ isolated from animals in Japan. J Appl
Microbiol. 2009;106:402–9.
62. Ahmed AM, Shimabukuro H, Shimamoto T. Isolation and molecular
characterization of multidrug-resistant strains of _Escherichia coli_ and _Sal-_
_monella_ from retail chicken meat in Japan. J Food Sci. 2009;74:1750–3841.
63. Antunes P, Machado J, Peixe L. Characterization of antimicrobial resist‑
ance and class 1 and 2 integrons in _Salmonella enterica_ isolates from
different sources in Portugal. J Antimicrob Chemother. 2006;58:297–304.
64. Doumith M, Godbole G, Ashton P, Larkin L, Dallman T, Day M, et al. Detec‑
tion of the plasmid-mediated _mcr_ -1 gene conferring colistin resistance
in human and food isolates of _Salmonella enterica_ and _Escherichia coli_ in
England and Wales. J Antimicrob Chemother. 2016;71:2300–5.
65. Quesada A, Ugarte-Ruiz M, Iglesias M, Porrero M, Martinez R, Florez-Cuad‑
rado D, et al. Detection of plasmid mediated colistin resistance (MCR-1) in
_Escherichia coli_ and _Salmonella enterica_ isolated from poultry and swine in
Spain. Res Vet Sci. 2016;105:134–5.
66. Jayol A, Nordmann P, Lehours P, Poirel L, Dubois V. Comparison of meth‑
ods for detection of plasmid-mediated and chromosomally encoded
colistin resistance in _Enterobacteriaceae_ . Clin Microbiol Infect Dis.
2017;S1198-743X:30291–4.
67. Seidavi A, Mirhosseini SZ, Shivazad M, Chamani M, Sadeghi AA, Pourseify
R. Detection and investigation of _Escherichia coli_ in contents of duode‑
num, jejunum, ileum and cecum of broilers at different ages by PCR.
Asia-Pac J Mol Biol Biotechnol. 2010;18:321–6.




Moawad _et al. Gut Pathog (2017) 9:57_ Page 13 of 13



68. Alvarez J, Sota M, Vivanco AB, Perales I, Cisterna R, Rementeria A, et al.
Development of a multiplex PCR technique for detection and epidemio‑
logical typing of _Salmonella_ in human clinical samples. J Clin Microbiol.
2004;42:1734–8.



69. Ahmed AM, Motoi Y, Sato M, Maruyama A, Watanabe H, Fukumoto Y,
et al. Zoo animals as reservoirs of gram-negative bacteria harboring
integrons and antimicrobial resistance genes. Appl Environ Microbiol.
2007;73:6686–90.




