Gharieb _et al. Gut Pathog (2015) 7:34_ Page 9 of 11



**Conclusions**
The higher resistance rates displayed by _Salmonella_ isolates in this study to the antibiotics and the detection of
class 1 integron gene cassettes harboring resistance genes
among MDR strains shows the potential of integrons to
carry and spread resistance genes to other _S. enterica_ isolates or to other bacteria. Therefore, a more prudent use
of antibiotics in both humans and animals is required.


**Methods**
**Ethical approval**
This study was approved by Zagazig University, Egypt
ethical board (Protocol No. 10122). Permission to collect
stool samples was obtained from the administrators of
Al-Ahrar General hospital.


**Specimen collection**
The present work was conducted during the period from
March, 2014 to February, 2015. A total of one hundred
fresh poultry carcasses comprising broiler chicken and
ducks (50, each) were randomly purchased from poultry retail outlets at Zagazig City, Egypt. In addition, fifty
stool samples were collected from the patients attending
the outpatients’ clinic of Al-Ahrar General Hospital. The
inclusion criteria were restricted to the patients who suffered from diarrhoea, fever and gave informed written
consent to participate in the study. Moreover, patients
received antibiotic treatments were excluded. A swab was
obtained from each freshly passed stool sample, labeled,
ice packed and transported to the Bacteriology Laboratory, Faculty of Veterinary Medicine, Zagazig University,
Egypt within 24 h for bacteriological analysis.


**Isolation and identification of** _**Salmonella**_ **spp**
_Salmonella_ isolation was carried out according to the
standard methods recommended by ISO 6579 [33].
Twenty-five grams of poultry meat were excised from
each sample and minced with 225 ml of buffered peptone water (BPW) enrichment broth. Also, each stool
swab was inserted in sterile tube containing 9 ml BPW.
All samples were incubated at 37 °C for 24 h. Aliquots
(0.1 ml) of the pre-enriched culture was inoculated into
10 ml of Rappaport Vasiliadis (RV) enrichment broth
and incubated at 42 °C for 24 h. A loopful of the enriched
culture was streaked on Xylose-Lysine-Deoxycholate
(XLD) agar plates (Oxoid, Basingstoke, UK) and incubated at 37 °C for 24 h. Suspected colonies with typical
_Salmonella_ morphology were confirmed biochemically
by indole, citrate utilization, urease, triple sugar iron
(TSI) and lysine iron agar (LIA) tests. All biochemically
confirmed _Salmonella_ isolates were serologically identified on the basis of somatic (O) and flagellar (H) antigens
by slide agglutination using commercial antisera (SISIN,



Berlin) following Kauffman-White scheme [34] in collaboration with Serology Unit, Animal Health Research
Institute, Dokki, Egypt.


**Antimicrobial susceptibility testing (antibiogram)**
Susceptibility of _Salmonella_ isolates to various routine
antimicrobial drugs was tested by the standard disc diffusion technique [35] using Mueller–Hinton agar and
commercial antibiotic discs (Oxoid, UK). The antibiotics
used were ampicillin (10 μg), erythromycin (15 μg), streptomycin (10 μg), ciprofloxacin (5 μg), tetracycline (30 μg),
ceftriaxone (30 μg), ampicillin-sulbactam (20 μg), trimethoprim-sulfamethoxazole (25 μg), chloramphenicol
(30 μg), nalidixic acid (30 μg), gentamicin (10 μg), amoxicillin-clavulanic acid (30 μg). The inhibition zones in mm
were measured and scored as sensitive, intermediate and
resistant categories in accordance with the critical breakpoints recommended by the Clinical and Laboratory
Standards Institute [36].


**Molecular detection of** _**Salmonella**_ **virulence genes using**
**uniplex PCR**
A total of twelve _Salmonella_ isolates were screened for
the presence of virulence genes using PCR. These genes
including; plasmid encoded fimbriae ( _pef_ A), hyper invasive locus ( _hil_ A), _Salmonella_ outer protein ( _sop_ B) and
_Salmonella_ enterotoxin gene ( _stn_ ). Briefly, DNA was
extracted from the overnight bacterial culture using
ABIO pure Genomic DNA extraction kit with modifications from the manufacturer’s recommendation. DNA
amplification was carried out using specific primers supplied from Metabion, Germany (Table 4). Primers were
utilized in a 25 µl reaction containing 12.5 µl of Emerald
Amp Max PCR Master Mix (Takara, Japan), one µl of
each primer (20 pmol concentration), 4.5 µl of nuclease
free water, and 6 µl of DNA template. The reaction was
performed in a T3 Biometra thermal cycler. Positive and
negative controls were included in each reaction. Aliquot
of each amplicon along with a 100 bp molecular weight
DNA ladder (Fermentas, USA) were separated by electrophoresis on 1.5 % agarose gel (Applichem, Germany,
GmbH) stained with 0.5 μg/ml ethidium bromide (Sigma,
USA) in 1 × TBE buffer on a mini slab horizontal electrophoresis unit (Bio-Rad, USA) at 100 V for 30 min. The gel
was photographed by a gel documentation system (Alpha
Innotech, Biometra) and the data was analyzed through
computer software.


**Molecular detection and sequencing of class 1 integrons**
All MDR _Salmonella_ isolates (n = 12) were tested for
the presence of conserved regions (5′-CS and 3′-CS) of
class Ι integrons using specific primers with the following sequences: 5′ CS-GGC ATC CAA GCA GCA AG and


