Osman et al.

Meat MSSA, MRSA, MS-NSA, MR-NSA

that were resistant and intermediate were categorized as non-susceptible while the sensitive strains were categorized as susceptible.

## DNA Extraction

Bacterial DNA from the 27 staphylococci were isolated from an overnight cultures using the boiling protocol previously described (Sowmya et al., 2012). DNA precipitates were resuspended in an appropriate volume of Tris-EDTA buffer solution.

## PCR for Detection of Genes Encoding Antimicrobial Resistance

The 27 Staphylococcus species isolates were confirmed as MSSA, MRSA, methicillin-susceptible non-Staphylococcus aureus (non-MSSA), and methicillin-resistant non-Staphylococcus aureus (non-MRSA) molecularly to identify mecA (the gold-standard reference method for this analysis). In addition, four antimicrobial resistance genes responsible for quinolone resistance (gyrA, gyrB, and grlA genes) which are habitually found in S. aureus according to resistance in the quinolone-resistance-determining-regions (QRDRs) and responsible for quinolone resistance (gyrA, gyrB, and grlA genes) and the cfr gene which confers resistance to five antibiotic classes (oxazolidinones, phenicols, streptogramin compounds, lincosamidins, and pleuromutilins usually referred to as PhLOPSA phenotype) were also amplified. Annealing temperatures and oligonucleotides used are presented in Table 1. The PCR runs included a reagent positive and negative control (without template DNA). Three independent PCR amplifications were carried out with a GeneAmp PCR System 2400 (Perkin-Elmer, Weiterstadt, Germany). Two microliters of template DNA was added to 23 mL of distilled sterile water, and finally 25 mL of reaction mixture containing 10mM Tris-HCl (pH8.3), 50mM KCl, 2.5mM MgCl₂, 100 mM deoxynucleoside triphosphates, 1 mM of each primer and 3U of Taq polymerase were added. The amplicons were analyzed by electrophoresis on 1.5% (w/v) agarose gel in TBE buffer.



## RESULTS

## I dentification of Staphylococci

We identified 27 staphylococcal isolates belonging to five different staphylococcal species. These included S. hyicus (n = 10, 37%), S. aureus (n = 9, 33.3%), S. schleiferi subsp. coagulans (n = 4, 14.8%), S. intermedius (n = 3, 11.1%), and S. lentus (n = 1, 3.7%).

## Phenotypic Virulence Assays

Summarized results of examining the 27 staphylococci isolates for factors that possibly contribute to their virulence are presented in Table 2.

## Haemolysin Production

Haemolytic activity on plates with 5% defibrinated rabbit or sheep blood was established. In the strains showing hemolytic activity, the three patterns of haemolysis ( $ \beta $ ,  $ \alpha $ , and  $ \gamma $ ) were observed, 66.7% were demonstrated to be  $ \beta $ -hemolytic, 22.2% were  $ \alpha $ -hemolytic, and 10.1% were  $ \gamma $ -hemolytic.

## Cytotoxicity Assay

All of the 27 Staphylococcus isolates were cytotoxic to Vero cells.

## Biofilm Producing Phenotype

We found a highly resistant pattern among biofilm producers in comparison with non-biofilm producers. Among the 7/9 vancomycin resistant S. aureus isolates, they were also a

<div style="text-align: center;">TABLE 1 | Antimicrobial resistance genes targets, primers and nucleotide sequences.</div>



<table border=1 style='margin: auto; width: max-content;'><tr><td style='text-align: center;'>Gene designation</td><td style='text-align: center;'>Oligonucleotide sequences (5′-3′)</td><td style='text-align: center;'>Amplion size (bp)</td><td style='text-align: center;'>PCR conditions</td><td style='text-align: center;'>References</td></tr><tr><td style='text-align: center;'>mecA</td><td style='text-align: center;'>F: GTAGAAATGACTGAACGTCCGATAA
R: CCAATTCCACATTGTTTTCGGTCTAA</td><td style='text-align: center;'>310</td><td style='text-align: center;'>Initial denaturation: 95°C for 1 min
Amplification (30 cycles of)
Denaturation: 95°C for 30 s
Annealing: 50°C for 30 s
Extension: 72°C for 60 s
Final extension: 72°C for 4 min</td><td style='text-align: center;'>Spanu et al., 2004</td></tr><tr><td style='text-align: center;'>gyrA</td><td style='text-align: center;'>F: ATGGCTGAATTACCTCAATC
R:CATCATAGTTATCGATGAAATC</td><td style='text-align: center;'>399</td><td rowspan="2">Initial denaturation: 95°C for 1 min
Amplification (30 cycles of)
Denaturation: 95°C for 30 s
Annealing: 55°C for 30 s
Extension: 72°C for 60 s</td><td style='text-align: center;'>Dubin et al., 1999</td></tr><tr><td style='text-align: center;'>grlA</td><td style='text-align: center;'>F: AATACGTATGATAAGAATTTCCG
R: GTTGTGTCATCATAGTTTGG</td><td style='text-align: center;'>429</td><td style='text-align: center;'>Dubin et al., 1999</td></tr><tr><td style='text-align: center;'>gyrB</td><td style='text-align: center;'>F: CAGCGTTAGATGTAGCAAGC
R: CCGATTCCTGTACCAAATGC</td><td style='text-align: center;'>250</td><td style='text-align: center;'>Final extension: 72°C for 4 min</td><td style='text-align: center;'>Linde et al., 2001</td></tr><tr><td style='text-align: center;'>cfr</td><td style='text-align: center;'>F: TGAAGTATAAAGCAGGTTGGGAGTCAR: ACCATATAATTGACCACAAGCAGC</td><td style='text-align: center;'>746</td><td style='text-align: center;'>Initial denaturation: 94°C for 1 min
Amplification (34 cycles of)
Denaturation: 94°C for 1 min
Annealing: 48°C for 30 s
Extension: 72°C for 3 min
Final extension: 72°C for 7 min</td><td style='text-align: center;'>Kehreenberg and Schwarz, 2006</td></tr></table>

Frontiers in Microbiology | www.frontiersin.org

4

February 2016 | Volume 7 | Article 222