Nguyen et al. Gut Pathog (2016) 8:39

Page 4 of 9

<div style="text-align: center;">Table 1 List of primers and primer sequences used for detection of antimicrobial resistance genes</div>



<table border=1 style='margin: auto; width: max-content;'><tr><td style='text-align: center;'>Antibiotic</td><td style='text-align: center;'>Method</td><td style='text-align: center;'>Primer</td><td style='text-align: center;'>Sequence (5′-3′)</td><td style='text-align: center;'>Amplicon length (bp)</td><td style='text-align: center;'>Reference</td></tr><tr><td rowspan="2">Erythromycin</td><td style='text-align: center;'>MAMA-PCR $ ^{a} $</td><td style='text-align: center;'>23SRNA-F ERY2075-R ERY2074-R</td><td style='text-align: center;'>TTA GCT AAT GTT GCC CGT ACC G TAG TAA AGG TCC ACG GGGTCG C AGT AAA GGT CCA CGG GGT CTG G</td><td style='text-align: center;'>485</td><td style='text-align: center;'>[17]</td></tr><tr><td style='text-align: center;'>PCR-RFLP $ ^{a} $</td><td style='text-align: center;'>F2-campy-23S R2-campy-23S</td><td style='text-align: center;'>AAT TGA TGG GGT TAG CAT TAG C CAA CAA TGG CTC ATA TAC AACTGG</td><td style='text-align: center;'>316</td><td style='text-align: center;'>[20]</td></tr><tr><td rowspan="2">Ciprofloxacin</td><td style='text-align: center;'>MAMA-PCR $ ^{b} $</td><td style='text-align: center;'>CampyMAMAgyrA1 CampyMAMAgyrA5</td><td style='text-align: center;'>TTT TTA GCA AAG ATT CTG AT CAA AGC ATC ATA AAC TGC AA</td><td style='text-align: center;'>265</td><td style='text-align: center;'>[19]</td></tr><tr><td style='text-align: center;'>MAMA-PCR $ ^{c} $</td><td style='text-align: center;'>GZgyrACcoli3F CanpyMAMAgyrA8</td><td style='text-align: center;'>TAT GAG CGA TAT TAT CGG TC TAA GGC ATC GTA AAC AGC CA</td><td style='text-align: center;'>192</td><td style='text-align: center;'>[24]</td></tr><tr><td rowspan="3">Tetracycline</td><td style='text-align: center;'>tet(O) PCR</td><td style='text-align: center;'>DMT 1 DMT 2</td><td style='text-align: center;'>GGC GTT TTG TTT ATG TGC G ATG GAC AAC CCG ACA GAA GC</td><td style='text-align: center;'>559</td><td style='text-align: center;'>[25]</td></tr><tr><td style='text-align: center;'>tet(A) PCR</td><td style='text-align: center;'>Tet(A)-F Tet(A)-R</td><td style='text-align: center;'>GTG AAA CCC AAC ATA CCC C GAA GGC AAG CAG GAT GTA G</td><td style='text-align: center;'>888</td><td style='text-align: center;'>[18]</td></tr><tr><td style='text-align: center;'>tet(A) PCR</td><td style='text-align: center;'>tet-A-1 tet-A-2</td><td style='text-align: center;'>GCT CAC GTT GAC GCA GGA AAG ATC GTC ATT GTC CGT TAC</td><td style='text-align: center;'>486</td><td style='text-align: center;'>This study</td></tr></table>

 $ ^{a} $  23S rRNA gene mutation

 $ ^{b} $  gyrA gene mutation Campylobacter jejuni

c gyrA gene mutation Campylobacter coli

JX891463 and JX891464)). Briefly, primers tet-A-1 and tet-A-2 (Table 1; Jena Bioscience GmbH) were used with the following PCR programme: An initial denaturation at 96 °C for 60 s was followed by 35 cycles of denaturation (96 °C for 15 s), annealing (49 °C for 60 s) and extension (72 °C for 30 s). PCR was terminated by final extension at 72 °C for 60 s. The PCR resulted in a 486 bp product.

All PCR products were analyzed by electrophoresis on 1.5 % agarose gels, staining with ethidium bromide and visualization under UV light.

## DNA sequencing

PCR products obtained by tet(A) PCRs were sequenced by cycle sequencing with BigDye Terminator v1.1 Cycle Sequencing Kit (Applied Biosystems, Darmstadt, Germany) according to the instructions of the manufacturer. In addition to the amplification primers of the Tet(A)-F/R fragment tet-A-A (5'-AAT TTT CTT CAA ATA AGG-3') and tet-A-B (5'-GTC ATT CTT ATA TTA AGT GG-3') were used as sequencing primers. Sequencing products were analyzed with an ABI PRISM 3130 genetic analyzer.

## MALDI-TOF mass spectrometry

Cultured bacteria were suspended in 300  $ \mu $ l of bi-distilled water and mixed with 900 ml of ethanol (Carl Roth GmbH). Further treatment of samples and measurement were described by El-Ashker et al. [30].

## Results

## I dentification of bacteria

In total, 58 isolates suspected as Campylobacter were recovered from faeces and cloacal swabs of chicken flocks in Kenya. After storage in skimmed milk medium and transportation to Germany only 40 of these isolates could be re-cultivated. Four C. coli and 31 C. jejuni were identified by mPCR (Table 2). Five other isolates were identified by MALDI-TOF mass spectrometry as members of genera Bacillus, Staphylococcus, Ochrobactrum as well as two Bordetella isolates.



Eighteen skimmed milk tubes contained Campylobacter DNA  $ [14\ C.\ coli,\ 3\ C.\ jejuni\ and\ one\ sample\ harboured\ both\ C.\ coli\ and\ C.\ jejuni\ (Table\ 2)] $ .

## Antimicrobial susceptibility profiles and multidrug resistance

The results of antimicrobial susceptibility testing of C. jejuni and C. coli isolates and the rate of resistance to seven antimicrobial agents are given in Tables 3 and 4, respectively. The C. jejuni isolates showed a high rate of resistance to nalidixic acid, tetracycline and ciprofloxacin with 77.4, 71.0 and 71.0 %, respectively. Low resistance rates were detected for gentamicin and chloramphenicol, both with 25.8 % of the isolates. For the low number of C. coli isolates (n = 4) a similar pattern was observed. Only two isolates were susceptible to all tested antimicrobial agents, one isolate was resistant to all tested antibiotics.

The multidrug resistance profiles of 31 C. jejuni isolates are shown in Table 5. Multidrug resistance to three or more classes of antibiotics was found in 19 isolates (61.3 %) and was observed in a range between 5.3 and 26.3 %.

## Molecular biological detection of antibiotic resistance determinants

DNA of 35 viable Campylobacter isolates and of 18 non-growing samples was investigated by PCR to detect antibiotic resistance. Mismatch amplification mutation assay