# Bacteriophages Isolated from Chicken Meat and the Horizontal Transfer of Antimicrobial Resistance Genes

Amira Shousha, $ ^{a} $  Nattakarn Awaiwanont, $ ^{a} $  Dmitrij Sofka, $ ^{a} $  Frans J. M. Smulders, $ ^{a} $  Peter Paulsen, $ ^{a} $  Michael P. Szostak, $ ^{b} $  Tom Humphrey, $ ^{c} $  Friederike Hilbert $ ^{a} $ 

Institute of Meat Hygiene, Meat Technology and Food Science, University of Veterinary Medicine, Vienna, Austria $ ^{a} $ ; Institute of Microbiology, University of Veterinary Medicine, Vienna, Austria $ ^{b} $ ; Medical Microbiology and Infectious Diseases Group, College of Medicine, Swansea University, Swansea, Wales, United Kingdom $ ^{c} $ 

Antimicrobial resistance in microbes poses a global and increasing threat to public health. The horizontal transfer of antimicrobial resistance genes was thought to be due largely to conjugative plasmids or transposons, with only a minor part being played by transduction through bacteriophages. However, whole-genome sequencing has recently shown that the latter mechanism could be highly important in the exchange of antimicrobial resistance genes between microorganisms and environments. The transfer of antimicrobial resistance genes by phages could underlie the origin of resistant bacteria found in food. We show that chicken meat carries a number of phages capable of transferring antimicrobial resistance. Of 243 phages randomly isolated from chicken meat, about a quarter (24.7%) were able to transduce resistance to one or more of the five antimicrobials tested into Escherichia coli ATCC 13706 (DSM 12242). Resistance to kanamycin was transduced the most often, followed by that to chloramphenicol, with four phages transducing tetracycline resistance and three transducing ampicillin resistance. Phages able to transduce antimicrobial resistance were isolated from 44% of the samples of chicken meat that we tested. The statistically significant  $ (P = 0.01) $  relationship between the presence of phages transducing kanamycin resistance and E. coli isolates resistant to this antibiotic suggests that transduction may be an important mechanism for transferring kanamycin resistance to E. coli. It appears that the transduction of resistance to certain antimicrobials, e.g., kanamycin, not only is widely distributed in E. coli isolates found on meat but also could represent a major mechanism for resistance transfer. The result is of high importance for animal and human health.

The use of antimicrobial agents in veterinary medicine and the development of antimicrobial resistance, especially in food-producing animals, are currently giving rise to intense discussion  $ (1) $ . Food of animal origin represents an important and direct link between animals and humans. Despite approved recommendations regarding the prudent use of antimicrobial substances in production animals, the rate of resistance to certain classes of antibiotics (e.g., extended-spectrum cephalosporins) used in human medicine continues to rise  $ (2, 3) $ . Escherichia coli, an important indicator bacterium for antimicrobial resistance, can frequently be found on food surfaces and has been reported to be resistant to many different classes of antimicrobial agents. This microorganism easily acquires and carries different antimicrobial resistance elements (in the last report on antimicrobial resistance in the European Union, at least one  $ E.\ coli $  isolate was found to be resistant to every single antimicrobial substance tested  $ [4] $ ) and not only transfers these elements within the species but also receives these elements from and transfers them to related microorganisms  $ (5–7) $ . Resistance gene exchange between Salmonella and  $ E.\ coli $  has also been described  $ (8) $ .

Bacterial resistance to antimicrobial agents develops by a variety of mechanisms: (i) by changes within the existing genome of a bacterial cell (mutations) and changes within the proteome (phase variation)  $ (9, 10) $ , (ii) by the formation of bacterial cell interactions (mixed bacterial biofilms)  $ (11) $ , and (iii) most importantly, by the acquisition of new genes through horizontal transfer  $ (12) $ . Bacteria have evolved numerous mechanisms to take up new genes to help them withstand harsh environmental conditions or settle in a new environmental niche. The same mechanism now also enables them to develop antimicrobial resistance  $ (13, 14) $ . Horizontal transfer of antimicrobial resistance genes was thought to be largely the result of conjugative plasmids, transposons, or the uptake of naked DNA, with only a minor part being played by transduction by bacteriophages  $ (12, 15, 16) $ .



Phages (bacterial viruses) depend on the bacterial cell to produce viral proteins or peptides. They are very common in all environments, outnumbering their bacterial hosts by 10:1 (17, 18). Bacteriophages are highly diverse and can be classified on the basis of their genome, morphology, and host specificity. Most phages are host specific, but some broad-host-range ones are able to invade different bacterial species and even different genera (19, 20). Infection of bacterial cells occurs when the phage attaches itself to the host cell and injects viral and sometimes foreign DNA (RNA) into it in a penetration step. Phages can be lytic or nonlytic, leading in the former case to the death of the host and in the latter case to the production of bacteria with phage DNA incorporated into their genomes. Temperate phages lead to lysogeny, where the bacterial cell becomes infected for an indefinite period of time. While

Received 16 March 2015 Accepted 22 April 2015

Accepted manuscript posted online 1 May 2015

Citation Shousha A, Awaiwanont N, Sofka D, Smulders FJM, Paulsen P, Szostak MP, Humphrey T, Hilbert F. 2015. Bacteriophages isolated from chicken meat and the horizontal transfer of antimicrobial resistance genes. Appl Environ Microbiol 81:4600–4606. doi:10.1128/AEM.00872-15.

Editor: J. Björkroth

Address correspondence to Friederike Hilbert, friederike.hilbert@vetmeduni.ac.at. Copyright © 2015, American Society for Microbiology. All Rights Reserved. doi:10.1128/AEM.00872-15

4600

aem.asm.org

Applied and Environmental Microbiology

July 2015 Volume 81 Number ___

14