|Col1|Col2|Col3|Col4|Col5|
|---|---|---|---|---|
||||||



the prevalence of _Salmonella_ all over the world, but most common

are _Salmonella_ Enteritidis, _Salmonella_ Typhimurium, and _Salmonella_

Pullorum (Batista et al., 2015). For example, _S._ Senftenberg has

been proved to be the dominant serovar in cooked meat products

in Chinese Henan province (Yu, Jiang, Zhou, Wu, & Wu, 2014). The

various serotypes have different virulence characteristics. Serotype

Enteritidis is mainly transmitted through contaminated meat and egg

products, which is the primary cause of human salmonellosis (Foley

et al., 2011). This serotype was also the most widespread in our study

(Figure 1). Serotype Typhi is defined as often being resistant to some

first‐line recommended antibiotics (Hardjo Lugito & Cucunawangsih,

2017). There are 11 serovars found in our _Salmonella_ isolates. These

serovars have also been frequently isolated from different sources in

China (Gong et al., 2014). _S._ Pullorum, which formed a large propor
tion of the isolates, are similar to strains isolated from gastroenteritis

(Luo, Yi, Yao, Zhu, & Qin, 2017). As _Salmonella_ serovars are closely

related to pathogenicity, it is necessary to monitor changes in their

distribution.

Molecular subtyping is widely used to analyze genetic diversity.

MLST divided the _Salmonella_ isolates into three clonal groups from

different sources, which differed in all seven alleles profiled. We

found that diverse STs mean that the isolates originated from more

than one common ancestor and possess several distantly related

STs. In the database, our STs were first isolated from diverse sources.

For example, ST31, ST39, and ST40 were from pig farms, pig slaugh
terhouses, retail markets, and humans in China (Zheng et al., 2017).

ST533, ST813, and ST640 were from clinics in Ethiopia and France.

Our MLST results were used for analyzing correlations among the

_Salmonella_ strains. ERIC‐PCR fingerprinting has also been used to

confirm epidemiological relationships between various _Salmonella_

isolates (Li, Liu, et al., 2017). In Korea, researchers claimed that ERIC‐

PCR combined with virulence profiling offered a rapid approach to

characterize antimicrobial‐resistant _Salmonella_ (Jin & Lee, 2017). We

did not find a significant association between ERIC‐PCR clusters,

source, and other characteristics. However, isolates in the last two

MLST and ERIC‐PCR were similar. In general, MLST and ERIC‐PCR

are useful as phylogenetic tools for investigations outbreaks of this

pathogen.


**5** | **CONCLUSION**


_Salmonella enterica_ food poisoning due to egg consumption is a

longstanding problem. Our study is the first to comprehensively

analyze the prevalence, antibiotic resistance, serotype, and molecu
lar subtype of strains isolated from Guangdong eggs, which are an

important local food of native. The results showed contamination

ratio of _Salmonella_ was 5.40%. Antibiotic resistance was widespread,

especially for β‐lactam antibiotics. The isolates belonged to multiple

serovars. MLST and ERIC‐PCR typing showed genetic similarity and

diversity within those isolates. These results provide useful informa
tion to improve strategies to control and treat _Salmonella_ infections.



**ACKNOWLEDGMENTS**


Our work was supported by Guangdong Forestry Science and

Technology Innovation Project (2016KJCX037) and Quality

Engineering Training Base Project of Guangdong Eco‐engineering

Polytechnic (2017ZLGC‐XJ‐GZS03).


**CONFLICT OF INTEREST**


The authors declare that they do not have any conflict of interest.


**AUTHOR CONTRIBUTIONS**


Tengfei Xie, Gang Wu are the common first authors, finished

this article experiment and write the article together. Jing Zhao

(Corresponding Author) give the idea and experiments support.

Xujun He, Zengzhe Lai, Huatong Zhang help to finish the experiment

on article.


**ETHICAL STATEMENT**


Ethical Review: This study does not involve any human or animal

testing.


**INFORMED CONSENT**


Written informed consent was obtained from all study participants.


**ORCID**


_Jing Zhao_ [https://orcid.org/0000-0003-2032-4371](https://orcid.org/0000-0003-2032-4371)


**REFERENCES**


Abbott, S. L., Ni, F. C., & Janda, J. M. (2012). Increase in extraintestinal
infections caused by _Salmonella enterica_ subspecies II–IV. _Emerging_
_Infectious Diseases_, _18_, 637–639.
Al, M. E. P. E. (2004). _Salmonella Enteritidis_ infections, United States,
1985–1999 – Volume 10, Number 1—January 2004 – Emerging
Infectious Disease journal – CDC. _Emerging Infectious Diseases_, _10_,
1–7.
Andrews, J. R., & Ryan, E. T. (2015). Diagnostics for invasive _Salmonella_
infections: Current challenges and future directions. _Vaccine_, _33_,
[C8–C15. https​://doi.org/10.1016/j.vacci​ne.2015.02.030](https://doi.org/10.1016/j.vaccine.2015.02.030)
Badouei, M. A., Ghalejooghi, B. M., & Madadgar, O. (2012). Study on
_Salmonella_ contamination of traditionally produced edible poultry
eggs. _Comparative Clinical Pathology_, _21_ [, 1093–1097. https​://doi.](https://doi.org/10.1007/s00580-011-1238-z)
[org/10.1007/s00580-011-1238-z](https://doi.org/10.1007/s00580-011-1238-z)
Batista, D. F. A., Freitas Neto, O. C., Barrow, P. A., de Oliveira, M. T.,
Almeida, A. M., Ferraudo, A. S., & Berchieri Jr., A. (2015). Identification
and characterization of regions of difference between the Salmonella
Gallinarum biovar Gallinarum and the Salmonella Gallinarum biovar
Pullorum genomes. _Infection Genetics and Evolution_, _30_ [, 74–81. https​](https://doi.org/10.1016/j.meegid.2014.12.007)
[://doi.org/10.1016/j.meegid.2014.12.007](https://doi.org/10.1016/j.meegid.2014.12.007)
Ben, S. R., Abbassi, M. S., García, V., Garcíafierro, R., Fernández, J., Kilani,
H., … Rodicio, M. R. (2017). Antimicrobial drug resistance and genetic properties of _Salmonella enterica_ serotype Enteritidis circulating


