Nguyen _et al. Gut Pathog (2016) 8:39_ Page 4 of 9


**Table 1 List of primers and primer sequences used for detection of antimicrobial resistance genes**


**Antibiotic** **Method** **Primer** **Sequence (5’–3’)** **Amplicon length (bp)** **Reference**



Erythromycin MAMA–PCR [a] 23SRNA-F
ERY2075-R
ERY2074-R



TTA GCT AAT GTT GCC CGT ACC G
TAG TAA AGG TCC ACG GGG TCG C
AGT AAA GGT CCA CGG GGT CTG G



485 [17]



PCR–RFLP [a] F2-campy-23S AAT TGA TGG GGT TAG CAT TAG C 316 [20]
R2-campy-23S CAA CAA TGG CTC ATA TAC AAC TGG

Ciprofloxacin MAMA–PCR [b] CampyMAMAgyrA1 TTT TTA GCA AAG ATT CTG AT 265 [19]
CampyMAMAgyrA5 CAA AGC ATC ATA AAC TGC AA

MAMA–PCR [c] GZgyrACcoli3F TAT GAG CGA TAT TAT CGG TC 192 [24]
CanpyMAMAgyrA8 TAA GGC ATC GTA AAC AGC CA

Tetracycline _tet_ (O) PCR DMT 1 GGC GTT TTG TTT ATG TGC G 559 [25]
DMT 2 ATG GAC AAC CCG ACA GAA GC

_tet_ (A) PCR Tet(A)-F GTG AAA CCC AAC ATA CCC C 888 [18]
Tet(A)-R GAA GGC AAG CAG GAT GTA G

_tet_ (A) PCR tet-A-1 GCT CAC GTT GAC GCA GGA AAG 486 This study
tet-A-2 ATC GTC ATT GTC CGT TAC


a 23S rRNA gene mutation

b _gyr_ A gene mutation _Campylobacter jejuni_

c _gyr_ A gene mutation _Campylobacter coli_



JX891463 and JX891464)). Briefly, primers tet-A-1 and
tet-A-2 (Table 1; Jena Bioscience GmbH) were used with
the following PCR programme: An initial denaturation at
96 °C for 60 s was followed by 35 cycles of denaturation
(96 °C for 15 s), annealing (49 °C for 60 s) and extension
(72 °C for 30 s). PCR was terminated by final extension at
72 °C for 60 s. The PCR resulted in a 486 bp product.
All PCR products were analyzed by electrophoresis on
1.5 % agarose gels, staining with ethidium bromide and
visualization under UV light.


**DNA sequencing**
PCR products obtained by _tet_ (A) PCRs were sequenced
by cycle sequencing with BigDye Terminator v1.1 Cycle
Sequencing Kit (Applied Biosystems, Darmstadt, Germany) according to the instructions of the manufacturer.
In addition to the amplification primers of the Tet(A)-F/R
fragment tet-A-A (5’-AAT TTT CTT CAA ATA AGG-3’)
and tet-A-B (5’-GTC ATT CTT ATA TTA AGT GG-3’)
were used as sequencing primers. Sequencing products
were analyzed with an ABI PRISM 3130 genetic analyzer.


**MALDI‑TOF mass spectrometry**
Cultured bacteria were suspended in 300 µl of bi-distilled water and mixed with 900 ml of ethanol (Carl Roth
GmbH). Further treatment of samples and measurement
were described by El-Ashker et al. [30].


**Results**
**Identification of bacteria**
In total, 58 isolates suspected as _Campylobacter_ were
recovered from faeces and cloacal swabs of chicken flocks
in Kenya. After storage in skimmed milk medium and



transportation to Germany only 40 of these isolates could
be re-cultivated. Four _C. coli_ and 31 _C. jejuni_ were identified by mPCR (Table 2). Five other isolates were identified by MALDI-TOF mass spectrometry as members of
genera _Bacillus_, _Staphylococcus_, _Ochrobactrum_ as well as
two _Bordetella_ isolates.
Eighteen skimmed milk tubes contained _Campylobac-_
_ter_ DNA [14 _C. coli_, 3 _C. jejuni_ and one sample harboured
both _C. coli_ and _C. jejuni_ (Table 2)].


**Antimicrobial susceptibility profiles and multidrug**
**resistance**
The results of antimicrobial susceptibility testing of _C._
_jejuni_ and _C. coli_ isolates and the rate of resistance to
seven antimicrobial agents are given in Tables 3 and 4,
respectively. The _C. jejuni_ isolates showed a high rate of
resistance to nalidixic acid, tetracycline and ciprofloxacin
with 77.4, 71.0 and 71.0 %, respectively. Low resistance
rates were detected for gentamicin and chloramphenicol,
both with 25.8 % of the isolates. For the low number of _C._
_coli_ isolates (n = 4) a similar pattern was observed. Only
two isolates were susceptible to all tested antimicrobial
agents, one isolate was resistant to all tested antibiotics.
The multidrug resistance profiles of 31 _C. jejuni_ isolates
are shown in Table 5. Multidrug resistance to three or
more classes of antibiotics was found in 19 isolates (61.3 %)
and was observed in a range between 5.3 and 26.3 %.


**Molecular biological detection of antibiotic resistance**
**determinants**
DNA of 35 viable _Campylobacter_ isolates and of 18 nongrowing samples was investigated by PCR to detect antibiotic resistance. Mismatch amplification mutation assay


