Fashae _et al._ _–_ _Salmonella_ from chickens and humans in Nigeria _J Infect Dev Ctries_ 2010; 4(8):484-494.


**Table 1.** Number of blood samples collected from humans (male/female) and the number and prevalence of _Salmonella_ spp.


|by sites.|Col2|Col3|
|---|---|---|
|Location|No. blood samples collected|No. (%) blood samples positive|
|Location|Total<br>Male<br>Female|Total<br>Male<br>Female|
|Hospital A<br>Hospital B<br>Hospital C<br>Hospital D<br>Hospital E<br>Hospital F<br>Hospital G<br>Diagnostic lab.|204<br>116<br>88<br>119<br>66<br>53<br>120<br>66<br>54<br>102<br>52<br>50<br>104<br>54<br>50<br>107<br>38<br>69<br>157<br>83<br>74<br>78<br>44<br>34|9 (4)<br>8 (7)<br>1 (1)<br>12 (10)<br>10 (15)<br>2 (4)<br>5 (4)<br>1 (2)<br>4 (7)<br>4 (4)<br>2 (4)<br>2 (4)<br>3 (3)<br>1 (2)<br>2 (4)<br>2 (2)<br>2 (5)<br>0 (0)<br>3 (2)<br>2 (2)<br>1 (1)<br>1 (1)<br>0 (0)<br>1 (3)|
|Total|991<br>519<br>472|39 (4)<br>26 (5)<br>13 (3)|



Copenhagen, Denmark (DTU-FOOD) on all
_Salmonella_ isolates as minimum inhibitory
concentration (MIC) determinations according to the
previously described method [27]. In addition to the
previously described method, the following
antimicrobial and resistance epidemiological cut-off
values were used in the study for cefotaxime; (R > 2
mg/mL) (www.eucast.org).


_Detection of plasmid mediated quinolone resistance_
_genes_

Strains showing reduced susceptibility to
ciprofloxacin and susceptible to nalidixic acid were
further characterized through the use of a PCR assay
with primers specific for _qnr_ A, _qnr_ B, _qnr_ C, _qnr_ D,
_qnr_ S, and _aac(6’)-lb_ [23,28,29]. PCR products were
purified using the GFX™ PCR DNA kit (GE
Healthcare, Chalfont St. Giles, UK) and submitted to
Macrogen Inc. for sequencing. Sequence analysis and
alignment was performed using the Vecton NTI suite
9 (InforMax Inc., Bethesda, Maryland, US) and the
resulting nucleotide sequences were compared to
sequences obtained from GenBank
(http://www.lahey.org/studies/webt.html).


**Results**
_Bacterial isolates from humans_

Of the 991 blood samples collected, 519 samples
originated from males. Thirty-nine samples yielded
_Salmonella_, which gave an overall prevalence among
humans submitting a blood sample at 4%. The
prevalence ranged substantially between the different
sample sites from 1% (diagnostic laboratory) to 10%
(hospital B) (Table 1). The prevalence between males
and females differed with 5% among males in
contrast to 3% among females (not significant p =



0.07). The overall age of patients from whom the
positive blood samples were collected ranged from
seven days to 28 years; however, large differences
were observed between the different hospitals: 7 days
to 4 years in hospital A; 3 months to 2 years and 6
months in hospital B; 18 days to 2 years in hospital
C; 2 years and 5 months to 7 years in hospital D; 1
year and 7 months to 10 years and 7 months in
hospital E; 1 year and 3 months to 16 years in
hospital F; 11 years to 28 years in hospital G; and 14
years at the diagnostic laboratory. The median age of
the febrile patients that yielded _Salmonella_ was 1
year and 7 months.


_Bacterial isolates from chickens_

Out of the 641 chicken fecal samples collected,
70 were positive for _Salmonella_, giving an overall
prevalence of 11%. The number of samples
investigated and the number of samples positive for
_Salmonella_ fluctuated among the different farms,
resulting in a prevalence ranging from 2% (farm C
and G) to 26% (farm J) (Table 2).


_Serotyping of human isolates_

The 39 human isolates revealed nine different
_Salmonella_ serovars of which 32 (82%) were NTS
and seven (18%) were _S._ Typhi. The most commonly
isolated serovars were _S_ . Enteritidis (n = 13; 33%), _S_ .
Dublin (n = 7; 18%), _S_ . Typhimurium (n = 7; 18), and
_S_ . Typhi (n = 7; 18%). Several uncommon serovars
were among the isolated _Salmonella_, such as _S_ .
Jukestown (n = 1; 3%), _S_ . Monschaui (n = 1; 3%), _S._
Oritamerin (n = 1; 3%), and _S._ Apapa (n = 1; 3%),
which were all from different sample sites (Table 3).


486


