Gharieb et al. Gut Pathog (2015) 7:34

Page 10 of 11

<div style="text-align: center;">Table 4 Primers' sequences and amplification's conditions used for detection of Salmonella virulence genes</div>



<table border=1 style='margin: auto; width: max-content;'><tr><td rowspan="2">Target gene</td><td rowspan="2">Primers’ sequences 5′-3′</td><td rowspan="2">Amplified segment (bp)</td><td rowspan="2">Primary denaturation</td><td colspan="3">Amplification (35 cycles)</td><td rowspan="2">Final extension</td><td rowspan="2">References</td></tr><tr><td style='text-align: center;'>Secondary denaturation</td><td style='text-align: center;'>Annealing</td><td style='text-align: center;'>Extension</td></tr><tr><td style='text-align: center;'>pefA</td><td style='text-align: center;'>TGTTTCCGGGGCTTGTGCT</td><td style='text-align: center;'>700</td><td style='text-align: center;'>94 °C 10 min</td><td style='text-align: center;'>94 °C 45 s</td><td style='text-align: center;'>55 °C 45 s</td><td style='text-align: center;'>72 °C 45 s</td><td style='text-align: center;'>72 °C 10 min</td><td style='text-align: center;'>[4]</td></tr><tr><td style='text-align: center;'>hilA</td><td style='text-align: center;'>CGGAAGCTTATTTGCGCCATGCT GAGGTAG</td><td style='text-align: center;'>854</td><td style='text-align: center;'>94 °C 10 min</td><td style='text-align: center;'>94 °C 45 s</td><td style='text-align: center;'>60 °C 45 s</td><td style='text-align: center;'>72 °C 45 s</td><td style='text-align: center;'>72 °C 10 min</td><td style='text-align: center;'>[37]</td></tr><tr><td style='text-align: center;'>sopB</td><td style='text-align: center;'>TCAGAAGRCGTCTAACCACTC</td><td style='text-align: center;'>517</td><td style='text-align: center;'>94 °C 5 min</td><td style='text-align: center;'>94 °C 30 s</td><td style='text-align: center;'>58 °C 30 s</td><td style='text-align: center;'>72 °C 30 s</td><td style='text-align: center;'>72 °C 7 min</td><td style='text-align: center;'>[38]</td></tr><tr><td style='text-align: center;'>stn</td><td style='text-align: center;'>TTGTGTCGCTATCACTGGCAACC</td><td style='text-align: center;'>617</td><td style='text-align: center;'>94 °C 10 min</td><td style='text-align: center;'>94 °C 45 s</td><td style='text-align: center;'>59 °C 45 s</td><td style='text-align: center;'>72 °C 45 s</td><td style='text-align: center;'>72 °C 10 min</td><td style='text-align: center;'>[4]</td></tr><tr><td style='text-align: center;'>ATTCGTAACCCGCTCTCGTCC</td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td><td style='text-align: center;'></td></tr></table>

3'CS- AAG CAG ACT TGA CCT GA [30]. Chromosomal DNA extraction, amplification and gel analysis were performed as mentioned before except to amplify the DNA in the thermal cycler we used a three-step profile (cycling conditions): 1 min of denaturation at 94 °C, 1 min of annealing at 55 °C, and 5 min of extension at 72 °C for a total of 35 cycles. Five seconds were added to the extension time at each cycle. Amplified products with variable sizes were purified using QIA quick PCR Product extraction kit (Qiagen, Valencia). Each purified amplicon was sequenced in both forward and reverse directions using the amplification primers. DNA sequences were obtained by Applied Biosystems 3130 genetic analyzer (HITACHI, Japan). Nucleotide sequence analysis was compared with published GenBank DNA sequences using NCBI BLAST program (http://www.ncbi.nlm.nih.gov/BLAST/) and alignment was performed using MEGA6 program.

## Nucleotide sequence accession numbers

Nucleotide sequences from this study were deposited into the GenBank under accession numbers [KT427378, KT449569, KT449570, KT449571, KT581253, KT581254, KT581255, KT581256, KT581257 and KT598359].

## Authors' contributions

RMG, YHT and MHE participated in the design of the study, collected and analyzed the samples. YHT participated in the sequence alignment. RMG and YHT drafted the manuscript. All authors read and approved the final manuscript.

## Author details

 $ ^{1} $  Department of Zoonoses, Faculty of Veterinary Medicine, Zagazig University, 44511 Zagazig, Egypt.  $ ^{2} $  Department of Bacteriology, Mycology and Immunology, Faculty of Veterinary Medicine, Zagazig University, 44511 Zagazig, Egypt.  $ ^{3} $  Department of Veterinary Public Health, Faculty of Veterinary Medicine, Zagazig University, 44511 Zagazig, Egypt.

## Acknowledgements

We are grateful to thank all administrators and clinicians in Al-Ahrar General Hospital, Zagazig, Egypt for their assistance in collecting stool samples.

## Competing interests

The authors declare that they have no competing interests.

Received: 17 September 2015  Accepted: 10 December 2015  

Published online: 23 December 2015