## **Bacteriophages Isolated from Chicken Meat and the Horizontal** **Transfer of Antimicrobial Resistance Genes**

**Amira Shousha,** **[a]** **Nattakarn Awaiwanont,** **[a]** **Dmitrij Sofka,** **[a]** **Frans J. M. Smulders,** **[a]** **Peter Paulsen,** **[a]** **Michael P. Szostak,** **[b]**

**Tom Humphrey,** **[c]** **Friederike Hilbert** **[a]**


Institute of Meat Hygiene, Meat Technology and Food Science, University of Veterinary Medicine, Vienna, Austria [a] ; Institute of Microbiology, University of Veterinary
Medicine, Vienna, Austria [b] ; Medical Microbiology and Infectious Diseases Group, College of Medicine, Swansea University, Swansea, Wales, United Kingdom [c]


**Antimicrobial resistance in microbes poses a global and increasing threat to public health. The horizontal transfer of antimicro-**
**bial resistance genes was thought to be due largely to conjugative plasmids or transposons, with only a minor part being played**
**by transduction through bacteriophages. However, whole-genome sequencing has recently shown that the latter mechanism**
**could be highly important in the exchange of antimicrobial resistance genes between microorganisms and environments. The**
**transfer of antimicrobial resistance genes by phages could underlie the origin of resistant bacteria found in food. We show that**
**chicken meat carries a number of phages capable of transferring antimicrobial resistance. Of 243 phages randomly isolated from**
**chicken meat, about a quarter (24.7%) were able to transduce resistance to one or more of the five antimicrobials tested into**
_**Escherichia coli**_ **ATCC 13706 (DSM 12242). Resistance to kanamycin was transduced the most often, followed by that to chlor-**
**amphenicol, with four phages transducing tetracycline resistance and three transducing ampicillin resistance. Phages able to**
**transduce antimicrobial resistance were isolated from 44% of the samples of chicken meat that we tested. The statistically signifi-**
**cant (** _**P**_ - **0.01) relationship between the presence of phages transducing kanamycin resistance and** _**E. coli**_ **isolates resistant to**
**this antibiotic suggests that transduction may be an important mechanism for transferring kanamycin resistance to** _**E. coli**_ **. It**
**appears that the transduction of resistance to certain antimicrobials, e.g., kanamycin, not only is widely distributed in** _**E. coli**_
**isolates found on meat but also could represent a major mechanism for resistance transfer. The result is of high importance for**
**animal and human health.**



he use of antimicrobial agents in veterinary medicine and the
# T development of antimicrobial resistance, especially in food
producing animals, are currently giving rise to intense discussion
(1). Food of animal origin represents an important and direct link
between animals and humans. Despite approved recommendations regarding the prudent use of antimicrobial substances in
production animals, the rate of resistance to certain classes of antibiotics (e.g., extended-spectrum cephalosporins) used in human
medicine continues to rise (2, 3). _Escherichia coli_, an important
indicator bacterium for antimicrobial resistance, can frequently
be found on food surfaces and has been reported to be resistant to
many different classes of antimicrobial agents. This microorganism easily acquires and carries different antimicrobial resistance
elements (in the last report on antimicrobial resistance in the European Union, at least one _E. coli_ isolate was found to be resistant
to every single antimicrobial substance tested [4]) and not only
transfers these elements within the species but also receives these
elements from and transfers them to related microorganisms (5–
7). Resistance gene exchange between _Salmonella_ and _E. coli_ has
also been described (8).
Bacterial resistance to antimicrobial agents develops by a variety of mechanisms: (i) by changes within the existing genome of a
bacterial cell (mutations) and changes within the proteome (phase
variation) (9, 10), (ii) by the formation of bacterial cell interactions (mixed bacterial biofilms) (11), and (iii) most importantly,
by the acquisition of new genes through horizontal transfer (12).
Bacteria have evolved numerous mechanisms to take up new
genes to help them withstand harsh environmental conditions or
settle in a new environmental niche. The same mechanism now
also enables them to develop antimicrobial resistance (13, 14).
Horizontal transfer of antimicrobial resistance genes was thought



to be largely the result of conjugative plasmids, transposons, or the
uptake of naked DNA, with only a minor part being played by
transduction by bacteriophages (12, 15, 16).
Phages (bacterial viruses) depend on the bacterial cell to produce viral proteins or peptides. They are very common in all environments, outnumbering their bacterial hosts by 10:1 (17, 18).
Bacteriophages are highly diverse and can be classified on the basis
of their genome, morphology, and host specificity. Most phages
are host specific, but some broad-host-range ones are able to invade different bacterial species and even different genera (19, 20).
Infection of bacterial cells occurs when the phage attaches itself to
the host cell and injects viral and sometimes foreign DNA (RNA)
into it in a penetration step. Phages can be lytic or nonlytic, leading in the former case to the death of the host and in the latter case
to the production of bacteria with phage DNA incorporated into
their genomes. Temperate phages lead to lysogeny, where the bacterial cell becomes infected for an indefinite period of time. While


Received 16 March 2015 Accepted 22 April 2015


Accepted manuscript posted online 1 May 2015


Citation Shousha A, Awaiwanont N, Sofka D, Smulders FJM, Paulsen P, Szostak MP,
Humphrey T, Hilbert F. 2015. Bacteriophages isolated from chicken meat and the
horizontal transfer of antimicrobial resistance genes. Appl Environ Microbiol
[81:4600–4606. doi:10.1128/AEM.00872-15.](http://dx.doi.org/10.1128/AEM.00872-15)


Editor: J. Björkroth


Address correspondence to Friederike Hilbert, friederike.hilbert@vetmeduni.ac.at.


Copyright © 2015, American Society for Microbiology. All Rights Reserved.


[doi:10.1128/AEM.00872-15](http://dx.doi.org/10.1128/AEM.00872-15)



**4600** [aem.asm.org](http://aem.asm.org) Applied and Environmental Microbiology July 2015 Volume 81 Number 14




most bacterial cells eventually lyse, a few are persistently infected
(for a review, see reference 21).
A phage can transfer bacterial DNA from one host to the other
in a process known as transduction. Specialized transduction occurs frequently in the environment during lysogeny, when a prophage incorrectly excises not only its own DNA but also attached
bacterial DNA from the host genome. Phages can store this DNA
in the capsid, and during the next infection cycle the bacterial host
might integrate the foreign DNA by homologous recombination
(22). The process has been documented in different environments, such as in environmental water and on plant surfaces (17).
New studies, using whole-genome sequencing, have shown that
transduction events have occurred in almost every bacterial isolate
sequenced (23). In theory, generalized transduction, in which either a virulent or a temperate phage accidentally packs bacterial
DNA instead of phage DNA into its capsid and delivers it to the
next host, may occur without the need for homologous recombination between different species of bacteria, although there has
not yet been any definitive proof of the phenomenon (16). Broadhost-range plasmids have also been transduced between bacteria
of nonrelated species (20).
We tested chicken meat for the presence of phages that infect _E._
_coli_ (coliphages) and that are able to transduce important antimicrobial resistances. Our results suggest that the transduction of
antimicrobial resistance is not, as has been assumed to date, a rare
event but is, rather, more common for certain resistance elements.
Phages able to transfer antimicrobial resistance are rather diverse,
ranging from host-specific ones to those with a broad host range,
and the process is not dependent on homologous recombination.
This study raises the question about new strategies in combating
the transfer of antimicrobial resistance in the food environment.


**MATERIALS AND METHODS**

**Sample collection and preparation.** From May to December 2013, 50
packed, chilled (4°C) chicken meat products were collected from supermarkets, street markets, and butchers in Vienna, Austria. A 25-g portion
of each sample was removed from the package and placed into a sterile
filter stomacher bag (BBAG03; Corning Gosselin, Hazebrouck, France)
under aseptic conditions. Fifty milliliters of sterile 0.9% NaCl was used as
a rinsing solution (Merck, Darmstadt, Germany) and was added to the
bag, which was shaken manually for 3 min. After the rinsing, the saline was
used for the isolation of phages and resistant bacteria.
**Isolation of resistant** _**Escherichia coli**_ **colonies.** One milliliter from
each of the rinsing solutions was spread onto Coli-ID agar plates (bioMérieux, Marcy l’Etoile, France) containing appropriate antimicrobials
(ampicillin at 35 �g/ml, tetracycline at 20 �g/ml, kanamycin at 30 �g/ml,
and chloramphenicol at 35 �g/ml [Sigma-Aldrich, St. Louis, MO, USA])
and incubated at 42°C for 24 h. Four presumptive resistant _E. coli_ colonies
were further subcultured from the colonized plates onto plate count agar
(Merck, Darmstadt, Germany) and incubated at 37°C for 24 h. These _E._
_coli_ isolates were stored in Luria broth (Merck, Darmstadt, Germany)
with the addition of 20% (wt/vol) glycerol (Sigma-Aldrich, St. Louis, MO,
USA) at �80°C. Isolates were tested for antimicrobial resistance using
standard methods (24).
**Isolation of bacteriophages and preparation of lysates.** Bacteriophages (virulent and temperate, based on plaque morphology) were isolated as described in standard ISO 10705-2:2000 (25). Briefly, 0.5 ml from
an overnight culture of _E. coli_ ATCC 13706 (DSM 12242) was mixed with
50 ml of modified Scholtens’ broth, and the mixture was incubated at
37°C with shaking for 1 to 5 h until the culture reached an optical density
at 600 nm of 0.4. One milliliter of this culture and 1 ml of sample rinsing
solution were added to 2.5 ml preheated (45°C) modified Scholtens’ soft



Phage Transfer of Antibiotic Resistance in Chicken Meat


agar (Scholtens’ broth plus 0.8% agar), and the mixture was vortexed and
immediately spread as an overlay on modified Scholtens’ agar plates
(Scholtens’ broth plus 1.5% agar) at room temperature. These plates were
incubated at 37°C for 18 to 20 h. Lysates of plaques were prepared as
described by Groisman (26). In brief, plaques (five from each sample)
were individually cut out from the soft agar plate and diluted in chloroform (1%, vol/vol), MgSO4 (2 mM), and CaCl2 (0.2 mM) (Sigma-Aldrich,
St. Louis, MO, USA). The solution was vortexed for 10 s and centrifuged
at 20,000 � _g_ for 10 min. The supernatants were filtered through a 0.2�m-pore-size filter (Whatman; GE Healthcare, Buckinghamshire, United
Kingdom) and kept at 4°C for further use.
**Titration of bacteriophage.** Tenfold serial dilutions (10 [�][1] to 10 [�][6] )
were prepared from each phage lysate. Mixtures from 1 ml of an overnight
culture of _E. coli_ ATCC 13706 and 10 �l from the diluted lysate were added
to 2.5 ml preheated (45°C) modified Scholtens’ soft agar, vortexed, and
immediately spread as an overlay on modified Scholtens’ agar plates to
solidify at room temperature. After incubation at 37°C for 18 to 20 h,
plaques were counted and appropriate dilutions were prepared for transduction and typing. Phage particles were expressed as the number of PFU
per milliliter using the following formula: (number of plaques � dilution
factor)/ml. Titers were calculated for each phage and ranged from 1.5 �
10 [8] to 3.1 � 10 [10] PFU/ml.
**Transduction.** Transduction was performed as described by Groisman with slight variations (26). In brief, 100 �l of an _E. coli_ ATCC 13706
overnight culture was mixed with 10 �l of properly diluted (10 [4] PFU/ml)
phage lysate, and the mixture was incubated at 37°C for 30 min without
shaking to allow phage absorption. Two milliliters of modified Scholtens’
broth was then added, and the mixture was incubated at 37°C for 1 to 2 h
with shaking. The tubes were centrifuged at 12,000 � _g_ for 3 min. The
supernatant was discarded, and the pellet was suspended in 150 �l modified Scholtens’ broth and plated on Mueller-Hinton agar plates (Merck,
Darmstadt, Germany) containing either ampicillin at 35 �g/ml, tetracycline at 20 �g/ml, kanamycin at 30 �g/ml, or chloramphenicol at 35 �g/
ml. The plates were incubated at 37°C for 24 to 48 h. Additionally, to
detect the transfer of extended-spectrum �-lactamase (ESBL) production
to _E. coli_, ESBL-Bx plates (bioMérieux, Marcy l’Etoile, France) were incubated. Colonies were subcultured and stored at �80°C in Luria broth with
the addition of 20% (wt/vol) glycerol for further analysis.
**PCR for identification of resistance genes and Southern blot analy-**
**sis.** Resistance genes in the transduced _E. coli_ isolates were detected by
amplification of a partial sequence of the resistance gene by PCR (27)
using different primers (Table 1) (primers were synthesized at Microsynth
AG, Balgach, Switzerland). DNA was extracted from _E. coli_ using the alkaline lysis method. The PCRs were carried out with a 12.5-�l reaction
mixture containing 10 ng DNA, 6.25 �l Kapa2G polymerase from a Fast
hot start kit (Kapa Biosystems, Wilmington, MA, USA), 1 mM MgCl2,
and 0.1 �M each primer. For Southern blot analysis, whole-genome DNA
was restricted with EcoRI, separated on a 1% agarose gel in 1� TAE
(Tris-acetate-EDTA) buffer, and transferred to a Hybond N [�] membrane
using a vacuum blotter (Hoefer Scientific Instruments, San Francisco, CA,
USA). The PCR product of _aphA1_ was labeled with a DIG High Prime
DNA labeling and detection starter kit II (Roche, Mannheim, Germany).
The hybridization was carried out according to the manufacturer’s specifications, and the hybridized product was visualized using an illuminator
(ChemiDoc XRS� system; Bio-Rad Laboratories, Munich, Germany).
**Characterization of coliphage.** Mixtures from 100-�l overnight cultures of different well-characterized, selected, laboratory _E. coli_ strains
with different properties (ATCC 13706, JM109, W3110, MC1061, and
DH5�) were used to gain a suitable lysogeny spectrum for the phages, and
2.5 ml preheated (45°C) modified Scholtens’ soft agar was spread as an
overlay on modified Scholtens’ agar plates at room temperature. After
solidification, 5 �l of appropriately diluted phage lysates was spotted on
these plates. The plates were incubated at 37°C for 18 to 24 h, and the
lysogeny spectra of each lysate were recorded.



July 2015 Volume 81 Number 14 Applied and Environmental Microbiology [aem.asm.org](http://aem.asm.org) **4601**




Shousha et al.


**TABLE 1** Primers and conditions for PCR of resistance genes


Annealing PCR amplification
Target gene Primer name Sequence temp (°C) product size (bp) Reference


_tet_ (A) tet(A)-f 5=-GCTACATCCTGCTTGCCTTC-3= 57 210 44
tet(A)-r 5=-CATAGATCGCCGTGAAGAGG-3=
_tet_ (B) tet(B)-f 5=-TTGGTTAGGGGCAAGTTTTG-3= 57 659 44
tet(B)-r 5=-GTAATGGGCCAATAACACCG-3=
_tet_ (C) tet(C)-f 5=-CTTGAGAGCCTTCAACCCAG-3= 55 418 44
tet(C)-r 5=-ATGGTCGTCATCTACCTGCC-3=
_aphA1_ aphA-1a-f 5=-AACGTCTTGCTCGAGGCCGCG-3= 65 670 45
aphA-1a-r 5=-GGCAAGATCCTGGTATCGGTCTGC-3=
_bla_ CMY blaCMY-f 5=-TGGCCGTTGCCGTTATCTAC-3= 55 870 46
blaCMY-r 5=-CCCGTTTTATGCACCCATGA-3=
_bla_ IMP blaIMP-f 5=-GAATAGAGTGGATTAATTCTC-3= 55 232 47
blaIMP-r 5=-GGTTTAAYAAAACAACCACC-3=
_bla_ OXA blaOXA-2-f 5=-CAAGCCAAAGGCACGATAGTTG-3= 56 561 47
blaOXA-2-r 5=-CTCAACCCATCCTACCCACC-3=
_bla_ PSE blaPSE-f 5=-TGCTTCGCAACTATGACTAC-3= 55 438 46
blaPSE-r 5=-AGCCTGTGTTTGAGCTAGAT-3=
_bla_ TEM-1 blaTEM-1-f 5=-CAGCGGTAAGATCCTTGAGA-3= 55 643 46
blaTEM-1-r 5=-ACTCCCCGTCGTGTAGATAA-3=
_catI_ catI-f 5=-AGTTGCTCAATGTACCTATAACC-3= 50 547 48
catI-r 5=-TTGTAATTCATTAAGCATTCTGCC-3=
_floR_ floR-f 5=-CGCCGTCATTCCTCACCTTC-3= 50 215 48
floR-r 5=-GATCACGGGCCACGCTGTGTC-3=



**Statistical analysis.** The relation of the presence of transferable elements encoding antimicrobial resistance in coliphages isolated from
chicken meat samples to the recovery of phenotypically antimicrobialresistant _E. coli_ isolates from the same samples was studied for four different antimicrobial substances. For this purpose, information on resistance
transfer from all phages (typically, five) per sample was combined; i.e., a
sample was considered to carry transferable resistance against an antimicrobial substance if at least one of the phages isolated from this sample was
able to transfer the respective resistance. The relation of the presence/
absence of resistant _E. coli_ to the presence/absence of phages able to transduce the respective antimicrobial resistance was then assessed by Fisher’s
exact test. The significance level was established at a _P_ value of �0.05.


**TABLE 2** Numbers of chicken meat samples by type of resistant _E. coli_
isolate


Antimicrobial resistance _[a]_ No. of samples

Amp [r] 6
Amp [r] Cm [r] 0
Amp [r] Km [r] 0
Amp [r] Tet [r] 12
Amp [r] Cm [r] Km [r] 1
Amp [r] Cm [r] Tet [r] 5
Amp [r] Km [r] Tet [r] 10
Amp [r] Cm [r] Km [r] Tet [r] 10
Cm [r] 0
Cm [r] Km [r] 0
Cm [r] Tet [r] 1
Cm [r] Km [r] Tet [r] 1
Km [r] 0
Km [r] Tet [r] 0
Tet [r] 1
Sensitive 3


Total 50


_a_ Ampr, Cmr, Kmr, and Tetr, ampicillin, chloramphenicol, kanamycin, and tetracycline
resistant, respectively.



**RESULTS**
**Detection of antimicrobial-resistant** _**E. coli**_ **isolates in chicken**
**meat.** One or more resistant _E. coli_ isolates whose resistance was
determined using antimicrobial susceptibility testing by broth
microdilution could be isolated from 47 out of 50 chicken meat
samples using selective plating. Most _E. coli_ isolates were resistant to ampicillin (44 of 50 samples), followed by tetracycline
(40 of 50 samples). Kanamycin-resistant isolates were found in
22 chicken meat samples, and chloramphenicol-resistant isolates were found in 18. Table 2 shows the numbers of samples
with the different types of resistant _E. coli_ isolates. Ampicillinresistant as well as tetracycline-resistant _E. coli_ strains were found
the most often in the chicken meat samples ( _n_ - 12, 24%); this
was followed by _E. coli_ isolates resistant to all different antimicrobials tested and _E. coli_ isolates resistant to ampicillin, kanamycin,
and tetracycline ( _n_ - 10 [20%] for each group). Six samples harbored ampicillin-resistant _E. coli_ isolates only. Five samples carried _E. coli_ isolates with resistance to ampicillin, chloramphenicol,
as well as tetracycline. Three samples did not harbor any _E. coli_
isolates resistant to the tested antimicrobial substances. One sample each held _E. coli_ strains resistant to tetracycline only; chloramphenicol and tetracycline; chloramphenicol, kanamycin,
and tetracycline; and ampicillin, chloramphenicol, and kanamycin.
**Resistance-transducing phage isolated from chicken meat.**
Coliphages were detected in all but one chicken meat sample.
The sample with no coliphage did not hold any resistant _E. coli_
isolates. About 90% of the phages isolated revealed a clear plaque
morphology, and 10% produced a cloudy center. Of all phages
randomly isolated from chicken meat ( _n_ - 243), about a quarter
( _n_ - 60) were able to transduce one or more of the tested antimicrobial resistances into _E. coli_ ATCC 13706. Phages able to transduce antimicrobial resistance were isolated from 22 out of a total
of 50 chicken meat samples. Phages which transduced kanamycin



**4602** [aem.asm.org](http://aem.asm.org) Applied and Environmental Microbiology July 2015 Volume 81 Number 14




**FIG 1** Proportion of resistance-transducing phages among all isolated phages.
Kanamycin resistance-transducing phages were detected the most often ( _n_ 43 of 243 phages detected), followed by chloramphenicol ( _n_ - 8) and tetracycline ( _n_ - 4) resistance-transducing phages. Three phages were found to transduce kanamycin plus ampicillin resistance, and two phages transduced kanamycin plus chloramphenicol resistance. �Cm [r], �Km [r], �Tet [r], �Km [r] Amp [r],
and �Km [r] Cm [r], chloramphenicol, kanamycin, tetracycline, kanamycin and
ampicillin, and kanamycin and chloramphenicol resistance-transducing
phages, respectively.


resistance only were isolated from seven chicken meat samples.
Phagestransducingkanamycinandchloramphenicolresistancewere
found in six samples. Phages from three samples could transduce
either kanamycin or ampicillin resistance. Chloramphenicol resistance-transducingphageswererecoveredfromthreesamples.Phages
with the ability to transduce resistance to tetracycline only were seen
in another two samples, and kanamycin and tetracycline resistancetransducing phages were isolated from one sample.
**Antimicrobial resistance transduced by phages.** Kanamycin
resistance could be transduced the most often, i.e., by 43
phages. Eight phages were able to transduce chloramphenicol
resistance, and four transduced tetracycline resistance. Three
phages could transduce ampicillin and kanamycin resistance,


**TABLE 3** Lysotypes of antimicrobial resistance-transducing phages



Phage Transfer of Antibiotic Resistance in Chicken Meat


and two transduced chloramphenicol and kanamycin resistance (Fig. 1). No phages were able to transduce extendedspectrum �-lactamase production to _E. coli_ ATCC 13706.
**Coliphage characterization.** An optimized titer was defined
by titration of phages ( _n_ - 243) isolated from chicken meat. The
titer of each phage was calculated to range from 1.5 � 10 [8] to 3.1 �
10 [10] PFU/ml. For a brief characterization of the antimicrobial resistance-transducing phages, a lysotype pattern was determined
for five different _E. coli_ strains (ATCC 13706, JM109, W3110,
MC1061, and DH5�) using lysates of each phage whose titers had
been determined. Table 3 lists the different lysotypes of all antimicrobial resistance-transducing phages. A high degree of variability, given by 13 different lysotypes, was seen within these phages
isolated from chicken meat.
**Detection of resistance genes in transduced** _**E. coli**_ **isolates.**
The _tet_ (A) gene fragment could be amplified from all transduced
tetracycline-resistant _E. coli_ isolates. All transduced kanamycinresistant _E. coli_ isolates harbored the _aphA1_ gene. All but one _E. coli_
isolate to which ampicillin resistance was transduced harbored the
_bla_ TEM gene. The _floR_ gene was detected in three transduced chloramphenicol-resistant _E. coli_ isolates. Southern blot analysis of five
kanamycin-resistant isolates of _E. coli_ ATCC 13706 transduced
with phage ��5c uncovered that the _aphA1_ gene integrated at
different sites in the genome; thus, homologous recombination
was not required.
**Association of resistance-transducing phages and** _**E. coli**_ **in**
**meat samples.** A statistically significant association between the
presence/absence of resistant _E. coli_ isolates and the presence/
absence of phages able to transduce the same antimicrobial
resistance into _E. coli_ ATCC 13706 in chicken meat samples was
established for kanamycin ( _P_ - 0.01) (Table 4).



LT _[a]_

pattern



Lysis of the indicated _E. coli_ strains Phage(s) transducing resistance to the indicated antimicrobial _[b]_


ATCC
13706 JM109 W3110 MC1061 DH5� Kanamycin Chloramphenicol Ampicillin Tetracycline



LT1 ��� ��� ��� ��� - �E2b, �E5a, �E5b, �E5c, �E5d, �E5e �E6b, �E16e, �E30e �E5c, �E9b �E10c, �E10d
�E7a, �E7b, �E7c, �E7d, �E7e, �E9b �E49d �E13d �E15d, �E40e
�E9d, �E12b, �E12e, �E13a, �E13b
�E13c, �E13d, �E13e, �E16a, �E16d
�E19b, �E19c, �E22c, �E23b, �E23c
�E28a, �E28b, �E28c, �E28d, �E30a, �E49a


LT2 ��� �� - ��� - �E2c �E43e
LT3 ��� �� �� �� - �E2d
LT4 ��� - �� �� - �E2e
LT5 ��� - ��� �� - �E6a, �E19a
LT6 ��� - ��� ��� - �E6d, �E9c, �E27c
LT7 ��� - - �� - �E9a, �E15c
LT8 ��� ��� - ��� - �E12c, �E12d, �E34e �E12c
LT9 ��� - �� - - �E19e
LT10 ��� �� �� ��� - �E34a �E34a
LT11 ��� - - ��� - �E35d
LT12 ��� �� ��� ��� - �E35e
LT13 ��� - - - - �E39d


_a_ LT, lysotype.
_b_ The phage designations indicate the chicken meat sample (E) number (indicated by Arabic numerals) and the phage isolate number (indicated by lowercase letters); for example,
�E5a represents the first phage isolate from chicken meat sample number 5.


July 2015 Volume 81 Number 14 Applied and Environmental Microbiology [aem.asm.org](http://aem.asm.org) **4603**




Shousha et al.


**TABLE 4** Association between isolation of antimicrobial-resistant _E. coli_
and resistance-transducing phages from the same chicken meat sample


No. of samples positive for:



Antimicrobial
resistance _[a]_



Transducing
_E. coli_ phage Both _P_



Amp [r] 44 3 3 1.0000
Cm [r] 18 9 1 0.1304
Km [r] 22 17 12 0.0148 _[b]_

Tet [r] 40 3 2 0.4959


_a_ Ampr, Cmr, Kmr, and Tetr, ampicillin, chloramphenicol, kanamycin, and tetracycline
resistant, respectively.
_b_ Statistically significant relation between isolation of antimicrobial-resistant _E. coli_ and
resistance-transducing phages.


**DISCUSSION**
Although transduction of the antibiotic resistance island in _Sal-_
_monella enterica_ serovar Typhimurium phage type DT104 has
been effectively shown (28), until very recently, transduction via
bacteriophages was assumed to play a minor part in the horizontal
transfer of antimicrobial resistance genes (12, 15). Food of animal
origin is a well-known vehicle allowing the transfer of bacteria
from food-producing animals to humans, as reflected by the increasing incidence of foodborne zoonotic diseases in humans and
the rising numbers of antimicrobial-resistant bacteria found on
food of animal origin. Such food thus represents a possible source
of antimicrobial resistance in the human environment and is included in many antimicrobial screening programs. The increasing
numbers of resistant bacteria, both commensals and foodborne
pathogens, on food are a cause of concern to public health authorities, veterinarians, and physicians. Strategies to combat antimicrobial resistance have enjoyed only limited success, and there are
still many questions relating to how and when resistance transfer
occurs. The presence of phages that transfer antimicrobial resistance could explain the failures to combat antimicrobial resistance
described above. We now show, based on mathematical modeling,
that chicken meat can carry a considerable number of phages capable of transferring antimicrobial resistance and that certain antimicrobial resistance genes may be transferred primarily by
phages. We tested randomly selected coliphages from chicken
meat samples for their ability to transduce resistance to tetracycline (an antimicrobial widely used in veterinary medicine, in
aquaculture, and on plants and thus extensively released into the
environment, although its photodegradation is fairly rapid), ampicillin (a �-lactam antibiotic and, thus, a compound typical of a
range of compounds that are important in veterinary and human
medicine and that have a very short half-life in the environment),
chloramphenicol (which is banned in veterinary medicine but
which is still found in the environment), and kanamycin (an aminoglycoside which is commonly used as a second-choice antibiotic in human medicine and which can be found in aquatic environments because of its long degradation time) (29, 30). About a
quarter of the randomly selected phages were able to transduce
resistance to one or more of these antimicrobials to _E. coli_ ATCC
13706. Thus, the actual number of phages found on chicken meat
able to transduce antibiotic resistance genes must be much higher,
as we tested for resistance to only five different antibiotics and
randomly chose only five phages per sample (from a plate with 10
to 300 plaques). Accordingly, our results confirm speculations
that transduction may have a role in the exchange of antimicrobial



resistance between microorganisms and, thus, between environments (17, 31–34). Chicken meat has been reported to harbor a
high number of resistant _E. coli_ isolates (63% of _E. coli_ strains
randomly isolated from chicken meat in Austria were resistant)
(7). Resistance to tetracycline has been described to be the most
prevalent. In this study, we detected tetracycline resistance-transducing phages in three different types of chicken meat samples (a
total of 50 samples).
By means of a mathematical model (16) of the transfer of resistance to _E. coli_ by transduction in the gut of cattle, it has been
estimated that transduction is on the order of 1,000-fold less efficient than bacterial conjugation at transferring genes (16). It is
thus not surprising that resistance to tetracycline and ampicillin,
which are the antimicrobials to which _E. coli_ strains isolated from
food are the most frequently resistant (27, 35–38), is not significantly associated with phages that transfer resistance. Thus, phage
transduction may not be the major mechanism for tetracycline
resistance transfer to _E. coli_ strains found on chicken meat. In
contrast, phages that transfer kanamycin resistance were more
frequently recovered and were significantly associated with the
presence of resistant _E. coli_ isolates in the samples that we investigated. These findings indicate not only that phages appear to represent a substantial reservoir of antimicrobial resistance transfer
but also demonstrate that different mechanisms of resistance depend on different transfer strategies. Although kanamycin-resistant _E. coli_ isolates are not common in food of animal origin (7,
35), our data show, for the first time, that phage transduction
might be a major mechanism for transferring this resistance trait.
Although the elements used to transfer tetracycline and ampicillin
resistance appear to be dominated by conjugative plasmids or
transposons, phages have been shown able to transfer all types of
resistance tested, except that associated with ESBL production.
ESBL-producing _E. coli_ strains can be found on chicken meat, but
data suggest that clonal dissemination is the most prevalent means
(39). The resistance genes transduced to _E. coli_ were those most
commonly found [ _aphA1_, _bla_ TEM, _floR_, and _tet_ (A)] in these bacteria on chicken meat (27, 36–38). As an example, one gene found
to be transduced and to deliver chloramphenicol resistance is the
_E. coli floR_ gene, which is almost identical to the _floR_ gene of
_Salmonella enterica_ serovar Typhimurium phage type DT104 and
is located in the antimicrobial resistance island flanked by two
class 1 integrons.
Recently, lytic phages have been used in phage therapy for the
treatment of colibacillosis caused by multiresistant chickenpathogenic _E. coli_ strains (40) or for reducing _Campylobacter_ loads
in chickens (41). Considering our results, there could possibly be
a risk of transduction of antibiotic resistance genes by these
phages. Testing the transduction ability of these therapeutic
phages could be a tool for avoiding the transduction of antibiotic
resistance by these phages.
Phages differ from their bacterial hosts in a number of important properties, such as their ability to persist in the presence of
disinfectants (42, 43). Thus, disinfectants used to kill bacteria are,
in many cases, not able to eliminate bacteriophages. Some phages
are highly resistant to sodium hypochlorite at the concentrations
used in the food industry, and this disinfectant does not eliminate
or reduce the numbers of most bacteriophages (42). Alcoholic
solutions, such as 70% ethanol, have been reported to be unable to
harm most bacteriophages (42, 43). It appears that only peracetic
acid is a reliable disinfectant for most bacteriophages (42), but at



**4604** [aem.asm.org](http://aem.asm.org) Applied and Environmental Microbiology July 2015 Volume 81 Number 14




present, this is not extensively used for disinfection. It is thus
highly likely that phages survive under routine conditions of disinfection not only in the food industry.
Our results show for the first time that generalized transduction without the need for homologous recombination is a neglected but nevertheless efficient way to transfer antimicrobial resistance to _E. coli_ in different environments, including in food of
animal origin. This conclusion has important consequences for
hygiene, sanitation, and public health when it is important to
combat antimicrobial resistance.


**ACKNOWLEDGMENTS**


Amira Shousha was granted a Vetmeduni Success Scholarship from the
University of Veterinary Medicine, Vienna, Austria, to perform this study.
Nattakarn Awaiwanont was funded by the Technologiestipendien
Südostasien in the framework of ASEA-UNINET, granted by the OeAD
(Austrian Agency for International Cooperation in Education & Research), financed by the Austrian Federal Ministry for Science and Research.
We thank Stefan Hertwig for discussions and Graham Tebb for critical
reading of and suggestions on the manuscript.


**REFERENCES**

1. **Laube H, Friese A, von Salviati C, Guerra B, Käsbohrer A, Kreienbrock**
**L, Roesler U.** 2013. Longitudinal monitoring of extended-spectrum-betalactamase/AmpC-producing _Escherichia coli_ at German broiler chicken
fattening farms. Appl Environ Microbiol **79:** [4815–4820. http://dx.doi.org](http://dx.doi.org/10.1128/AEM.00856-13)
[/10.1128/AEM.00856-13.](http://dx.doi.org/10.1128/AEM.00856-13)
2. **Bronzwaer S, Lönnroth A, Haigh R.** 2004. The European Community
strategy against antimicrobial resistance. Euro Surveill **9** (1) **:** [pii�441. http:](http://www.eurosurveillance.org/ViewArticle.aspx?ArticleId=441)
[//www.eurosurveillance.org/ViewArticle.aspx?ArticleId�441.](http://www.eurosurveillance.org/ViewArticle.aspx?ArticleId=441)
3. **Vogt D, Overesch G, Endimiani A, Collaud A, Thomann A, Perreten V.**
2014. Occurrence and genetic characteristics of third-generation cephalosporin-resistant _Escherichia coli_ in Swiss retail meat. Microb Drug Resist
**20:** [485–494. http://dx.doi.org/10.1089/mdr.2013.0210.](http://dx.doi.org/10.1089/mdr.2013.0210)
4. **European Food Safety Authority, European Centre for Disease Preven-**
**tion and Control.** 2015. EU summary report on antimicrobial resistance
in zoonotic and indicator bacteria from humans, animals and food in
2013. EFSA J **13:** [4036. http://dx.doi.org/10.2903/j.efsa.2015.4036.](http://dx.doi.org/10.2903/j.efsa.2015.4036)
5. **Ibekwe AM, Murinda SE, Graves AK.** 2011. Genetic diversity and antimicrobial resistance of _Escherichia coli_ from human and animal sources
uncovers multiple resistances from human sources. PLoS One **6:** e20819.
[http://dx.doi.org/10.1371/journal.pone.0020819.](http://dx.doi.org/10.1371/journal.pone.0020819)
6. **Kruse H, Sørum H.** 1994. Transfer of multiple drug resistance plasmids
between bacteria of diverse origins in natural microenvironments. Appl
Environ Microbiol **60:** 4015–4021.
7. **Mayrhofer S, Paulsen P, Smulders FJM, Hilbert F.** 2006. Antimicrobial
resistance in commensal _Escherichia coli_ isolated from muscle foods as
related to the veterinary use of antimicrobial agents in food-producing
animals in Austria. Microb Drug Resist **12:** [278–283. http://dx.doi.org/10](http://dx.doi.org/10.1089/mdr.2006.12.278)
[.1089/mdr.2006.12.278.](http://dx.doi.org/10.1089/mdr.2006.12.278)
8. **Gunell M, Webber MA, Kotilainen P, Lilly AJ, Caddick JM, Jalava J,**
**Huovinen P, Siitonen A, Hakanen AJ, Piddock LJV.** 2009. Mechanisms
of resistance in nontyphoidal _Salmonella enterica_ strains exhibiting a nonclassical quinolone resistance phenotype. Antimicrob Agents Chemother
**53:** [3832–3836. http://dx.doi.org/10.1128/AAC.00121-09.](http://dx.doi.org/10.1128/AAC.00121-09)
9. **Woodford N, Ellington MJ.** 2007. The emergence of antibiotic resistance
by mutation. Clin Microbiol Infect **13:** [5–18. http://dx.doi.org/10.1111/j](http://dx.doi.org/10.1111/j.1469-0691.2006.01492.x)
[.1469-0691.2006.01492.x.](http://dx.doi.org/10.1111/j.1469-0691.2006.01492.x)
10. **Mempel M, Feucht H, Ziebuhr W, Endres M, Laufs R, Grüter L.** 1994.
Lack of _mecA_ transcription in slime-negative phase variants of methicillinresistant _Staphylococcus epidermidis_ . Antimicrob Agents Chemother **38:**
[1251–1255. http://dx.doi.org/10.1128/AAC.38.6.1251.](http://dx.doi.org/10.1128/AAC.38.6.1251)
11. **Mah TF.** 2012. Biofilm-specific antibiotic resistance. Future Microbiol
**7:** [1061–1072. http://dx.doi.org/10.2217/fmb.12.76.](http://dx.doi.org/10.2217/fmb.12.76)
12. **Boerlin P, Reid-Smith RJ.** 2008. Antimicrobial resistance: its emergence
and transmission. Anim Health Res Rev **9:** [115–126. http://dx.doi.org/10](http://dx.doi.org/10.1017/S146625230800159X)
[.1017/S146625230800159X.](http://dx.doi.org/10.1017/S146625230800159X)
13. **Martinez JL, Fajardo A, Garmendia L, Hernandez A, Linares JF, Mar-**



Phage Transfer of Antibiotic Resistance in Chicken Meat


**tínez-Solano L, Sánchez MB.** 2009. A global view of antibiotic resistance.
FEMS Microbiol Rev **33:** [44–65. http://dx.doi.org/10.1111/j.1574-6976](http://dx.doi.org/10.1111/j.1574-6976.2008.00142.x)
[.2008.00142.x.](http://dx.doi.org/10.1111/j.1574-6976.2008.00142.x)
14. **Wiedenbeck J, Cohan FM.** 2011. Origins of bacterial diversity through
horizontal genetic transfer and adaptation to new ecological niches. FEMS
Microbiol Rev **35:** [957–976. http://dx.doi.org/10.1111/j.1574-6976.2011](http://dx.doi.org/10.1111/j.1574-6976.2011.00292.x)
[.00292.x.](http://dx.doi.org/10.1111/j.1574-6976.2011.00292.x)
15. **Normark BH, Normark S.** 2002. Evolution and spread of antibiotic resistance. J Intern Med **252:** [91–106. http://dx.doi.org/10.1046/j.1365-2796](http://dx.doi.org/10.1046/j.1365-2796.2002.01026.x)
[.2002.01026.x.](http://dx.doi.org/10.1046/j.1365-2796.2002.01026.x)
16. **Volkova VV, Lu Z, Besser T, Gröhn YT.** 2014. Modeling the infection
dynamics of bacteriophages in enteric _Escherichia coli_ : estimating the contribution of transduction to antimicrobial gene spread. Appl Environ Microbiol **80:** [4350–4362. http://dx.doi.org/10.1128/AEM.00446-14.](http://dx.doi.org/10.1128/AEM.00446-14)
17. **Fineran PC, Petty NK, Salmond GPC.** 2009. Transduction: host DNA
transfer by bacteriophages, p 666–679. _In_ Schaechter M (ed), The desk
encyclopedia of microbiology. Elsevier Academic Press, San Diego, CA.
18. **Hendrix RW, Smith MCM, Burns RN, Ford ME, Hatfull GF.** 1999.
Evolutionary relationships among diverse bacteriophages and prophages:
all the world’s a phage. Proc Natl Acad Sci U S A **96:** [2192–2197. http://dx](http://dx.doi.org/10.1073/pnas.96.5.2192)
[.doi.org/10.1073/pnas.96.5.2192.](http://dx.doi.org/10.1073/pnas.96.5.2192)
19. **Schwarzer D, Buettner FFR, Browning C, Nazarov S, Rabsch W, Bethe**
**A, Oberbeck A, Bowman VD, Stummeyer K, Mühlenhoff M, Leiman**
**PG, Gerardy-Schahn R.** 2012. A multivalent adsorption apparatus explains the broad host range of phage phi92: a comprehensive genomic and
structural analysis. J Virol **86:** [10384–10398. http://dx.doi.org/10.1128/JVI](http://dx.doi.org/10.1128/JVI.00801-12)
[.00801-12.](http://dx.doi.org/10.1128/JVI.00801-12)
20. **Evans TJ, Crow MA, Williamson NR, Orme W, Thomson NR, Komi-**
**topoulou E, Salmond GPC.** 2010. Characterization of a broad-host-range
flagellum-dependent phage that mediates high-efficiency generalized
transduction in, and between, _Serratia_ and _Pantoea_ . Microbiology **156:**
[240–247. http://dx.doi.org/10.1099/mic.0.032797-0.](http://dx.doi.org/10.1099/mic.0.032797-0)
21. **Weitz JS, Poisot T, Meyer JR, Flores CO, Valverde S, Sullivan MB,**
**Hochberg ME.** 2013. Phage-bacteria infection networks. Trends Microbiol **21:** [82–91. http://dx.doi.org/10.1016/j.tim.2012.11.003.](http://dx.doi.org/10.1016/j.tim.2012.11.003)
22. **Zhang Z, Kottadiel VI, Vafabakhsh R, Dai L, Chemla YR, Ha T, Rao**
**VB.** 2011. A promiscuous DNA packaging machine from bacteriophage T4. PLoS Biol **9:** [e1000592. http://dx.doi.org/10.1371/journal](http://dx.doi.org/10.1371/journal.pbio.1000592)
[.pbio.1000592.](http://dx.doi.org/10.1371/journal.pbio.1000592)
23. **Schmieder R, Edwards R.** 2012. Insights into antibiotic resistance
through metagenomic approaches. Future Microbiol **7:** [73–89. http://dx](http://dx.doi.org/10.2217/fmb.11.135)
[.doi.org/10.2217/fmb.11.135.](http://dx.doi.org/10.2217/fmb.11.135)
24. **EUCAST.** [2010. Antimicrobial susceptibility testing. http://www.eucast](http://www.eucast.org/ast_of_bacteria/)
[.org/ast_of_bacteria/.](http://www.eucast.org/ast_of_bacteria/)
25. **International Organization for Standardization.** 2000. Water quality—
detection and enumeration of bacteriophages. Part 2. Enumeration of
somatic coliphages. ISO 10705-2:2000. International Organization for
Standardization, Geneva, Switzerland.
26. **Groisman EA.** 1991. In vivo genetic engineering with bacteriophage Mu.
Methods Enzymol **204:** [180 –212. http://dx.doi.org/10.1016/0076-](http://dx.doi.org/10.1016/0076-6879(91)04010-L)
[6879(91)04010-L.](http://dx.doi.org/10.1016/0076-6879(91)04010-L)
27. **Sacher A.** 2008. Ph.D. thesis. Veterinärmedizinischen Universität Wien,
Vienna, Austria.
28. **Schmieger H, Schicklmaier P.** 1999. Transduction of multiple drug
resistance of _Salmonella enterica_ serovar _typhimurium_ DT104. FEMS Microbiol Lett **170:** [251–256. http://dx.doi.org/10.1111/j.1574-6968.1999.tb1](http://dx.doi.org/10.1111/j.1574-6968.1999.tb13381.x)
[3381.x.](http://dx.doi.org/10.1111/j.1574-6968.1999.tb13381.x)
29. **Milic´ N, Milanovic´ M, Grujic´ Letic´ N, Sekulic´ MT, Radonic´ J,**
**Mihajlovic´ I, Miloradov MV.** 2013. Occurrence of antibiotics as emerging contaminant substances in aquatic environment. Int J Environ Health
Res **23:** [296–310. http://dx.doi.org/10.1080/09603123.2012.733934.](http://dx.doi.org/10.1080/09603123.2012.733934)
30. **Zhang J, Chang VWC, Giannis A, Wang J-Y.** 2013. Removal of cytostatic
drugs from aquatic environment: a review. Sci Total Environ **445-446:**
[281–298. http://dx.doi.org/10.1016/j.scitotenv.2012.12.061.](http://dx.doi.org/10.1016/j.scitotenv.2012.12.061)
31. **Balcazar JL.** 2014. Bacteriophages as vehicles for antibiotic resistance
genes in the environment. PLoS Pathog **10:** [e1004219. http://dx.doi.org/10](http://dx.doi.org/10.1371/journal.ppat.1004219)
[.1371/journal.ppat.1004219.](http://dx.doi.org/10.1371/journal.ppat.1004219)
32. **Colomer-Lluch M, Imamovic L, Jofre J, Muniesa M.** 2011. Bacteriophages carrying antibiotic resistance genes in fecal waste from cattle, pigs,
and poultry. Antimicrob Agents Chemother **55:** [4908–4911. http://dx.doi](http://dx.doi.org/10.1128/AAC.00535-11)
[.org/10.1128/AAC.00535-11.](http://dx.doi.org/10.1128/AAC.00535-11)
33. **Colomer-Lluch M, Jofre J, Muniesa M.** 2011. Antibiotic resistance genes



July 2015 Volume 81 Number 14 Applied and Environmental Microbiology [aem.asm.org](http://aem.asm.org) **4605**




Shousha et al.


in the bacteriophage DNA fraction of environmental samples. PLoS One
**6:** [e17549. http://dx.doi.org/10.1371/journal.pone.0017549.](http://dx.doi.org/10.1371/journal.pone.0017549)
34. **Modi SR, Lee HH, Spina CS, Collins JJ.** 2013. Antibiotic treatment
expandstheresistancereservoirandecologicalnetworkofthephagemetagenome. Nature **499:** [219–222. http://dx.doi.org/10.1038/nature12212.](http://dx.doi.org/10.1038/nature12212)
35. **AURES.** 2012. Resistenzbericht O [¨ ] sterreich. Federal Ministry of Health,
[Vienna, Austria. http://www.referenzzentrum.at.](http://www.referenzzentrum.at)
36. **Sheikh AA, Checkley S, Avery B, Chalmers G, Bohaychuk V, Boerlin P,**
**Reid-Smith R, Aslam M.** 2012. Antimicrobial resistance and resistance
genes in _Escherichia coli_ isolated from retail meat purchased in Alberta,
Canada. Foodborne Pathog Dis **9:** [625–631. http://dx.doi.org/10.1089/fpd](http://dx.doi.org/10.1089/fpd.2011.1078)
[.2011.1078.](http://dx.doi.org/10.1089/fpd.2011.1078)
37. **Van TTH, Chin J, Chapman T, Tran LT, Coloe PJ.** 2008. Safety of raw
meat and shellfish in Vietnam: an analysis of _Escherichia coli_ isolations for
antibiotic resistance and virulence genes. Int J Food Microbiol **124:** 217–
[223. http://dx.doi.org/10.1016/j.ijfoodmicro.2008.03.029.](http://dx.doi.org/10.1016/j.ijfoodmicro.2008.03.029)
38. **Zhao S, Blickenstaff K, Bodeis-Jones S, Gaines SA, Tong E, McDermott**
**PF.** 2012. Comparison of the prevalences and antimicrobial resistances of
_Escherichia coli_ isolates from different retail meats in the United States,
2002 to 2008. Appl Environ Microbiol **78:** [1701–1707. http://dx.doi.org/10](http://dx.doi.org/10.1128/AEM.07522-11)
[.1128/AEM.07522-11.](http://dx.doi.org/10.1128/AEM.07522-11)
39. **Wu G, Day MJ, Mafura MT, Nunez-Garcia J, Fenner JJ, Sharma M, van**
**Essen-Zandbergen A, Rodríguez I, Dierikx C, Kadlec K, Schink AK,**
**Wain J, Helmuth R, Guerra B, Schwarz S, Threlfall J, Woodward MJ,**
**Woodford N, Coldham N, Mevius D.** 2013. Comparative analysis of
ESBL-positive _Escherichia coli_ isolates from animals and humans from the
UK, The Netherlands and Germany. PLoS One **8:** [e75392. http://dx.doi](http://dx.doi.org/10.1371/journal.pone.0075392)
[.org/10.1371/journal.pone.0075392.](http://dx.doi.org/10.1371/journal.pone.0075392)
40. **Zhang C, Wenli L, Wenhua L, Ling Z, Chen Y, Kai L, Huiying R.** 2013.
T4-like phage Bp7, a potential antimicrobial agent for controlling drugresistant _Escherichia coli_ in chickens. Appl Environ Microbiol **79:** 5559–
[5565. http://dx.doi.org/10.1128/AEM.01505-13.](http://dx.doi.org/10.1128/AEM.01505-13)
41. **Kittler S, Fischer S, Abdulmawjood A, Glünder G, Klein G.** 2013. Effect



of bacteriophage application on _Campylobacter jejuni_ loads in commercial
broiler flocks. Appl Environ Microbiol **79:** [7525–7533. http://dx.doi.org](http://dx.doi.org/10.1128/AEM.02703-13)
[/10.1128/AEM.02703-13.](http://dx.doi.org/10.1128/AEM.02703-13)
42. **Mercanti DJ, Guglielmotti DM, Patrignani F, Reinheimer JA, Quibe-**
**roni A.** 2012. Resistance of two temperate _Lactobacillus paracasei_ bacteriophages to high pressure homogenization, thermal treatments and
chemical biocides of industrial application. Food Microbiol **29:** 99–104.
[http://dx.doi.org/10.1016/j.fm.2011.09.003.](http://dx.doi.org/10.1016/j.fm.2011.09.003)
43. **Capra ML, Quiberoni A, Reinheimer JA.** 2004. Thermal and chemical
resistance of _Lactobacillus casei_ and _Lactobacillus paracasei_ bacteriophages.
Lett Appl Microbiol **38:** [499–504. http://dx.doi.org/10.1111/j.1472-765X](http://dx.doi.org/10.1111/j.1472-765X.2004.01525.x)
[.2004.01525.x.](http://dx.doi.org/10.1111/j.1472-765X.2004.01525.x)
44. **Ng LK, Martin I, Alfa M, Mulvey M.** 2001. Multiplex PCR for the
detection of tetracycline resistant genes. Mol Cell Probes **15:** 209–215.
[http://dx.doi.org/10.1006/mcpr.2001.0363.](http://dx.doi.org/10.1006/mcpr.2001.0363)
45. **Sandvang D, Aarestrup FM.** 2000. Characterization of aminoglycoside
resistance genes and class 1 integrons in porcine and bovine gentamicinresistant _Escherichia coli_ . Microb Drug Resist **6:** [19–27. http://dx.doi.org](http://dx.doi.org/10.1089/mdr.2000.6.19)
[/10.1089/mdr.2000.6.19.](http://dx.doi.org/10.1089/mdr.2000.6.19)
46. **Chen S, Zhao S, White DG, Schroeder CM, Lu R, Yang H, McDermott**
**PF, Ayers S, Meng J.** 2004. Characterization of multiple-antimicrobialresistant _Salmonella_ serovars isolated from retail meats. Appl Environ Microbiol **70:** [1–7. http://dx.doi.org/10.1128/AEM.70.1.1-7.2004.](http://dx.doi.org/10.1128/AEM.70.1.1-7.2004)
47. **Henriques I, Moura A, Alves A, Saavedra MJ, Correia A.** 2006. Analysing diversity among �-lactamase encoding genes in aquatic environments.
FEMS Microbiol Ecol **56:** [418–429. http://dx.doi.org/10.1111/j.1574-6941](http://dx.doi.org/10.1111/j.1574-6941.2006.00073.x)
[.2006.00073.x.](http://dx.doi.org/10.1111/j.1574-6941.2006.00073.x)
48. **Maynard C, Fairbrother JM, Bekal S, Sanschagrin F, Levesque RC,**
**Brousseau R, Masson L, Lariviére S, Harel J.** 2003. Antimicrobial resistance genes in enterotoxigenic _Escherichia coli_ O149:K91 isolates obtained
over a 23-year period from pigs. Antimicrob Agents Chemother **47:** 3214–
[3221. http://dx.doi.org/10.1128/AAC.47.10.3214-3221.2003.](http://dx.doi.org/10.1128/AAC.47.10.3214-3221.2003)



**4606** [aem.asm.org](http://aem.asm.org) Applied and Environmental Microbiology July 2015 Volume 81 Number 14


