[International Journal of Food Microbiology 189 (2014) 39–44](http://dx.doi.org/10.1016/j.ijfoodmicro.2014.07.031)


[Contents lists available at ScienceDirect](http://www.sciencedirect.com/science/journal/01681605)

# International Journal of Food Microbiology


journal homepage: www.elsevier.com/locate/ijfoodmicro

## Characterization of integrons and resistance genes in multidrug-resistant Salmonella enterica isolated from meat and dairy products in Egypt


Ashraf M. Ahmed [a], Toshi Shimamoto [b], Tadashi Shimamoto [b][,] ⁎


a Department of Bacteriology, Mycology and Immunology, Faculty of Veterinary Medicine, Kafrelsheikh University, Kafr El-Sheikh 33516, Egypt
b Laboratory of Food Microbiology and Hygiene, Graduate School of Biosphere Science, Hiroshima University, Higashi-Hiroshima 739-8528, Japan


a r t i c l e i n f o a b s t r a c t



Article history:
Received 28 February 2014
Received in revised form 22 July 2014
Accepted 23 July 2014
Available online 31 July 2014


Keywords:
β-lactamase
Enteritidis
Infantis
Plasmid-mediated
Typhimurium


1. Introduction



Foodborne pathogens are a leading cause of illness and death, especially in developing countries. The problem is
exacerbated if bacteria attain multidrug resistance. Little is currently known about the extent of antibiotic resistance in foodborne pathogens and the molecular mechanisms underlying this resistance in Africa. Therefore, the
current study was carried out to characterize, at the molecular level, the mechanism of multidrug resistance in
Salmonella enterica isolated from 1600 food samples (800 meat products and 800 dairy products) collected
from different street venders, butchers, retail markets and slaughterhouses in Egypt. Forty-seven out of 69 isolates (68.1%) showed multidrug resistance phenotypes to at least three classes of antimicrobials. The incidence
of multidrug-resistant isolates was higher in meat products (37, 69.8%) than in dairy products (10, 62.5%). The
multidrug-resistant serovars included, S. enterica serovar Typhimurium (24 isolates, 34.8%), S. enterica serovar
Enteritidis, (15 isolates, 21.8%), S. enterica serovar Infantis (7 isolates, 10.1%) and S. enterica non-typable serovar
(1 isolate, 1.4%). The highest resistance was to ampicillin (95.7%), then to kanamycin (93.6%), spectinomycin
(93.6%), streptomycin (91.5%) and sulfamethoxazole/trimethoprim (91.5%). PCR and DNA sequencing were
used to screen and characterize integrons and antibiotic resistance genes and 39.1% and 8.7% of isolates were positive for class 1 and class 2 integrons, respectively. β-lactamase-encoding genes were identified in 75.4% of isolates and plasmid-mediated quinolone resistance genes were identified in 27.5% of isolates. Finally, the
florphenicol resistance gene, floR, was identified in 18.8% of isolates. PCR screening identified S. enterica serovar
Typhimurium DT104 in both meat and dairy products. This is the first study to report many of these resistance
genes in dairy products. This study highlights the high incidence of multidrug-resistant S. enterica in meat and
dairy products in Egypt, with the possibility of their transfer to humans leading to therapeutic failure. Therefore,
the overuse of antibiotics in animals should be drastically reduced in developing countries.
© 2014 Elsevier B.V. All rights reserved.



Increasing resistance to antimicrobial agents remains a major
challenge to public health professionals in both developed and developing countries. However, in developing countries, antimicrobial
resistance is exacerbated by over-prescription of antibiotics and
increased use in both human and animal healthcare. Furthermore, strategies to combat and prevent resistance are not at the top of the list of
priorities in these countries (da Costa et al., 2013). More recently, the
Centers for Disease Control and Prevention of the USA estimates that,
in the USA, more than two million people are made ill every year with
antibiotic-resistant infections, with at least 23,000 dying as a result of
these infections (CDC, 2013). Food may act as a vector for the transfer
of antimicrobial resistant bacteria and antimicrobial resistance genes
to humans (Verraes et al., 2013). Multidrug-resistant (MDR) bacteria


⁎ Corresponding author. Tel./fax: +81 82 424 7897.
[E-mail address: tadashis@hiroshima-u.ac.jp (T. Shimamoto).](mailto:tadashis@hiroshima-u.ac.jp)


[http://dx.doi.org/10.1016/j.ijfoodmicro.2014.07.031](http://dx.doi.org/10.1016/j.ijfoodmicro.2014.07.031)
0168-1605/© 2014 Elsevier B.V. All rights reserved.



can spread to humans either via the food supply (e.g., meat, fish, eggs
and dairy products), direct contact with animals or, more indirectly,
through environmental pathways (Angulo et al., 2004). The consequences of antimicrobial resistance are particularly important when
pathogens are resistant to antimicrobials that are critically important
in the treatment of human disease. This concern includes infections acquired in hospitals, community infections acquired in outpatient care
settings, and also, resistant foodborne disease associated with drug
use in food-producing animals (WHO, 2009).
The spread of MDR bacteria via meat and dairy products poses serious public health concerns. In the United States, the newly-released
10th National Antimicrobial Resistance Monitoring System (NARMS)
report describes alarming increases in antibiotic-resistant bacteria
found on retail meats. The report also confirmed that 80% of all antibiotics used in the United States are used not on humans but on food
animals, most of which are perfectly healthy (NARMS, 2011). The increasing prevalence of multidrug resistance among Salmonella, not
only against the first-line antibiotics, ampicillin, chloramphenicol and
trimethoprim/sulfamethoxazole, but also, against clinically important




40 A.M. Ahmed et al. / International Journal of Food Microbiology 189 (2014) 39–44



antimicrobial agents, such as fluoroquinolones and third generation
cephalosporins, is also an emerging problem (Lunguya et al., 2013). In
developed countries, many surveys have been conducted at the molecular level to monitor the incidence of MDR Salmonella enterica in meat
and dairy products (Ahmed et al., 2009a; Barlow and Gobius, 2006;
Clemente et al., 2013; Li et al., 2013; Mohamed et al., 2014; Thong and
Modarressi, 2011; Van Keesel et al., 2013; Wong and Chen, 2013;
Zhao et al., 2009). However, the extent of antibiotic resistance to
S. enterica in many developing countries and the molecular mechanisms
underlying this resistance remain unclear. The misuse of antibiotics in
humans and animals in these countries is not uncommon. Therefore,
the aim of this study was to perform a large scale survey to determine
the incidence of MDR strains of S. enterica isolated from meat and
dairy products in Egypt and to characterize the molecular mechanisms
of antimicrobial resistance.


2. Materials and methods


2.1. Bacterial isolates


A total of 69 isolates of S. enterica (28 isolates of S. enterica serovar
Typhimurium, 22 isolates of S. enterica serovar Enteritidis, 16 isolates
of S. enterica serovar Infantis and 3 isolates of non-typable serovars)
were used in this study. They were isolated in Egypt from 800 meat
products (480 beef and 320 chickens) and 800 dairy products (480
milk, 240 cheese and 80 yogurt) as previously described (Ahmed and
Shimamoto, 2014).


2.2. Antimicrobial susceptibility testing


The antimicrobial sensitivity phenotypes of bacterial isolates were
determined using a Kirby–Bauer disk diffusion assay according to the
standards and interpretive criteria described by the Clinical and Laboratory Standards Institute (CLSI, 2011). The following antibiotics were
used: ampicillin (AMP), 10 μg; amoxicillin–clavulanic acid (AMC), 20/
10 μg; cefoxitin (FOX), 30 μg; cefotetan (CTT), 30 μg; cefotaxime
(CTX), 30 μg; cefpodoxime (CPD), 10 μg; ceftriaxone (CRO), 30 μg; aztreonam (ATM), 30 μg; nalidixic acid (NAL), 30 μg; ciprofloxacin (CIP),
5 μg; chloramphenicol (CHL), 30 μg; gentamicin (GEN), 10 μg; kanamycin (KAN), 30 μg; oxacillin (OXA) 30 μg; streptomycin (STR), 10 μg;
spectinomycin (SPX), 10 μg; sulfamethoxazole/trimethoprim (SXT),
23.75/1.25 μg and tetracycline (TET), 30 μg. The disks were purchased
from Oxoid, UK, and the results were recorded based on CLSI guidelines
(CLSI, 2011). The reference strain, Escherichia coli ATCC 25922, was
included as a quality control.


2.3. Bacterial DNA preparation


DNA was prepared using boiled lysates, as previously described
(Ahmed et al., 2009a). All selected single colonies were subcultured in
LB broth. An overnight bacterial culture (200 μl) was mixed with
800 μl of distilled water and boiled for 10 min. The resulting solution
was centrifuged and the supernatant used as the DNA template. DNA
template was stored at −20 °C until use.


2.4. PCR screening for integrons and antimicrobial resistance genes


Conserved primers were used for the detection and identification of
class 1 and class 2 integrons as previously described (Ahmed et al.,
2013). PCR screening for TEM, SHV, CTX-M, OXA and CMY β-lactamaseencoding genes was performed using universal primers for the TEM,
SHV, OXA, CTX-M and CMY families, as described previously (Ahmed
et al., 2013). Furthermore, PCR amplification was used to screen for
plasmid-mediated quinolone resistance genes, qnrA, qnrB, qnrS and
aac(6′)-Ib-cr, using previously described primers (Ahmed et al., 2013).
Finally, the florfenicol resistance gene, floR, was detected using StCM-L



and StCM-R primers, as described previously (Ahmed et al., 2013). The
PCR reaction products were subjected to electrophoresis in a 1.0% agarose
gel, stained with ethidium bromide and visualized under UV light. PCR
fragments were then purified from the agarose gel using a QIAquick Gel
Extraction Kit (Qiagen, Japan). Both DNA strands of the PCR product
were sequenced using an ABI automatic DNA sequencer (Model 373;
Perkin-Elmer).


2.5. Identification of S. enterica serovar Typhimurium DT104 by PCR


The PCR amplification of an internal segment of the 16S-to-23S
spacer region of bacterial rRNA genes (size = 162 bp) was used to
identify S. enterica serovar Typhimurium DT104 as previously described
(Pritchett et al., 2000).


2.6. Transconjugation experiments and plasmid incompatibility grouping


Transferability of plasmids was determined by mating-out assay
using S. enterica isolates as donors and a rifampicin-resistant mutant
of E. coli HB101 as a recipient as described previously (Ahmed et al.,
2013). Some S. enterica isolates which have no potent resistance gene
marker were not tested. Transconjugants were selected on agar
supplemented with 250 mg/l rifampicin and 100 mg/l ampicillin. Plasmid DNA was extracted from both S. enterica isolates and E. coli
transconjugants using the Kado and Liu method (Kado and Liu, 1981).
Plasmid incompatibility grouping was determined by PCR-based replicon typing as previously described (Carattoli et al., 2005). The transfer
of integrons and resistance genes was confirmed by PCR assays on the
transconjugants as described previously (Ahmed et al., 2013).


2.7. Computer analysis of the sequence data


A similarity search was carried out using the BLAST program
[available at the NCBI BLAST homepage (http://blast.ncbi.nlm.nih.gov/](http://blast.ncbi.nlm.nih.gov/Blast.cgi)
[Blast.cgi).](http://blast.ncbi.nlm.nih.gov/Blast.cgi)


3. Results and discussion


3.1. Incidence of MDR S. enterica in meat and dairy products


The emergence and spread of antimicrobial resistance in bacteria
constitute a threat to human health and present a major financial burden. Food is generally considered to be the most important vector for
the spread of resistance between humans and animals (WHO, 2007).
In this study 47 out of 69 isolates (68.1%) showed multidrug resistance
phenotypes to at least three classes of antimicrobials (Table 1). The incidence of MDR isolates was higher in meat products (69.8%) than in
dairy products (62.5%). The MDR S. enterica serovars were: S. enterica
serovar Typhimurium (34.8%), S. enterica serovar Enteritidis, (21.8%),
S. enterica serovar Infantis (10.1%) and S. enterica non-typable serovar
(1.4%) (Table 1). The highest resistance was to ampicillin (95.7%),
then to kanamycin (93.6%), spectinomycin (93.6%), streptomycin
(91.5%) and sulfamethoxazole/trimethoprim (91.5%) (Table 1). Interestingly, the results of multidrug resistance phenotypes in this study
are quite similar to that we previously reported in S. enterica isolated
from diarrheic calves in Egypt as 66.7% of S. enterica showed multidrug
resistance phenotypes to ampicillin, amoxicillin, streptomycin, spectinomycin, sulfamethoxazole/trimethoprim and kanamycin (Ahmed
et al., 2009b). Our results confirm the high level of resistance to the traditional antibiotics used widely in Egypt. Unfortunately, in developing
countries, these antibiotics are widely used to treat diarrhea because
of their low cost and availability (Van et al., 2012).




A.M. Ahmed et al. / International Journal of Food Microbiology 189 (2014) 39–44 41


Table 1
Resistance phenotypes and incidence of integrons and resistance genes in Salmonella enterica isolated from meat and dairy products.


No. Isolate Serovar Meat product Resistance phenotype [b] Integrons/resistance genes


1 ST-M1 S. Typhimurium [a] Beef AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, GEN, KAN, Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaCMY-2,
NAL, OXA, SPX, STR, SXT, TET blaCTX-M-3, qnrB, aac(6′)-Ib-cr, floR
2 ST-M2 S. Typhimurium [a] Beef AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, KAN, NAL, Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaCMY-2,
OXA, SPX, STR, SXT, TET blaCTX-M-15, blaSHV-12, qnrA, floR
3 ST-M3 S. Typhimurium [a] Beef AMC, AMP, ATM, CHL, CIP, CRO, CTT, CTX, FOX, GEN, KAN, NAL, Class 1 (aadA2 and blaPSE-1), blaTEM-1,
OXA, SPX, STR, SXT, TET blaSHV-12, qnrB, floR
4 ST-M4 S. Typhimurium [a] Beef AMP, ATM, CHL, CIP, CTT, FOX, GEN, KAN, NAL, OXA, SPX, Class 1 (aadA2 and blaPSE-1), qnrS, floR
STR, SXT, TET

5 ST-M5 S. Typhimurium Beef AMP, ATM, CIP, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, Class 1 (dfrA12-orf-aadA2), blaCMY-2,
STR, SXT, TET aac(6′)-Ib-cr
6 ST-M6 S. Typhimurium Chicken AMP, ATM, CHL, CTT, CTX, FOX, GEN, KAN, NAL, OXA, SPX, Class 1 (aadB-catB3), blaTEM-1, blaOXA-1, floR
STR, SXT

7 ST-M7 S. Typhimurium Beef AMC, AMP, ATM, CHL, CIP, CTT, CTX, FOX, KAN, NAL, OXA, Class 1 (dfrA1-aadA1), blaTEM-1, blaSHV-12,
SPX, STR, SXT, TET aac(6′)-Ib-cr
8 ST-M8 S. Typhimurium Beef AMC, MP, ATM, CPD, CRO, CTT, CTX, FOX, KAN, GEN, KAN, Class 1 (dfrA12-orf-aadA2), blaTEM-1, blaCTX-M-3
OXA, SPX, STR, SXT, TET

9 ST-M9 S. Typhimurium Beef AMC, AMP, ATM, CHL, CPD, CRO, CTT, CTX, FOX, GEN, KAN, Class 1 (dfrA17-aadA5), blaTEM-1, blaCTX-M-3, floR
NAL, OXA, SPX, STR, SXT, TET

10 ST-M10 S. Typhimurium Beef AMP, ATM, CHL, CIP, CTT, FOX, GEN, KAN, NAL, OXA, SPX, Class 1 (dfrA17-aadA5), blaTEM-1, qnrS
STR, SXT, TET

11 ST-M11 S. Typhimurium Beef AMP, ATM, CIP, FOX, KAN, NAL, OXA, SPX, STR, SXT, TET Class 1 (dfrA1-aadA1), blaTEM-1, qnrB
12 ST-M12 S. Typhimurium Beef AMP, ATM, CHL, KAN, OXA, SPX, STR, SXT, TET Class 1 (dfrA1-aadA1), blaTEM-1
13 ST-M13 S. Typhimurium Chicken AMP, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET Class 1 (aadA2), blaTEM-1
14 ST-M14 S. Typhimurium Beef AMC, AMP, CPD, CRO, CTT, CTX, FOX, GEN, KAN, OXA, Class 1 (dfrA15), blaTEM-1, blaCTX-M-15
SPX, STR, SXT, TET

15 ST-M15 S. Typhimurium Beef AMC, AMP, CHL, CTT, CTX, FOX, KAN, NAL, OXA, SPX, Class 2 (dfrA1-sat2-aadA1), blaCMY-2
STR, SXT, TET

16 ST-M16 S. Typhimurium Beef AMP, CHL, KAN, NAL, OXA, SPX, STR, SXT, TET Class 2 (dfrA1-sat2-aadA1)
17 ST-M17 S. Typhimurium Beef CHL, CIP, GEN, NAL, SPX, STR, SXT, TET qnrB
18 ST-M18 S. Typhimurium Beef AMP, CIP, CTT, FOX, NAL, OXA, SPX, STR, SXT, TET blaTEM-1, aac(6′)-Ib-cr
19 ST-M19 S. Typhimurium Beef AMP, CHL, KAN, NAL, SXT, TET floR
20 SE-M1 S. Enteriditis Beef AMC, AMP, ATM, CHL, CIP, CTT, CTX, FOX, GEN, KAN, Class 1 (dfrA1-aadA1), blaCMY-2,
NAL, OXA, SPX, STR, SXT, TET qnrB, floR
21 SE-M2 S. Enteriditis Chicken AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, GEN, Class 1 (aadA2), blaTEM-1,
KAN, NAL, OXA, SPX, STR, SXT, TET blaCTX-M-3, qnrS
22 SE-M3 S. Enteriditis Chicken AMP, ATM, CHL, CIP, CTT, CTX, FOX, GEN, KAN, NAL, OXA, Class 1 (dfrA17-aadA5), blaTEM-1,
SPX, STR, SXT aac(6′)-Ib-cr
23 SE-M4 S. Enteriditis Beef AMP, ATM, CTT, FOX, KAN, OXA, SPX, STR, SXT, TET Class 1 (dfrA5), blaTEM-1
24 SE-M5 S. Enteriditis Beef AMP, CTT, FOX, GEN, KAN, NAL, OXA, SPX, STR, SXT, TET Class 1 (estX-aadA1), blaTEM-1
25 SE-M6 S. Enteriditis Beef AMP, CHL, OXA, SPX, STR, SXT blaTEM-1, floR
26 SE-M7 S. Enteriditis Chicken AMP, CTT, FOX, KAN, OXA, SPX, STR, SXT, TET Class 2 (estX-sat2-aadA1), blaTEM-1
27 SE-M8 S. Enteriditis Beef AMC, AMP, ATM, CHL, CPD, CRO, CTT, CTX, FOX, GEN, KAN, Class 2 (estX-sat2-aadA1), blaSHV-12
OXA, SPX, STR, SXT, TET

28 SE-M9 S. Enteriditis Chicken AMP, GEN, KAN, SPX, STR, SXT, TET Class 2 (estX-sat2-aadA1)
29 SE-M10 S. Enteriditis Beef AMP, KAN, OXA, SPX, STR, SXT,TET blaTEM-1
30 SE-M11 S. Enteriditis Beef AMC, AMP, ATM, CHL, CPD, CRO, CTT, CTX, FOX, GEN, KAN, blaCMY-2
OXA, SPX, STR, SXT, TET

31 SE-M12 S. Enteriditis Beef AMP, ATM, KAN, OXA, SPX, STR blaOXA-1
32 SI-M1 S. Infantis Chicken AMP, ATM, CTT, CTX, FOX, GEN, KAN, OXA, SPX, STR, SXT, TET Class 1 (aadA1), blaTEM-1
33 SI-M2 S. Infantis Chicken AMP, ATM, CHL, GEN, KAN, SPX, STR, SXT, TET Class 1 (aadA1)
34 SI-M3 S. Infantis Chicken AMP, CHL, CIP, NAL, OXA, SPX, STR, TET floR
35 SI-M4 S. Infantis Chicken AMP, CIP, GEN, KAN, SPX, STR, SXT, TET qnrB
36 SI-M5 S. Infantis Beef AMP, CHL, KAN, OXA, SPX, STR, SXT, TET blaTEM-1
37 SN-M1 S. non-typable Beef AMP, CIP, GEN, KAN, NAL, OXA blaTEM-1
38 ST-D1 S. Typhimurium [a] Cheese AMC, AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, GEN, KAN, Class 1 (aadA2 and blaPSE-1), blaTEM-1,
NAL, OXA, SPX, STR, SXT, TET blaSHV-12, aac(6′)-Ib-cr, qnrB, floR
39 ST-D2 S. Typhimurium Milk AMP, ATM, CHL, CIP, CPD, CRO, CTT, CTX, FOX, Class 1 (dfrA17-aadA5), blaTEM-1,
GEN, KAN, NAL, blaCTX-M-3, qnrS
OXA, SPX, STR, SXT, TET

40 ST-D4 S. Typhimurium Milk AMC, CHL, CTT, FOX, GEN, KAN, NAL, OXA, STR, Class 1 (dfrA15b-cm1A4-aadA2),
SPX, SXT, TET blaTEM-1
41 ST-D3 S. Typhimurium Milk AMC, ATM, CTT, FOX, OXA, SPX, STR, SXT, TET Class 2 (dfrA1-sat2), blaCMY-2
42 ST-D5 S. Typhimurium Milk AMP, GEN, NAL, SPX, STR, SXT blaTEM-1
43 SE-D1 S. Enteriditis Milk AMP, ATM, CHL, CTT, FOX, GEN, KAN, OXA, SPX, STR, SXT, TET Class 1 (aac(3)-Id-aadA7), blaTEM-1, blaOXA-1, floR
44 SE-D2 S. Enteriditis Milk AMC, AMP, ATM, CPD, CRO, CTT, CTX, FOX, KAN, NAL, Class 1 (dfrA12-orf-aadA2), blaCMY-2
OXA, SPX, STR, SXT, TET

45 SE-D3 S. Enteriditis Cheese CHL, CIP, GEN, NAL, SXT, TET qnrS
46 SI-D1 S. Infantis Milk AMP, CTT, FOX, KAN, OXA, SPX, STR, SXT, TET Class 1 (aadA1), blaTEM-1
47 SI-D2 S. Infantis Milk AMP, CHL, GEN, KAN, NAL, SPX,TET floR


a S. enterica serovar Typhimurium DT104.
b AMC, amoxicillin–clavulanic acid; AMP, ampicillin; ATM, aztreonam; CHL, chloramphenicol; CIP, ciprofloxacin; CPD, cefpodoxime; CRO, ceftriaxone; CTT, cefotetan; CTX, cefotaxime;
FOX, cefoxitin; GEN, gentamicin; KAN, kanamycin; NAL, nalidixic acid; OXA, oxacillin; SPX, spectinomycin; STR, streptomycin; SXT, sulfamethoxazole–trimethoprim; TET, tetracycline.




42 A.M. Ahmed et al. / International Journal of Food Microbiology 189 (2014) 39–44



3.2. Incidence of class 1 and class 2 integrons in S. enterica from meat and
dairy products


It is well known that genes encoding antimicrobial resistance are
often linked to mobile genetic elements. One of the most important genetic elements responsible for spreading antimicrobial resistance genes
among bacteria is the integron (Mazel, 2006). Most integrons were
found to be located on plasmids and could be transferred to other
strains by conjugation (Van et al., 2012). In this study, PCR identified
class 1 integrons in 39.1% S. enterica isolates (Table 1). The incidence
of class 1 integrons was higher in meat products (39.6%) than in dairy
products (37.5%) (Table 1). DNA-sequencing results for the inserted
gene cassettes identified 12 types of class 1 integron with 16 different
antimicrobial resistance gene cassettes (Table 1). Our results relating
to the incidence of class 1 integrons are higher than those reported in
the USA, where only 15% of S. enterica serovars (including Typhimurium
and Enteritidis) isolated from retail meats harbored class 1 integrons
(Zhao et al., 2009). This incidence of class 1 integrons in dairy products
is significantly higher than that recently reported for the USA by Van
Keesel et al. (2013), as class 1 integrons were identified in only 2.8% of
the MDR S. enterica isolates isolated from bulk milk and milk filters.
Class 2 integron is also an important vehicle for spreading antimicrobial
resistance genes in S. enterica strains (Ahmed et al., 2005). In this study,
PCR identified class 2 integrons in only 8.7% S. enterica isolates as follows: S. enterica serovar Typhimurium (4.4%) and S. enterica serovar
Enteritidis (4.4%) (Table 1). The incidence of class 2 integrons was
higher in meat products (9.4%) than in dairy products (6.2%). DNAsequencing results for the inserted gene cassettes identified three
types of class 2 integron (Table 1). In Egypt, class 2 integrons have
been reported previously in 5.4% of Gram-negative bacteria isolated
from bovine mastitis (Ahmed and Shimamoto, 2011). Interestingly, to
the best of our knowledge, this is the first report of the detection and
identification of class 2 integrons in raw milk.


3.3. Incidence of β-lactamase-encoding genes in S. enterica from meat and
dairy products


Production of β-lactamases is considered the main mechanism of
resistance in Gram-negative bacteria to overcome penicillin-derived
(β-lactam) antibiotics. Resistance to expanded-spectrum cephalosporins in S. enterica is a special concern, since these antimicrobials
are a front-line therapeutic for the treatment of numerous Gramnegative infections (Bradford, 2001). In this study, PCR identified
β-lactamase-encoding genes in 75.4% S. enterica isolates. The incidence of β-lactamase-encoding genes was very similar in meat products (75.5%) and dairy products (75.0%) (Table 1). These genes
include: blaTEM-1 (41.5%), blaCMY-2 (11.3%), blaCTX-M-3 and blaCTX-M
15 [(11.3%),][ bla] SHV-12 [(7.5%) and][ bla] OXA-1 [(3.7%) (][Table 1][). In China,]
blaOXA-1 was the most commonly identified β-lactamase gene in
S. enterica isolates from pigs, ducks and chickens from abattoirs and
retail markets, followed by blaTEM-1, blaPSE-1 and blaCMY-2 (Li et al.,
2013). In Portugal, blaCTX-M-1, blaCTX-M-14, blaCTX-M-15 and blaCTX-M
32 [,][ bla] SHV-12 [and][ bla] TEM-1 [genes were detected in][ S. enterica][ isolates]
from poultry, swine and food products of animal origin (bovine,
swine and poultry) (Clemente et al., 2013). In the USA, 11.4% of
S. enterica isolates isolated from bulk milk and milk filters were positive for blaCMY (Van Keesel et al., 2013).


3.4. Incidence of plasmid-mediated quinolone resistance genes in S. enterica
from meat and dairy products


Quinolones are used extensively in veterinary practice worldwide to
combat bacterial diseases. S. enterica with reduced susceptibility to ciprofloxacin is of serious concern as fluoroquinolone (e.g., ciprofloxacin)
is the drug of first choice for the treatment of invasive and systemic salmonellosis that occurs in humans and animals (Dimitrov et al., 2007). In



this study, the plasmid-mediated quinolone resistance genes: qnrA,
qnrB, qnrS and aac(6′)-Ib-cr were identified in 28.3% of tested
S. enterica isolates from meat products. In Hong Kong, it was reported
that S. enterica strains isolated from retail meats with different resistance profiles harbored plasmid-mediated quinolone resistance genes,
one carrying the qnrS gene and the other carrying qnrS and aac(6′)-Ibcr genes (Wong and Chen, 2013). In China, qnrA, qnrB, qnrS and
aac(6′)-Ib-cr genes were identified in S. enterica strains isolated from retail foods (including chicken meat) with the incidence of 46.6%, 12.7%,
19.5% and 13.6%, respectively (Yang et al., 2013). In Colombia, qnrB
was identified in S. enterica strains isolated from retail meats (including
ground and chicken meats) (Karczmarczyk et al., 2010). Also qnrB, qnrS
and aac(6′)-Ib-cr were identified in 25% of tested S. enterica isolates
from dairy products. In Egypt, qnrA, qnrB, qnrS and aac(6′)-Ib-cr genes
were identified in 14.3% of Gram-negative bacteria isolated from bovine
mastitis (Ahmed and Shimamoto, 2011). It is well known that qnr genes
confer only low-level resistance to fluoroquinolones and accumulations
of quinolone resistance-determining region (QRDR) mutations is necessary for S. enterica to be resistant to fluoroquinolone especially ciprofloxacin (Eaves et al., 2004; Robicsek et al., 2006). To the best of our
knowledge, this is the first report of the detection and identification of
plasmid-mediated quinolone resistance genes in S. enterica isolates
from dairy products.


3.5. Incidence of florfenicol resistance gene, floR, in S. enterica from meat
and dairy products


Florfenicol was approved by the FDA in 1996 for veterinary use in
food animals in the United States. Florfenicol is not approved for
human use; however, it is related to chloramphenicol and can select
for cross-resistance among bacterial pathogens. Florfenicol resistance
is mediated by the floR gene, which confers non-enzymatic crossresistance to chloramphenicol (White et al., 2000). In this study, PCRand DNA-sequence screening identified floR in 18.8% S. enterica isolates.
The incidence of floR was higher in dairy products (25.0%) than in meat
products (17.0%) (Table 1). In Malaysia, floR was detected in MDR
S. enterica strains isolated from meat products (raw beef, chicken meat
and street foods) (Thong and Modarressi, 2011). In Egypt, floR, has
been detected in 1% of S. enterica strains isolated from diseased broilers
(Ahmed and Shimamoto, 2012). Also, in this study, floR, was identified
in 25.0% of S. enterica strains isolated from dairy products. In Egypt,
floR, has been detected in 6.3% of Gram-negative bacteria isolated
from cases of bovine mastitis (Ahmed and Shimamoto, 2011). To the
best of our knowledge, this is the first report of the detection and identification of floR in dairy products.


3.6. Incidence of S. enterica serovar Typhimurium DT104 in meat and dairy
products


S. enterica serovar Typhimurium DT104 is a truly international
multiresistant clone of S. enterica that emerged in the UK in the early
1990s and then spread worldwide (Threlfall, 2000). S. enterica serovar
Typhimurium DT104 usually shows a pentadrug-resistance phenotype
to ampicillin, chloramphenicol, streptomycin, sulphonamides and tetracyclines (ACSSuT resistance type). In this study, S. enterica serovar
Typhimurium DT104 was identified at high levels in beef (7.5%) and
also in cheese (6.3%) (Table 1). Our findings are very important from
the public health point of view as a beef-associated outbreak of
S. enterica serovar Typhimurium DT104 occurred in September–November 2005 in the Netherlands (Kivi et al., 2007). Also, an outbreak
of human salmonellosis caused by S. enterica serovar Typhimurium
DT104 occurred in Denmark between July and August 2005 due to
imported beef served as carpaccio (Ethelberg et al., 2007). Furthermore,
in the USA, numerous outbreaks of S. enterica serovar Typhimurium
DT104 have been associated with the consumption of raw-milk cheese
(Cody et al., 1999).




A.M. Ahmed et al. / International Journal of Food Microbiology 189 (2014) 39–44 43


Table 2
Results of conjugation experiments and plasmid replicon typing for Salmonella enterica isolated from meat and dairy products.


No. Isolate Serovar Resistance genotype Plasmid replicon [a] Conjugable [b] Transconjugant resistance genotype [b]


1 ST-M1 S. Typhimurium Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaCMY-2, N Yes blaTEM-1, blaCMY-2, blaCTX-M-3, qnrB,
blaCTX-M-3, qnrB, aac(6′)-Ib-cr, floR aac(6′)-Ib-cr
2 ST-M2 S. Typhimurium Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaCMY-2, A/C Yes blaTEM-1, blaCMY-2, blaCTX-M-15,
blaCTX-M-15, blaSHV-12, qnrA, floR blaSHV-12, qnrA,
3 ST-M3 S. Typhimurium Class 1 (aadA2 and blaPSE-1), blaTEM-1, blaSHV-12, I1 Yes blaTEM-1, blaSHV-12, qnrB
qnrB, floR

4 ST-M4 S. Typhimurium Class 1 (aadA2 and blaPSE-1), qnrS, floR NT NA NA
5 ST-M5 S. Typhimurium Class 1 (dfrA12-orf-aadA2), blaCMY-2, A/C Yes Class 1 (dfrA12-orf-aadA2),
aac(6′)-Ib-cr blaCMY-2, aac(6′)-Ib-cr
6 ST-M6 S. Typhimurium Class 1 (aadB-catB3), blaTEM-1, blaOXA-1, floR I1 Yes Class 1 (aadB-catB3), blaTEM-1
7 ST-M7 S. Typhimurium Class 1 (dfrA1-aadA1), blaTEM-1, blaSHV-12, HI1 Yes Class 1 (dfrA1-aadA1), blaTEM-1,
aac(6′)-Ib-cr blaSHV-12, aac(6′)-Ib-cr
8 ST-M8 S. Typhimurium Class 1 (dfrA12-orf-aadA2), blaTEM-1, L/M Yes Class 1 (dfrA12-orf-aadA2),
blaCTX-M-3 blaTEM-1, blaCTX-M-3

9 ST-M9 S. Typhimurium Class 1 (dfrA17-aadA5), blaTEM-1, L/M Yes Class 1 (dfrA17-aadA5),
blaCTX-M-3, floR blaTEM-1, blaCTX-M-3
10 ST-M10 S. Typhimurium Class 1 (dfrA17-aadA5), blaTEM-1, qnrS HI1 No NA
11 ST-M11 S. Typhimurium Class 1 (dfrA1-aadA1), blaTEM-1, qnrB I1 Yes Class 1 (dfrA1-aadA1),
blaTEM-1, qnrB
12 ST-M12 S. Typhimurium Class 1 (dfrA1-aadA1), blaTEM-1 I1 Yes Class 1 (dfrA1-aadA1), blaTEM-1
13 ST-M13 S. Typhimurium Class 1 (aadA2), blaTEM-1 I1 Yes Class 1 (aadA2), blaTEM-1
14 ST-M14 S. Typhimurium Class 1 (dfrA15), blaTEM-1, blaCTX-M-15 I1 Yes Class 1 (dfrA15), blaTEM-1, blaCTX-M-15
15 ST-M15 S. Typhimurium Class 2 (dfrA1-sat2-aadA1), blaCMY-2 A/C Yes Class 2 (dfrA1-sat2-aadA1), blaCMY-2
16 ST-M16 S. Typhimurium Class 2 (dfrA1-sat2-aadA1) NT NA NA
17 ST-M17 S. Typhimurium qnrB NT NA NA
18 ST-M18 S. Typhimurium blaTEM-1, aac(6′)-Ib-cr N No NA
19 ST-M19 S. Typhimurium floR NT NA NA
20 SE-M1 S. Enteriditis Class 1 (dfrA1-aadA1), blaCMY-2, qnrB, floR A/C Yes Class 1 (dfrA1-aadA1), blaCMY-2, qnrB
21 SE-M2 S. Enteriditis Class 1 (aadA2), blaTEM-1, blaCTX-M-3, qnrS N Yes Class 1 (aadA2), blaTEM-1, blaCTX-M-3, qnrS
22 SE-M3 S. Enteriditis Class 1 (dfrA17-aadA5), blaTEM-1, aac(6′)-Ib-cr A/C Yes Class 1 (dfrA17-aadA5), blaTEM-1, aac(6′)-Ib-cr
23 SE-M4 S. Enteriditis Class 1 (dfrA5), blaTEM-1 A/C Yes Class 1 (dfrA5), blaTEM-1
24 SE-M5 S. Enteriditis Class 1 (estX-aadA1), blaTEM-1 N Yes Class 1 (estX-aadA1), blaTEM-1
25 SE-M6 S. Enteriditis blaTEM-1, floR I1 Yes blaTEM-1
26 SE-M7 S. Enteriditis Class 2 (estX-sat2-aadA1), blaTEM-1 A/C Yes blaTEM-1
27 SE-M8 S. Enteriditis Class 2 (estX-sat2-aadA1), blaSHV-12 A/C Yes blaSHV-12
28 SE-M9 S. Enteriditis Class 2 (estX-sat2-aadA1) NT NA NA
29 SE-M10 S. Enteriditis blaTEM-1 I1 No NA
30 SE-M11 S. Enteriditis blaCMY-2 A/C Yes blaCMY-2
31 SE-M12 S. Enteriditis blaOXA-1 A/C Yes blaOXA-1
32 SI-M1 S. Infantis Class 1 (aadA1), blaTEM-1 HI1 No NA
33 SI-M2 S. Infantis Class 1 (aadA1) HI1 No NA
34 SI-M3 S. Infantis floR NT NA NA
35 SI-M4 S. Infantis qnrB NT NA NA
36 SI-M5 S. Infantis blaTEM-1 I1 Yes blaTEM-1
37 SN-M1 S. non-typable blaTEM-1 I1 Yes blaTEM-1
38 ST-D1 S. Typhimurium Class 1 (aadA2 and blaPSE-1), blaTEM-1, N Yes blaTEM-1, blaSHV-12, aac
blaSHV-12, aac(6′)-Ib-cr, qnrB, floR (6′)-Ib-cr, qnrB
39 ST-D2 S. Typhimurium Class 1 (dfrA17-aadA5), blaTEM-1, L/M Yes Class 1 (dfrA17-aadA5), blaTEM-1, blaCTX-M-3, qnrS
blaCTX-M-3, qnrS

40 ST-D4 S. Typhimurium Class 1 (dfrA15b-cm1A4-aadA2), blaTEM-1 I1 Yes Class 1 (dfrA15b-cm1A4-aadA2), blaTEM-1
41 ST-D3 S. Typhimurium Class 2 (dfrA1-sat2), blaCMY-2 A/C Yes Class 2 (dfrA1-sat2), blaCMY-2
42 ST-D5 S. Typhimurium blaTEM-1 I1 Yes blaTEM-1
43 SE-D1 S. Enteriditis Class 1 (aac(3)-Id-aadA7), blaTEM-1, I1 Yes Class 1 (aac(3)-Id-aadA7), blaTEM-1
blaOXA-1, floR

44 SE-D2 S. Enteriditis Class 1 (dfrA12-orf-aadA2), blaCMY-2 A/C Yes Class 1 (dfrA12-orf-aadA2), blaCMY-2
45 SE-D3 S. Enteriditis qnrS NT NA NA
46 SI-D1 S. Infantis Class 1 (aadA1), blaTEM-1 HI1 No NA
47 SI-D2 S. Infantis floR NT NA NA


a NT, not tested.
b NA, not applicable.



3.7. Transferability and replicon typing of plasmids


In this study, plasmid analysis showed that most of plasmid carrying integrons and resistance genes were conjugable with replicon
types: IncI1, IncA/C, IncHI1, IncN and IncL/M (Table 2). These types
of plasmid replicon are more frequently detected in plasmids
among Enterobacteriaceae and play a crucial role in spreading of specific resistance genes especially extended-spectrum β-lactamase
genes and acquired AmpC genes (Carattoli, 2011). Also, these types
of plasmid are considered to be “epidemic resistance plasmids”



that are being worldwide detected in Enterobacteriaceae of different
origins and sources (Carattoli, 2011).


4. Conclusions


Our study highlights the high incidence of MDR S. enterica in meat
and dairy products from Egypt and provides molecular characterization
of different mechanisms of antimicrobial resistance. Also, some resistance genes were identified in dairy products for the first time. Information on antibiotic resistance phenotypes and genotypes of foodborne




44 A.M. Ahmed et al. / International Journal of Food Microbiology 189 (2014) 39–44



pathogens in different countries and geographic regions is necessary to
track the change in resistance pattern and to follow changes in antimicrobial sensitivity patterns that may require a reassessment of treatment and control strategy.


Conflict of interest statement


The authors declare no conflicts of interest.


Acknowledgment


This work was supported financially by the Science and Technology
Development Fund (STDF), Egypt, Grant No. 540 to A.M.A.


References


[Ahmed, A.M.,Shimamoto, T., 2011. Molecular characterization of antimicrobial resistance](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0015)
[in Gram-negative bacteria isolated from bovine mastitis in Egypt. Microbiol.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0015)
[Immunol. 55, 318–327.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0015)
[Ahmed, A.M.,Shimamoto, T., 2012. Genetic analysis of multiple antimicrobial resistance in](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0020)
[Salmonella isolated from diseased broilers in Egypt. Microbiol. Immunol. 56, 254–261.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0020)
[Ahmed, A.M.,Shimamoto, T., 2014. Isolation and molecular characterization of Salmonella](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0030)
[enterica, Escherichia coli O157:H7 and Shigella spp. from meat and dairy products in](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0030)
[Egypt. Int. J. Food Microbiol. 168–169, 57–62.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0030)
[Ahmed, A.M.,Nakano, H.,Shimamoto, T., 2005. Molecular characterization of integrons in](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0005)
[non-typhoid Salmonella serovars isolated in Japan: description of an unusual class 2](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0005)
[integron. J. Antimicrob. Chemother. 55, 371–374.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0005)
[Ahmed, A.M.,Shimabukuro, H.,Shimamoto, T., 2009a. Isolation and molecular characteri-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0010)
[zation of multidrug-resistant strains of Escherichia coli and Salmonella from retail](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0010)
[chicken meat in Japan. J. Food Sci. 74, M405–M410.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0010)
[Ahmed, A.M.,Younis, E.E.,Ishida, Y.,Shimamoto, T., 2009b. Genetic basis of multidrug re-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0035)
[sistance in Salmonella enterica serovars Enteritidis and Typhimurium isolated from](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0035)
[diarrheic calves in Egypt. Acta Trop. 111, 144–149.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0035)
[Ahmed, A.M., Shimamoto, T., Shimamoto, T., 2013. Molecular characterization of](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0025)
[multidrug-resistant avian pathogenic Escherichia coli isolated from septicemic](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0025)
[broilers. Int. J. Med. Microbiol. 303, 475–483.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0025)
[Angulo, F.,Nargund, V.,Chiller, T., 2004. An evidence of an association between use of anti-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0040)
[microbial agents in food animals and anti-microbial resistance among bacteria isolat-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0040)
[ed from humans and the human health consequences of such resistance. J. Vet. Med.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0040)
[B 51, 374–379.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0040)
[Barlow, R.S., Gobius, K.S., 2006. Diverse class 2 integrons in bacteria from beef cattle](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0045)
[sources. J. Antimicrob. Chemother. 58, 1133–1138.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0045)
[Bradford, P.A., 2001. Extended-spectrum β-lactamases in the 21st century: characteriza-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0050)
[tion, epidemiology, and detection of this important resistance threat. Clin. Microbiol.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0050)
[Rev. 14, 933–951.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0050)
[Carattoli, A., 2011. Plasmids in Gram negatives: molecular typing of resistance plasmids.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0055)
[Int. J. Med. Microbiol. 301, 654–658.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0055)
[Carattoli, A.,Bertini, A.,Villa, L.,Falbo, V.,Hopkins, K.L.,Threlfall, E.J., 2005. Identification of](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0060)
[plasmids by PCR-based replicon typing. J. Microbiol. Methods 63, 219–228.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0060)
CDC (Centers for Disease Control, Prevention), 2013. Antibiotic resistance threats to the
[United States, 2013Available from: http://www.cdc.gov/drugresistance/threat-re-](http://www.cdc.gov/drugresistance/threat-report-2013/pdf/ar-threats-2013-508.pdf)
[port-2013/pdf/ar-threats-2013-508.pdf (Accessed on 10th February 2014).](http://www.cdc.gov/drugresistance/threat-report-2013/pdf/ar-threats-2013-508.pdf)
Clemente, L.,Manageiro, V.,Ferreira, E.,Jones-Dias, D.,Correia, I.,Themudo, P.,Albuquerque,
[T.,Caniça, M., 2013. Occurrence of extended-spectrum β-lactamases among isolates](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0070)
[of Salmonella enterica subsp. enterica from food-producing animals and food prod-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0070)
[ucts, in Portugal. Int. J. Food Microbiol. 167, 221–228.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0070)
[CLSI, 2011. Performance standards for antimicrobial susceptibility testing. Twenty-First](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0195)
[Informational Supplement, vol. 31. Clinical and Laboratory Standards Institute M02-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0195)
[A10 and M07-A08.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0195)
[Cody, S.H.,Abbott, S.L.,Marfin, A.A.,Schulz, B.,Wagner, P.,Robbins, K., 1999. Two outbreaks](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0075)
[of multidrug-resistant Salmonella serotype Typhimurium DT104 infections linked to](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0075)
[raw-milk cheese in northern California. J. Am. Med. Assoc. 281, 1805–1810.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0075)
[da Costa, P.M.,Loureiro, L.,Matos, A.J., 2013. Transfer of multidrug-resistant bacteria be-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0080)
[tween intermingled ecological niches: the interface between humans, animals and](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0080)
[the environment. Int. J. Environ. Res. Public Health 10, 278–294.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0080)
[Dimitrov, T.,Udo, E.E.,Albaksami, O.,Kilani, A.A.,Shehab, E.M.R., 2007. Ciprofloxacin treat-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0085)
[ment failure in a case of typhoid fever caused by Salmonella enterica serotype](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0085)
[Paratyphi A with reduced susceptibility to ciprofloxacin. J. Med. Microbiol. 56,](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0085)
[277–279.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0085)
Eaves, D.J.,Randall, L.,Gray, D.T.,Buckley, A.,Woodward, M.J.,White, A.P.,Piddock, L.J., 2004.

[Prevalence of mutations within the quinolone resistance-determining region of gyrA,](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0090)



[gyrB, parC, and parE and association with antibiotic resistance in quinolone-resistant](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0090)
[Salmonella enterica. Antimicrob. Agents Chemother. 48, 4012–4015.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0090)
Ethelberg, S.,Sørensen, G.,Kristensen, B.,Christensen, K.,Krusell, L.,Hempel-Jørgensen, A.,
[Perge, A.,Nielsen, E.M., 2007. Outbreak with multiresistant Salmonella Typhimurium](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0095)
[DT104 linked to carpaccio, Denmark, 2005. Epidemiol. Infect. 135, 900–907.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0095)
[Kado, C.I.,Liu, S.-T., 1981. Rapid procedure for detection and isolation of large and small](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0100)
[plasmids. J. Bacteriol. 145, 1365–1373.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0100)
[Karczmarczyk, M.,Martins, M.,McCusker, M.,Mattar, S.,Amaral, L.,Leonard, N., 2010. Char-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0105)
[acterization of antimicrobial resistance in Salmonella enterica food and animal isolates](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0105)
[from Colombia: identification of a qnrB19-mediated quinolone resistance marker in](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0105)
[two novel serovars. FEMS Microbiol. Lett. 313, 10–19.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0105)
Kivi, M.,Hofhuis, A.,Notermans, D.W.,Wannet, W.J.,Heck, M.E.,Van De Giessen, A.W.,Van
[Duynhoven, Y.T.,Stenvers, O.F.,Bosman, A.,Van Pelt, W., 2007. A beef-associated out-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0110)
[break of Salmonella Typhimurium DT104 in The Netherlands with implications for](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0110)
[national and international policy. Epidemiol. Infect. 135, 890–899.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0110)
[Li, R.,Lai, J.,Wang, Y.,Liu, S.,Li, Y.,Liu, K.,Shen, J.,Wu, C., 2013. Prevalence and characteri-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0115)
[zation of Salmonella species isolated from pigs, ducks and chickens in Sichuan Prov-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0115)
[ince, China. Int. J. Food Microbiol. 163, 14–18.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0115)
Lunguya, O.,Lejon, V.,Phoba, M.F.,Bertrand, S.,Vanhoof, R.,Glupczynski, Y.,Verhaegen, J.,
[Muyembe-Tamfum, J.J., Jacobs, J., 2013. Antimicrobial resistance in invasive non-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0120)
[typhoid Salmonella from the Democratic Republic of the Congo: emergence of de-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0120)
[creased fluoroquinolone susceptibility and extended-spectrum beta lactamases.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0120)
[PLoS Negl. Trop. Dis. 7, e2103.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0120)
[Mazel, D., 2006. Integrons: agents of bacterial evolution. Nat. Rev. Microbiol. 4, 608–620.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0125)
[Mohamed, T.,Zhao, S.,White, D.G.,Parveen, S., 2014. Molecular characterization of antibi-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0130)
[otic resistant Salmonella Typhimurium and Salmonella Kentucky isolated from pre-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0130)
[and post-chill whole broilers carcasses. Food Microbiol. 38, 6–15.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0130)
National Antimicrobial Resistance Monitoring System (NARMS), 2011. NARMS Retail
[Meat Annual Report, 2011Available online: http://www.fda.gov/AnimalVeterinary/](http://www.fda.gov/AnimalVeterinary/SafetyHealth/AntimicrobialResistance/NationalAntimicrobialResistanceMonitoringSystem/ucm334828.htm)
[SafetyHealth/AntimicrobialResistance/NationalAntimicrobialResistanceMonitoring](http://www.fda.gov/AnimalVeterinary/SafetyHealth/AntimicrobialResistance/NationalAntimicrobialResistanceMonitoringSystem/ucm334828.htm)
[System/ucm334828.htm (Accessed on 10th February 2014).](http://www.fda.gov/AnimalVeterinary/SafetyHealth/AntimicrobialResistance/NationalAntimicrobialResistanceMonitoringSystem/ucm334828.htm)
[Pritchett, L.C.,Konkel, M.E.,Gay, J.M.,Besser, T.E., 2000. Identification of DT104 and U302](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0135)
[phage types among Salmonella enterica serotype Typhimurium isolates by PCR. J.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0135)
[Clin. Microbiol. 38, 3484–3488.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0135)
[Robicsek, A., Jacoby, G.A., Hooper, D.C., 2006. The worldwide emergence of plasmid-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0140)
[mediated quinolone resistance. Lancet Infect. Dis. 6, 629–640.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0140)
[Thong, K.L.,Modarressi, S., 2011. Antimicrobial resistant genes associated with Salmonella](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0145)
[from retail meats and street foods. Food Res. Int. 44, 2641–2646.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0145)
[Threlfall, E.J., 2000. Epidemic Salmonella typhimurium DT 104-a truly international](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0150)
[multiresistant clone. J. Antimicrob. Chemother. 46, 7–10.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0150)
[Van Keesel, J.S.,Sonnier, J.,Zhao, S.,Karns, J.S., 2013. Antimicrobial resistance of Salmonella](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0160)
[enterica isolates from bulk tank milk and milk filters in the United States. J. Food Prot.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0160)
[76, 18–25.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0160)
[Van, T.T.H.,Nguyen, H.N.K.,Smooker, P.M.,Coloe, P.J., 2012. The antibiotic resistance char-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0155)
[acteristics of non-typhoidal Salmonella enterica isolated from food-producing ani-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0155)
[mals, retail meat and humans in South East Asia. Int. J. Food Microbiol. 154, 98–106.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0155)
Verraes, C., Van Boxstael, S., Van Meervenne, E., Van Coillie, E., Butaye, P., Catry, B., de
Schaetzen, M.A., Van Huffel, X., Imberechts, H., Dierick, K., Daube, G., Saegerman, C.,
[De Block, J.,Dewulf, J.,Herman, L., 2013. Antimicrobial resistance in the food chain:](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0165)
[a review. Int. J. Environ. Res. Public Health 10, 2643–2669.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0165)
White, D.G., Hudson, C., Maurer, J.J., Ayers, S., Zhao, S., Lee, M.D., Bolton, L., Foley, T.,
[Sherwood, J., 2000. Characterization of chloramphenicol and florfenicol resistance](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0170)
[in Escherichia coli associated with bovine diarrhea. J. Clin. Microbiol. 38, 4593–4598.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0170)
WHO (World Health Organization), 2007. Critically important antimicrobials for
human medicine: categorization for the development of risk management strategies to contain antimicrobial resistance due to non human antimicrobial use:
report of the second WHO Expert Meeting, Copenhagen, 29-31 May 2007Avail[able online: http://www.who.int/foodborne_disease/resistance/antimicrobials_](http://www.who.int/foodborne_disease/resistance/antimicrobials_human.pdf)
[human.pdf (Accessed on 10th February 2014).](http://www.who.int/foodborne_disease/resistance/antimicrobials_human.pdf)
WHO (World Health Organization). 2009. Report of the 1st meeting of the WHO advisory
group on Integrated Surveillance of Antimicrobial Resistance (AGISAR). Part 1: Third
edition of the WHO list of critically important antimicrobials. Part 2: Strategic framework for WHO activities on integrated surveillance of antimicrobial resistance; Co[penhagen. June 15–19; Geneva, Switzerland: WHO; 2009. http://www.who.int/](http://www.who.int/foodborne_disease/resistance/agisar_June09/en/)
[foodborne_disease/resistance/agisar_June09/en/. (Accessed on 10th February 2014).](http://www.who.int/foodborne_disease/resistance/agisar_June09/en/)
[Wong, M.H.,Chen, S., 2013. First detection of oqxAB in Salmonella spp. isolated from food.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0175)
[Antimicrob. Agents Chemother. 57, 658–660.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0175)
Yang, B.,Qiao, L.,Zhang, X.,Cui, Y.,Xia, X.,Cui, S.,Wang, X.,Meng, X.,Ge, W.,Shi, X.,Wang, D.,
[Meng, J., 2013. Serotyping, antimicrobial susceptibility, pulse field gel electrophoresis](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0180)
[analysis of Salmonella isolates from retail foods in Henan Province, China. Food Con-](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0180)
[trol 32, 228–235.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0180)
Zhao, S.,Blickenstaff, K.,Glenn, A.,Ayers, L.,Friedman, S.L.,Abbott, J.W.,McDermott, P.F.,
[2009. β-lactam resistance in Salmonella strains isolated from retail meats in the](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0185)
[United States by the National Antimicrobial Resistance Monitoring System between](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0185)
[2002 and 2006. Appl. Environ. Microbiol. 75, 7624–7630.](http://refhub.elsevier.com/S0168-1605(14)00374-2/rf0185)


